---
layout: post
title: FungiDB 36 Released
category: [FungiDB]
---

<p><strong>We are pleased to announce the release of FungiDB 36</strong></p>
      
      <section>
        <h3><b>New data in this release</b></h3>
        <div style="margin-left: 3em;">
        
        <h4>Genome Sequence and Annotation: these annotated genomes appear for the first time in FungiDB.</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_0c8f7592b7"><i>Fusarium oxysporum</i> f. sp. cubense tropical race 4 54006</a> &ndash; 
                 Provider: Li-Jun Ma; genome provided prior to publication.  
                 <a href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA73539">NCBI BioProject PRJNA73539</a>.
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_7f555aad50"><i>Fusarium oxysporum</i> Fo47</a> &ndash; 
                 Provider: Li-Jun Ma; genome provided prior to publication. 
                 <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA67069">NCBI BioProject PRJNA67069</a>.
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_385db6b935"><i>Histoplasma capsulatum</i> G217B</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/21958208">Voorhies et al. 2015</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_76ede52023"><i>Histoplasma capsulatum</i> H88 (<i>Ajellomyces capsulatus</i> H88)</a> &ndash; 
                 The Broad Institute. 
                 <a href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA29163">NCBI Bioproject PRJNA29163</a>.
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_9a46569d2f"><i>Penicillium rubens</i> Wisconsin 54-1255</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/18820685">van den Berg M. A. et al. 2011</a>
             </li>
           </ul>
           
        <h4>Transcriptomics data</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4ffc3f21b9">Transcriptome profiling of <i>Coccidioides posadasii</i> WT and &#x394;<i>cps1</i> mutant</a> &ndash; Spherules were collected from both strains. Paired&#45;end RNA&#45;Seq analysis was performed in duplicates. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/27481239" target="_blank">Narra et al. 2008</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4bd3a11499">Transcriptome analysis of the filamentous fungus <i>Aspergillus nidulans</i> directed to the global identification of promoters</a> &ndash; 
                 <i>A. nidulans</i> transcriptome was accessed at 5 different growth conditions: nitrate, complete, ammonia, -N (4hr), and -N (72hr) media.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24299161" target="_blank">Sibthorp et al. 2013</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_f5b8d4ff37">Microarray of <i>Fusarium graminearum</i> transcriptome during symptomless, symptomatic, and intermediate wheat disease symptoms</a> &ndash; Three replicates were collected from rachis internodes and compared to spikelet samples.

                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/28466509" target="_blank">Brown et al. 2017</a>
             </li>
           </ul>
           
        
       <h4>Phenotype data</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_c334877c42">Phenotypic analysis of <i>Neurospora crassa</i> knockout mutants</a> &ndash; A phenotypic pipeline was designed for high&#45;throughput analysis of 10 different traits during hyphal growth, asexual sporulation (macroconidiation), and the sexual cycle in knockout mutants from <i>Neurospora crassa</i>. Provider: Katherine Borkovich; dataset provided prior to publication.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/17352902" target="_blank">Borkovich et al.</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_f10e32c6b8"><i>Neurospora</i> Genome Project Phenotype Image Collection</a> &ndash; Analysis of colony growth and morphology was performed on solid Vogel&#39;s minimal (MIN) medium. For each strain, the hyphae at the edge of the colony and plate growth were photographed. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/17352902" target="_blank">Dunlap et al. 2007</a>,
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16801547" target="_blank">Colot et al. 2006</a>, and
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/21590422" target="_blank">Turner G.E. 2011</a>.
             </li>
           </ul>
           
        <h4>Proteomics data</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_12b70afbf9">Proteogenomic Re-Annotation of <i>Coccidioides posadasii</i> str. Silveira</a> &ndash; 
                 Protein fractions of <i>in vitro</i> grown mycelia and parasitic phase spherules from <i>C. posadasii </i>str. Silveira were analysed using GeLC-MS/MS. Proteomic analysis was performed to provide empirical evidence of the proteins produced in both the mycelial (CDN) and parasitic (SPH) phases.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29130603" target="_blank">Mitchell et al.</a>
             </li>
           </ul>
           
        <h4>Updated Functional Annotation</h4>
        <div style="margin-left: 3em;">
        
            
        <a href="https://docs.google.com/spreadsheets/d/19V1EBijs67XCTgBW7UBvY5vEMHR1DAGVWVHIOC578ws/edit?usp=sharing" target="_blank">Summary of annotation updates</a> which include products, GO terms, EC numbers, gene names, synonyms, publications and notes from curators.
        <br><br>
        Functional annotations in FungiDB 36 have been updated for the following genomes:

        <style>
           #FunctAnnot_Bld36 table, #FunctAnnot_Bld36 td, #FunctAnnot_Bld36 th, #FunctAnnot_Bld36 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld36 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld36"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293</a></td>
             <td>210 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99</a></td>
             <td>220 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4d171eb9ea"><i>Cryptococcus neoformans</i> JEC21</a></td>
             <td>632 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_62f0e6170f"><i>Neurospora crassa</i> OR74A </a></td>
             <td>2890 genes</td>
           </tr>
           </table>
           <br>
       
       </div>
       

        <br><br>
       <h4>Curated User Comments:</h4>  
        Gene page comments that are entered by users are curated at GeneDB and regularly integrated as 
        functional annotations in FungiDB.  22 user comments have been reviewed and updated in FungiDB 36.
       <br><br>
        
        <div style="margin-left: 3em;">
        
        <style>
           #UComment_Bld36 table, #UComment_Bld36 td, #UComment_Bld36 th, #UComment_Bld36 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #UComment_Bld36 {
           margin-left : 5 em;
           }
         </style> 
           <table id="UComment_Bld36"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Comments Integrated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293 </a></td>
             <td> 2</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4d171eb9ea"><i>Cryptococcus neoformans</i> JEC21</a></td>
             <td> 15</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_62f0e6170f"><i>Neurospora crassa </i> OR74A</a></td>
             <td> 5</td>
           </tr>
          </table>
          <br>
          </div> 
          
          
        <h4>Updated Phenotype data for <i>Aspergillus</i> species:</h4>  
            <ul> 
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_7cdba4f78a">Manually curated <i>Aspergillus</i> Phenotypes </a> &ndash; 
                 The literature-curated phenotype data from AspGD have now been completely loaded onto FungiDB. 
                 The data have been reformatted in-line with EuPathDB's approach for phenotype annotation, where 
                 we follow an "entity + quality" approach. We have converted the phenotypes from AspGD-curated 
                 Ascomycete Phenotype Ontology (APO) to multiple ontologies: Phenotypic Quality Ontology (PATO) 
                 for "quality" terms and Gene Ontology (GO), Fungal Gross Anatomy Ontology (FAO), Chemical Entities 
                 of Biological Interest (ChEBI) and Ascomycete Phenotype Ontology (APO) for "entity" terms. We also 
                 plan to improve the phenotype table (in gene pages) and the query structure in the coming releases. 
                 See table below for details.
             </li>
           </ul>  
       <br><br>
        
        <div style="margin-left: 3em;">
        
        <style>
           #AspPheno_Bld36 table, #AspPheno_Bld36 td, #AspPheno_Bld36 th, #AspPheno_Bld36 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #AspPheno_Bld36 {
           margin-left : 5 em;
           }
         </style> 
           <table id="AspPheno_Bld36"> 
           <tr>
             <th>Species</th>
             <th>Phenotype mappings</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td><i>Aspergillus fumigatus</i> Af293</td>
             <td>1205</td>
             <td>282</td>
           </tr>
           <tr>
             <td><i>Aspergillus nidulans</i> FGSC A4</td>
             <td> 4881</td>
             <td> 1064</td>
           </tr>
           <tr>
             <td><i>Aspergillus niger</i> CBS 513.88</td>
             <td> 262</td>
             <td> 92</td>
           </tr>
           <tr>
             <td><i>Aspergillus oryzae</i> RIB40</td>
             <td> 219</td>
             <td> 73</td>
           </tr>
          </table>
          <br>
          </div> 
      </div>    
      </section>
      
	<br><br>
       <h4>Functional Annotations from Other Databases - <a href="https://scabusa.org/FgMutantDb" target="_blank">FgMutantDB</a></h4>
	A total 1,247 user comments matching 1,158 unique GeneIDs were loaded from FgMutantDb. Comments contain information pertaining <i>F. graminearum</i> mutants, gene products, publication references, <i>etc</i>. User comments can be searched by deploying a <i>New Search</i> for <i>Genes</i>, <i>Annotations, Curation, and Identifiers</i>, <i>User comments</i> and then limiting the search results on <i>F. graminearum</i>.
	<br><br>

       <section>
      <h3><b>New features in this release</b></h3>
            <div style="margin-left: 3em;">
      
      <h4>Search Page Improvements: search pages were redesigned to include an organism filter and provide better access to help information.</h4>
      
          <ul>
            <li>Search parameters are now left justified. The Get Answer button remains center justified below the parameters. Parameter help 
                icons, which offer information concerning the parameters when hovered over, now appear before the parameter name instead of after.
                </li>
            <li>The Organism Parameter, the first parameter on search pages, has a new look and now contains a filter box for quickly finding 
                your organism of interest.  Type any portion of an organism name (at leaset two letters of the genus, species, or strain) 
                in the filter box and the tree collapses to show 
                only those organisms containing your entry.  To return to the full tree, click the X in the filter box.  
                </li>
          </ul>
          
          <h4>New Paralog Count search</h4>
      
          <ul>
            <li>The new 
                <a href="/a/showQuestion.do?questionFullName=GeneQuestions.GenesByParalogCount">Paralog Count search</a>
                returns genes based on the number of assigned paralogs. Find genes that have a number of paralogs within the specified range. 
                Paralogs are defined as genes from the same species that belong to the same OrthoMCL group.
                </li>
            <li>Find the search under the Orthology and Synteny category.
                </li>
          </ul>
       
       <h4>Gene Record Page Improvements:</h4>
 
          <ul>
            <li>A new GO Slim table now appears above the GO Term table on the record pages of genes with GO associated data. 
                The GO Slim table maps GO Terms form the GO Term table to their GO Slim terms based on the 
                <a href="http://www.geneontology.org/page/go-slim-and-subset-guide" target="_blank">Generic GO slim developed by GO Consortium</a>.
                </li>
            <li>Column headers in gene page tables now show a help icon (
                <img src="/a/images/HelpIcon.png"  height="15px" width="16px">)
                when additional information or definitions are available 
                for the column. Hover over the help icons to reveal additional information.  
                </li>
            <li>Metabolic pathway and reaction tables on gene pages now include reaction compound names, with CHEBI IDs on mouse over, 
                and a column that indicates if the EC number associated with a gene is an exact match to the EC number in metabolic pathway.   
                </li>    
          </ul>

        
        <h4>Search Strategy Improvement: new option for combining steps in a search strategy. </h4>
       
          <ul>
            <li>Search strategies now offer an option to 'ignore' a step when editing a multi-step strategy. To remove a search 
                from the strategy without deleting the step, click Edit for the combine graphic of the step you are interested in ignoring. 
                Then choose 'IGNORE step #' from the Revise Operation popup. For more information see this 
                <a href="/a/images/ignoreStep.jpg">Ignore Step help document</a>.
                </li>
          </ul>
        </div>
      </section>
     
      <section>
      <h3><b>Introducing ClinEpiDB, a new member of the EuPathDB family of databases:</b></h3>
      <div style="margin-left: 3em;">
      
          <ul>
            <li>We are excited to announce the launch of 
                <a href="http://www.clinepidb.org/" target="_blank">ClinEpiDB</a>, 
                which hosts clinical and epidemiological data for the infectious disease research community. The current release contains data from the 
                <a href="https://www.niaid.nih.gov/research/east-africa-icemr" target="_blank">PRISM</a> 
                study on malaria in Uganda. Check out this 
                <a href="https://www.youtube.com/watch?v=535PcFrBH8M&feature=youtu.be" target="_blank">video</a> 
                to learn more about our newest EuPathDB resource!
                </li>
          </ul>
       </div>
      </section>  
     
     
      </section>
      
     <section>
     <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
