---
layout: post
title: FungiDB 42 Released
category: [EuPathDB,VEuPathDB,FungiDB]
---

<p><strong>We are pleased to announce the release of FungiDB 42</strong></p>
    
    <section>
    <h3><b>Release 42 Webinar</b></h3>
    <div style="margin-left: 3em;">
      Join us on March 20, 2019 10:00-11:00 AM EDT for our Release 42 Highlights webinar where we will discuss and 
      demonstrate new features and data available in release 42 of EuPathDB sites. 
      <a href="https://attendee.gotowebinar.com/register/6755367952348158722">Register here</a>
    </div>
    </section>
      
      <section>
        <h3><b>New data in FungiDB 42</b></h3>
        <div style="margin-left: 3em;">
        
        <h4>Genome Sequence and Annotation: these annotated genomes appear for the first time in FungiDB.</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_778f9987a4"><i>Kwoniella dejecticola</i> CBS10117</a>  
             </li>
             
             <li><a href="/a/app/record/dataset/DS_7acdb4b9a1"><i>Kwoniella bestiolae</i> CBS 10118</a>  
             </li>
             
             
             <li><a href="/a/app/record/dataset/DS_e983d1d491"><i>Fusarium oxysporum</i> FOSC 3-a</a> 
             </li>
             
             
            </ul>
           <br>
           
        <h4>Genome Sequence and Annotation: updated genomes</h4> 
         	<ul>
        	  <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_f670f03393"><i>Cryptococcus neoformans</i> var. grubii KN99</a> &ndash; 
                 Gene mapping and functional annotation were re-mapped using the update files from GenBank
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29305388">Friedman et al. 2018</a>.
             </li>  
             
             <li><a href="/a/app/record/dataset/DS_d1767fcbd3"><i>Candida parapsilosis</i> CDC 317</a> &ndash;  
                 The genome was reloaded to include the mitochondrial genome 
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/19465905">Butler et al. 2009</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_62f0e6170f"><i>Neurospora crassa</i> OR74A</a> &ndash;  
                 The genome was reloaded. Incorrect gene aliases reflecting The Broad Institution v4 (June 2010) gene set 
                 were previously loaded to FungiDB due to two dates of issue (February and June). 
                 This was rectified with remapping of these aliases using only the February version gene ids. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/12712197">Galagan et al. 2009</a>.
             </li>
            </ul>
           <br>    
           
        <h4>New Genome Sequence: available for the first time in FungiDB:</h4>
        <ul>
        	<li><a href="/a/app/record/dataset/DS_ed04aca700"><i>Naganishia albida</i> NRRL Y-1402</a> &ndash;  
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/27198024">Vajpeyi et al. 2016</a>.
        	</li>
        
        	<li><a href="/a/app/record/dataset/DS_812930c840"><i>Phytopthora plurivora</i> AV1007</a> &ndash;  
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/30060094">Vetukuri et al. 2018</a>.
        	</li>
                        
            </ul>
           <br>
           
        <h4>Transcriptomics</h4>
           <ul>
           
           <li><a href="/a/app/record/dataset/DS_ecba2dec39">Transcriptome profiling of <i>Candida auris</i> clinical isolates forming biofilm</a> &ndash;
             Transcriptomic investigation of isolates collected by Public Health England, UK using Illumina HiSeq X Ten technology.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29997121" target="_blank">Kean et al. 2018</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_b6fe0c017e">Transcriptome analysis of <i>Aspergillus fumigatus</i> grown in blood</a> &ndash;
             Gene expression profile at various stages of an <i>in vitro</i> model of aspergillosis.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/26311470" target="_blank">Irmer et al. 2015</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_3190d69ecb">Transcriptomes of itraconazole-resistant strains derived from A1160, HapB and 29.9.</a> &ndash;
             RNA-Seq of <i>Aspergillus fumigatus</i> mutants resistant to itraconazole.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_139dce39e2">Stranded RNA-Seq of WT and cqs1delta strains during sexual development stage</a> &ndash;
             Stranded RNA-Seq of <i>C. neoformans</i> JEC21 WT (XL280&alpha;) and cqs1&Delta; strains grown on V8 media.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29784977" target="_blank">Tian et al. 2018</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_9c905047fb"><i>Cryptococcus neoformans</i> sexual reproduction is controlled by a quorum sensing peptide</a> &ndash;
             Transcriptomic analysis of gene expression in <i>Cryptococcus neoformans</i> var neoformans targeting pheromone pathways and quorum sensing.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29784977" target="_blank">Tian et al. 2018</a>.
             </li>
             <li><a href="/a/app/record/dataset/DS_3de13093fd">Transcriptomics of banana pathogen <i>F. oxysporum f.sp. cubense</i> race 4</a> &ndash; 
                 Strand specific RNA-Seq analysis of <i>Fusarium oxysporum f. sp. cubense</i> race 4 infecting <i>Musa acuminata</i> at 0h and 48h.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24743270" target="_blank">Guo et al. 2014</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_00b5344451">Transcriptome of heat-stressed <i>Fusarium graminearum</i> Z-3639</a> &ndash;
             Illumina HiSeq 2000 of 24 h-old mycelia from complete medium incubated in 25C and 37C for 15 min. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/27306495" target="_blank">Bui et al. 2016</a>.
             </li>
        	</ul>
        	<br>
        	          
        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_d33a14b7e6">Aligned genomic sequence reads - <i>Candida auris</i> isolates</a> &ndash;
                 Multidrug-resistant <i>Candida auris</i> isolates were collected from on 3 continents and sequenced Illumina HiSeq 2500 technology.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/27988485">Lockhart et al. 2017</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_11cc0909dd">SNP calls on <i>Paracoccidioides lutzii</i> WGS</a> &ndash;
                 Whole genome sequencing of <i>P. lutzii</i> isolates.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/27704050">Mu&ntilde;oz et al. 2018</a>.
             </li>
           </ul>
           <br>
           
        <h4>Phenotype data</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_61efdacf5b">Manually Curated <i>Fusarium</i> Phenotypes</a> &ndash; 
                 The phenotypes for <i>Fusarium graminearum</i> PH-1 are manually curated from published literature describing phenotypes obtained from mutant studies. 
                 We have curated and added phenotypes from <a href="https://www.ncbi.nlm.nih.gov/pubmed/22028654" target="_blank">Son et al. 2011</a> publication 
                 on transcription factors to the currently existing Fusarium phenotypes from <a href="https://www.ncbi.nlm.nih.gov/pubmed/25758923" target="_blank">Yun et al. 2007</a>.
                 We followed an &quot;entity &#43; quality&quot; approach and curated phenotypes with multiple ontologies: 
                 Phenotypic Quality Ontology (PATO) for &quot;quality&quot; terms and Gene Ontology (GO), Fungal Gross Anatomy Ontology (FAO),
                 Chemical Entities of Biological Interest (ChEBI) and Ascomycete Phenotype Ontology (APO) for &quot;entity&quot; terms. 
             </li>
             </ul>
             <br>
               
        <div style="margin-left: 3em;">
        
        <style>
           #Phenotype_Bld42 table, #Phenotype_Bld42 td, #Phenotype_Bld42 th, #Phenotype_Bld42 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #Phenotype_Bld42 {
           margin-left : 5 em;
           }
         </style> 
           <table id="Phenotype_Bld42"> 
           <tr>
             <th>Species</th>
             <th>Phenotype mapping</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td><i>Fusarium graminearum</i> PH-1</td>
             <td>11,556</td>
             <td>642</td>
           </tr>
          </table>
          <br>
          </div> 
             
           <br>
           
           <ul>
             <li><a href="/a/app/record/dataset/DS_43c139682a">Manually Curated <i>Cryptococcus</i> Phenotypes</a> &ndash; 
                 The phenotypes for <i>Cryptococcus neoformans</i> JEC21 and H99 and <i>Cryptococcus gattii</i> WM276 genomes are manually 
                 curated from published literature describing phenotypes obtained from mutant studies. The curated phenotypes from 
                 <a href="https://docs.google.com/spreadsheets/d/1m7hLDZjphgKJTFJeDx20nA7CVjm0eLQfyMjjRK64G5g/edit?usp=sharing" target="_blank"><i>Cryptococcus</i> RO3 Project </a> are available in FungiDB as controlled vocabularies since release 25. 
                 We have mapped these phenotypes into ontologies in-line with EuPathDB's approach for phenotype annotation.
                 We followed an &quot;entity &#43; quality&quot; approach and curated phenotypes with multiple ontologies: 
                 Phenotypic Quality Ontology (PATO) for &quot;quality&quot; terms and Gene Ontology (GO), Fungal Gross Anatomy Ontology (FAO),
                 Chemical Entities of Biological Interest (ChEBI) and Ascomycete Phenotype Ontology (APO) for &quot;entity&quot; terms. 
             </li>
              </ul>
             <br>
              
        <div style="margin-left: 3em;">
        
        <style>
           #Phenotype_Bld42 table, #Phenotype_Bld42 td, #Phenotype_Bld42 th, #Phenotype_Bld42 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #Phenotype_Bld42 {
           margin-left : 5 em;
           }
         </style> 
           <table id="Phenotype_Bld42"> 
           <tr>
             <th>Species</th>
             <th>Phenotype mapping</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td><i>Cryptococcus gattii</i> WM276</td>
             <td>14</td>
             <td>3</td>
           </tr>
           
           <tr>
             <td><i>Cryptococcus neoformans</i> var grubii H99</td>
             <td>1,968</td>
             <td>422</td>
           </tr>
           
           <tr>
             <td><i>Cryptococcus gattii</i> var neoformans JEC21</td>
             <td>526</td>
             <td>128</td>
           </tr>
          </table>
          <br>
          </div> 
             
           </ul>
           <br>
           
           <h4>Link Outs</h4>
           Link outs to the NCBI RefSeq, CAZy, and MEROPS databases were integrated for <i>Aspergillus nidulans</i>, <i>A. niger</i> and <i>A. oryzae</i>, and link outs to RefSeq and MEROPS databases were integrated for 
           <i>A. fumigatus</i>. 
           <br>
           
           
           <h4>Updated Functional Annotation</h4>
        <div style="margin-left: 3em;">
                    
        <a href="https://docs.google.com/spreadsheets/d/1RynjM06XLt05ijeFQpA-cbJtUnQoHbex8zs2YKj-IOI/edit?usp=sharing" target="_blank">Summary of annotation updates</a> which include products, GO terms, EC numbers, gene names, synonyms, publications, external database references, and notes from curators.
        <br><br>
        Functional annotations have been updated for the following genomes:

        <style>
           #FunctAnnot_Bld42 table, #FunctAnnot_Bld42 td, #FunctAnnot_Bld42 th, #FunctAnnot_Bld42 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld42 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld42"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293 </a></td>
             <td>9621 </td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4b35c88aa1"><i>Aspergillus nidulans</i> FGSC A4 </a></td>
             <td>9063</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_e3b988685c"><i>Aspergillus niger </i> CBS 513.88 </a></td>
             <td>10138</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_07d4b5c58b"><i>Aspergillus oryzae </i> ORIB40</a></td>
             <td>11783</td>
           </tr>
           </table>
           <br>
       </div>
 
       <h4>Curated User Comments:</h4>  
        Gene page comments that are entered by users are curated and integrated as functional annotations in FungiDB. 19 user comments have been reviewed and updated in FungiDB 42.
       <br><br>
        
        <div style="margin-left: 3em;">
        
        <style>
           #UComment_Bld42 table, #UComment_Bld42 td, #UComment_Bld42 th, #UComment_Bld42 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #UComment_Bld42 {
           margin-left : 5 em;
           }
         </style> 
           <table id="UComment_Bld42"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Comments Integrated</th>
           </tr>
           <tr>
             <td title="Click here to view this dataset's full description."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4b35c88aa1"><i>Aspergillus nidulans</i> FGSC A4</a></td>
             <td>18</td>
           </tr>
           
           <tr>
             <td title="Click here to view this dataset's full description."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_e3b988685c"><i>Aspergillus niger</i> CBS 513.88</a></td>
             <td>1</td>
           </tr>
          </table>
          <br>
          </div> 
           <br>
           
       </div>    
       </section>
       
       <section>
      <h3><b>New features in FungiDB 42</b></h3>
      <div style="margin-left: 3em;">
         <ul>
           <li>
             Updated Alignment Tool on Gene Pages &ndash; <br>
             The Clustal Omega alignment tool in the 'Orthologs and Paralogs within EuPathDB' table now 
             accommodates protein, CDS and genomic sequence alignments. To run Clustal Omega, select genes from the table. Then choose the sequence 
             type and initiate the alignment with the 'Run Clutal Omega for selected genes' button. Output formats include Mismatches highlighted, FASTA, PHYLIP, STOCKHOLM, 
             and VIENNA. This tool replaces the 'Retrieve multiple 
             sequence alignment or multi-FASTA' tool.
           </li>
           <li>
             Beta access to JBrowse &ndash; <br>
             A beta implementation of the 
             <a href="https://jbrowse.org/" target="_blank">JBrowse Genome Browser</a>, is available from gene pages. Use the 
             JBrowse Genome Browser link under the graphic Shortcuts on the gene page. 
             <a href="/assets/JBrowseLink.png">Example Link Location</a>
           </li>
           <li>
             New Gene Page STRING tool &ndash; <br>
             The 'Protein features and properties' section now includes an option to submit the protein sequence 
             to the <a href="https://string-db.org/" target="_blank">STRING</a> functional protein association network. 
             <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210637/" target="_blank">Szklarczyk et al</a>.
           </li>
           <li>
             Result Table Improvements &ndash; <br>
             Result column configurations and sorting are now saved for each strategy step and will be retained 
             when you come back or when you share a strategy with someone else.
           </li>
           <li>
             New Options for FASTA downloads &ndash; <br>
             The header line of FASTA formatted downloads can be configured to contain only the gene ID.  The sequence can 
             be configured to appear on a signle line or the standard 60 characters per line. 
           </li>
         </ul>
       </div> 
      </section>
       
       <section>
     <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
