---
layout: post
title: FungiDB 44 Released
category: [FungiDB]
---

<p><strong>We are pleased to announce the release of FungiDB 44</strong></p>
    
    <section>
    <h3><b>Release 44 Webinar</b></h3>
    <div style="margin-left: 3em;">
      Join us on Tuesday July 16th at 10:00 am (EST) for our Release 44 Highlights webinar where we will discuss and 
      demonstrate new features and data available in release 44 of EuPathDB sites. 
      <a href="https://attendee.gotowebinar.com/register/7430028289888360460">Register here</a>
    </div>
    </section>
      
      <section>
        <h3><b>New data in FungiDB 44</b></h3>
        <div style="margin-left: 3em;">
        

        <h4>Genome Sequence and Annotation: these annotated genomes appear for the first time in FungiDB.</h4>
           <ul>
             
             <li><a href="/a/app/record/dataset/DS_6438c3f349"><i>Cladophialophora immunda</i> CBS 83496</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>.  
             </li>
             
             <li><a href="/a/app/record/dataset/DS_248d79bc8e"><i>Rhizophagus irregularis</i> DAOM 181602 (DAOM 197198)</a> &ndash;  
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29355972">Chen et al. 2018</a>,
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/27572831">Ropars et al. 2016</a>,
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24277808">Tisserant et al. 2013</a>.
             </li>
            
            </ul>
           <br>
           
           
           <h4>New Genome Sequence: available for the first time in FungiDB:</h4>
        <ul>
        	<li><a href="/a/app/record/dataset/DS_ab94e95077"><i>Amauroascus niger</i> UAMH 3544</a> &ndash;  
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26613950">Whiston et al. 2015</a>.
        	</li>
        
                        
            </ul>
           <br>
                      
        <h4>Transcriptomics</h4>
           <ul>
                          
             <li><a href="/a/app/record/dataset/DS_795f56c575"><i>Aspergillus fumigatus</i> transcriptome under normoxia and hypoxia conditions</a> &ndash;
             RNAseq analysis of the <i>Aspergillus fumigatus</i> was used to determine its response to early hypoxic conditions.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/24996522" target="_blank">Losada et al. 2014</a>.
             </li>
             
                         
            <li><a href="/a/app/record/dataset/DS_4fe013954b">Transcriptome landscape of <i>Candida glabrata</i> during pH change and nitrosative stress</a> &ndash;
             RNA-Seq-based analysis of the transcriptomic landscape of <i>C. glabrata</i> in nutrient-rich media, under nitrosative stress, and during pH shift.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/25586221" target="_blank">Linde et al. 2015</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_b9dffe2a12">Microarray analysis of fruiting body development in <i>Coprinopsis cinerea</i></a> &ndash;
             Meiosis was compared in <i>rad50-1</i> and wild type strains of <i>C. cinerea</i> at 6 time points spanning the meiotic timecourse.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/23050232" target="_blank">Anderson et al. 2012</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_9572ee47dc">O2-responsive genes in the human pathogenic fungus <i>Aspergillus fumigatus</i></a> &ndash;
             Transcriptomic investigation of low oxygen adaptation of <i>A. fumigatus</i>.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/24948085" target="_blank">Hillmann et al. 2014</a>.
             </li>
             
             
            <li><a href="/a/app/record/dataset/DS_b8793f4345"><i>Histoplasma capsulatum</i> G186A and G217A yeast and mycelial transcriptomes</a> &ndash;
             Yeast and mycelial transcriptome of <i>Histoplasma capsulatum</i> G186A and G217A strains.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/24112604" target="_blank">Edwards et al. 2013</a>.
             </li>
             
            <li><a href="/a/app/record/dataset/DS_c45818957f">Gene expression analysis of ADV-1 and PP-1 mutants in <i>Neurospora crassa</i></a> &ndash;
             The transcription factors ADV-1 and PP-1 are essential for germling communication and fusion and <i>Neurospora crassa</i>.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29678830" target="_blank">Fischer et al. 2018</a>.
             </li>
             
                          
        	</ul>
        	<br>
        	          
                   
        <h4>Phenotype data</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_43c139682a">Manually curated <i>Cryptococcus</i> Phenotypes</a> &ndash; 
                 The phenotypes for <i>Cryptococcus neoformans</i> JEC21 and H99 and <i>Cryptococcus gatii</i> WM276 are manually curated from published literature describing 
                 phenotypes obtained from mutant studies. We have curated and added phenotypes from <a href="https://www.ncbi.nlm.nih.gov/pubmed/27677328" target="_blank">Lee et al</a>.  
                 publication on kinases to the currently existing phenotypic dataset for <i>C. neoformans</i> H99.
                 We followed an &quot;entity &#43; quality&quot; approach and curated phenotypes with multiple ontologies: 
                 Phenotypic Quality Ontology (PATO) for &quot;quality&quot; terms and Gene Ontology (GO), Fungal Gross Anatomy Ontology (FAO),
                 Chemical Entities of Biological Interest (ChEBI) and Ascomycete Phenotype Ontology (APO) for &quot;entity&quot; terms. 
             </li>
             </ul>
             <br>
               
        <div style="margin-left: 3em;">
        
        <style>
           #Phenotype_Bld44 table, #Phenotype_Bld44 td, #Phenotype_Bld44 th, #Phenotype_Bld44 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #Phenotype_Bld44 {
           margin-left : 5 em;
           }
         </style> 
           <table id="Phenotype_Bld44"> 
           <tr>
             <th>Species</th>
             <th>New phenotype mappings</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td><i>Cryptococcus neoformans</i> var grubii H99</td>
             <td>4027</td>
             <td>130</td>
           </tr>
          </table>
          <br>
          </div> 
             
             <br>
             
      <h4>Updated Functional Annotation</h4>
               <div style="margin-left: 3em;">
             
             <a href="https://docs.google.com/spreadsheets/d/1IoBVcv4WeydzkLs531eRQmI7JZY0bHLZ8yj-xmPQNaQ/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include products, GO terms, EC numbers, gene names, synonyms, publications, external database references and notes from curators. These functional annotations are current with GeneDB June 2019 GFF versions. <br><br>
             
             <style>
                #FunctAnnot_Bld44 table, #FunctAnnot_Bld44 td, #FunctAnnot_Bld44 th, #FunctAnnot_Bld44 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld44 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld44"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99 </a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-06/CneoformansH99/CneoformansH99.gff.gz">GeneDB version June, 2019</a></td>
                  <td>255</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4d171eb9ea"><i>Cryptococcus neoformans</i> JEC21 </a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-06/CneoformansneoformansJEC21/CneoformansneoformansJEC21.gff.gz  ">GeneDB version June, 2019</a></td>
                  <td>267</td>
                </tr>
                </table>
                <br>    
                </div>

       
       <h4>Annotations from Gene Ontology Consortium (GOC) resources:</h4>
       <div style="margin-left: 3em;"> 
        
        Curated and computed gene ontology (GO) annotations from Gene Ontology Consortium (GOC) resources have been integrated into FungiDB in Release 44 for 
        <i>Candida</i> genomes. These GO annotations are current with GOC version 17 April 2019. 
        The processed GAF files from which GO annotations were loaded onto FungiDB can be found in the public folder linked  
        <a href="https://github.com/Achchuthan/FungiDB-Curation/tree/master/Gene%20Ontology%20Annotations%20from%20GOC%20resources/Release%2044" target="_blank">here</a>.
       <br><br>
        
        
        <style>
           #GOC_Bld44 table, #GOC_Bld44 td, #GOC_Bld44 th, #GOC_Bld44 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #GOC_Bld44 {
           margin-left : 5 em;
           }
         </style> 
           <table id="GOC_Bld44"> 
           <tr>
             <th>FungiDB Genome</th>
             <th>Major contributing resource</th>
             <th>GO annotations</th>
           </tr>
           <tr>
             <td title="Click here to view this dataset's full description"><a href="https://fungidb.org/fungidb/app/record/dataset/DS_0d9ee4ff86"><i>Candida albicans</i></i> SC5314</a></td>
             <td title="Click here to be redirected to CGD"><a href="http://www.candidagenome.org/">CGD</a></td>
             <td>56,203</td>
           </tr>
           
           <tr>
             <td title="Click here to view this dataset's full description"><a href="https://fungidb.org/fungidb/app/record/dataset/DS_e43fc5921f"><i>Candida albicans</i></i> SC5314_B</a></td>
             <td title="Click here to be redirected to CGD"><a href="http://www.candidagenome.org/">CGD</a></td>
             <td>38</td>
           </tr>
           
           <tr>
             <td title="Click here to view this dataset's full description"><a href="https://fungidb.org/fungidb/app/record/dataset/DS_033b09f437"><i>Candida albicans</i></i> WO-1</a></td>
             <td title="Click here to be redirected to CGD"><a href="http://www.candidagenome.org/">CGD</a></td>
             <td>19,037</td>
           </tr>
           
           <tr>
             <td title="Click here to view this dataset's full description"><a href="https://fungidb.org/fungidb/app/record/dataset/DS_c8c07fac9f"><i>Candida glabrata</i></i> CBS 138</a></td>
             <td title="Click here to be redirected to CGD"><a href="http://www.candidagenome.org/">CGD</a></td>
             <td>36,531</td>
           </tr>
           
           <tr>
             <td title="Click here to view this dataset's full description"><a href="https://fungidb.org/fungidb/app/record/dataset/DS_d1767fcbd3"><i>Candida parapsilosis</i></i> CDC317</a></td>
             <td title="Click here to be redirected to CGD"><a href="http://www.candidagenome.org/">CGD</a></td>
             <td>39,202</td>
           </tr>
           
           <tr>
             <td title="Click here to view this dataset's full description"><a href="https://fungidb.org/fungidb/app/record/dataset/DS_adb102b7b5"><i>Candida tropicalis</i></i> MYA_3404</a></td>
             <td title="Click here to be redirected to CGD"><a href="http://www.candidagenome.org/">CGD</a></td>
             <td>19,122</td>
           </tr>
           
           <tr>
             <td title="Click here to view this dataset's full description"><a href="https://fungidb.org/fungidb/app/record/dataset/DS_783046b845"><i>Clavispora lusitaniae</i></i> ATCC 42720</a></td>
             <td title="Click here to be redirected to CGD"><a href="http://www.candidagenome.org/">CGD</a></td>
             <td>17,756</td>
           </tr>
          </table>
          <br>
          </div> 
           <br>
           
       </div>    
       </section>
       
       <section>
      <h3><b>New features in FungiDB 44</b></h3>
      <div style="margin-left: 3em;">
         <ul>
           <li>
             Synteny and the Protein Browser are availabe in our beta implementation of the 
             <a href="https://jbrowse.org/" target="_blank">JBrowse Genome Browser</a>. 
           </li>
           <li>
             New EuPathDB RNA Seq Tool. The EuPathDB Galaxy Workspace now offers a tool that converts 
             read counts from HTSeq-count output to FPKM values. 
           </li>
           <li>
             New EuPathDB Workflows. The EuPathDB Galaxy Workspace is updated with several new 
             pre-configured workflows that are designed to allow export of strand specific data 
             to EuPathDB for visualization and searching.
           </li>
         </ul>
       </div> 
      </section>
       
       <section>
     <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
