---
layout: post
title: FungiDB 55 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 55.</h3>

<section>
  <h3>Free online workshop </h3>
  <div style="margin-left: 3em;">
  Free 4-day virtual VEuPathDB workshop: December 13-16, 2021, 9AM-1PM US Eastern time. Preliminary schedule is available on <a href="/a/app/static-content/workshops.html#DECveupath2021">our workshops page</a>. If you are interested and able to attend, please <b><a href="https://upenn.co1.qualtrics.com/jfe/form/SV_0Il5yFqOJ056ENU">register</a></b> by December 6th, 2021.
</div>
</section>

<section>
  <h3>Another way to stay in touch:</h3>
  <div style="margin-left: 3em;">
    The bioinformatic resource centers have established a <a href="https://www.reddit.com/r/BRC_users" target="_blank">Reddit community</a> for free and open discussions. We welcome everyone to join the discussion.
  </div>
</section>

<section>
  <h3>Release 55 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday December 9, 2021 at 10am EDT for our Release 55 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 55.  
    <a href="https://attendee.gotowebinar.com/register/3655307034315220750" target="_blank">Register here</a>  
  </div>
</section>

    <section>
      <h3>New data in FungiDB 55</h3>
      <div style="margin-left: 3em;">



      <h4>Genome Sequence and Annotation: </h4>
      <ul>
        
        <li><a href="/a/app/record/dataset/DS_d36b2dd162"><i>Arthrobotrys flagrans</i> CBS H-5679</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30917129">Youssar et al. 2019</a>.  
           </li>

        <li><a href="/a/app/record/dataset/DS_2d927856d1"><i>Candida glabrata</i> BG2</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33713372">Xu et al. 2021</a>.  
            </li>
  
          <li><a href="/a/app/record/dataset/DS_856ead0bd3"><i>Coprinopsis cinerea</i> #326</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/33253902">Xie et al. 2021</a>.  
              </li>
  
        <li><a href="/a/app/record/dataset/DS_3047cb4c05"><i>Debaryomyces hansenii</i> CBS767</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/15229592">Dujon et al. 2014</a>.  
            </li>

        <li><a href="/a/app/record/dataset/DS_c7fbbf0487"><i>Diutina rugosa</i> CBS 613</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/31575637">Mixão et al. 2019</a>.  
              </li>

         <li><a href="/a/app/record/dataset/DS_84d147e42f"><i>Emergomyces pasteurianus</i> Ep9510</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/28176377">Dukik et al. 2017</a>.  
              </li>

        <li><a href="/a/app/record/dataset/DS_f030a38781"><i>Pneumocystis canis</i> CanA</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/33686174">Cissé et al. 2021</a>.  
              </li>

        <li><a href="/a/app/record/dataset/DS_712eb6bf46"><i>Pneumocystis canis</i> P2C</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/33686174">Cissé et al. 2021</a>.  
              </li>

        <li><a href="/a/app/record/dataset/DS_f176967403"><i>Phytophthora capsici </i> LT1534 (2021)</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/34042486">Stajich et al. 2021</a>.  
                </li>

        <li><a href="/a/app/record/dataset/DS_6397b8a66b"><i>Taphrina deformans</i> PYCC 5710</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23631913">Cissé et al. 2013</a>.  
              </li>    
      </ul>
      <br>

      <h4>Genome Sequence: </h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_f3975acb19"><i>Apophysomyces variabilis</i> BNCCPF 102052</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/28923009">Prakash et al. 2017</a>.  
            </li>
            
        <li><a href="/a/app/record/dataset/DS_b59dfb5341"><i>Lichtheimia ramosa</i> KPH11</a> &ndash;
          Jeong-Ah Seo et al. 2019.  
           </li>
      </ul>
      <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>

        <ul>
          <li><i>Candida albicans</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_a104c5de28">SNP calls on WGS of Candida albicans collected from old oaks and NCYC 597 clinical strain</a> &ndash;
              Short-read genome data were generated for the type strain of <i>C. albicans</i> (NCYC 597) and three strains isolated from the bark of oak trees in the New Forest in the United Kingdom.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/30463870">Bensasson et al. 2019</a>.
            </li>

            <li>
                <a href="/a/app/record/dataset/DS_325f546077">SNP calls on WGS of 182 isolates of Candida albicans collected worldwide</a> &ndash;
                Isolates were collected worldwide, and include commensal isolates, as well as ones responsible for superficial and invasive infections.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/29884848">Ropars et al. 2018</a>.
            </li>
            </ul>
        </ul>    

        <ul>
          <li><i>Candida auris</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_ca480bb83f">SNP calls on WGS of Candida auris isolates from the coastal wetlands of Andaman Islands, India</a> &ndash;
              13 Candida auris strains isolated in the present study from the South Andaman district, Andaman & Nicobar Islands, Union Territory of India.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/33727354">Aurora et al. 2021</a>.
            </li>

              <li>
                <a href="/a/app/record/dataset/DS_baf9ed83c4">SNP calls on WGS of laboratory evolved C. auris strains</a> &ndash;
                Experimental evolution assays using a fluconazole-susceptible isolate of C. auris (BJCA001).
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33077664">Bing et al. 2020</a>.
              </li>
            </ul>
        </ul>
        
        <ul>
          <li><i>Fusarium graminearum</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_ef7873e854">SNP calls on WGS of six F. graminearum strains collected from wheat fields in France</a> &ndash;
                Sequencing analysis of F. graminearum strains collected in France.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/28231761">Ropars et al. 2017</a>.
              </li>
            </ul>
          
        </ul>
          
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Clavispora lusitaniae </i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_37390faca3">Expression profiles of fluconazole resistant and fluconazole sensitive clinical isolates</a> &mdash;
          The expression profiles of fluconazole resistant (U) and fluconazole sensitive (L) clinical isolates, from the same patient, all containing mutation in the trancription factor Mrr1.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30389707" target="_blank">Demers et al. 2018</a>.  
      </li>
      </ul>

     <li><i>Aspergillus fumigatus</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_f6dd8e44a3">Gene expression in WT, hrmA deletion, hrmA OE, hrmA_REV, EVOL under hypoxia and normoxia conditions</a> &mdash;
          Gene expression differences in Aspergillus fumigatus wild type, hrmA mutant, and overexpression under hypoxia and normoxia conditions.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31548684" target="_blank">Kowalski et al. 2019</a>.  
      </li>
      </ul>
    </ul>

        <h4>ChIP-chip</h4>
      <ul>
        <li><i>Histoplasma capsulatum</i></li>
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_2dd5798f04">ChIP-chip investigation of loci bound by Ryp1, Ryp2, Ryp3, and Ryp4</a> &mdash;  
            Genome binding/occupancy profiling by genome tiling array.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23935449" target="_blank">Beyhan et al. 2013</a>.  
        </li>
      </ul>
  
    </ul>   
    <br>
             
        <h4>Updated Functional Annotation</h4>
       		<ul>
             <li>
             <a href="https://docs.google.com/spreadsheets/d/1VjDxtbhuYq_AxaLMBVKYkdWINYiWT2XiDnP2YePwoU0/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include gene names, products, GO terms, publications and notes from curators. These functional annotations are current with GeneDB October 2021 GFF versions.
            </li>
            </ul> 
            <br>
               
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld55 table, #FunctAnnot_Bld55 td, #FunctAnnot_Bld55 th, #FunctAnnot_Bld55 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld55 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld55"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-10/AfumigatusAf293/AfumigatusAf293.gff.gz">GeneDB version October, 2021</a></td>
                  <td>208</td>
                </tr>
                </table>
                </div>
          <br>
     </div>
</section>
        
<section>
  <h3>New features in {{project}} 55</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>My Organism Preferences is a new option for filtering menus to a subset of organisms in {{project}}. Use the <a href="/a/app/preferred-organisms">My Organism Preferences page</a> to choose the organisms you are interested in.  {{project}} will function as if the database contained only your chosen organisms.  The feature is easily disabled/enabled at any time from a button located under the header on every page. <a href="/a/app/static-content/faq.html#organismpreferences"> More Details</a> </li>

    <li>The Reference sequence track in JBrowse has a new color scheme. <a href="{{'/assets/images/JBrowse-new-colors.gif' | absolute_url}}">View a tutorial GIF.</a>
      <ul>
        <li>Purines appear in shades of blue while pyrimidines appear in shades of yellow.</li>
        <li>Stop codons are red while methionines are green.</li>
        <li>The six frame translations appear in gray.</li>
        <li>Nucleotides and amino acids are distinguished by color at low zoom.  Under high zoom they are also labeled with single letter codes.</li>
      </ul>
    </li>

    <li>Galaxy Version updated on 8 Oct, 2021: This updated version of the Galaxy software improves job and workflow performance. In addition to the previous set of tools and workflows, we now offer additional tools, such as those for scRNA-seq (Scanpy, Seurat), species ID (CryptoGenotyper), table manipulation (Datamash), and proteomics (MSstats, Search GUI, Peptide Shaker, MaxQuant). Certain tools have been upgraded and now function without errors: bamCoverage, bgzip, tabix and Upload File. </li>

    <li>Enhanced searching on gene record pages &ndash; The content navigation tool now accesses detailed descriptions of {{project}} gene page content, making it easier to find the content that you need.</li>
    </ul>
  </div>
</section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
