---
layout: post
title: FungiDB 60 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 60.</h3>

<section>
  <h3>Release 60 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday November 17th, 2022 at 10am EST for our Release 60 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 60.  
    <a href="https://attendee.gotowebinar.com/register/8994775200159032332" target="_blank">Register here</a>  
  </div>
  <br>
</section>

<section>
  <h3>Pseudogene Classification</h3>
  <div style="margin-left: 3em;">
    In release 51 (16 March 2021) pseudogenes with an internal stop codon automatically included predicted 3'UTRs. The UTR prediction was due to an incorrect pipeline which has now been rectified. UTRs have now been removed and all pseudogenes are correctly displayed without CDS and UTR. The affected pseudogenes were shown in VEuPathDB with the Biotype classification pseudogene_with_CDS. This has now been changed to the correct biotype pseudogene. <a href="https://veupathdb.org/veupathdb/app/workspace/strategies/import/f9db1d88ac085282" target="_blank">List of affected genes</a>. </div>
</section>

    <section>
      <h3>New data in FungiDB 60</h3>
      <div style="margin-left: 3em;">

        <h4>Genome Updates:</h4>  The following genomes were updated to the INSDC versions to avoid small discrepancies between public versions.
        <ul>
          <li><a href="/a/app/record/dataset/DS_0d9ee4ff86"><i><i>Coccidioides posadasii</i> strain Silveira 2020</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_018416015.2" target="_blank"> GCA_018416015.2</a>.</li>

       </ul>
      <br>

      <h4>Genome Sequence and Annotation </h4>
      <ul>
       <li><a href="/a/app/record/dataset/DS_5d4f2125d2"><i>Aspergillus parasiticus</i> CBS 117618</a> &ndash;
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/32107379">Vesth et al. 2018</a>.  
          </li>

       <li><a href="/a/app/record/dataset/DS_f6bf2f9a44"><i>Fusarium vanettenii</i> 77-13-4</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/19714214">Coleman et al. 2009</a>.  
               </li>

       <li><a href="/a/app/record/dataset/DS_dfd1dbb3d2"><i>Lichtheimia corymbifera</i> JMRC:FSU:9682</a> &ndash;
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/25121733">Schwartze et al. 2014</a>.
           </li>

        <li><a href="/a/app/record/dataset/DS_1e6b9dc02c"><i>Phaeosphaeria nodorum</i> SN15</a> &ndash;
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/34034667">Bertazzoni et al. 2021</a>.
            </li>
  
         <li><a href="/a/app/record/dataset/DS_68a1bc4efa"><i>Phytophthora cinnamomi </i> GKB4</a> &ndash;
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/33902447">Engelbrecht et al. 2021</a>. 
             </li>
  
        <li><a href="/a/app/record/dataset/DS_a75d620d3c"><i>Podospora comata</i> strain T mat+</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30288581">Silar et al. 2019</a>. 
         </li>
          
         <li><a href="/a/app/record/dataset/DS_9fc597f917"><i>Rhizopus microsporus</i> var. <i>microsporus</i></i> ATCC 52814</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33245689">Lastovetsky et al. 2016</a>
           </li>

         <li><a href="/a/app/record/dataset/DS_8a651348e2"><i>Saccharomyces paradoxus</i> CBS432</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/22830625">Yue et al. 2017</a>.
             </li>

         <li><a href="/a/app/record/dataset/DS_ca64d9eec1"><i>Stachybotrys chartarum</i> IBT 40293 </a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/25015739">Semeiks et al. 2014</a>.
               </li>
      </ul>
      <br>

        <h4>Whole genome sequence of isolates for SNP analysis</h4>
        <ul>
          <li><i>Aspergillus fumigatus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_773518f5a9">SNP calls on WGS of <i>Aspergillus fumigatus</i> clinical, environmental, AsTEC and CDC study strains</a> &ndash;
              The strain collection includes strains from CF patients followed over multiple years of infection (AF100 and AF106); environmental strains; AsTEC and CDC study strains. 
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/34465017">Ross et al. 2021</a>.
            </li>

            <li>
              <a href="/a/app/record/dataset/DS_f72ff9e9e7">SNP calls on <i>Aspergillus fumigatus</i> environmental isolates from Southeast USA</a> &ndash;
              Environmental isolates from agricultural settings in Southeast USA.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/34897421">Kang et al. 2022</a>.
            </li>
            </ul>

            <li><i>Botrytis cinerea</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_8360a983e6">SNP calls on WGS of <i>Botrytis cinerea</i> isolates</a> &ndash;
                SNP calls on WGS of field isolates of <i>Botrytis cinerea</i>, largely collected from a local set of plant species.
                <a href="https://www.biorxiv.org/content/10.1101/489799v1">Atwell et al. 2018</a>.
              </li>
            </ul>

          <li><i>Candida auris</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_0f19274214">SNP calls on WGS of <i>Candida auris</i> identified in the Netherlands</a> &ndash;
                Clinical strains collected from patients treated in the Netherlands.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/31574934">Vogelzang et al. 2019</a>.
              </li>
            </ul>

            <li><i>Pyricularia oryzae</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_c7ef4d99b1">SNP calls on WGS of <i>Pyciularia oryzae</i> isolates from different hosts</a> &ndash;
                Isolates collected from the following hosts: <i>Oryza sativa</i>, <i>Setaria italica</i>, and <i>Eleusine indica</i>.
                <a href="https://zenodo.org/record/2564950">Win et al. 2019</a>.
              </li>

              <li>
                <a href="/a/app/record/dataset/DS_d9cbcddc1a">SNP calls on WGS of <i>Pyciularia oryzae</i> isolates from Zambia</a> &ndash;
                13 <i>Magnaporthe oryzae</i> (syn. <i>Pyricularia oryzae</i>) isolates collected in Zambia from 2018 to 2020 during outbreaks of wheat blast.
                Tha Sainsbury Laboratory.
              </li>
            </ul>

          </ul>
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Aspergillus fumigatus</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_c339242e7e">Transcriptomics analysis of <i>Aspergillus fumigatus</i> co-cultivated with <i>Pseudomonas aeruginosa</i></a> &mdash;
          Gene expression was determined at 45, 90 and 180 minutes after co-incubation.
          Vito Valiante 2018.  
      </li>
      </ul>

     <li><i>Candida albicans</i></li>
     <ul>
       <li>
       <a href="/a/app/record/dataset/DS_7da7acc835">Global translational landscape of the <i>Candida albicans</i> morphological transition</a> &mdash;
       <i>Candida albicans</i> grown under 2 conditions (YEPD + serum at 37°C (filament-inducing) and YEPD at 30°C (non filament-inducing)).
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/33585865" target="_blank">Mundodi et al. 2020</a>.   
       </li>

       <li>
        <a href="/a/app/record/dataset/DS_79549674a3">Global transcriptional response to various environmental stresses, alone and in combinations</a> &mdash;
        Transcriptome of <i>C. albicans</i> in response to exposure to H2O2, NaCL, and H2O2 & NaCl.
        Brown et al. 2015.   
        </li>
     </ul>

     <li><i>Cryptococcus neoformans</i></li>
     <ul>
     <li>
       <a href="/a/app/record/dataset/DS_2d477b2338">Stranded mRNA-Seq for delineation of transcript ends in <i>C. neoformans</i> JEC21 </a> &mdash;
       Stranded RNA-Seq from exponentially and stationary phases of growth.
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/32020195" target="_blank">Wallace et al. 2020</a>.  
     </li>
     </ul>

     <li><i>Fusarium mangiferae</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_91ca610be2">In planta expression profiles of <i>Fusarium magniferae</i> </a> &mdash;
        Transcriptional changes during maize root infection. Root samples were taken 4, 10, and 20 dpi for RNA isolation. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/28040774" target="_blank">Niehaus et al. 2015</a>.  
      </li>
      </ul>

      <li><i>Neurospora crassa</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_d5cb6ab9a2">Transcriptome of <i>N. crassa</i> strains grown on different carbon sources (complex polysaccharides)</a> &mdash;
        Transcriptional profiling dataset series determining how fungi sense simple to complex carbohydrates
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32111691" target="_blank">Wu et al. 2020</a>.  
      </li>
      </ul>

      <li><i>Puccinia striiformis f. sp. tritici</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_14b0e873fb">RNA sequencing of stripe rust infected wheat leaves and isolated haustoria</a> &mdash;
        Total RNA was extracted from a pool of stripe rust infected wheat leaves and from two biological replicates of haustoria isolates. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23607900" target="_blank">Cantu et al. 2013</a>.  
      </li>
      </ul>

      <li><i>Zymoseptoria tritici</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_d93bf0945b">Small RNA profiling of the infection cycle of <i>Zymoseptoria tritici</i> on wheat</a> &mdash;
        Samples were collected at 4, 9, 13, and 21 days post infection.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31333714" target="_blank">Kettles et al. 2019</a>.  
      </li>    
      </ul>

    </ul>   
    <br>

    <h4>Genome Browser Tracks</h4>
    <ul>
      <li><i>Candida albicans</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_c339242e7e">Global translational landscape of the <i>Candida albicans</i> morphological transition</a> &mdash;
          Ribo-Seq under 2 conditions (YEPD + serum at 37°C (filament-inducing) and YEPD at 30°C (non filament-inducing)).
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33585865" target="_blank">Mundodi et al. 2021</a>.   
      </li>
      </ul>
    </ul>
    <br>

      
    <h4>Phenotype data</h4>
            
        <ul>
        <li><a href="/a/app/record/dataset/DS_f2312514ac">Manually curated Phenotypes (VEuPathDB)</a> &ndash; 
            We have curated and integrated phenotypes from <a href="https://pubmed.ncbi.nlm.nih.gov/26529322/" target="_blank">Lin et al. 2015</a> and 
            <a href="https://pubmed.ncbi.nlm.nih.gov/29150592/" target="_blank">Dos Reis et al. 2018</a> for <i>Aspergillus fumigatus</i> Af293 in this release.    
         </li>
        </ul>
        <br>
          <div style="margin-left: 3em;">
                    <style>
                      #Phenotype_Bld60Asp table, #Phenotype_Bld60Asp td, #Phenotype_Bld60Asp th, #Phenotype_Bld60Asp tr {
                      text-align : left;
                      padding-left: 5px;
                      padding-right: 5px;
                      padding-top: 5px;
                      padding-bottom: 5px;
                      border: 1px solid black;
                      }
                      #Phenotype_Bld60Asp {
                      margin-left : 5 em;
                      }
                    </style> 
                      <table id="Phenotype_Bld60Asp"> 
                      <tr>
                        <th>Species</th>
                        <th>New phenotype mappings</th>
                        <th>Genes with phenotypes</th>
                      </tr>
                      <tr>
                        <td><i>Aspergillus fumigatus</i> Af293</td>
                        <td>47</td>
                        <td>4</td>
                      </tr>
                      </table>
            </div>
            <br>
    <ul>
        <li><a href="/a/app/record/dataset/ DS_8cb36b3135">Manually curated Phenotypes from MagnaGenes database</a> &ndash; 
            <a href="https://zenodo.org/record/4647766#.Yz6zFi8w3UI" target="_blank">MagnaGenes (Foster et al. 2021)</a> is a database that compiles and summarizes 
            all the available studies reporting phenotypic data about gene function in the blast fungus <i>Pyricularia oryzae</i> (syn. <i>Magnaporthe oryzae</i>). The currently integrated 
            v2.0 of MagnaGenes database provides reported phenotypic annotations for 1,679 genes from <i>Pyricularia oryzae</i> 70-15. After consultation with the MagnaGenes team, 
            these annotations were mapped to OBO-foundry consistent ontologies by the VEuPathDB team.
         </li>
    </ul>
          <br>
          
           <h4>Updated Functional Annotation</h4>
       		<ul>
             <li>
             <a href="https://docs.google.com/spreadsheets/d/1H7CN8BYKNoqMrG88bRdnBRr4nD3tXab7BYS5VBEww4g/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include gene names, products, GO terms, publications and notes from curators. These functional annotations are current with GeneDB September 2022 GFF version.
            </li>
            </ul> 
            <br>
               
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld60 table, #FunctAnnot_Bld60 td, #FunctAnnot_Bld60 th, #FunctAnnot_Bld60 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld60 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld60"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-09/AfumigatusAf293/AfumigatusAf293.gff.gz">GeneDB version September, 2022</a></td>
                  <td>306</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_4b35c88aa1"><i>Aspergillus nidulans</i> FGSC A4</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-09/AnidulansFGSCA4/AnidulansFGSCA4.gff.gz">GeneDB version September, 2022</a></td>
                  <td>8</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_cd90ec7602"><i>Fusarium graminearum</i> PH-1</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-09/FgraminearumPH1/FgraminearumPH1.gff.gz">GeneDB version September, 2022</a></td>
                  <td>3598</td>
                </tr>
                </table>
                </div>
                <br>
                
        <ul>
             <li>
             Functional annotations from MagnaGenes database (Foster et al) &ndash; Functional annotations that are currently not available in FungiDB and are available in 
             <a href="https://zenodo.org/record/4647766#.Yz6zFi8w3UI">MagnaGenes database</a> were integrated in this release. These include 778 new gene names, 100 gene 
             synonyms and 374 new publications. The parsed annotations can be found <a href="https://docs.google.com/spreadsheets/d/19fhSPUJ-WerKmN-9xKMHnTIUINcxh2kMsECvTUIRUKs/edit?usp=sharing">here</a>.
        	</li>
        </ul> 
                <br>
      <h4>Structure Predictions</h4> 
      <ul>
        <li><a href="/a/app/record/dataset/DS_dc1c8cb24c">AlphaFold Predictions</a> &ndash; are updated to AlphaFold Release 4 (July 2022) which offers predictions for significantly more proteins than Release 3.  Use {{project}}'s <a href="/a/app/search/transcript/GenesWithAlphafold">Genes with AlphaFold Predictions</a> search to explore genes with AlphaFold predictions or visit the Structure Analysis section of your favorite gene page to view data relevant to you.</li> 
      </ul>
      <br>
       </div>
        
<section>
  <h3>New features in {{project}} 60</h3>
  <div style="margin-left: 3em;">
    <h4>Associated Data Sets</h4>
      <ul>
        <li>Data sets that come from the same publication are now linked.  The new 'Associated Data Sets' section on any data set record page will link to other data from the same publication within {{project}} as well as data from organisms supported in different VEuPathDB sites.  <a target="_blank" href="{{'/assets/images/news/AssociatedDataSets.gif' | absolute_url}}">View tutorial GIF.</a></li>
      </ul>
  </div>
</section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
