---
layout: post
title: FungiDB 58 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 58.</h3>

<section>
  <h3>Release 57 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday April 28th, 2022 at 10am EDT for our Release 57 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 57.  
    <a href="https://attendee.gotowebinar.com/register/1664706106928504331" target="_blank">Register here</a>  
  </div>
</section>

    <section>
      <h3>New data in FungiDB 58</h3>
      <div style="margin-left: 3em;">

       <h4>Genome Sequence and Annotation: </h4>
       <ul>
        <li><a href="/a/app/record/dataset/DS_f28dfc1390"><i>Aspergillus eucalypticola</i> CBS 122712</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30349117">Vesth et al. 2018</a>.  
           </li>

         <li><a href="/a/app/record/dataset/DS_ec6d40428f"><i>Aspergillus tanneri</i> NIH1004</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/31948962">Mounaud et al. 2020</a>.
            </li>
 
        <li><a href="/a/app/record/dataset/DS_611a31686e"><i>Cladophialophora carrionii</i> CBS 160.54</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>.  
              </li>

         <li><a href="/a/app/record/dataset/DS_14b629a086"><i>Exophiala spinifera</i> CBS 89968</a> &ndash;
           The Broad Institute.  
             </li>
   
          <li><a href="/a/app/record/dataset/DS_377ed4b120"><i>Gaeumannomyces tritici</i> R3-111a-1</a> &ndash;
            The Broad Institute.  
              </li>
   
         <li><a href="/a/app/record/dataset/DS_88df210e7f"><i>Hortaea werneckii</i> EXF-2000</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/28500048">Sinha et al. 2017</a>. 
          </li>
           
          <li><a href="/a/app/record/dataset/DS_2d8753866f"><i>Microsporum canis</i> CBS 113480</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/22951933">Martinez et al. 2012</a>.
            </li>
       </ul>
       <br>

       <h4>Annotation added to Genome Sequence: </h4>
       <ul>
        <li><a href="/a/app/record/dataset/DS_efbafbcb21"><i>Rafaela lauricola</i> RL4</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/26899007">Ma et al. 2016</a>.  
              </li>
       </ul>
      <br>
        
       <h4>Genome Sequence: </h4>
       <ul>

         <li><a href="/a/app/record/dataset/DS_ef14b36372"><i>Talaromyces marneffei</i> TM4</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/1497334">Guého et al. 1992</a>.  
           </li>
         </ul>
       <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><i>Aspergillus fumigatus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_46ca1ca0a7">SNP calls on WGS of clinical and environmental isolates of <i>Aspergillus fumigatus</i> in Germany</a> &ndash;
              Genome sequencing of environmental <i>Aspergillus fumigatus</i> isolates collected from farms in central Germany and clinical isolates submitted to the German National Reference Center for Invasive Fungal Disease.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/34819642">Barber et al. 2021</a>.
            </li>
            </ul>

          <li><i>Cryptococcus gattii</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_9aff260283">SNP calls on WGS of <i>Cryptoccocus</i> from Central Zambezian miombo woodlands (VGV isolates)</a> &ndash;
                <i>Cryptoccocus</i>  VGV DNAseq from Central Zambezian miombo woodlands.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/31719178">Farrer et al. 2019</a>.
              </li>

              <li>
                <a href="/a/app/record/dataset/DS_71148e3658">SNP calls on WGS of <i>Cryptoccocus</i> from Central Zambezian miombo woodlands (non-VGV isolates - VGI, VGII, VGIII, VGIV, VGVI)</a> &ndash;
                six isolates were collected from three locations spanning the Central Miombo Woodlands of Zambia, Africa.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/31719178">Farrer et al. 2019</a>.
              </li>

              <li>
                <a href="/a/app/record/dataset/DS_91c53e61a3">SNP calls on WGS of <i>Cryptoccocus gattii</i> isolates collected from the North American Pacific Northwest</a> &ndash;
                WGS of <i>C. gattii</i> populations (VGIIa, VGIIb, and VGIIc).
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/29359190">Roe et al. 2021</a>.
              </li>
            </ul>

        </ul>
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Aspergillus fumigatus</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_faa5674b55">Transcriptome analysis of conidium germination of <i>Aspergillus fumigatus</i> in different growth conditions</a> &mdash;
            Gene expression level of Aspergillus fumigatus CEA17_ΔakuBKU80 during germination in two different medium  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33419224" target="_blank">Danion et al. 2021</a>.  
      </li>
      </ul>


     <li><i>Cryptococcus neoformans</i></li>
     <ul>
       <li>
       <a href="/a/app/record/dataset/DS_b652373fe0">Stranded mRNA-Seq for delineation of transcript ends in <i>C. neoformans</i> alphaKN99</a> &mdash;
       Differential translational control by start codon context across the fungal kingdom.
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/32020195" target="_blank">Wallace et al. 2020</a>.   
       </li>
     </ul>

     <li><i>Fusarium graminearum</i></li>
     <ul>
     <li>
       <a href="/a/app/record/dataset/DS_36c1c0082f">RNA-seq Analysis of FlbD-, AbaA-, and WetA-dependent Genes in <i>Fusarium graminearum</i> Z-3639</a> &mdash;
       RNA-seq Analysis of FlbD-, AbaA-, and WetA-dependent Genes in Fusarium graminearum Z-3639.
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/24039821" target="_blank">Son et al. 2013</a>.  
     </li>
     </ul>

     <li><i>Neurospora crassa</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_1dd4f50bad">Transcriptome of N. crassa strains grown on different carbon sources (citrus peel, Miscanthus, corn stover, Wing Nut, Energy Cane, Switchgrass)</a> &mdash;
        Transcriptional profiling of N. crassa on 40 different carbon sources, including plant biomass, to provide data on how fungi sense simple to complex carbohydrates.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32111691" target="_blank">Wu et al. 2020</a>.  
      </li>
      </ul>


      </ul>   
    <br>
          
    <h4>ChiP-Seq</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_87b5c26e2a">RNAP II ChIP-Seq data for <i>Talaromyces marneffei</i></a>&mdash;
            ChIP-seq was performed on WT conidia to map the genome-wide occupancy of elongating RNA polymerase II (RNAP II).   
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/34183813" target="_blank">Wang et al. 2021</a>.  
            </li>
            </ul>

          <br>


    <h4>Function prediction</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_4a1633821f">CalCENv1 co-expression network UMAP clusters</a> &mdash;
            CalCENv1 co-expression network was projected to two dimensions using UMAP and 18 clusters were identified and annotated through gene set enrichment analysis.  
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/33472984" target="_blank">O'Meara and O'Meara, 2021</a>.  
            </li>
            </ul>

          <br>


       <h4>Phenotype data</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_e51cb08950">Phenotypes from PHI-base database</a> &mdash;
            <a href="http://www.phi-base.org/" target="_blank">PHI-base</a> is a web-accessible database that catalogues experimentally verified pathogenicity, virulence and effector genes from 
            fungal, oomycete and bacterial pathogens, which infect animal, plant, fungal and insect hosts. The pathogen-host interactions (PHI) phenotypes and pathogen-specific 
           phenotypes from PHI-base were integrated for 61 different genomes available in FungiDB in this release. This dataset is current with PHI-base version 4.12. We have 
           integrated 5,381 phenotypic annotations mapping to 3,140 genes in total. The original data file from PHI-base can be found on
           <a href="https://github.com/PHI-base/data/blob/master/releases/phi-base_v4-12_2021-09-02.csv" target="_blank">PHI-base GitHub</a> site. 
            </li>
            </ul>

          <br>
          
           <h4>Updated Functional Annotation</h4>
       		<ul>
             <li>
             <a href="https://docs.google.com/spreadsheets/d/1im0lQyZ6E1Qec48dOQjZDI0eGXKv_fY8TeE7_RptB4U/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include gene names, products, GO terms, publications and notes from curators. These functional annotations are current with GeneDB March 2022 GFF version. Functional annotation 
               suggestions made by our users via the <a href="https://apollo.veupathdb.org/annotator/index" target="_blank">VEuPathDB Apollo</a> instance are reviewed by our curators and are included 
               in the official annotation when appropriate.
            </li>
            </ul> 
            <br>
               
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld57 table, #FunctAnnot_Bld57 td, #FunctAnnot_Bld57 th, #FunctAnnot_Bld57 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld57 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld57"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/AfumigatusAf293/AfumigatusAf293.gff.gz">GeneDB version March, 2022</a></td>
                  <td>113</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_4b35c88aa1"><i>Aspergillus nidulans</i> FGSC A4</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/AnidulansFGSCA4/AnidulansFGSCA4.gff.gz">GeneDB version March, 2022</a></td>
                  <td>2</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_d22dd5953a"><i>Pyricularia oryzae</i> 70-15</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/Moryzae7015/Moryzae7015.gff.gz">GeneDB version March, 2022</a></td>
                  <td>91</td>
                </tr>
                </table>
                </div>
                
                <br>
           
           <h4>External database links</h4>

           <ul>
             <li>Links to <a href="http://www.phi-base.org/">PHI-base</a> database in the gene pages have been updated in this release, providing links for 3,140 genes from 61 genomes in FungiDB. 
             </li>
           </ul>
           
           <br>
       
       </div>
        
  <section>
  <h3>New features in {{project}} 57</h3>
  <div style="margin-left: 3em;">
  <h4>New Searches</h4>
    <ul>
    <li>The new <a href="/a/app/search/transcript/GenesByIntronJunctions">Unannotated Intron Junctions</a> search enables users to identify genes that contain, or are flanked by, unannotated high confidence intron junction-spanning reads from RNA-seq data. These genes may be incompletely or inaccurately annotated due to missing introns/exons and/or alternative splice variants.  The search is categorized under Search for Genes, Function Prediction.  See our <a href="{{'/documents/Intron_junctions_search_16_04_2022.pdf' | absolute_url}}">PDF Tutorial</a> or our <a href="https://www.youtube.com/watch?v=-kjjsB3eqxA" target="_blank">Video Tutorial</a> for more information. </li>
    <li>The new <a href="/a/app/search/transcript/GenesByFunctionalGeneList">Dataset Gene List</a> search enables users to search for genes in co-expression clusters associated with <a href="http://www.ncbi.nlm.nih.gov/pubmed/33472984" target="_blank">O'Meara et al 2021</a>.   </li>
    </ul>
  </div>
  </section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
