---
layout: post
title: FungiDB 57 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 57.</h3>

<section>
  <h3>Release 57 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday February 24th, 2022 at 10am EDT for our Release 56 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 56.  
    <a href="https://attendee.gotowebinar.com/register/580233597692794126" target="_blank">Register here</a>  
  </div>
</section>

    <section>
      <h3>New data in FungiDB 57</h3>
      <div style="margin-left: 3em;">

      <h4>Genome Sequence and Annotation: updated genomes </h4>
      <ul>
        
        <li><a href="/a/app/record/dataset/DS_5136d77df4"><i>Trichoderma reesei</i> QM6a</a> &ndash;
          We are pleased to announce the first update for a FungiDB genome through Apollo, a web-based genomic annotation editing platform. We incorporated 225 changes, including creation of new genes, modification of existing genes, merges and splits. See a <a href='https://docs.google.com/spreadsheets/d/1IipM-c9D2VRBfVlfUFGrdft8zKQZzBEoiGxGw3RoXME/edit?usp=sharing'>breakdown of the changes and the data incorporated</a>.
          </li>
        </ul>
       <br>

          <img style="width:20em" src="{{'/assets/images/fungidb/apollofungidb56.png' | absolute_url}}" />
        <br>

       <h4>Genome Sequence and Annotation: </h4>
       <ul>

        <li><a href="/a/app/record/dataset/DS_08c86b558d"><i>Aspergillus fijiensis</i> CBS 313.89</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/32665272">Drott et al. 2020</a>.  
           </li>

         <li><a href="/a/app/record/dataset/DS_0360a7ae66"><i>Candida tropicalis</i> MYA-3404 (2020)</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/19745113">Jackson et al. 2009</a>.  
            </li>
 
         <li><a href="/a/app/record/dataset/DS_d18b70822b"><i>Cladophialophora psammophila</i> CBS 110553</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/31105748">Mixão et al. 2019</a>.  
             </li>
   
           <li><a href="/a/app/record/dataset/DS_c2284632aa"><i>Debaryomyces fabryi</i> CBS 789</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>.  
               </li>
   
         <li><a href="/a/app/record/dataset/DS_c454ebb27c"><i>Emmonsia africanus</i> CBS 136260</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/28385833">van Rijswijck et al. 2017</a>.  
             </li>
 
         <li><a href="/a/app/record/dataset/DS_a363c3f7a3"><i>Pneumocystis jirovecii</i> RU7</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/32843521">Fokkens et al. 2020</a>.  
               </li>
 
          <li><a href="/a/app/record/dataset/DS_3f9de63eb2"><i>Trichophyton mentagrophytes</i> TIMM 2789</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/19465905">Butler et al. 2009</a>.  
               </li>
 
         <li><a href="/a/app/record/dataset/DS_05a7d4a66a"><i>Trichophyton tonsurans</i> CBS 112818</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/30462193">Seixas et al. 2019</a>.  
               </li>
        </ul>
       <br>
        
       <h4>Genome Sequence: </h4>
       <ul>

         <li><a href="/a/app/record/dataset/DS_1e397f04b9"><i>Raffaelea lauricola</i> RL4</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/32819276">Zhang et al. 2020</a>.  
           </li>
         </ul>
       <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><i>Aspergillus fumigatus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_2216592524">Whole genome association study for itraconazole sensitivity in <i>Aspergillus fumigatus</i></a> &ndash;
              Clinical isolates of <i>A. fumigatus</i> from Japan.
              <a href="https://doi.org/10.3389/ffunb.2020.617338">Zhao et al. 2021</a>.
            </li>
            <li>
              <a href="/a/app/record/dataset/DS_f280f1be3e">Strain specific <i>A. fumigatus</i> persistence in the murine lung</a> &ndash;
              Characterisation of W72310 strain by WGS and comparison to a common reference strain, CEA10.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/33597172">Jones et al. 2021</a>.
            </li>
            </ul>

          <li><i>Candida albicans</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_caec2bd9bb"><i>Candida albicans</i> adaptation to azole antifungal drugs</a> &ndash;
                A study of the rapid adaptation to antifungal drugs.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/32687060">Todd et al. 2020</a>.
              </li>
            </ul>

          <li><i>Candida auris</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_c557df700f"><i>C. auris</i> B11220 mono- and multidrug resistance evolution experiment</a> &ndash;
                Experimentally evolved MDR clade II <i>C. auris</i>.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33820824">Carolus et al. 2021</a>.
              </li>
            </ul>
          
            <li><i>Cryptococcus neoformans</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_b57833f9f4">Genetic basis of within-host microevolution underlying recurrent Cryptococcal meningitis infection</a> &ndash;
                Whole-genome sequencing assessing the genetic basis of Cryptococcal infection in 17 HIV-infected individuals with recurrent cryptococcal meningitis.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33820824">Rhodes et al. 2017</a>.
              </li>
            </ul>
          
        </ul>
          
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Aspergillus fumigatus</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_2b76656bb1">Determining <i>Aspergillus fumigatus</i> transcription factor expression and function during invasion of the mammalian lung</a> &mdash;
          RNA-seq analysis was performed on wild-type and an ACE1 deletion mutant.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33780518" target="_blank">Liu et al. 2021</a>.  
      </li>
      </ul>

     <li><i>Candida albicans</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_075a4b4e1f">Transcriptomes of clinical isolate strains and mutant derivatives</a> &mdash;
        Transcriptome of the fungi in infected maize roots growing in CM agar.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31116789" target="_blank">Huang et al. 2019</a>.  
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_24f03e74c5">Transcriptomic analysis of wild type and GCN5-deficient <i>Candida albicans</i> cells prior to and after treatment with caspofungin</a> &mdash;
        Investigation of the virulence factors in <i>C. albicans</i> 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31263212" target="_blank">Shivarathri et al. 2019</a>.   
        </li>

        <li>
          <a href="/a/app/record/dataset/DS_429e31c02f">CalCEN: Candida albicans co-expression Network v1.0.0</a> &mdash;
          Raw reads from 853 RNA-seq runs of 18 large-scale <i>Candida albicans</i> studies were collected from NCBI and re-mapped to 6226 open reading frames of the SC5314 strain. The CalCEN co-expression network was then estimated by the pairwise spearman rank-correlation of FPKM expression values across all runs. 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33472984" target="_blank">Teresa O'Meara and Matthew O'Meara, 2021</a>.   
          </li>
      </ul>

      <li><i>Candida auris</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_cd56220e4b">Experimental evolution of fluconazole resistance in <i>Candida auris</i></a> &mdash;
          Transcriptomic profiles of drug resistant and sensitive strains of <i>C. auris</i>. 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33077664" target="_blank">Bing et al. 2020</a>.   
          </li>
        </ul>

      <li><i>Candida glabrata</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_0024327063">DNA damage-induced transcriptome changes in <i>Candida glabratae</i></a> &mdash;
          RNA-Seq analysis of transcriptional rewiring of the DNA damage response in <i>C. glabrata</i>.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33323516" target="_blank">Shor et al. 2020</a>. 
       </li>
      </ul>

      <li><i>Candida lusitaniae</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_9a20a7bfa0">Transcriptional profiling of strains with different levels of Mrr1 activity in <i>Candida lusitaniae</i></a> &mdash;
          Expression profiling by high throughput sequencing.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33785623" target="_blank">Demers et al. 2021</a>.   
          </li>
        </ul>

        <li><i>Coccidioides posadasii</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_e68c49ad2b">Transcriptomics of environmental (mycelia) and parasitic (spherule 96 hr) stages of the Δcts2/Δard1/Δcts3 strain</a> &mdash;
          An attenuated mutant strain Δcts2/Δard1/Δcts3 in which chitinase genes 2 and 3 were deleted was previously created for vaccine development and profiled by RNA-Seq at two conditions.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/32499817" target="_blank">Mead et al. 2020</a>.   
          </li>
        </ul>


        <li><i>Cryptococcus neoformans</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_eb589156b3">Transcriptomic profiling of the <i>Cryptococcus neoformans</i> transition to titan-like cells</a> &mdash;
          RNA-Seq study of different time points of the transition: 7 and 24 h.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/29775477" target="_blank">Trevijano-Contador et al. 2018</a>.   
          </li>
        </ul>

        <li><i>Saccharomyces cerevisiae</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_6de7930665">DNA damage-induced transcriptome changes in <i>Saccharomyces cerevisiae</i></a> &mdash;
          RNA-Seq analysis of transcriptional rewiring of the DNA damage response.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33323516" target="_blank">Shor et al. 2020</a>.   
          </li>
        </ul>
    
    </ul>   
    <br>
          
       <h4>Phenotype data</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_e51cb08950">Phenotypes from PHI-base database</a> &mdash;
           <a href="http://www.phi-base.org/" target="_blank">PHI-base</a> is a web-accessible database that catalogues experimentally verified pathogenicity, virulence and effector genes from 
           fungal, Oomycete and bacterial pathogens, which infect animal, plant, fungal and insect hosts. The pathogen – host interactions (PHI) phenotypes and pathogen-specific 
           phenotypes from PHI-base were integrated for 61 different genomes available in FungiDB in this release. This dataset is current with PHI-base version 4.12. We have 
           integrated 5,381 phenotypic annotations mapping to 3,140 genes in total. The original data file from PHI-base can be found in their GitHub site 
           <a href="https://github.com/PHI-base/data/blob/master/releases/phi-base_v4-12_2021-09-02.csv" target="_blank">linked here</a>.   
            </li>
            </ul>

          <br>
          
           <h4>Updated Functional Annotation</h4>
       		<ul>
             <li>
             <a href="https://docs.google.com/spreadsheets/d/1im0lQyZ6E1Qec48dOQjZDI0eGXKv_fY8TeE7_RptB4U/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include gene names, products, GO terms, publications and notes from curators. These functional annotations are current with GeneDB March 2022 GFF version. These annotation updates 
               also include some of the functional annotations made by our users via the <a href="https://apollo.veupathdb.org/annotator/index" target="_blank">VEuPathDB Apollo</a> instance.
            </li>
            </ul> 
            <br>
               
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld57 table, #FunctAnnot_Bld57 td, #FunctAnnot_Bld57 th, #FunctAnnot_Bld57 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld57 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld57"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/AfumigatusAf293/AfumigatusAf293.gff.gz">GeneDB version March, 2022</a></td>
                  <td>113</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_4b35c88aa1"><i>Aspergillus nidulans</i> FGSC A4</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/AnidulansFGSCA4/AnidulansFGSCA4.gff.gz">GeneDB version March, 2022</a></td>
                  <td>2</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_d22dd5953a"><i>Pyricularia oryzae</i> 70-15</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/Moryzae7015/Moryzae7015.gff.gz">GeneDB version March, 2022</a></td>
                  <td>91</td>
                </tr>
                </table>
                </div>
                
                <br>
           
           <h4>External database links</h4>

           <ul>
             <li>Links to <a href="http://www.phi-base.org/">PHI-base</a> database in the gene pages have been updated in this release, providing links for 3,140 genes from 61 genomes in FungiDB. 
             </li>
           </ul>
           
           <br>
       
       
      </section>
     </div>
        
    <section>
      <h3>New features in {{project}} 56</h3>
      <div style="margin-left: 3em;">
      <h4>BLAST Tool Improvements</h4>
        <ul>
        <li>The <a href="/a/app/workspace/blast/new">BLAST (multi-query capable)</a> tool, known previously as BLAST<i>beta</i>, can perform queries with one or more FASTA-formatted sequences and offers many download types. The tool uses NCBI BLAST to find genes, genomic sequences, ESTs, or PopSet Sequences that have BLAST similarity to your input sequence. BLAST (multi-query capable) is <a href="{{'/assets/images/Find-blast-tool.png' | absolute_url}}">available from the tools menu.</a></li>
        <li>A new download option, <a href="{{'/assets/images/BLAST-This-table-download.png' | absolute_url}}">'This table (csv)'</a>, for the results of multi-sequence queries (combined result) retrieves the {{project}} result table for the results of multi-sequence queries (combined result). </li>
        <li><a href="{{'/assets/images/BLAST-56-improvements.gif' | absolute_url}}">View tutorial GIF.</a></li>
        </ul>
      <h4>Gene page sequence highlighting</h4>
        <ul>
          <li>Genomic and mRNA sequences on gene pages have a new color scheme.  Introns are highlighted in yellow and UTR sequence are highlighted in pink.  Previously they were gray and blue, respectively. <a href="{{'/assets/images/gene-seq-highlight.png' | absolute_url}}">Example</a></li>
        </ul>
      </div>
    </section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
