---
layout: post
title: FungiDB 59 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 59.</h3>

<section>
  <h3>Annual VEuPathDB Workshop (Virtual)</h3>
    The 15th Annual VEuPathDB workshop will be a virtual event taking place daily on <b>September 6 - 9th, 2022</b> from <b>9am - 1pm Eastern</b>. Join us for these 4 half-day sessions to learn how to effectively use VEuPathDB tools and features that support your hypothesis driven research. Short lectures will precede hands-on exercises via informal Zoom sessions where discussion and conversation are encouraged. <a href="/a/app/static-content/workshopSept2022.html">Details here</a> or <a href="https://ugeorgia.ca1.qualtrics.com/jfe/form/SV_elEnAmi5PvtUNwO" target="_blank">Register here</a>.
</section>

<section>
  <h3>Release 59 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday September 1st, 2022 at 10am EDT for our Release 59 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 59.  
    <a href="https://attendee.gotowebinar.com/register/3393660656180049424" target="_blank">Register here</a>  
  </div>
  <br>
</section>

    <section>
      <h3>New data in FungiDB 59</h3>
      <div style="margin-left: 3em;">

       <h4>Genome Sequence and Annotation: </h4>
       <ul>
        <li><a href="/a/app/record/dataset/DS_c8c6d0b908"><i>Alternaria alternata</i> SRC1lrK2f</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/27434633">Zeiner et al. 2016</a>.  
           </li>

         <li><a href="/a/app/record/dataset/DS_9ad6869a2a"><i>Batrachochytrium salamandrivorans</i> BS</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30473694">Grogan et al. 2018</a>.
            </li>
 
        <li><a href="/a/app/record/dataset/DS_f453f876da"><i>Cordyceps militaris</i> ATCC 34164</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/29178836">Kramer et al. 2017</a>.  
              </li>

         <li><a href="/a/app/record/dataset/DS_aa6c0c41fd"><i>Cryptococcus</i> cf. <i>gattii</i> MF34</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/31719178">Farrer et al. 2019</a>.
             </li>
   
          <li><a href="/a/app/record/dataset/DS_ea9df1c0c0"><i>Exophiala aquamarina</i> CBS 119918</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>. 
              </li>
   
         <li><a href="/a/app/record/dataset/DS_d8e276588e"><i>Hemileia vastatrix</i> Race XXXIII</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/30998802">Porto et al. 2019</a>. 
          </li>
           
          <li><a href="/a/app/record/dataset/DS_cd741aae10"><i>Magnaporthiopsis poae</i> ATCC 64411</a> &ndash;
            Li-Jun Ma 2011.
            </li>

          <li><a href="/a/app/record/dataset/DS_a2d60e11ca"><i>Saccharomycodes ludwigii</i> UTAD17</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30533777">Tavares et al. 2018</a>.
              </li>

          <li><a href="/a/app/record/dataset/DS_3a745be8fc"><i>Pythium insidiosum</i> Pi-s</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26089407">Rujirawat et al. 2015</a>.
                </li>

          <li><a href="/a/app/record/dataset/DS_82669379aa"><i>Rhizophagus irregularis</i> C2</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/34747029">Yildirir et al. 2022</a>.
                </li>
       </ul>
       <br>

       <h4>Genome Sequence: </h4>
       <ul>
        <li><a href="/a/app/record/dataset/DS_5ead846f15"><i>Beauveria bassiana</i> HN6</a> &ndash;
          Jiangsu University of Science and Technology, 2022.  
              </li>
       </ul>
      <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><i>Aspergillus fumigatus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_df3742f4a3">SNP calls on WGS of clinical <i>Aspergillus fumigatus</i> isolates from Portugal</a> &ndash;
              Genome sequencing of 8 clinical isolates of <i>Aspergillus fumigatus</i> from the Portuguese Oncology Institute in Porto, Portugal.
              Abigail Lind.
            </li>
            </ul>

          <li><i>Histoplasma capsulatum</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_a9eb7e00a0">SNPs on WGS of <i>Histoplasma</i> isolates from Alberta, Canada</a> &ndash;
                Clinical strains were collected by the AB ProvLab, Canada.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/35544208">Dingle et al. 2021</a>.
              </li>
            </ul>

            <li><i>Pneumocystis jirovecii</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_1ddb8babd8">SNP calls on WGS of <i>P. jirovecii</i> isolated from human bronchoalveolar lavage samples</a> &ndash;
                SNP calls on P. jirovecii isolates (54c, and 46).
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33686174">Cissé et al. 2021</a>.
              </li>
            </ul>

            <li><i>Pyricularia oryzae</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_9906c5c8f5">SNPs calls on WGS data of <i>Pyricularia oryzae</i> isolates from Africa</a> &ndash;
                SNPs calls on WGS data of 29 <i>Pyricularia oryzae</i> isolates.
                Talbot et al. 2019.
              </li>
            </ul>

            <li><i>Zymoseptoria tritici</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_486c6c20b9">SNP calls on WGS of <i>Zymoseptoria tritici</i> from infected wheat fields</a> &ndash;
                <i>Z. tritici</i> field sampling in Switzerland and Oregon, USA..
                Croll et al. 2019.
              </li>
            </ul>

        </ul>
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Aspergillus nidulans</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_e9d3e44418">Gene expression in <i>Aspergillus nidulans</i> grown on multiple nitrogen sources</a> &mdash;
          <i>Aspergillus nidulans</i> (MH1; biA1) was grown in liquid culture with ammonium, glutamine or alanine as the nitrogen source  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/34154419" target="_blank">Steyer et al. 2021</a>.  
      </li>
      </ul>

     <li><i>Botrytis cinerea</i></li>
     <ul>
       <li>
       <a href="/a/app/record/dataset/DS_3c23b11271">Transciptomic analysis of <i>B. cinerea</i> exposed to a bacterial biocontrol agent (secretome)</a> &mdash;
       Transcriptome of <i>B. cinerea</i> in response to exposure to Bacillus subtilis MBI 600, a new commercialized biopesticide
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/33839506" target="_blank">Samaras et al. 2021</a>.   
       </li>
     </ul>

     <li><i>Candida albicans</i></li>
     <ul>
     <li>
       <a href="/a/app/record/dataset/DS_65b833fa3a">Genome-wide transcriptional analysis of tomatidine-treated <i>C. albicans</i> cells</a> &mdash;
       <i>Candida albicans</i> was subjected to treatment with tomatidine and samples were taken at 2 different time points (1h, 3h).
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/28674054" target="_blank">Dorsaz et al. 2017</a>.  
     </li>
     </ul>

     <li><i>Fusarium proliferatum</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_1dd4f50bad">Transcriptome of N. crassa strains grown on different carbon sources (citrus peel, Miscanthus, corn stover, Wing Nut, Energy Cane, Switchgrass)</a> &mdash;
        Transcriptional profiling of N. crassa on 40 different carbon sources, including plant biomass, to provide data on how fungi sense simple to complex carbohydrates.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32111691" target="_blank">Wu et al. 2020</a>.  
      </li>
      </ul>

      <li><i>Fusarium graminearum</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_1dd4f50bad">Transcriptome of N. crassa strains grown on different carbon sources (citrus peel, Miscanthus, corn stover, Wing Nut, Energy Cane, Switchgrass)</a> &mdash;
        Transcriptional profiling of N. crassa on 40 different carbon sources, including plant biomass, to provide data on how fungi sense simple to complex carbohydrates.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32111691" target="_blank">Wu et al. 2020</a>.  
      </li>
      </ul>

      <li><i>Neurospora crassa</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_1dd4f50bad">Transcriptome of N. crassa strains grown on different carbon sources (citrus peel, Miscanthus, corn stover, Wing Nut, Energy Cane, Switchgrass)</a> &mdash;
        Transcriptional profiling of N. crassa on 40 different carbon sources, including plant biomass, to provide data on how fungi sense simple to complex carbohydrates.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32111691" target="_blank">Wu et al. 2020</a>.  
      </li>
      </ul>

      <li><i>Cryptococcus neoformans</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_1dd4f50bad">Transcriptome of N. crassa strains grown on different carbon sources (citrus peel, Miscanthus, corn stover, Wing Nut, Energy Cane, Switchgrass)</a> &mdash;
        Transcriptional profiling of N. crassa on 40 different carbon sources, including plant biomass, to provide data on how fungi sense simple to complex carbohydrates.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32111691" target="_blank">Wu et al. 2020</a>.  
      </li>
      </ul>


      </ul>   
    <br>
          
    <h4>ChiP-Seq</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_aeb210cd5d">RNAP II ChIP-Seq data for <i>Aspergillus fumigatus</i></a>&mdash;
            ChIP-seq was performed on WT conidia to map the genome-wide occupancy of elongating RNA polymerase II (RNAP II).   
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/34183813" target="_blank">Wang et al. 2021</a>.  
            </li>

          <li>
             <a href="/a/app/record/dataset/DS_aeb210cd5d">RNAP II ChIP-Seq data for <i>Aspergillus fumigatus</i></a>&mdash;
             ChIP-seq was performed on WT conidia to map the genome-wide occupancy of elongating RNA polymerase II (RNAP II).   
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/34183813" target="_blank">Wang et al. 2021</a>.  
              </li>
            </ul>

          <br>

          <h4>Proteomics</h4>
          <ul>
              <li><i>Candida albicans</i></li>
                <ul>
                  <li>
    
              <a href="/a/app/record/dataset/DS_d7b42bd592">Extracellular vesicle and whole cell lysate proteomes for DAY226 yeast/biofilm, ATCC90028 and ATCC10231 strains</a>&mdash;
              Extracellular vesicle (EV) and whole cell lysate (WCL) proteome analysis.   
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/32363014" target="_blank">Dawson et al. 2020</a>.  
              </li>
            </ul>

            <li><i>Candida auris</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_d609b47fc6">Whole cell proteomics of <i>Candida auris</i> MMC1 and MMC2 strains</a>&mdash;
                Whole cell proteomics of the clinical isolates isolated in New York, USA.   
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/31186339" target="_blank">Zamith-Miranda et al. 2019</a>.  
              </li>
              </ul>

          </ul>

          <br>

		<h4>Phenotype data</h4>
            
        <ul>
        <li><a href="/a/app/record/dataset/DS_f2312514ac">Manually curated Phenotypes (VEuPathDB)</a> &ndash; 
            We have curated and integrated phenotypes from five publications (<a href="https://pubmed.ncbi.nlm.nih.gov/25477510/" target="_blank">Lee et al. 2015</a>, 
            <a href="https://pubmed.ncbi.nlm.nih.gov/26112692/" target="_blank">Esher et al. 2015</a>,  <a href="https://pubmed.ncbi.nlm.nih.gov/27303728/" target="_blank">Esher et al. 2016</a>,  
            <a href="https://pubmed.ncbi.nlm.nih.gov/29864141/" target="_blank">Esher et al. 2018</a> and  <a href="https://pubmed.ncbi.nlm.nih.gov/32398313/" target="_blank">Thak et al. 2020</a>) 
            for <i>Cryptococcus neoformans</i> H99 in this release.    
         </li>
        </ul>
        <br>
          <div style="margin-left: 3em;">
                    <style>
                      #Phenotype_Bld59Asp table, #Phenotype_Bld59Asp td, #Phenotype_Bld59Asp th, #Phenotype_Bld59Asp tr {
                      text-align : left;
                      padding-left: 5px;
                      padding-right: 5px;
                      padding-top: 5px;
                      padding-bottom: 5px;
                      border: 1px solid black;
                      }
                      #Phenotype_Bld59Asp {
                      margin-left : 5 em;
                      }
                    </style> 
                      <table id="Phenotype_Bld59Asp"> 
                      <tr>
                        <th>Species</th>
                        <th>Phenotype mapping</th>
                        <th>Genes with phenotypes</th>
                      </tr>
                      <tr>
                        <td><i>Cryptococcus neoformans</i> H99</td>
                        <td>197</td>
                        <td>19</td>
                      </tr>
                      </table>
            </div>
          <br>
          
           <h4>Updated Functional Annotation</h4>
       		<ul>
             <li>
             <a href="https://docs.google.com/spreadsheets/d/1w8RfQtPm9Rez2G6TmDwFUoAmGiGQ8MIudj665oW7hIo/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include gene names, products, GO terms, publications and notes from curators. These functional annotations are current with GeneDB May 2022 GFF version.
            </li>
            </ul> 
            <br>
               
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld59 table, #FunctAnnot_Bld59 td, #FunctAnnot_Bld59 th, #FunctAnnot_Bld59 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld59 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld59"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-05/CneoformansH99/CneoformansH99.gff.gz">GeneDB version May, 2022</a></td>
                  <td>23</td>
                </tr>
                </table>
                </div>
                
                <br>
           
           <h4>Curated User Comments</h4> 
           <ul>
           <li>          
           Gene page comments that are entered by users are curated and integrated as functional annotations in FungiDB. 5 user comments have been reviewed and updated in this release.
           </li>
           </ul>
           <br>
           <div style="margin-left: 3em;">   
           <style>
            #UComment_Bld59 table, #UComment_Bld59 td, #UComment_Bld59 th, #UComment_Bld59 tr {
            text-align : center;
            padding-left: 5px;
            padding-right: 5px;
            padding-top: 5px;
            padding-bottom: 5px;
            border: 1px solid black;
            }
            #UComment_Bld59 {
            margin-left : 5 em;
            }
            </style> 
            <table id="UComment_Bld59"> 
            <tr>
            <th>FungiDB Genome Version</th>
            <th>Comments Integrated</th>
            </tr>
            <tr>
            <td title="Click here to view this dataset's full description."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99</a></td>
            <td>5</td>
            </tr>
            </table>
            </div>
           
           <br>
       
       </div>
        
       <section>
        <h3>New features in {{project}} 59</h3>
        <div style="margin-left: 3em;">
        <h4>'Send To' actions for the results of a gene search</h4>
          <ul>
            <li>Send to My Data Sets:  Save your search result as an ID list in My Data Sets. Open the 'Send To' dropdown menu and choose 'My Data Sets' to create your private data set with the IDs from the search result.  Your new data set record will appear on your My Data Sets page as a downloadable text file. You must be a registered {{project}} user and logged in to use this feature. <a target="_blank" href="{{'/assets/images/MyDataSets/SaveListMyDataSets.gif' | absolute_url}}">View tutorial GIF.</a></li><br>
            
            <li>Send to VEuPathDB: Transfer a gene list from {{project}} to VEuPathDB and analyze in context with all VEuPathDB organisms.  From the {{project}} gene result, open the 'Send To' dropdown menu and choose 'VEuPathDB.org Strategy'.  The VEuPathDB My Strategies page will open in a new window displaying a one-step strategy that returns the gene list just transferred from {{project}}. Once transferred, use the 'Add a step' tool to extend the strategy and take advantage of orthology across all VEuPathDB organisms. <a target="_blank" href="{{'/assets/images/MyDataSets/SendToVeupathdb.gif' | absolute_url}}">View tutorial GIF.</a>  </li>
          </ul>
        <h4>Upload a gene list to My Data Sets</h4>
          <ul>
            <li>In preparation to provide gene set functionality such as gene set enrichment analysis we are introducing this new feature. The 'New upload' tab of <a href="/a/app/workspace/datasets">My Data Sets</a> allows for the upload of a gene list into {{project}} directly from a file or URL. Your new list will appear in the All tab and will remain available between sessions. If the results of your last experiment or the last article you read included a gene list, you can upload it here for further analysis. <a target="_blank" href="{{'/assets/images/MyDataSets/UploadListMyDataSets.gif' | absolute_url}}">View tutorial GIF.</a></li>
          </ul>
        <h4>Sequence Retrieval Tool improvements</h4>
          <ul>
            <li>The <a href="/a/app/fasta-tool/genesequence">Sequence Retrieval Tool</a> now remembers your parameter settings during your session. You can leave and return to the tool without having to set every parameter again. </li>
          </ul>
        </div>
      </section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
