---
layout: post
title: FungiDB 56 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 56.</h3>

<section>
  <h3>Another way to stay in touch:</h3>
  <div style="margin-left: 3em;">
    The bioinformatic resource centers have established a <a href="https://www.reddit.com/r/BRC_users" target="_blank">Reddit community</a> for free and open discussions. We welcome everyone to join the discussion.
  </div>
</section>

<section>
  <h3>Release 54 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday September 16, 2021 at 10am EDT for our Release 54 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 54.  
    <a href="https://attendee.gotowebinar.com/register/699699930361785872" target="_blank">Register here</a>  
  </div>
</section>

    <section>
      <h3>New data in FungiDB 56</h3>
      <div style="margin-left: 3em;">

      <h4>Genome Sequence and Annotation: updated genomes </h4>
      <ul>
        
        <li><a href="/a/app/record/dataset/DS_3182ff461b"><i>Trichoderma reesei</i> QM6a 2017</a> &ndash;
          We are pleased to announce the first update for a FungiDB genome through Apollo, a web-based genomic annotation editing platform. We incorporated 225 changes, including creation of new genes, modification of existing genes, merges and splits.  
          </li>
        </ul>
       <br>

          <img style="width:20em" src="{{'/assets/images/fungidb/apollofungidb56.png' | absolute_url}}" />
        <br>

       <h4>Genome Sequence and Annotation: </h4>
       <ul>
         
         <li><a href="/a/app/record/dataset/DS_0360a7ae66"><i>Candida dubliniensis</i> CD36</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/19745113">Jackson et al. 2009</a>.  
            </li>
 
         <li><a href="/a/app/record/dataset/DS_d18b70822b"><i>Candida metapsilosis</i> BP57</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/31105748">Mixão et al. 2019</a>.  
             </li>
   
           <li><a href="/a/app/record/dataset/DS_c2284632aa"><i>Cladophialophora bantiana</i> CBS 173.52</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>.  
               </li>
   
         <li><a href="/a/app/record/dataset/DS_c454ebb27c"><i>Cyberlindnera fabianii</i> 65</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/28385833">van Rijswijck et al. 2017</a>.  
             </li>
 
         <li><a href="/a/app/record/dataset/DS_a363c3f7a3"><i>Fusarium oxysporum</i> f. sp. <i>conglutinans</i> Fo5176</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/32843521">Fokkens et al. 2020</a>.  
               </li>
 
          <li><a href="/a/app/record/dataset/DS_3f9de63eb2"><i>Lodderomyces elongisporus</i> NRRL YB-4239</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/19465905">Butler et al. 2009</a>.  
               </li>
 
         <li><a href="/a/app/record/dataset/DS_05a7d4a66a"><i>Meyerozyma guilliermondii</i> ATCC 6260</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/30462193">Seixas et al. 2019</a>.  
               </li>
 
         <li><a href="/a/app/record/dataset/DS_c9de871465"><i>Pneumocystis oryctolagi</i> CS1</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/33686174">Cissé et al. 2021</a>.  
               </li>
 
         <li><a href="/a/app/record/dataset/DS_8037a33828"><i>Pneumocystis wakefieldiae</i> 2A</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/33686174">Cissé et al. 2021</a>.  
                 </li>
 
         <li><a href="/a/app/record/dataset/DS_7304b45c56"><i>Trichomonascus ciferrii</i> CBS 4856</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/31575637">Mixão et al. 2019</a>.  
               </li>    

        <li><a href="/a/app/record/dataset/DS_206186a632"><i>Trichophyton rubrum</i> CBS 118892</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/22951933">Martinez et al. 2012</a>.  
               </li> 
       </ul>
       <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><i>Aspergillus fumigatus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_2216592524">Whole genome association study for itraconazole sensitivity in <i>Aspergillus fumigatus</i></a> &ndash;
              Clinical isolates of <i>A. fumigatus</i> from Japan.
              <a href="https://doi.org/10.3389/ffunb.2020.617338">Zhao et al. 2021</a>.
            </li>
            <li>
              <a href="/a/app/record/dataset/DS_f280f1be3e">Strain specific <i>A. fumigatus</i> persistence in the murine lung</a> &ndash;
              Characterisation of W72310 strain by WGS and comparison to a common reference strain, CEA10.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/33597172">Jones et al. 2021</a>.
            </li>
            </ul>

          <li><i>Candida albicans</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_caec2bd9bb"><i>Candida albicans</i> adaptation to azole antifungal drugs</a> &ndash;
                A study of the rapid adaptation to antifungal drugs.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/32687060">Todd et al. 2020</a>.
              </li>
            </ul>

          <li><i>Candida auris</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_c557df700f"><i>C. auris</i> B11220 mono- and multidrug resistance evolution experiment</a> &ndash;
                Experimentally evolved MDR clade II <i>C. auris</i>.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33820824">Carolus et al. 2021</a>.
              </li>
            </ul>
          
            <li><i>Cryptococcus neoformans</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_b57833f9f4">Genetic basis of within-host microevolution underlying recurrent Cryptococcal meningitis infection</a> &ndash;
                Whole-genome sequencing assessing the genetic basis of Cryptococcal infection in 17 HIV-infected individuals with recurrent cryptococcal meningitis.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33820824">Rhodes et al. 2017</a>.
              </li>
            </ul>
          
        </ul>
          
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Aspergillus fumigatus</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_2b76656bb1">Determining <i>Aspergillus fumigatus</i> transcription factor expression and function during invasion of the mammalian lung</a> &mdash;
          RNA-seq analysis was performed on wild-type and an ACE1 deletion mutant.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33780518" target="_blank">Liu et al. 2021</a>.  
      </li>
      </ul>

     <li><i>Candida albicans</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_075a4b4e1f">Transcriptomes of clinical isolate strains and mutant derivatives</a> &mdash;
        Transcriptome of the fungi in infected maize roots growing in CM agar.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31116789" target="_blank">Huang et al. 2019</a>.  
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_24f03e74c5">Transcriptomic analysis of wild type and GCN5-deficient <i>Candida albicans</i> cells prior to and after treatment with caspofungin</a> &mdash;
        Investigation of the virulence factors in <i>C. albicans</i> 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31263212" target="_blank">Shivarathri et al. 2019</a>.   
        </li>

        <li>
          <a href="/a/app/record/dataset/DS_429e31c02f">CalCEN: Candida albicans co-expression Network v1.0.0</a> &mdash;
          Raw reads from 853 RNA-seq runs of 18 large-scale <i>Candida albicans</i> studies were collected from NCBI and re-mapped to 6226 open reading frames of the SC5314 strain. The CalCEN co-expression network was then estimated by the pairwise spearman rank-correlation of FPKM expression values across all runs. 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33472984" target="_blank">Teresa O'Meara and Matthew O'Meara, 2021</a>.   
          </li>
      </ul>

      <li><i>Candida auris</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_cd56220e4b">Experimental evolution of fluconazole resistance in <i>Candida auris</i></a> &mdash;
          Transcriptomic profiles of drug resistant and sensitive strains of <i>C. auris</i>. 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33077664" target="_blank">Bing et al. 2020</a>.   
          </li>
        </ul>

      <li><i>Candida glabrata</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_0024327063">DNA damage-induced transcriptome changes in <i>Candida glabratae</i></a> &mdash;
          RNA-Seq analysis of transcriptional rewiring of the DNA damage response in <i>C. glabrata</i>.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33323516" target="_blank">Shor et al. 2020</a>. 
       </li>
      </ul>

      <li><i>Candida lusitaniae</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_9a20a7bfa0">Transcriptional profiling of strains with different levels of Mrr1 activity in <i>Candida lusitaniae</i></a> &mdash;
          Expression profiling by high throughput sequencing.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33785623" target="_blank">Demers et al. 2021</a>.   
          </li>
        </ul>

        <li><i>Coccidioides posadasii</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_e68c49ad2b">Transcriptomics of environmental (mycelia) and parasitic (spherule 96 hr) stages of the Δcts2/Δard1/Δcts3 strain</a> &mdash;
          An attenuated mutant strain Δcts2/Δard1/Δcts3 in which chitinase genes 2 and 3 were deleted was previously created for vaccine development and profiled by RNA-Seq at two conditions.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/32499817" target="_blank">Mead et al. 2020</a>.   
          </li>
        </ul>


        <li><i>Cryptococcus neoformans</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_eb589156b3">Transcriptomic profiling of the <i>Cryptococcus neoformans</i> transition to titan-like cells</a> &mdash;
          RNA-Seq study of different time points of the transition: 7 and 24 h.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/29775477" target="_blank">Trevijano-Contador et al. 2018</a>.   
          </li>
        </ul>

        <li><i>Saccharomyces cerevisiae</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_6de7930665">DNA damage-induced transcriptome changes in <i>Saccharomyces cerevisiae</i></a> &mdash;
          RNA-Seq analysis of transcriptional rewiring of the DNA damage response.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/33323516" target="_blank">Shor et al. 2020</a>.   
          </li>
        </ul>
    
    </ul>   
    <br>
          
       <h4>Phenotype data</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_0732741900"><i>Candida albicans</i> functional genomic screening</a> &mdash;
            Compiled and normalized <i>Candida albicans</i> functional genomic screening data from ~400 screens.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/34864983" target="_blank">Gervais et al. 2021</a>.   
            </li>

        <li><a href="/a/app/record/dataset/DS_f2312514ac">Manually curated Phenotypes (VEuPathDB)</a> &ndash; 
            We have curated and integrated phenotypes from <a href="https://pubmed.ncbi.nlm.nih.gov/32546620/" target="_blank">Valero et al. 2020</a> for <i>Aspergillus fumigatus</i> Af293 and 
            <a href="https://pubmed.ncbi.nlm.nih.gov/22589729/" target="_blank">Saitoh et al. 2012</a> for <i>Pyricularia oryzae</i> 70-15 in this release.    
             </li>
             </ul>
          <br>
          <div style="margin-left: 3em;">
                    <style>
                      #Phenotype_Bld56Asp table, #Phenotype_Bld56Asp td, #Phenotype_Bld56Asp th, #Phenotype_Bld56Asp tr {
                      text-align : center;
                      padding-left: 5px;
                      padding-right: 5px;
                      padding-top: 5px;
                      padding-bottom: 5px;
                      border: 1px solid black;
                      }
                      #Phenotype_Bld56Asp {
                      margin-left : 5 em;
                      }
                    </style> 
                      <table id="Phenotype_Bld56Asp"> 
                      <tr>
                        <th>Species</th>
                        <th>Phenotype mapping</th>
                        <th>Genes with phenotypes</th>
                      </tr>
                      <tr>
                        <td><i>Aspergillus fumigatus</i> Af293</td>
                        <td>83</td>
                        <td>12</td>
                      </tr>
                      <tr>
                        <td><i>Pyricularia oryzae</i> 70-15</td>
                        <td>86</td>
                        <td>77</td>
                      </tr>
                      </table>
          <br>
     </div>
        
<section>
  <h3>New features in {{project}} 54</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>A table describing <a href="/a/app/static-content/dataInprogress.html">Datasets we are working on loading</a> is now available from the 'About' menu in the {{project}} header. While this list describes data slated for the next few VEuPathDB releases, the list is fluid and will be updated frequently.  Please <a href="/a/app/contact-us">Contact Us</a> with comments or suggestions and let us know of data you would like to see integrated.  </li>
    </ul>
  </div>
</section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
