---
layout: post
title: FungiDB 55 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 55.</h3>

<section>
  <h3>Another way to stay in touch:</h3>
  <div style="margin-left: 3em;">
    The bioinformatic resource centers have established a <a href="https://www.reddit.com/r/BRC_users" target="_blank">Reddit community</a> for free and open discussions. We welcome everyone to join the discussion.
  </div>
</section>

<section>
  <h3>Release 54 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday September 16, 2021 at 10am EDT for our Release 54 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 54.  
    <a href="https://attendee.gotowebinar.com/register/699699930361785872" target="_blank">Register here</a>  
  </div>
</section>

    <section>
      <h3>New data in FungiDB 55</h3>
      <div style="margin-left: 3em;">

      <h4>Genome Sequence and Annotation: updated genomes </h4>
      <ul>
        
        <li><a href="/a/app/record/dataset/DS_3182ff461b"><i>Trichoderma reesei</i> QM6a 2017</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/28690679">Wan-Chen Li et al. 2017</a>.  
          </li>
        </ul>
       <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><i>Blastomyces percursus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_76baca140e">SNP calls on WGS isolates causing blastomycosis in Africa and the Middle East</a> &ndash;
              Sequencing of <i>Blastomyces percursus</i> isolates causing blastomycosis in Africa and the Middle East.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/32766820">Schwartz et al. 2020</a>.
            </li>
            </ul>

          <li><i>Cryptococcus neoformans</i></li>
            <ul>
              <li>
              DAR1401 isolate was added to the following dataset: <a href="/a/app/record/dataset/DS_d8fd7514d2">SNPs calls on WGS of clinical Cryptococcus neoformans isolates recovered from patients undergoing anti-fungal treatment</a> &ndash;
              Single-end reads of serial isolates from cryptococcal meningitis cases.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/30688656">Stone et al. 2019</a>.
              </li>

              <li>
                <a href="/a/app/record/dataset/DS_e03933e8eb">SNP calls on WGS of <i>Cryptococcus</i> isolates behind the outbreak in Queen Elizabeth University Hospital, Glasgow, Scotland, 2018</a> &ndash;
                Genome sequencing of Cryptococcus neoformans isolates.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33620303">Farrer et al. 2021</a>.
              </li>
            </ul>
          
        </ul>
          
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Coccidioides immitis</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_3c0865bc03">Transcriptional analysis of <i>Coccidioides immitis</i> mycelia and spherules</a> &mdash;
          Analysis of gene expression changes in mycelia and young and mature spherules.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/34067070" target="_blank">Carlin et al. 2021</a>.  
      </li>
      </ul>

     <li><i>Fusarium fujikuroi</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_bdd3c72769">In planta expression profiles of <i>Fusarium fujikuroi</i></a> &mdash;
        Transcriptome of the fungi in infected maize roots growing in CM agar.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/28040774" target="_blank">Niehaus et al. 2016</a>.  
      </li>
      </ul>

      <li><i>Mucor circinelloides</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_4703a409e8">Transcriptomics of <i>Mucor circinelloides</i> NCRIP deficient spores phagocytosed by mouse macrophages</a> &mdash;
          Transcriptomics of R3B2 and RDRP1 fungal spores and fungal spore-macrophage coculture. This data set was re-aligned against the genome of <i>Mucor lusitanicus</i> CBS 277.49 and now also contains the control sample L15 R7B 5h. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32658892" target="_blank">Pérez-Arques et al. 2020</a>.   
          </li>
        </ul>

      <li><i>Neurospora crassa</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_a03298576e">Comparative transcriptome of <i>Neurospora crassa</i> WT and delta mus52 strains</a> &mdash;
          RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/29668735" target="_blank">Martins et al. 2018</a>. 
       </li>

       <li>
        <a href="/a/app/record/dataset/DS_afadd9b553"><i>Neurospora crassa</i> gene co-expression network</a> &mdash;
        The co-expression network was used to determine the transcriptional response of <i>Neurospora crassa</i> to various carbohydrates. Philipp Benz et al. 2021 (submitted).  
       </li>
      </ul>
    
    </ul>   
    <br>
 
            
       <h4>Phenotype data</h4>
             
             <ul>
             <li><a href="/a/app/record/dataset/DS_f2312514ac">Manually curated Phenotypes (VEuPathDB)</a> &ndash; 
                 We have curated and integrated phenotypes from <a href="https://pubmed.ncbi.nlm.nih.gov/32546620/" target="_blank">Valero et al. 2020</a> for 
                 <i>Aspergillus fumigatus</i> Af293 and <a href="https://pubmed.ncbi.nlm.nih.gov/22589729/" target="_blank">Saito et al. 2012</a> for 
                 <i>Pyricularia oryzae</i> 70-15 in this release.    
             </li>
             </ul>
          <br>
          <div style="margin-left: 3em;">
                    <style>
                      #Phenotype_Bld56Asp table, #Phenotype_Bld56Asp td, #Phenotype_Bld56Asp th, #Phenotype_Bld56Asp tr {
                      text-align : center;
                      padding-left: 5px;
                      padding-right: 5px;
                      padding-top: 5px;
                      padding-bottom: 5px;
                      border: 1px solid black;
                      }
                      #Phenotype_Bld56Asp {
                      margin-left : 5 em;
                      }
                    </style> 
                      <table id="Phenotype_Bld56Asp"> 
                      <tr>
                        <th>Species</th>
                        <th>Phenotype mapping</th>
                        <th>Genes with phenotypes</th>
                      </tr>
                      <tr>
                        <td><i>Aspergillus fumigatus</i> Af293</td>
                        <td>83</td>
                        <td>12</td>
                      </tr>
                      <tr>
                        <td><i>Pyricularia oryzae</i> 70-15</td>
                        <td>86</td>
                        <td>77</td>
                      </tr>
                      </table>
          <br>
     </div>
</section>
        
<section>
  <h3>New features in {{project}} 54</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>A table describing <a href="/a/app/static-content/dataInprogress.html">Datasets we are working on loading</a> is now available from the 'About' menu in the {{project}} header. While this list describes data slated for the next few VEuPathDB releases, the list is fluid and will be updated frequently.  Please <a href="/a/app/contact-us">Contact Us</a> with comments or suggestions and let us know of data you would like to see integrated.  </li>
    </ul>
  </div>
</section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
