---
layout: post
title: FungiDB 59 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 59.</h3>

<section>
  <h3>Release 59 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday September 15th, 2022 at 10am EDT for our Release 59 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 59.  
    <a href="https://attendee.gotowebinar.com/register/3393660656180049424" target="_blank">Register here</a>  
  </div>
  <br>
</section>

    <section>
      <h3>New data in FungiDB 59</h3>
      <div style="margin-left: 3em;">

       <h4>Genome Sequence and Annotation: </h4>
       <ul>
        <li><a href="/a/app/record/dataset/DS_c8c6d0b908"><i>Aspergillus cristatus</i> GZAAS20.1005</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/27434633">Zeiner et al. 2016</a>.  
           </li>

         <li><a href="/a/app/record/dataset/DS_9ad6869a2a"><i>Chaetomium globosum</i> CBS 148.51</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30473694">Grogan et al. 2018</a>.
            </li>
 
        <li><a href="/a/app/record/dataset/DS_f453f876da"><i>Candida glabrata</i> CBS138 2020</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/29178836">Kramer et al. 2017</a>.  
              </li>

         <li><a href="/a/app/record/dataset/DS_aa6c0c41fd"><i>Coccidioides posadasii</i> strain Silveira 2020</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/31719178">Farrer et al. 2019</a>.
             </li>
   
          <li><a href="/a/app/record/dataset/DS_ea9df1c0c0"><i>Fusarium mangiferae</i> MRC7560</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>. 
              </li>
   
         <li><a href="/a/app/record/dataset/DS_d8e276588e"><i>Histoplasma capsulatum</i> G184AR</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/30998802">Porto et al. 2019</a>. 
          </li>
           
          <li><a href="/a/app/record/dataset/DS_cd741aae10"><i>Macrophomina phaseolina</i> MS6</a> &ndash;
            Li-Jun Ma 2011.
            </li>

          <li><a href="/a/app/record/dataset/DS_a2d60e11ca"><i>Metarhizium anisophilae</i> ARSEF 549</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30533777">Tavares et al. 2018</a>.
              </li>

          <li><a href="/a/app/record/dataset/DS_3a745be8fc"><i>Pyrenophora teres</i> f. <i>teres</i> 0-1</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26089407">Rujirawat et al. 2015</a>.
                </li>

          <li><a href="/a/app/record/dataset/DS_82669379aa"><i>Synchytrium endobioticum</i> MB42</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/34747029">Yildirir et al. 2022</a>.
                </li>
       </ul>
       <br>

       <h4>Updated genomes: </h4>
       <ul>
        <li><a href="/a/app/record/dataset/DS_5ead846f15"><i>Batrachochytrium dendrobatidis</i> JEL42</a> &ndash;
          Jiangsu University of Science and Technology, 2022.  
              </li>
        <li><a href="/a/app/record/dataset/DS_5ead846f15"><i>Cryptococcus gattii</i> VGII R265</a> &ndash;
                Jiangsu University of Science and Technology, 2022.  
                    </li>
       </ul>
      <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><i>Aspergillus fumigatus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_df3742f4a3">SNP calls on WGS of clinical <i>Aspergillus fumigatus</i> isolates from Portugal</a> &ndash;
              Genome sequencing of 8 clinical isolates of <i>Aspergillus fumigatus</i> from the Portuguese Oncology Institute in Porto, Portugal.
              Abigail Lind.
            </li>
            </ul>

          <li><i>Histoplasma capsulatum</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_a9eb7e00a0">SNPs on WGS of <i>Histoplasma</i> isolates from Alberta, Canada</a> &ndash;
                Clinical strains were collected by the AB ProvLab, Canada.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/35544208">Dingle et al. 2021</a>.
              </li>
            </ul>

            <li><i>Pneumocystis jirovecii</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_1ddb8babd8">SNP calls on WGS of <i>P. jirovecii</i> isolated from human bronchoalveolar lavage samples</a> &ndash;
                SNP calls on <i>P. jirovecii</i> isolates (54c, and 46).
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33686174">Cissé et al. 2021</a>.
              </li>
            </ul>

            <li><i>Pyricularia oryzae</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_9906c5c8f5">SNPs calls on WGS data of <i>Pyricularia oryzae</i> isolates from Africa</a> &ndash;
                SNPs calls on WGS data of 29 <i>Pyricularia oryzae</i> isolates.
                Talbot et al. 2019.
              </li>
            </ul>

            <li><i>Zymoseptoria tritici</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_486c6c20b9">SNP calls on WGS of <i>Zymoseptoria tritici</i> from infected wheat fields</a> &ndash;
                <i>Z. tritici</i> field sampling in Switzerland and Oregon, USA.
                Croll et al. 2019.
              </li>
            </ul>

        </ul>
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Aspergillus nidulans</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_e9d3e44418">Gene expression in <i>Aspergillus nidulans</i> grown on multiple nitrogen sources</a> &mdash;
          <i>Aspergillus nidulans</i> (MH1; biA1) was grown in liquid culture with ammonium, glutamine or alanine as the nitrogen source. 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/34154419" target="_blank">Steyer et al. 2021</a>.  
      </li>
      </ul>

     <li><i>Botrytis cinerea</i></li>
     <ul>
       <li>
       <a href="/a/app/record/dataset/DS_3c23b11271">Transciptomic analysis of <i>B. cinerea</i> exposed to a bacterial biocontrol agent (secretome)</a> &mdash;
       Transcriptome of <i>B. cinerea</i> in response to exposure to Bacillus subtilis MBI 600, a new commercialized biopesticide.
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/33839506" target="_blank">Samaras et al. 2021</a>.   
       </li>
     </ul>

     <li><i>Candida albicans</i></li>
     <ul>
     <li>
       <a href="/a/app/record/dataset/DS_65b833fa3a">Genome-wide transcriptional analysis of tomatidine-treated <i>C. albicans</i> cells</a> &mdash;
       <i>Candida albicans</i> was subjected to treatment with tomatidine and samples were taken at 2 different time points (1h, 3h).
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/28674054" target="_blank">Dorsaz et al. 2017</a>.  
     </li>
     </ul>

     <li><i>Fusarium proliferatum</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_91ca610be2">In planta expression profiles of <i>Fusarium proliferatum</i> NRRL 62905</a> &mdash;
        Transcriptional changes during maize root infection. Root samples were taken 4, 10 and 20 dpi for RNA isolation. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/28040774" target="_blank">Niehaus et al. 2015</a>.  
      </li>
      </ul>

      <li><i>Fusarium graminearum</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_326fa801af">RNA-seq analysis of FSS1-dependent genes in <i>Fusarium graminearum</i> Z-3639</a> &mdash;
        The wild-type, fss1 deletion, fss1 overexpression mutant strains were incubated in potato dextrose broth (PDB) with or without 1 M NaCl supplementation for an hour. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/25627458" target="_blank">Son et al. 201</a>.  
      </li>
      </ul>

      <li><i>Neurospora crassa</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_bb6e5ae0e5">Transcriptomes of strains grown on different carbon sources (mono-, di- and polysaccharides)</a> &mdash;
        Transcriptional profiling of <i>N. crassa</i> on different carbon sources. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32111691" target="_blank">Wu et al. 2020</a>.  
      </li>
      </ul>

      <li><i>Cryptococcus neoformans</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_09d688636a"><i>Cryptococcus neoformans</i> var. <i>grubii</i> KN99: Mapping TSS with TSS-Seq</a> &mdash;
        Positioning the 5′ ends of coding genes.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32020195" target="_blank">Wallace et al. 2020</a>.  
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_5c98701c8c"><i>Cryptococcus neoformans</i> JEC21: Mapping TSS with TSS-Seq</a> &mdash;
        Positioning the 5′ ends of coding genes. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32020195" target="_blank">Wallace et al. 2020</a>.  
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_766c6b1ce3"><i>Cryptococcus neoformans</i> var. <i>grubii</i> KN99: Mapping 3′ ends of coding genes with QuantSeq 3′mRNA-Seq (PAS-Seq)</a> &mdash;
        Positioning the 3′ ends of coding genes. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32020195" target="_blank">Wallace et al. 2020</a>.  
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_34f2ba224c"><i>Cryptococcus neoformans</i> JEC21: Mapping 3′ ends of coding genes with QuantSeq 3′mRNA-Seq (PAS-Seq)</a> &mdash;
        Positioning the 3′ ends of coding genes.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32020195" target="_blank">Wallace et al. 2020</a>.  
      </li>
     
      </ul>


      </ul>   
    <br>
          
    <h4>ChiP-Seq</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_93c08648a2">RNAP II ChIP-Seq for <i>Aspergillus nidulans</i></a> &mdash;
            ChIP-seq was performed on WT conidia to map the genome-wide occupancy of elongating RNA polymerase II (RNAP II).   
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/34183813" target="_blank">Wang et al. 2021</a>.  
            </li>

          <li>
             <a href="/a/app/record/dataset/DS_c608d4b120">Mapping histone modifications in <i>Aspergillus nidulans</i> spores</a> &mdash;
             Mapping histone modifications in <i>Aspergillus nidulans</i> spores by ChIPseq (H3K36me3, H3K4me3, H3AC, H3)   
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/34183813" target="_blank">Wang et al. 2021</a>.  
              </li>
            </ul>

          <br>
       
       </div>
        
<section>
  <h3>New features in {{project}} 59</h3>
  <div style="margin-left: 3em;">
  
  <h4>{{project}} site search accesses Apollo data</h4>
    <ul>
      <li>Gene annotation updates entered in <a target="_blank" href="/a/app/static-content/apollo_help.html">{{project}} Apollo</a> are now indexed and available to the site search.  Genes with Apollo annotations that contain your search term will be available in site search results under >Genes >Filter gene fields.</li>
    </ul>
  <h4>Gene page navigation improvements</h4>
    <ul>
      <li>The RNA-Seq Transcription Summary graph now links directly to the Transcript Expression table. Click a data set title in the RNA-Seq Transcription Summary graph to navigate directly to the details for the same data set in the Transcript Expression table.</li>
      <li>The gene page content navigation tool now contains links to every table or attribute on the gene page. Open each numbered data section to reveal links to all data within the section.  Or use the 'expand all' link at the top of the content tool to view all links.</li>
    </ul>
  <h4>AlphaFold data available</h4>
    <ul>
      <li>Protein structure predictions from <a href="https://alphafold.ebi.ac.uk/" target="_blank">AlphaFold</a> are now available in {{project}}.  The AlphaFold database (<a href="https://alphafold.ebi.ac.uk/faq" target="_blank">Version 3</a>) was downloaded and structures were associated to {{project}} genes via the UniProt ID. <a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" target="_blank">Jumper et al. 2021</a> and <a href="https://pubmed.ncbi.nlm.nih.gov/34791371/" target="_blank">Varadi et al. 2022</a></li>
      <li>The <a href="/a/app/search/transcript/GenesWithAlphafold">AlphaFold search</a> returns all genes with AlphaFold predictions in {{project}}.</li>
      <li>Gene pages include a table of AlphaFold data and the AlphaFold 3D viewer for dynamic visualization of the predicted structure. </li>
    </ul>
  </div>
</section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
