---
layout: post
title: FungiDB 57 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 57.</h3>

<section>
  <h3>Release 57 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday February 24th, 2022 at 10am EDT for our Release 56 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 56.  
    <a href="https://attendee.gotowebinar.com/register/580233597692794126" target="_blank">Register here</a>  
  </div>
</section>

    <section>
      <h3>New data in FungiDB 57</h3>
      <div style="margin-left: 3em;">

       <h4>Genome Sequence and Annotation: </h4>
       <ul>

        <li><a href="/a/app/record/dataset/DS_9e005f7b0b"><i>Aspergillus fijiensis</i> CBS 313.89</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30349117">Vesth et al. 2018</a>.  
           </li>

         <li><a href="/a/app/record/dataset/DS_09efa311b1"><i>Candida tropicalis</i> MYA-3404 (2020)</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/32469306">Guin et al. 2020</a>.  
            </li>
 
         <li><a href="/a/app/record/dataset/DS_44c7c4539b"><i>Cladophialophora psammophila</i> CBS 110553</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>.  
             </li>
   
           <li><a href="/a/app/record/dataset/DS_c839bb7a93"><i>Debaryomyces fabryi</i> CBS 789</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/26847909">Tafer et al. 2016</a>.  
               </li>
   
         <li><a href="/a/app/record/dataset/DS_76a40ba5c4"><i>Emergomyces (Emmonsia) africanus</i> CBS 136260</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/28176377">Dukik et al. 2017</a>.  
             </li>
 
         <li><a href="/a/app/record/dataset/DS_efbafbcb21"><i>Pneumocystis jirovecii</i> RU7</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/26899007">Ma et al. 2016</a>.  
               </li>
 
          <li><a href="/a/app/record/dataset/DS_d146ee5662"><i>Trichophyton mentagrophytes</i> TIMM 2789</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/30310114">Alshahni et al. 2018</a>.  
               </li>
 
         <li><a href="/a/app/record/dataset/DS_06f647c621"><i>Trichophyton tonsurans</i> CBS 112818</a> &ndash;
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/22951933">Martinez et al. 2012</a>.  
               </li>
        </ul>
       <br>
        
       <h4>Genome Sequence: </h4>
       <ul>

         <li><a href="/a/app/record/dataset/DS_ef14b36372"><i>Apiotrichum montevideense</i> JCM 9937</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/1497334">Guého et al. 1992</a>.  
           </li>

        <li><a href="/a/app/record/dataset/DS_99cf7bf998"><i>Emergomyces orientalis</i> 5z489</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/28409126">Yang et al. 2017</a>.  
            </li>
         </ul>
       <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><i>Aspergillus fumigatus</i></li>
            <ul>  
            <li>
              <a href="/a/app/record/dataset/DS_f9dd9912d5">SNP calls on WGS of <i>Aspergillus fumigatus</i> strains collected in Spain</a> &ndash;
              Genome-wide comparative analysis of <i>Aspergillus fumigatus</i> strains.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/30029559">Garcia-Rubio et al. 2018</a>.
            </li>
            <li>
              <a href="/a/app/record/dataset/DS_b7b6cff99c">SNP calls on WGS of <i>Aspergillus fumigatus</i> isolates from a CDC surveillance of triazole resistance in the United States</a> &ndash;
              Genomic Surveillance of <i>Aspergillus fumigatus</i> Triazole Resistance in the United States.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/34372699">Etienne et al. 2021</a>.
            </li>
            </ul>

          <li><i>Clavispora lusitaniae</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_91c53e61a3">SNPs calls on WGS of <i>Clavispora</i> (<i>Candida</i>) <i>lusitaniae</i> strains and populations from a patient with cystic fibrosis</a> &ndash;
                Population genomics study of <i>Clavispora</i> (<i>Candida</i>) <i>lusitaniae</i>.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/30389707">Demers et al. 2018</a>.
              </li>
            </ul>

          <li><i>Cryptococcus neoformans</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_93ead824d4">SNP calls on WGS of <i>Cryptococcus</i> isolates collected from an Ugandan patient cohort</a> &ndash;
                Isolates were collected from an Ugandan patient cohort infected with closely related <i>C. neoformans</i> strains.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/31311883">Gerstein et al. 2019</a>.
              </li>
            </ul>
          
            <li><i>Fusarium verticillioides</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_e23b9209c1">SNP calls on WGS of <i>Fusarium verticillioides</i> strains</a> &ndash;
                SNP calls on laboratory cross strains (RJP98.118, RJP98.111, RJP98.75) and Fv999 re-sequenced strain.
                Tommas Hammond 2016.
              </li>
            </ul>

            <li><i>Pneumocystis macacae</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_b4b53f9435">SNP calls on WGS of <i>Pneumocystis macacae</i> isolates</a> &ndash;
                <i>P. macacae</i> isolates ER17, UC86, GL92.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/33686174">Cissé et al. 2021</a>.

              </li>
            </ul>
        </ul>
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li><i>Aspergillus nidulans</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_44cccd9101">Transcriptomics of fungal condinia and hyphae</a> &mdash;
          Transcription in fungal conidia (3 and 17 days) and hyphae.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/34183813" target="_blank">Wang et al. 2021</a>.  
      </li>
      </ul>

     <li><i>Fusarium proliferatum</i></li>
      <ul>
      <li>
        <a href="/a/app/record/dataset/DS_4c6607753d">In planta expression profiles of Fusarium proliferatum ET1</a> &mdash;
        Transcriptional changes during maize root infection. Root samples were taken 4, 10 and 20 dpi for RNA isolation.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/28040774" target="_blank">Niehaus et al. 2016</a>.  
      </li>
      </ul>

      <li><i>Mucor lusitanicus</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_9384e9d657"><i>Mucor circinelloides</i> spores phagocytosed by mouse macrophages: single culture</a> &mdash;
          A study of the transcriptomics changes occuring in both host and pathogen upon the interaction of <i>Mucor circinelloides</i> spores with phagocytic cells of a mouse. 
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30723131" target="_blank">Pérez-Arques et al. 2019</a>.   
          </li>
        </ul>

      <li><i>Neurospora crassa</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_0024327063">Transcriptional Profiling of sulfur starvation response</a> &mdash;
          Using RNA sequencing to perform a global survey of the <i>N. crassa</i> sulfur starvation response and the role of CYS-3 in regulating sulfur responsive genes.  
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/34523983" target="_blank">Huberman et al. 2021</a>. 
       </li>
      </ul>

      <li><i>Pyricularia oryzae</i></li>
        <ul>
          <li>
          <a href="/a/app/record/dataset/DS_372b3a2dc0">RNA-Seq of wheat blast field samples from Bangladesh (collected March, 2017)</a> &mdash;
          RNA-Seq of 15 isolates collected in the wheat fields.
          <a href="https://figshare.com/articles/dataset/Bangladesh_Field_RNA-Seq_samples_-_July_2017/5202451/2" target="_blank">Open Wheat Blast</a>.   
          </li>
        </ul>

      </ul>   
    <br>
          
    <h4>Function prediction</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_4a1633821f">CalCENv1 co-expression network UMAP clusters</a> &mdash;
            CalCENv1 co-expression network was projected to two dimensions using UMAP and 18 clusters were identified and annotated through gene set enrichment analysis.  
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/33472984" target="_blank">O'Meara and O'Meara, 2021</a>.  
            </li>
            </ul>

          <br>


       <h4>Phenotype data</h4>
            
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_e51cb08950">Phenotypes from PHI-base database</a> &mdash;
           The pathogen – host interactions (PHI) phenotypes and pathogen-specific 
           phenotypes from PHI-base were integrated for 61 different genomes available in FungiDB in this release. This dataset is current with PHI-base version 4.12. We have 
           integrated 5,381 phenotypic annotations mapping to 3,140 genes in total. <a href="http://www.phi-base.org/" target="_blank">PHI-base</a> is a web-accessible database that catalogues experimentally verified pathogenicity, virulence and effector genes from 
           fungal, oomycete and bacterial pathogens, which infect animal, plant, fungal and insect hosts. The original data file from PHI-base can be found on
           <a href="https://github.com/PHI-base/data/blob/master/releases/phi-base_v4-12_2021-09-02.csv" target="_blank">PHI-base GitHub</a> site. 
            </li>
            </ul>

          <br>
          
           <h4>Updated Functional Annotation</h4>
       		<ul>
             <li>
             <a href="https://docs.google.com/spreadsheets/d/1im0lQyZ6E1Qec48dOQjZDI0eGXKv_fY8TeE7_RptB4U/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include gene names, products, GO terms, publications and notes from curators. These functional annotations are current with GeneDB March 2022 GFF version. Functional annotation 
               suggestions made by our users via the <a href="https://apollo.veupathdb.org/annotator/index" target="_blank">VEuPathDB Apollo</a> instance are reviewed by our curators and are included 
               in the official annotation when appropriate.
            </li>
            </ul> 
            <br>
               
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld57 table, #FunctAnnot_Bld57 td, #FunctAnnot_Bld57 th, #FunctAnnot_Bld57 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld57 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld57"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_071f05cd56"><i>Aspergillus fumigatus</i> Af293</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/AfumigatusAf293/AfumigatusAf293.gff.gz">GeneDB version March, 2022</a></td>
                  <td>113</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_4b35c88aa1"><i>Aspergillus nidulans</i> FGSC A4</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/AnidulansFGSCA4/AnidulansFGSCA4.gff.gz">GeneDB version March, 2022</a></td>
                  <td>2</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_d22dd5953a"><i>Pyricularia oryzae</i> 70-15</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2022-03/Moryzae7015/Moryzae7015.gff.gz">GeneDB version March, 2022</a></td>
                  <td>91</td>
                </tr>
                </table>
                </div>
                
                <br>
           
           <h4>External database links</h4>

           <ul>
             <li>Links to <a href="http://www.phi-base.org/">PHI-base</a> database in the gene pages have been updated in this release, providing links for 3,140 genes from 61 genomes in FungiDB. 
             </li>
           </ul>
           
           <br>
       
       </div>
        
    <section>
      <h3>New features in {{project}} 57</h3>
      <div style="margin-left: 3em;">
      <h4>BLAST Tool Improvements</h4>
        <ul>
        <li>The <a href="/a/app/workspace/blast/new">BLAST (multi-query capable)</a> tool, known previously as BLAST<i>beta</i>, can perform queries with one or more FASTA-formatted sequences and offers many download types. The tool uses NCBI BLAST to find genes, genomic sequences, ESTs, or PopSet Sequences that have BLAST similarity to your input sequence. BLAST (multi-query capable) is <a href="{{'/assets/images/Find-blast-tool.png' | absolute_url}}">available from the tools menu.</a></li>
        <li>A new download option, <a href="{{'/assets/images/BLAST-This-table-download.png' | absolute_url}}">'This table (csv)'</a>, for the results of multi-sequence queries (combined result) retrieves the {{project}} result table for the results of multi-sequence queries (combined result). </li>
        <li><a href="{{'/assets/images/BLAST-56-improvements.gif' | absolute_url}}">View tutorial GIF.</a></li>
        </ul>
      <h4>Gene page sequence highlighting</h4>
        <ul>
          <li>Genomic and mRNA sequences on gene pages have a new color scheme.  Introns are highlighted in yellow and UTR sequence are highlighted in pink.  Previously they were gray and blue, respectively. <a href="{{'/assets/images/gene-seq-highlight.png' | absolute_url}}">Example</a></li>
        </ul>
      </div>
    </section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
