---
layout: post
title: FungiDB 66 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 66.</h3>

<section>
  <h3>Release 66 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday Dec 14th, 2023 at 10 am Eastern for our Release 66 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 66. 
    <a href="https://attendee.gotowebinar.com/register/6066397511661949782" target="_blank">Register here</a>
  </div>
  <br>
</section>

<section>
  <h3>2024 Fungal Pathogen Genomics course (in-person)</h3>
  <div style="margin-left: 3em;">
    <b>APPLY NOW!!</b> Registration for the in-person Fungal Pathogen Genomics course, 2-7 June 2024 in Hinxton, UK is now open! Learn how to take advantage of the unique tools offered by FungiDB, Ensembl Fungi, SGD/CGD, and MycoCosm/JGI databases & more. 
    <br>Application deadline: 29 February 2024. More information: <a href="https://coursesandconferences.wellcomeconnectingscience.org/event/fungal-pathogen-genomics-20240602/" target="_blank">2024 Fungal Pathogen Genomics</a>.
  </div>
  <br>
</section>

<section>
      <h3>New data in FungiDB 66</h3>
      <div style="margin-left: 3em;">

        <h4>Genome Sequence and Annotation </h4>
        <ul>
          <li><a href="/a/app/record/dataset/DS_0c1029de83"><i>Chaetothyriales sp.</i> CBS 132003</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/TBD">Westerdijk Fungal Biodiversity Institute</a>.
            </li>
          <li><a href="/a/app/record/dataset/DS_2fa12a8a7a"><i>Leptosphaeria maculans </i> JN3</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/21326234">Rouxel et al. 2012</a>.
            </li>
        </ul>
        <br>

        <h4>Genome replacements </h4>
        <ul>
          <li><a href="/a/app/record/dataset/DS_1488fda3d6"><i>Coccidioides immitis </i> H538.4</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/20516208">Neafsey et al. 2010</a>.
          </li>

          <li><a href="/a/app/record/dataset/DS_0901e8597d"><i>Malassezia globosa </i> CBS 7966</a> &ndash;
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/20516208">Xu et al. 2007</a>.
              </li>

          <li><a href="/a/app/record/dataset/DS_248d79bc8e"><i>Rhizophagus irregularis</i> DAOM 197198</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/34747029">Yildirir et al. 2021</a>.
              </li>
        </ul>
        <br>

     <h4>Transcriptomics</h4>
     <ul>
      <li><i>Aspergillus fumigatus</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_47a1a2ad67"> Transcriptomics of <i>Aspergillus fumigatus</i> strains upon exposure to human airway epithelial cells</a> &ndash;
          <i>Aspergillus fumigatus</i> (strains Af293 or CEA10) were exposed to human cells for 6 or 16 hours.RNA-seq analysis was performed on both the host and fungal samples.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/29345613" target="_blank">Watkins et al. 2018</a>.
          This FungiDB data set has <a href="/a/app/record/dataset/DS_47a1a2ad67#AssociatedDatasets">associated data sets</a>.
       </li>
      </ul>

      <li><i>Aspergillus niger</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_47a1a2ad67"> Aspergillus niger gene co-expression network</a> &ndash;
          Co-expression network was generated from 155 RNA-Seq experiments.
          <a href="https://www.biorxiv.org/content/10.1101/2023.07.28.550800v1" target="_blank">Schäpe et al. 2023</a>.
        </li>
      </ul>

      <li><i>Botrytis cinerea</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_c37639bde3"> Transcriptome profiling data of <i>Botrytis cinerea</i> infection on whole tomato plant</a> &ndash;
          Transcriptome at 5 infection stages from <i>B. cinerea</i> infection sites.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/32552519" target="_blank">Srivastava et al. 2020</a>.
       </li>
      </ul>

      <li><i>Neurospora crassa</i> </li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_c764829c69"> Transcriptome under standard minimal and iron strarvation conditions</a> &ndash;
          <i>Neurospora crassa</i> grown under normal and iron starvation conditions.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31068340l" target="_blank">Bayram et al. 2018</a>.
       </li>
      </ul>
      
      <li><i>Candida albicans</i></li>
       <ul>
         <li>
           <a href="/a/app/record/dataset/DS_1ed8718060"> RNA-seq of C. ablicans from cecum of monocolonized WT and Rag1-/- mice</a> &ndash;
           Transcriptome of <i>Candida albicans</i> in response to antibody binding in the mouse (C57Bl/6) cecum.
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/34262174" target="_blank">Ost et al. 2021</a>.
           This FungiDB data set has <a href="/a/app/record/dataset/DS_1ed8718060#AssociatedDatasets">associated data sets</a>.
        </li>
       </ul>

       <li><i>Pyricularia oryzae</i></li>
       <ul>
         <li>
           <a href="/a/app/record/dataset/DS_24393028be"> RNA-seq of mycelial cultures of Dhox7 and Dmst12</a> &ndash;
           Mycelial cultures of Dhox7 and Dmst12.
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/36913579" target="_blank">Tang et al. 2023</a>.
        </li>
       </ul>
 
 
    </ul>
     <br>


     <h4>Whole genome sequence of isolates for SNP analysis</h4>
     <ul>
      <li><i>Candida auris</i></li>
        <ul>
         <li>
          <a href="/a/app/record/dataset/DS_b5431bbf9f">First five Candida auris isolates in Austria</a> &ndash;
          Four isolates were characterized as South Asian clade I and one isolate as African clade III.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/36836244" target="_blank">Spettel et al. 2023</a>.
         </li>
        </ul>
    </ul>
    <br>

    <h4>Phenotype data</h4>
     <ul>
      <li><i>Candida auris</i></li>
        <ul>
         <li>
          Manually curated phenotypes from <a href="http://www.candidagenome.org/" target="_blank">Candida Genome Database</a> now appear on FungiDB gene records pages for <i>Candida albicans</i>, <i>Candida auris</i>, <i>Candida tropicalis</i>, <i>Candida dubliniensis</i>, <i>Candida parapsilosis</i>, and <i>Nakaseomyces glabratus</i>.
           </li>
        </ul>

    </ul>
    <br>

    <h4>Link Outs</h4>
      <ul>
      <li>Link outs to <a href="https://mycocosm.jgi.doe.gov/mycocosm/home" target="_blank">JGI MycoCosm</a> were integrated for <i>Aspergillus campestris</i> IBT 28561, <i> Aspergillus steynii</i>  IBT 23096, and <i> Pleurotus ostreatus</i> PC15.
        </li>
      </ul>
   <br>

  </div>    
<section>
  <h3>New features in {{project}} 66</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li>The {{project}} <a href="/a/app/static-content/webinars.html" target="_blank">webinars page</a> has a new look. 'Upcoming' webinars announce the next few webinars. 'Releases' contains the recordings for our previous release webinars.  'On-Demand Training' provides embedded videos for special topic webinars. All webinar recordings are also available from the <a href="https://www.youtube.com/@EuPathDB/playlists" target="_blank">VEuPathDB YouTube Channel</a>.  </li>
      <li><a href="https://orthomcl.org/orthomcl/app" target="_blank">OrthoMCL DB</a> offers a new preconfigured download table called Phyletic Distribution Clade Abundance that offers the OMCL group distribution of sequences in Alveolata, Archaea, Amoeba, Bacteria, Fungi, Euglenozoa, Metazoa, Viridiplantae, and other Eukaryotes. <a href="{{'/assets/images/news/Download-OMCL-table.png' | absolute_url}}"> OMCL Download Tutorial</a> </li>
   </ul>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
