---
layout: post
title: FungiDB 62 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 62.</h3>

<section>
  <h3>Release 62 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday March 16, 2023 at 10 am Eastern for our Release 62 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 62.  
    <a href="https://attendee.gotowebinar.com/register/6612181883667626331" target="_blank">Register here</a>  
  </div>
  <br>
</section>

<section>
  <h3>VEuPathDB is a DataWorks! prize winner!</h3>
  <div style="margin-left: 3em;">
  <b>Thank you for voting!</b> VEuPathDB is grateful to be the <a href="https://datascience.nih.gov/director/directors-blog-dataworks-winners-2023" target="_blank">2023 DataWorks! Prize Winner</a> in two categories: <b>Significant Achievement Award for Data Reuse</b> and the <b>People's Choice Award</b>. The DataWorks! program was launched by a partnership between the NIH Office of Data Science Strategy (ODSS) and the Federation of American Societies for Experimental Biology (FASEB) to highlight examples of innovative data sharing and reuse. 
</div><br>
</section>

<section>
      <h3>New data in FungiDB 62</h3>
      <div style="margin-left: 3em;">

        <h4>Genome Sequence and Annotation </h4>
        <ul>
         <li><a href="/a/app/record/dataset/DS_284b293a31"><i>Aspergillus uvarum</i> CBS 121591</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/30349117">Vesth et al. 2018</a>.  
            </li>
  
         <li><a href="/a/app/record/dataset/DS_822f80a1d2"><i>Blumeria graminis</i> f. sp. <i>triticale</i> THUN-12</a> &ndash;
             University of Zurich.  
                 </li>
  
         <li><a href="/a/app/record/dataset/DS_fb78585c20"><i>Cladophialophora yegresii</i> CBS 114405</a> &ndash;
           The Broad Institute.
             </li>
  
          <li><a href="/a/app/record/dataset/DS_9eed86981e"><i>Cryptococcus depauperatus</i> CBS 7841</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/35713948">Passer et al. 2022</a>.
              </li>
    
           <li><a href="/a/app/record/dataset/DS_403a34d78a"><i>Emmonsia crescens</i> UAMH 3008</a> &ndash;
           The Broad Institute. 
               </li>

        </ul>
        <br>


     <h4>Transcriptomics</h4>
     <ul>
       <li><i>Botrytis cinerea</i></li>
       <ul>
         <li>
           <a href="/a/app/record/dataset/DS_3f4593a823">RNA-seq of <i>Botrytis cinerea</i> isolates from Spain and Italy</a> &mdash;
           RNA-Seq was performed on isolates collected from Vitis vinifera host in Spain and Italy.
           Centre for Plant Biotechnology and Genomics UPM-INIA.   
       </li>
       </ul>
 
      <li><i>Candida albicans</i></li>
      <ul>
         <li>
            <a href="/a/app/record/dataset/DS_9b979ad174"> Candida transcriptomes during oropharyngeal candidiasis infection in mouse</a> &mdash;
            Transcriptional responses to macrophage phagocytosis by <i>Candida albicans</i>.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30873177" target="_blank">Kirchneret al. 2019</a>.
              This FungiDB data set has <a href="/a/app/record/dataset/DS_9b979ad174#AssociatedDatasets">associated data sets</a>.
            </li>
      </ul>
 
      <li><i>Cordyceps militaris</i></li>
      <ul>
        <li>
        <a href="/a/app/record/dataset/DS_4d7e37d0bf"> RNA-Seq data of Cordyceps militaris cultivated on the germinated soybeans</a> &mdash;
        RNA-Seq was applied to the fungal cultures at seven-time points (2, 4, and 7-day and 2, 3, 5, 7-week old cultures) to investigate the global transcriptomic change.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/35291592" target="_blank">Yoo et al. 2022</a>. 
        </li>
      </ul>

      <li><i>Neurospora crassa</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_eb1f1d3a5c">Analysis of clock-regulated genes</a> &mdash;
          RNA was sampled every 2h over 48h to identify circadianly expressed genes in <i>Neurospora crassa</i>.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/25362047" target="_blank">Hurley al. 2014</a>.   
      </li>
      </ul>

    </ul>
     <br>


     <h4>Whole genome sequence of isolates for SNP analysis</h4>
     <ul>
       <li><i>Aspergillus fumigatus</i></li>
       <ul>
        <li>
          <a href="/a/app/record/dataset/DS_b596003550">SNP calls on WGS on avian isolates of Aspergillus fumigatus</a> &mdash;
          Isolates taken from aspergillosis-associated avian carcasses, representing 24 bird species from a wide geographic range, and representing individual bird mortalities as well as epizootic events.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/35377435" target="_blank">Lofgren al. 2022</a>.
         </li>
       </ul>
 
      <li><i>Candida auris</i></li>
      <ul>
        <li>
        <a href="/a/app/record/dataset/DS_edf8e6773c">SNP calls on WGS of Candida auris isolates obtained by NY Public Health's Wadsworth Center Mycology Laboratory</a> &mdash;
        Whole-genome comparisons of 19 isolates, collected over 72 days.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/35770999" target="_blank">Jacobs et al. 2022</a>. 
        </li>

        <li>
          <a href="/a/app/record/dataset/DS_fc8cffedb5">SNP calls on WGS of Candida auris identified in Canada, 2012-2019</a> &mdash;
          Canadian isolates belonged to the four main genomic clades: Clade I (formerly called South Asian clade, n = 12), Clade II (East Asian, n = 3), Clade III (African, n = 4), and Clade IV (South American, n = 5). Isolates within each clade were clonal.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/34910140" target="_blank">De Luca al. 2022</a>.
         </li>

         <li>
          <a href="/a/app/record/dataset/DS_426f7ac5d5">SNP calls on WGS of Candida auris isolated from apples</a> &mdash;
          Fruits were tested as a source of multidrug-resistant <i>C. auris</i>.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/35357170" target="_blank">Yadav al. 2022</a>.
         </li>

      </ul>

        <li><i>Cryptococcus gattii</i></li>
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_4eb29e611f">SNP calls on WGS of C. gattii from North American Pacific Northwest</a> &mdash;
            Cryptococcus gattii in North American Pacific Northwest: whole population genome analysis provides insights into species evolution and dispersal.
            TGen North.   
        </li>
        </ul>

        <li><i>Neurospora discreta</i></li>
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_7e69303d47">SNP calls on WGS of Neurospora discreta crosses</a> &mdash;
            Parental strains FGSC2223 and FGSC4825 were used to generate cross progeny.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/32017762" target="_blank">Waters al. 2020</a>.   
        </li>
        </ul>
    </ul>
    </div>
     <br>
     
    
<section>
  <h3>New features in {{project}} 62</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li>Site search 'Reference Only' organism filter &ndash; Large results (&#62; 50,000 hits) from the site search are common since it accesses all {{project}} data/page categories (genes, data types, data sets, organisms, etc.).  The new 'Reference Only' toggle offers a one-click filter to reduce the result from all organisms to include only results from VEuPathDB reference orgnanisms.  </li>

      <li>Search details and data sets &ndash; Details for the search and the data sets accessed by the search are now available in tabbed pages from the top of the search pages.  Previous versions of {{project}} presented this information at the bottom of the page where it was easily missed. </li>

      <li>Scroll and zoom on gene page browser frames &ndash; The dynamic browser views on gene pages now provide active scroll and zoom functions with a reset button to return to the default view. Dynamic browser views are available for Synteny, Alignments, SNPs, Transcriptomics Coverage, Protein Features and Proteomics.  </li>
   </ul>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
