---
layout: post
title: FungiDB 68 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 68.</h3>

<section>
  <h3>Release 68 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday May 23rd, 2024 at 10 am Eastern for our Release 68 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 68. 
    <a href="https://attendee.gotowebinar.com/register/362861265556682325" target="_blank">Register here</a>
  </div>
  <br>
</section>

<section>
      <h3>New data in FungiDB 68</h3>
      <div style="margin-left: 3em;">

        <h4>Genome Sequence and Annotation </h4>

        <li><a href="/a/app/record/dataset/DS_122336d025"><i>Phytophthora cactorum </i> 10300</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30278048">Armitage et al. 2018</a>.
             </li>
   
          <li><a href="/a/app/record/dataset/DS_8116a8b3fd"><i>Pseudocercospora fijiensis</i> CIRAD86</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/23236275">Ohm et al. 2012</a>.
                   </li>

          <li><a href="/a/app/record/dataset/DS_8d5877eec8"><i>Penicillium expansum</i> d1</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23236275">Ballester et al. 2015</a>.
                   </li>

          <li><a href="/a/app/record/dataset/DS_d8486c1276"><i>Aspergillus ellipticus</i> CBS 707.79</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30349117">Vesth et al. 2018</a>.
                   </li>

        <li><a href="/a/app/record/dataset/DS_22ae577d5a"><i>Phytophthora ramorum</i> Pr102</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/35285670">Carleson et al. 2021</a>.
             </li>

        <h4>Genome replacements </h4>
        <ul>
          <li><a href="/a/app/record/dataset/DS_8c9d407527"><i>Puccinia striiformis</i> 93-210</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23607900">Cantu et al. 2013</a>.
          </li>

          <li><a href="/a/app/record/dataset/DS_a4a5d44eab"><i>Puccinia striiformis</i> 93TX-2</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23607900">Cantu et al. 2013</a>.
              </li>
        </ul>
        <br>

    <h4>Transcriptomics</h4>
    <ul>
      <li><i>Batrachochytrium salamandrivorans</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_d2d11e9b9e"> Transcriptomes of <i>B. salamandrivorans</i> zoospores treated with different carbohydrate sources</a> &ndash;
            Zoospore transcriptomic response to different carbohydrates (glucose, mannose or galactose).
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/34608163" target="_blank">Wang et al. 2021</a>.
       </li>
      </ul>

       <li><i>Botrytis cinerea</i></li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_f5dab99c2a"> sRNA profiling data of <i>Botrytis cinerea</i> infection on tomato plant leaves</a> &ndash;
           Transcriptome profiling data of Botrytis cinerea infection on whole plant <i>Solanum lycopersicum</i>.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/36168919" target="_blank">Veloso et al. 2022</a>.
        </li>
      </ul>

        <li><i>Candida auris</i></li>
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_caa4eaf6df"> Transcriptional profiling of <i>Candida auris</i> B8441 grown on different carbon and nitrogen sources</a> &ndash;
             Transcriptional profiles were generated after 4 h incubation at 37C.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/37097196" target="_blank">Brandt et al. 2023</a>.
         </li>
      </ul>
 
          <li><i>Trichophyton rubrum</i></li>
          <ul>
            <li>
              <a href="/a/app/record/dataset/DS_1436095090"> Transcriptome profiling of <i>Trichophyton rubrum</i> on glucose and keratin</a> &ndash;
               Global analysis of the dermatophyte <i>Trichophyton rubrum</i> when shifted from glucose to keratin.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/35783403" target="_blank">Martins-Santana et al. 2023</a>.
           </li>
          </ul>

            <li><i>Mucor lusitanicus</i></li>
            <ul>
              <li>
                <a href="/a/app/record/dataset/DS_4703a409e8"> Transcriptomics of <i>Mucor circinelloides</i> NCRIP deficient spores phagocytosed by mouse macrophages.</a> &ndash;
                  Transcriptomics of R3B2 and RDRP1 fungal spores and fungal spore-macrophage coculture
                  <a href="http://www.ncbi.nlm.nih.gov/pubmed/32658892" target="_blank">Pérez-Arques et al. 2020</a>. These data were reloaded using <i>Mucor lusitanicus</i> CBS 277.49 (GCA_001638945.1).
             </li>
             <li>
              <a href="/a/app/record/dataset/DS_1436095090"><i>Mucor circinelloides</i> spores phagocytosed by mouse macrophages: single culture</a> &ndash;
              A study of the transcriptomics changes occuring in both host and pathogen upon the interaction of <i>Mucor circinelloides</i> spores with phagocytic cells of a mouse.
                <a href="http://www.ncbi.nlm.nih.gov/pubmed/30723131" target="_blank">Pérez-Arques et al. 219</a>. These data were reloaded using <i>Mucor lusitanicus</i> CBS 277.49 (GCA_001638945.1).
           </li>
            </ul>
      
      </ul>
     <br>


     <h4>Whole genome sequence of isolates for SNP analysis</h4>
     <ul>
      <li><i>Blastomyces dermatitidis</i></li>
        <ul>
         <li>
          <a href="/a/app/record/dataset/DS_686ebef186">SNP calls on WGS of <i>Blastomyces dermatitidis</i> isolates</a> &ndash;
          Whole genome sequencing of clinical isolates of <i>Blastomyces dermatitidis</i> of isolates from Quebec, Canada.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/34670643" target="_blank">Carignan et al. 2021</a>.
         </li>
        </ul>

       <li><i>Candida auris</i></li>
        <ul>
         <li>
          <a href="/a/app/record/dataset/DS_d0600e5230">SNP calls on WGS of <i>Candida auris</i> isolates from Romania</a> &ndash;
          Whole genome sequencing of <i>Candida auris</i> clinical bloodstream isolates.
          National Institute for Infectious Diseases, Bucharest, Romania.
         </li>
        </ul>

        <li><i>Parastagonospora nodorum</i></li>
        <ul>
         <li>
          <a href="/a/app/record/dataset/DS_3b67a4647c">SNP calls of WGS of <i>Parastagonospora nodorum</i> isolates</a> &ndash;
          Population genomics of <i>Parastagonospora nodorum</i>.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/32986802" target="_blank">Pereira et al. 2020</a>.
         </li>
        </ul>

      </ul>
      <br>

      <h4>Proteomics</h4>
        <ul>
         <li><i>Aspergillus spp.</i></li>
           <ul>
            <li>
             <a href="/a/app/record/dataset/DS_5eef8ec7c8">Conidial surface-associated proteome</a> &ndash;
             Surface proteome of germinating Aspergillus fischeri conidia.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/37662192" target="_blank">Valero et al. 2023</a>.
            </li>
            <li>
              <a href="/a/app/record/dataset/DS_6f8c2052cf">Conidial surface-associated proteome</a> &ndash;
              Surface proteome of germinating Aspergillus fumigatus conidia.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/37662192" target="_blank">Valero et al. 2023</a>.
             </li>
             <li>
              <a href="/a/app/record/dataset/DS_86881a3912">Conidial surface-associated proteomes</a> &ndash;
              Surface proteome of germinating Aspergillus lentulus conidia.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/37662192" target="_blank">Valero et al. 2023</a>.
             </li>
           </ul>
  

      </ul>
      <br> 

      <section>
        <h3>New FungiDB publication</h3>
        <div style="margin-left: 3em;">
         <table border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td colspan="0"><hr></td></tr>
          <tr class="rowLight" style="display: none;"><td>
            Basenko, E. Y., Shanmugasundram, A., Böhme, U., Starns, D., Wilkinson, P. A., Davison, H. R., Crouch, K., Maslen, G., Harb, O. S., Amos, B., McDowell, M. A., Kissinger, J. C., Roos, D. S., Jones, A. R.
            <br>
            <b>What is new in FungiDB: a web-based bioinformatics platform for omics-scale data analysis for fungal and oomycete species.</b>
            <br>
            GENETICS, 2024 (Mar 26:iyae035)<br>
             <a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyae035/7634838"> https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyae035/7634838</a>
            </td>
          </tr></table>  
      </section>

<section>
  <h3>New features in {{project}} 68</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li>Long gene product descriptions are now trucated in relation to the width of your browser window.  The full product description is available on mouseover and individual transcript product descriptions are available in the All Product Descriptions table on gene pages. <a target="_blank" href="{{'/assets/images/news/Truncated_descriptions.gif' | absolute_url}}">Truncated descriptions GIF</a>. </li>

      <li>The Orthologs and Paralogs table on {{project}} gene pages also serves as a Clustal Omega tool for retrieving alignments between orthologs. This table now includes 'Selection Count' and 'Selection Clear' options for improved tool function. <a target="_blank" href="{{'/assets/images/news/Clustal_Omega_Counts.gif' | absolute_url}}">Clustal Omega Counts GIF</a>.   </li>

      <li>The Send To tool is now offered in VEuPathDB.org Strategies. Results created in VEuPathDB.org Strategies can now be ported to {{project}} My Strategies for further investigation. Only genes present in {{project}} species will be included in the ported strategy results. <a target="_blank" href="{{'/assets/images/news/VEuPath_Port_to_Component.gif' | absolute_url}}">VEuPathDB.org Send To GIF</a>.</li>

      <li>Newly sorted strategy result tables will now default to page 1 &ndash; Search result tables can be sorted based on any column of data. When a table is sorted while viewing page 3, users are returned to page 1 to view the sorted data table. </li>
   </ul>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
