---
layout: post
title: TrichDB 65 Released
category: [TrichDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 65.</h3>

<section>
  <h3>Release 65 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday July 20th, 2023 at 10 am Eastern for our Release 65 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 65. 
    <a href="https://attendee.gotowebinar.com/register/4226674862077131357" target="_blank">Register here</a>
  </div>
  <br>
</section>

<section>
  <h3>New data in {{project}} 65</h3>
  <div style="margin-left: 3em;">
    <h3>Proteomics</h3>
    <ul>
      <li><a href="/a/app/record/dataset/DS_c679be0232">Dynamic Secretome</a> &ndash; <i>Trichomonas vaginalis</i> extracellular proteome at 10, 30, 60, 120 minutes of incubation. Controls incubated at 4C were acquired at 60 and 120 minutes. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29233912" target="_blank"> Štáfková et al. 2018</a>.</li>

      <li><a href="/a/app/record/dataset/DS_c0d0c750e2">Metronidazole treated sensitive or resistant trichomonas proteomes</a> &ndash; iTRAQ-based analysis to profile the proteomes of MTZ-sensitive (MTZ-S) and MTZ-resistant (MTZ-R) cultures of <i>Trichomonas vaginalis</i>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32487244" target="_blank">Lin et al. 2020</a>.</li>

      <li><a href="/a/app/record/dataset/DS_d782442710">Phagolysosomal proteome (Quantitative)</a> &ndash; Phagolysosomal proteome of the human parasite Trichomonas vaginalis. The organelles were purified using Percoll and OptiPrep gradient centrifugation and a novel purification protocol based on the phagocytosis of lactoferrin-covered magnetic nanoparticles. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34763061" target="_blank">Zimman et al. 2022</a>.</li>

      <li><a href="/a/app/record/dataset/DS_2012741b82">TOM complex-associated proteins</a> &ndash; Co-immunoprecipitations (coIPs) of HA-tagged TvTom40-2 under crosslinking and native conditions, and the eluted proteins were analysed using label-free quantitative MS (LFQ-MS). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30608924" target="_blank">Makki et al. 2019</a>.</li>
    </ul>
  </div>
</section>

<section>
  <h3>New features in {{project}} 65</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="https://veupathdbprod.globusgenomics.org/" target="_blank">VEuPathDB Galaxy</a> now offers a preconfigured ChIP Seq workflow. This workflow processes ChIP Seq FASTQ files by aligning to your chosen reference genome with BWA then calling peaks using MACS2. Peaks calculated from the input FASTQ files are converted to bigWig format which can be exported from Galaxy to your VEuPathDB account.</li>
      <li>Strategy Improvement  &ndash; A new option in the gene result table provides a view of search specific data columns for your preferred subset of genes. Click the 'Show only the Genes in my basket' option from any gene result to filter the result based on the genes in your basket.  This can be used in combination with the 'Send to Basket (replace)' option to view search specific data columns for multi-step strategy results. <a target="_blank" href="https://youtu.be/SCCxNTGu-dQ">Filter by Basket tutorial.</a> </li>
   </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
