---
layout: post
title: MicrobiomeDB 27 Released
category: [MicrobiomeDB]
---
<div style="font-size:130%"><i>We are pleased to announce the release of MicrobiomeDB 27</i></div>
      <section>
        <h3><b>New data in this release</b></h3>
        <ul>
        <li>The MORDOR study (Macrolides Oraux pour Réduire les Décès avec un Oeil sur la Résistance) phase I is a large, double-masked, placebo-controlled, cluster randomized trial that took place between 2014 and 2017 in three countries: Niger; Malawi; and Tanzania.  Of the three sites evaluated as part of MORDOR I, Niger showed by far the highest reduction in mortality among preschool children (aged 1–59 months) who received biannual oral azithromycin mass drug administration (MDA) compared to those who received biannual placebo. The mechanism for the reduction in all-cause mortality was not clear. To test the hypothesis that azithromycin causes an alteration in the gut pathogen burden, metagenomic RNA sequencing was carried out on rectal samples obtained from preschool children (n = 600) from 30 villages in Niger. These villages were randomly selected from the same pool as the mortality MORDOR I trial in Niger and randomized to receive either one oral administration of azithromycin MDA or placebo every 6 months for 24 months.
        </li><li>The Malaysia Helminth Study sought to understand the impact of helminth infections on diversity and composition of the gut microbiome. Whole-metagenome shotgun sequencing was carried out on 650 stool samples from indigenous Malaysians from 5 villages that vary in their prevalence of helminth infections.
        </li></ul>
      </section>

      <section>
        <h3><b>New features in this release</b></h3>
        <ul>
        <li>There are no new features in this release.
        </li>
        </ul>
      </section>

      <section>
        <h3><b>Publications</b></h3>
        <ul>
        <li>The MicrobiomeDB team has developed a new method that leverages Markov clustering and network metrics to identify microbial eukaryotes (e.g., fungi, protozoa, and helminths).  Later this year, we plan to incorporate CORRAL into our data loading workflow so that this type of data is available for any curated study on our site.  You can learn more about CORRAL and see its performance on simulated and real metagenomic datasets in our recent publication (https://pubmed.ncbi.nlm.nih.gov/37032329/)
        </li></ul>
      </section>

      <section>
        <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When sample IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
        </aside>
      </section>

