---
layout: post
title: TriTrypDB 53 Released
category: [TriTrypDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h4>We are pleased to announce the release of {{project}} 53.</h4>

<section>
  <h3>Release 53 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on May 27, 2021 at 10am EDT for our Release 53 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 53.  
    <a href="https://attendee.gotowebinar.com/register/8223585429100223502" target="_blank">Register here</a>  
  </div>
</section>

<section>
  <h3>New data in {{project}} 53</h3>
  <div style="margin-left: 3em;">
  <h4>Whole genome sequence of isolates for SNP and CNV</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_65779f34e0"><i>WGS of Trypanosoma brucei</i> RUMP 503</a> &ndash; Illumina sequence analysis of <i>T. brucei</i> brucei RUMP 503 DNA<a href="https://www.ncbi.nlm.nih.gov/biosample/?term=SAMN07659300" target="_blank">Davey et al. 2021</a> </li>
      </ul>
      <br>
<!--
  <h4>Updated Functional Annotation</h4>

        <a href="https://docs.google.com/spreadsheets/d/1HFwfPuIEKZGyUYBWhcJfrw35om_CPQUKB0lNuZqXjt8/edit?usp=sharing" target="_blank">Summary of annotation updates</a> which include products, GO terms, gene names, synonyms, publications, and notes from curators. These functional annotations are current with GeneDB April 2021 GFF versions.
        <br><br>
        Functional annotations in TriTrypDB 53 have been updated for the following genomes:
        <br><br>
        <style>
           #FunctAnnot_Bld53 table, #FunctAnnot_Bld53 td, #FunctAnnot_Bld53 th, #FunctAnnot_Bld53 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld53 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld53"> 
           <tr>
             <th>TriTrypDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           
           <tr>
            <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_839623217a"><i>Leishmania major </i>Friedlin</a></td>
            <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Lmajor/Lmajor.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
            <td>9 genes</td>
           </tr>
           <tr>
            <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_e318592c8e"><i>Leishmania mexicana </i>MHOM/GT/2001/U1103</a></td>
            <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Lmexicana/Lmexicana.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
            <td>7 genes</td>
           </tr>            
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_31282613b1"><i>Trypanosoma brucei </i>927</a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Tbruceibrucei927/Tbruceibrucei927.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
             <td>275 genes</td>
           </tr>
           <tr>
            <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_1a6c8bdaac"><i>Trypanosoma cruzi </i>CL Brener</a></td>
            <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Tcruzi/Tcruzi.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
            <td>206 genes</td>
          </tr>
           </table>
    <br><br>
           
		   
	<h4>Curated User Comments</h4>
       
          Gene page comments that are entered by users are curated at GeneDB and regularly integrated as functional  annotations in TriTrypDB. 103 user comments have been integrated in this release.<br><br>
          
         <style>
           #UComment_Bld53 table, #UComment_Bld53 td, #UComment_Bld53 th, #UComment_Bld53 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #UComment_Bld53 {
           margin-left : 5 em;
           }
         </style> 
           <table id="UComment_Bld53"> 
           <tr>
             <th>TriTrypDB Genome Version</th>
             <th>Comments Integrated</th>
           </tr>
           <tr>
            <td><a href="/a/app/record/dataset/DS_839623217a"><i>Leishmania major </i>Friedlin</a></td>
            <td>7</td>
           </tr>
           <tr>
             <td><a href="/a/app/record/dataset/DS_31282613b1"><i>Trypanosoma brucei</i> 927</a></td>
             <td>96</td>
           </tr>
          </table>  
  -->
  </div>
</section>

<section>
  <h3>New features in {{project}} 53</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>The gene pages of <a href="https://en.wikipedia.org/wiki/Pseudogene" target="_blank">pseudogenes</a> are modified to improved clarity: <a href="{{'/assets/images/pseudogene-page.png' | absolute_url}}">(Pseudogene page changes)</a>
      <ul>
        <li>The Transcripts table in the Gene Models section now indicates that no protein products are produced. </li>
        <li>The exons in JBrowse views of pseudogenes are colored gray.  Blue represents genes on the forward strand and red represents genes on the reverse strand. </li>
        <li>The Orthology and Synteny section indicates that orthology is not determined for pseudogenes.</li>
        <li>Protein sequences are no longer available from the Sequences section.</li>
      </ul>
    </li>
    <li>A new <a href="/a/app/static-content/faq.html">FAQs page</a> is available from the <a href="/a/app/static-content/landing.html">Learn How to use VEuPathDB page</a>. Open each section to reveal step-by-step instructions for common tasks.  </li>
    <li>Reference organisms are indicated in the Organism parameter on search pages.  VEuPathDB chooses a representative genome from each species and aligns all functional genomics data to the reference.  In doing so, all data can be viewed and compared in the genome browser against the reference.  In some cases, data are aligned to the source organism in addition to the reference. (<a href="{{'/assets/images/Reference.png' | absolute_url}}">Reference-organism.png</a>)</li>
    <li>The algorithm for predicting EC Numbers of genes has been improved. Previously, we predicted EC numbers by using the most prevalent EC number in an ortholog group (defined by OrthoMCL) and transferring that number to all other members in the group. In the new algorithm, we use orthology as well as protein sequence/domain comparison to infer EC number assignments. Additionally, we provide multiple forms of evidence that are used to inform each prediction, allowing users to determine their confidence in the prediction. Specifically, the algorithm evaluates three features (length, domain architecture and BLAST similarity to other proteins) of each protein in an OrthoMCL ortholog group. Those features are compared to the features of proteins already assigned an EC number, in order to infer which proteins should be assigned a specific EC number.  We also show the number of proteins and the number of genera that are assigned this EC number within the ortholog group, so that the user can assess the prevalence of a given EC number. This new algorithm performs extremely well in predicting previously assigned EC numbers, demonstrating that the algorithm is accurate.</li>
    <li>Community annotations made in {{project}} via Apollo are now represented on gene pages in the Gene Models graphic. In addition, links from the gene page to Apollo are offered to facilitate community annotations. <a href="{{'/assets/images/51-apollo-news.png' | absolute_url}}">Example image.</a>  Please see our <a href="https://youtu.be/RMXZDBMh8Fo" target="_blank">Apollo webinar</a> for instruction. </li>
    </ul>
  </div>
</section>


<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data September slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
