---
layout: post
title: TriTrypDB 59 Released
category: [TriTrypDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 59.</h3>

<section>
  <h3>Annual VEuPathDB Workshop (Virtual)</h3>
    The 15th Annual VEuPathDB workshop will be a virtual event taking place daily on <b>September 6 - 9th, 2022</b> from <b>9am - 1pm Eastern</b>. Join us for these 4 half-day sessions to learn how to effectively use VEuPathDB tools and features that support your hypothesis driven research. Short lectures will precede hands-on exercises via informal Zoom sessions where discussion and conversation are encouraged. <a href="/a/app/static-content/workshopSept2022.html">Details here</a> or <a href="https://ugeorgia.ca1.qualtrics.com/jfe/form/SV_elEnAmi5PvtUNwO" target="_blank">Register here</a>.
</section>

<section>
  <h3>Release 59 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday September 1st, 2022 at 10am EDT for our Release 59 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 59.  
    <a href="https://attendee.gotowebinar.com/register/3393660656180049424" target="_blank">Register here</a>  
  </div>
  <br>
</section>

<section>
  <h3>New data in {{project}} 59</h3>
  <div style="margin-left: 3em;">

    <h4>Genome Sequence and Annotation</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_bb973be480"><i>Leishmania major</i> Friedlin 2021</a> &ndash;  The <i>L. major</i> Friedlin genome was re-sequenced using the Pacific Biosciences (PacBio) technology, which provides long reads able to span long repeats, and the Illumina technology to generate paired-end short-reads useful to join fragmented chromosomes, extend chromosomes ends and correct homopolymer indel errors. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34573340" target="_blank"> Camacho et al. 2021</a></li>

        <li><a href="/a/app/record/dataset/DS_7fc2d6f660"><i>Trypanosoma equiperdum</i> OVI</a> &ndash; the causative agent of dourine, a sexually-transmitted infection of horses.<a href="http://www.ncbi.nlm.nih.gov/pubmed/34573340" target="_blank"> H&eacute;bert et al. 2017</a></li>
      </ul>
      <br>
    <h4>ChIP Sequence</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_ee27843472">ChIP-seq of <i>Leishmania mexicana</i> BDF5</a> &ndash; Bomodomain binding factor sites in <i>Leishmania mexicana</i>.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/35831302">Jones et al. 2022</a>.</li>
      </ul>
    
  </div>
</section>

<section>
  <h3>New features in {{project}} 59</h3>
  <div style="margin-left: 3em;">
  
  <h4>{{project}} site search accesses Apollo data</h4>
    <ul>
      <li>Gene annotation updates entered in <a target="_blank" href="/a/app/static-content/apollo_help.html">{{project}} Apollo</a> are now indexed and available to the site search.  Genes with Apollo annotations that contain your search term will be available in site search results under >Genes >Filter gene fields.</li>
    </ul>
  <h4>Single cell RNA-seq &Beta;eta version</h4>
    <ul>
      <li>Single cell RNA-seq data is now represented in {{project}} via <a href="https://arxiv.org/abs/1802.03426" target="_blank">UMAP</a> in <a href="https://cellxgene.cziscience.com/" target="_blank">cellxgene</a> . </li>
      <li>A new search that returns genes represented in scRNA-seq data sets is categorized under Search For, Genes, Transcriptomics, Single Cell RNA-Seq Evidence.  Use this search to retrieve a list of all genes with expression in a particular scRNA-seq data set, as well as a link to explore the data set in the cellxgene application. </li>
      <li>A new table, Single Cell RNA-Seq (cellxgene), appears in the Transcriptomics section of gene record pages with scRNA-Seq data. Rows represent individual data sets.  Expand the rows to reveal static cellxgene images and links to dynamically view the data in the cellxgene application.</li>
    </ul>
  <h4>AlphaFold data available</h4>
    <ul>
      <li>Protein structure predictions from <a href="https://alphafold.ebi.ac.uk/" target="_blank">AlphaFold</a> are now available in {{project}}.  The AlphaFold database was downloaded and structures were associated to {{project}} genes via the UniProt ID. <a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" target="_blank">Jumper et al. 2021</a> and <a href="https://pubmed.ncbi.nlm.nih.gov/34791371/" target="_blank">Varadi et al. 2022</a></li>
      <li>The <a href="/a/app/search/transcript/GenesWithAlphafold">AlphaFold search</a> returns all genes with AlphaFold predictions in {{project}}.</li>
      <li>Gene pages include a table of AlphaFold data and the AlphaFold 3D viewer for dynamic visualization of the predicted structure. </li>
    </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data September slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
