---
layout: post
title: TriTrypDB 64 Released
category: [TriTrypDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 64.</h3>

<section>
  <h3>Release 64 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday July 20th, 2023 at 10 am Eastern for our Release 64 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 64. 
    <a href="https://attendee.gotowebinar.com/register/4226674862077131357" target="_blank">Register here</a>
  </div>
  <br>
</section>


<section>
  <h3>New data in {{project}} 64</h3>
  <div style="margin-left: 3em;">

    <h4>Genome Sequence</h4>
    <div style="margin-left: 3em;">
      <ul>
        <li><a href="/a/app/record/dataset/NCBITAXON_2592485"><i>Blastocrithidia sp.</i> p57</a> &ndash; Illumina and PacBio sequence assembly. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36631608" target="_blank">Kachale et al. 2023</a>  &ndash; <a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_028554745.1" target="_blank">GCA_028554745.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_968c0a92de"><i>Trypanosoma congolense</i> Tc1/148</a> &ndash; <a href="http://www.ncbi.nlm.nih.gov/pubmed/30006414" target="_blank">Silva Pereira et al. 2018</a>  &ndash; <a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_002287245.1" target="_blank">GCA_002287245.1</a>.</li>
      </ul>
    </div>
    <br>

    <h4>TrypTag data updated</h4>
    <div style="margin-left: 3em;">
      <ul>
        <li><a href="/a/app/record/dataset/DS_c357287e29">Cellular localization of tagged <i>T. brucei</i> proteins in procyclic forms</a> &ndash; The data were updated with mitochondrial proteome GO terms.  The data are reflected in the 'Protein targeting and localization' and 'GO Terms' sections of the record pages of updated genes.</li>
      </ul>
    </div>
    <br>
  </div>
</section>

<section>
  <h3>New features in {{project}} 64</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="https://veupathdbprod.globusgenomics.org/" target="_blank">VEuPathDB Galaxy</a> now offers a preconfigured ChIP Seq workflow. This workflow processes ChIP Seq FASTQ files by aligning to your chosen reference genome with BWA then calling peaks using MACS2. Input a pair of treated and control FASTQ data files. Peaks are converted to bigWig format. Coverage files are created in BigWig format for each input file. In addition, a differential coverage track is created in BigWig format with BamCompare. BigWig files can be exported from Galaxy to your VEuPathDB account.  </li>
      <li>Strategy Improvement  &ndash; A new option in the gene result table provides a view of search specific data columns for your preferred subset of genes. Click the 'Show only the Genes in my basket' option from any gene result to filter the result based on the genes in your basket.  This can be used in combination with the 'Send to Basket (replace)' option to view search specific data columns for multi-step strategy results. <a target="_blank" href="{{'/assets/images/news/BETAdownload.gif' | absolute_url}}">See Filter by Basket GIF TODO.</a> </li>
   </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data September slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
