---
categories: []
layout: post
title: PlasmoDB 53 Released
category: [PlasmoDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 53.</h3>  

<section>
  <h3>Release 53 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on May 27, 2021 at 10am EDT for our Release 53 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 53.  
    <a href="https://attendee.gotowebinar.com/register/8223585429100223502" target="_blank">Register here</a>  
  </div>
</section>


<section>
  <h3>New data in {{project}} 53</h3>
  <div style="margin-left: 3em;">
 
  <h4>Genome Sequence and Annotation</h4>
  <p>The following chromosome level assemblies with genome annotation appear for the first time in {{project}} 53 and are described in <a href="http://www.ncbi.nlm.nih.gov/pubmed/33888092" target="_blank">Ramaprasad et al. 2021</a>.</p>
    <ul>
      <li><a href="/a/app/record/dataset/DS_0a1b2f6f63"><i>Plasmodium vinckei</i> Cameroon EL</a> </li>
      <li><a href="/a/app/record/dataset/DS_b18630dd54"><i>Plasmodium vinckei</i> brucechwatti DA</a> </li>
      <li><a href="/a/app/record/dataset/DS_759b3f6731"><i>Plasmodium vinckei</i> lentum DE</a> </li>
      <li><a href="/a/app/record/dataset/DS_679e1e4f4c"><i>Plasmodium vinckei</i> petteri CR 2020</a> </li>
      <li><a href="/a/app/record/dataset/DS_eab17936f0"><i>Plasmodium vinckei</i> vinckei CY</a> </li>
    </ul>
    <br>

  <h4>Transcriptomics</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_fe2b5a278f">Intraerythrocytic development cycle transcriptome (2019)</a>  Blood-stage transcriptome determined by RNA-Seq analysis at 8 developmental stages including young ring stage (8 hpi), late ring stage/early trophozoite (16 hpi), mid-age trophozoite (24 hpi), late trophozoite (32 hpi), early schizont (40 hpi), schizont (44 hpi), late schizont (48 hpi) and purified merozoites (0 hpi). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31363031" target="_blank">Wichers et al. 2019</a>.</li>
      <li><a href="/a/app/record/dataset/DS_aae60e6bf7">Transcriptome in severe vs uncomplicated malaria</a> &ndash; RNA sequencing analysis of gene expression in 44  parasite isolates that caused severe or uncomplicated malaria in Papuan patients (23 severe, 21 uncomplicated cases). Data are aligned to <i>Plasmodium falciparum</i> 3D7. <a href="http://www.ncbi.nlm.nih.gov/pubmed/29529020" target="_blank">Tonkin-Hill et al. 2018</a>.</li>
      <li><a href="/a/app/record/dataset/DS_5137426106">Blood Stage Transcriptome</a> &ndash; Transcriptome sequencing of <i>Hepatocystis sp. ex Piliocolobus tephrosceles</i> strain 2019 from a <i>Piliocolobus tephrosceles</i> blood sample was performed as part of the red colobus monkey genome sequencing project. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32745123">Aunin et al. 2020.</a> </li>
      <li><a href="/a/app/record/dataset/DS_7dae78ae0b">Transcriptome during early and mid-stage <i>P. berghei</i> liver infection</a> &ndash; Transcriptome sequencing (RNA-Seq) analysis throughout the Plasmodium berghei liver stage, covering as early as 2 hours post-infection (hpi) and extending to 48 hpi in HuH7 and HepG2 cells. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32019802">Toro-Moreno et al.</a></li>
    </ul>
    <br>

  <h4>Whole genome sequence of isolates for SNPs and CNV</h4>
    <ul>
      <li>UTR calls based on DAFT sequencing</li>
      <li>Transcription start sites based on 5UTR seqencing</li>
    </ul>
    <br>

  <h4>Updated Functional Annotation</h4>
        
    <p><a href="https://docs.google.com/spreadsheets/d/1WBR_gRyfiufvOWn12GiFD6z_QuCUyvDRZNNXdeMpj6w/edit#gid=1112700546" target="_blank">Summary of annotation updates</a>
      which include products, GO terms, gene names, synonyms and PMID. Functional annotations in PlasmoDB 53 are current with the <a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/" target="_blank">GeneDB GFF versions from May 2021</a>. </p>
    
    <style>
       #FunctAnnot_Bld53 table, #FunctAnnot_Bld53 td, #FunctAnnot_Bld53 th, #FunctAnnot_Bld53 tr {
       text-align : center;
       padding-left: 5px;
       padding-right: 5px;
       padding-top: 5px;
       padding-bottom: 5px;
       border: 1px solid black;
       }
       #FunctAnnot_Bld53 {
       margin-left : 5 em;
       }
     </style> 
       <table id="FunctAnnot_Bld53"> 
       <tr>
         <th>PlasmoDB Genome Version</th>
         <th>Download GeneDB GFF file</th>
         <th>Genes Updated</th>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_f2d80510c7"><i>P. berghei</i> ANKA </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Pberghei/">GeneDB version May 2021</a></td>
         <td>62 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_5dab6e4a9f"><i>P. chabaudi</i> AS </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Pchabaudi/">GeneDB version May 2021</a></td>
         <td>56 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_1d17c1883c"><i>P. falciparum</i> 3D7 </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Pfalciparum/">GeneDB version May 2021</a></td>
         <td>360 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_22b4dadc51"><i>P. gallinaceum</i> 8A </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Pgallinaceum/">GeneDB version May 2021</a></td>
         <td>59 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_8793a259f4"><i>P. knowlesi</i> H </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Pknowlesi/">GeneDB version May 2021</a></td>
         <td>57 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_3986a0b578"><i>P. malariae</i> UG01 </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Pmalariae/">GeneDB version May 2021</a></td>
         <td>57 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_99a2d5bf10"><i>P. ovale curtisi</i> GH01 </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Povale/">GeneDB version May 2021</a></td>
         <td>58 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_91fa338f8f"><i>P. reichenowi</i> CDC </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Preichenowi/">GeneDB version May 2021</a></td>
         <td>58 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_c3f289d21a"><i>P. relictum</i> SGS1 </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Prelictum/">GeneDB version May 2021</a></td>
         <td>58 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_ad07e73d42"><i>P. vivax</i> P01 </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/PvivaxP01/">GeneDB version May 2021</a></td>
         <td>60 genes</td>
       </tr>
       <tr>
         <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="/a/app/record/dataset/DS_b5a76b5939"><i>P. yoelii yoelii</i> 17X </a></td>
         <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-05/Pyoelii/">GeneDB version May 2021</a></td>
         <td>56 genes</td>
       </tr>
       </table>
    </div>
</section>



<section>
  <h3>New features in {{project}} 53</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>The gene pages of <a href="https://en.wikipedia.org/wiki/Pseudogene" target="_blank">pseudogenes</a> are modified to improved clarity: <a href="{{'/assets/images/pseudogene-page.png' | absolute_url}}">(Pseudogene page changes)</a>
      <ul>
        <li>The Transcripts table in the Gene Models section now indicates that no protein products are produced. </li>
        <li>The exons in JBrowse views of pseudogenes are colored gray.  Blue represents genes on the forward strand and red represents genes on the reverse strand. </li>
        <li>The Orthology and Synteny section indicates that orthology is not determined for pseudogenes.</li>
        <li>Protein sequences are no longer available from the Sequences section.</li>
      </ul>
    </li>
    <li>A new <a href="/a/app/static-content/faq.html">FAQs page</a> is available from the <a href="/a/app/static-content/landing.html">Learn How to use VEuPathDB page</a>. Open each section to reveal step-by-step instructions for common tasks.  </li>
    <li>Reference organisms are indicated in the Organism parameter on search pages.  VEuPathDB chooses a representative genome from each species and aligns all functional genomics data to the reference.  In doing so, all data can be viewed and compared in the genome browser against the reference.  In some cases, data are aligned to the source organism in addition to the reference. (<a href="{{'/assets/images/Reference.png' | absolute_url}}">Reference-organism.png</a>)</li>
    <li>The algorithm for predicting EC Numbers of genes has been improved. Previously, we predicted EC numbers by using the most prevalent EC number in an ortholog group (defined by OrthoMCL) and transferring that number to all other members in the group. In the new algorithm, we use orthology as well as protein sequence/domain comparison to infer EC number assignments. Additionally, we provide multiple forms of evidence that are used to inform each prediction, allowing users to determine their confidence in the prediction. Specifically, the algorithm evaluates three features (length, domain architecture and BLAST similarity to other proteins) of each protein in an OrthoMCL ortholog group. Those features are compared to the features of proteins already assigned an EC number, in order to infer which proteins should be assigned a specific EC number.  We also show the number of proteins and the number of genera that are assigned this EC number within the ortholog group, so that the user can assess the prevalence of a given EC number. This new algorithm performs extremely well in predicting previously assigned EC numbers, demonstrating that the algorithm is accurate.</li>
    </ul>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
