---
categories: []
layout: post
title: PlasmoDB 56 Released
category: [PlasmoDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 56.</h3>

<section>
  <h3>Release 56 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday February 24th, 2022 at 10am EDT for our Release 56 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 56.  
    <a href="https://attendee.gotowebinar.com/register/580233597692794126" target="_blank">Register here</a>  
  </div>
</section>

<section>
  <h3>New data in {{project}} 56</h3>
  <div style="margin-left: 3em;">

    <h4>Transcriptomics</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_e973eadd57">Heat shock response in sensitive mutants (LRR5, DHC)</a> &ndash; RNA-Seq transcriptomes of wild-type and heat shock sensitive mutant clonal lines (&Delta;DHC and &Delta;LRR5) during normal (37&#8451;) and febrile (41&#8451;) conditions. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34315897" target="_blank">Zhang et al. 2021</a>.</li>

      <li><a href="/a/app/record/dataset/DS_87e4fcafff">Intraerythrocytic development cycle transcriptome by UTR-Seq (2020)</a> &ndash; Transcriptome throughout the intraerythrocytic development cycle. RNA sequence analysis of 7 time points (0, 8, 16, 24, 32, 40, 48 hrs) post infection of tightly synchronized <i>Plasmodium falciparum</i> 3D7 <a href="http://www.ncbi.nlm.nih.gov/pubmed/32513207" target="_blank">Chappell et al. 2020</a>.</li>
    </ul>

    <h4>Phenotype</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_166e521549">Heat-shock response phenotypic screen</a> &ndash; Heat-shock sentitivity phenotype screen of 922 functionally uncharacterized piggy back mutants. Three rounds of temperture cycling followed by quantitative insertion site sequencig were used to establish the heat shock phenotype (sensitive, neutral, tolerant) for each gene. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34315897" target="_blank">Zhang et al. 2021</a>.</li>
    </ul>
  </div>
</section>

<section>
  <h3>New features in {{project}} 56</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>My Organism Preferences is a new option for filtering menus to a subset of organisms in {{project}}. Use the <a href="/a/app/preferred-organisms">My Organism Preferences page</a> to choose the organisms you are interested in.  {{project}} will function as if the database contained only your chosen organisms.  The feature is easily disabled/enabled at any time from a button located under the header on every page. <a href="/a/app/static-content/faq.html#organismpreferences"> More Details</a> </li>

    <li>The Reference sequence track in JBrowse has a new color scheme. <a href="{{'/assets/images/JBrowse-new-colors.gif' | absolute_url}}">View a tutorial GIF.</a>
      <ul>
        <li>Purines appear in shades of blue while pyrimidines appear in shades of yellow.</li>
        <li>Stop codons are red while methionines are green.</li>
        <li>The six frame translations appear in gray.</li>
        <li>Nucleotides and amino acids are distinguished by color at low zoom.  Under high zoom they are also labeled with single letter codes.</li>
      </ul>
    </li>

    <li>Galaxy Version updated on 8 Oct, 2021: This updated version of the Galaxy software improves job and workflow performance. In addition to the previous set of tools and workflows, we now offer additional tools, such as those for scRNA-seq (Scanpy, Seurat), species ID (CryptoGenotyper), table manipulation (Datamash), and proteomics (MSstats, Search GUI, Peptide Shaker, MaxQuant). Certain tools have been upgraded and now function without errors: bamCoverage, bgzip, tabix and Upload File. </li>

    <li>Enhanced searching on gene record pages &ndash; The content navigation tool now accesses detailed descriptions of {{project}} gene page content, making it easier to find the content that you need.</li>
    </ul>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
