---
categories: []
layout: post
title: PlasmoDB 65 Released
category: [PlasmoDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 65.</h3>

<section>
  <h3>Release 65 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday July 20th, 2023 at 10 am Eastern for our Release 65 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 65. 
    <a href="https://attendee.gotowebinar.com/register/4226674862077131357" target="_blank">Register here</a>
  </div>
  <br>
</section>


<section>
  <h3>New data in {{project}} 65</h3>
  <div style="margin-left: 3em;">
    <h4>Genome Sequence and Annotation</h4>
    <div style="margin-left: 3em;">
      <ul>
        <li><a href="/a/app/record/dataset/DS_0a9930bb09"><i>Plasmodium yoelii</i> yoelii 17XNL 2023 Genome Sequence and Annotation</a> &ndash; high-quality genome assembly for <i>P. yoelii 17XNL</i> using HiFi PacBio long-read DNA sequencing. In addition Nanopore long-read direct RNA-seq, Illumina short-read sequencing of mixed blood stages was used to create complete gene models that include not only coding sequences but also alternate transcript isoforms, and 5' and 3' UTR designations. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37247760" target="_blank">Godin and Sebastian et al. 2022</a>  &ndash; <a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_020844765.3" target="_blank">GCA_020844765.3</a>.</li>

        <li><a href="/a/app/record/dataset/DS_7ac217da8c"><i>Plasmodium ovale</i> wallikeri strain PowCR01 Genome Sequence and Annotation</a> &ndash; <i>de novo</i> genome assembly of <i>P. ovale</i> wallikeri strain PowCR01 was produced using 100bp paired-end Illumina sequencing on a HiSeq 2000 version 3. The isolate was sourced from a patient attending the Navrongo War Memorial hospital, Ghana. <a href="http://www.ncbi.nlm.nih.gov/pubmed/28117441" target="_blank">Rutledge et al. 2017</a>  &ndash; <a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_900090025.2" target="_blank">GCA_900090025.2</a>.</li>
      </ul>
    </div>
    <br>

    <h4>Genetic Variation (WGS)</h4>
    <div style="margin-left: 3em;">
      <ul>
        <li><a href="/a/app/record/dataset/DS_3aa0ae7aee">WGS of 8 Patient Isolates from Peru</a> &ndash; DNA from the leukocyte-depleted Peruvian blood samples was extracted and sequenced with illumina methods. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36250048" target="_blank">De Meulenaere et al. 2022</a>. </li>
      </ul>
    </div>
    <br>

    <h4>RNA Sequence</h4>
    <div style="margin-left: 3em;">
      <ul>
        <li><a href="/a/app/record/dataset/DS_1f0d95797a"><i>Plasmodium vivax</i> patient isolate transcriptomes</a> &ndash; RNA seq transcriptomes of schizont and trophozoite stage cultures of <i>P. vivax</i> isolates from patients in Peru and Papua New Guinea. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36250048" target="_blank">De Meulenaere et al. 2022</a>. </li>
      </ul>
    </div>
    <br>

    <h4>Single-cell RNA Sequence</h4>
    <div style="margin-left: 3em;">
      <ul>
        <li><a href="/a/app/record/dataset/DS_f9d0d0082d">A spatiotemporally resolved single cell atlas of the <i>Plasmodium</i> liver stage - Parasite Only</a> &ndash; Parasite portion of the single cell liver stage atlas.  Molecular characterization of thousands of infected hepatocytes in different time points to infer spatial coordinates, and characterize the parasite's temporal expression programs in a zonally-controlled manner. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36352220" target="_blank">Afriat et al. 2022</a>. </li>
        <li><a href="/a/app/record/dataset/DS_fa220f10ed">A spatiotemporally resolved single cell atlas of the <i>Plasmodium</i> liver stage - Combined</a> &ndash; Host and parasite portion of the single cell liver stage atlas.  Molecular characterization of thousands of infected hepatocytes in different time points to infer spatial coordinates, and characterize the parasite's temporal expression programs in a zonally-controlled manner. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36352220" target="_blank">Afriat et al. 2022</a>. </li>
      </ul>
    </div>
    <br>
  </div>
</section>

<section>
  <h3>New features in {{project}} 65</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="https://veupathdbprod.globusgenomics.org/" target="_blank">VEuPathDB Galaxy</a> now offers a preconfigured ChIP Seq workflow. This workflow processes ChIP Seq FASTQ files by aligning to your chosen reference genome with BWA then calling peaks using MACS2. Peaks calculated from the input FASTQ files are converted to bigWig format which can be exported from Galaxy to your VEuPathDB account.</li>
      <li>Strategy Improvement  &ndash; A new option in the gene result table provides a view of search specific data columns for your preferred subset of genes. Click the 'Show only the Genes in my basket' option from any gene result to filter the result based on the genes in your basket.  This can be used in combination with the 'Send to Basket (replace)' option to view search specific data columns for multi-step strategy results. <a target="_blank" href="https://youtu.be/SCCxNTGu-dQ">Filter by Basket tutorial.</a> </li>
   </ul>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
