---
layout: plain
title: VEuPathDB Workshops
permalink: /ags2022
tags: [workshop-exercise]
---
<style>
  h1 {
    font-size: 2.5em;
  }
  div.contents {
    margin-left: 1em;
    margin-bottom: 3em;
  }
  
  div.workshop {
    margin: 2em 1em;
  }

details summary, details ul {
  margin-top: 1em;
}
details summary {
  font-size: 120%;
  color: #069;
}
details p, details table {
  margin-left: 2em;
}
details table {
  margin-right: 6em;
}

table {
  margin-top: 1em;
  border-collapse: collapse;
}
/*
table, th, td {
  border: 1px solid black;
  padding: 0.5em;
}
*/
tr.break td {
  background-color: #DCDCDC;
}

table.hor-minimalist-a {
  text-align: left;
}
table.hor-minimalist-a th {
  font-size: 110%;
  font-weight: 400;
  color: #039;
  border-bottom: 2px solid #6678b1;
  padding: 0.5em;
  text-align: left;
}
table.hor-minimalist-a tr {
  border-bottom: 1px solid #ddd;
}
table.hor-minimalist-a tr:hover td {
  color: #039; 
}
table.hor-minimalist-a tr.other td {
  background-color: #fafafa;         
}
table.hor-minimalist-a tbody {
  display: table-row-group;
  vertical-align: middle;
  border-color: inherit;
}
table.hor-minimalist-a td {
  color: #669; 
  padding: 0.5em 0.5em 0.5em;
  vertical-align: middle;
}
table.hor-minimalist-a tfoot {
  font-size: 90%;
}
table.hor-minimalist-a tfoot tr {
  border:0;
}
th.time {
  width: 10%;
}
th.event {
  width: 50%;
}
th.author {
  width: 20%;
}
th.recording {
  width: 20%;
}
div.centered-title {
    border: 1px solid black;
    border-radius: 0.8em;
    text-align: center;
    margin-left: 15em;
    margin-right: 15em;
    background: #F8F8F8;
}
</style>

<p><a href="/">Home</a> >> Help >> 
   <a href="/a/app/static-content/landing.html">Learn how to use VEuPathDB</a> >> 
   <a href="/a/app/static-content/workshops.html">Workshops</a></p>

<h1>VEuPathDB Workshops</h1>
<div class="static-content">


  <div class="centered-title">     
    <h4>Arthropod Genomicis Symposium 2022: pre-meeting VectorBase hands-on Workshop</h4>
    <p>6:00 pm on Sunday, 5th June to 12:00 pm (noon) on Thursday, 9th June 2022</p>
    <p>Jordan Hall of Science, University of Notre Dame, Notre Dame, IN</p>
    <p>In-person (sessions will not be virtual or recorded)</p>
  </div>

  

<div class="contents">

  <div class="anchor"><a name="AGSvectorbase2022"></a></div>
  <div class="workshop">

    <p><h2>About the workshop</h2></p>
    <p>This is a 5-day intensive workshop that will provide you with hands-on training on how to effectively use VectorBase to mine the available data and to analyze your own data. Since VectorBase is part of the VEuPathDB Bioinformatics Resource Center (BRC), all learned knowledge can be applied to its other components such as those supporting vector-transmitted pathogens <i>Plasmodium</i> (PlasmoDB), <i>Leishmania</i>, <i>Trypanosoma</i> (TriTrypDB), <i>Babesia</i>, and <i>Theileria</i> (PiroplasmaDB).     </p>
  
    <p>Expected learning objectives:</p>
    <ul>
      <li>Site Search: use keywords (<i>e.g.</i>, gene metadata and gene IDs), filter and explore results, export to strategies</li>
      <li>Organism preferences: enable and disable to explore data sets </li>
      <li>Gene pages: become familiar with the information on it, navigate to and from it, use the table of contents to navigate </li>
      <li>Search strategies: how to construct and combine the more than 100 available searches to produce multi-step strategies, results can be in the format of gene IDs </li>
      <li>RNAseq and microarrays data searches: use differential expression, fold change, and percentile searches to explore gene expression</li>
      <li>Proteomics data searches: use mass spec data to identify genes based on expression evidence</li>
      <li>Genome browser: use menu and navigation options, add and configure data tracks, interpret data (<i>e.g.</i> RNAseq)</li>
      <li>Gene Ontology (GO) enrichment: run and analyze a GO enrichment analysis </li>
      <li>Apollo: learn how to manually annotate a gene, which involves both structural changes (intron/exon boundaries) and metadata (gene name and symbol)</li>
      <li>Galaxy: perform your own bioinformatic analysis using our servers, <i>e.g.</i> RNAseq transcript differential expression or SNP calling  </li> 
    </ul>
  
    <p>This workshop is scheduled as a symposium pre-event, and even if you are not attending AGS, you are more than welcome to attend the workshop. </p> 
    <p><h2>COVID-19 safety</h2></p>
    <p>We encourage all workshop attendees to wear face coverings while indoors, regardless of vaccination status. Please see the <a href="https://cvent.me/qvLA93">AGS website</a> for more information.</p>

    <p><h2>About us</h2></p>
    <p>If you are a new user and will like to know more details about our resources to decide if you want to apply for the workshop, we invite you to read our more recent VEuPathDB (<a href="https://pubmed.ncbi.nlm.nih.gov/34718728/">Amos et al 2022</a>) and VectorBase (<a href="https://pubmed.ncbi.nlm.nih.gov/34864248/">Giraldo-Calderón et al 2022</a>) papers, and explore our <b><u>free</u></b> tutorials and previously recorded <a href="https://vectorbase.org/vectorbase/app/static-content/landing.html">webinars and workshops</a>. Regardless of if you attend or not the workshop, please feel free to email us (<a href="https://vectorbase.org/vectorbase/app/contact-us">help@vectorbase.org</a>), our goal is to maintain a fruitful and ongoing conversation between our team and you, the scientists working in medical entomology, to obtain more efficient, effective ways to help you reach your goals.</p>

    <p><h2>Computer and internet access</h2></p>
    <p>A room with computers will not be provided, please bring your own laptop. It should have the latest Firefox, Google Chrome, Safari or Microsoft edge version installed. 
 </p>

    <p><h2>Registration</h2></p>
    <ul>
        <li>Number of course participants: maximum 30</li>
        <li>Registration will open soon via the <a href="https://cvent.me/qvLA93">AGS website</a>. After you complete your registration, you will receive an email from ggiraldo@nd.edu (Gloria I. Giraldo-Calderón) to fill out an online form, this process completes your registration. The form information is confidential and it helps us know our prospective audience to customize examples and exercises.</li>
        <li>Registration deadline: it is the 29th of May 2022. It will be taken on a first-come, first-served basis</li>
        <li>Who should register: the ideal candidate should have a need to learn how to use VectorBase and be interested in using the database to answer their own research questions. Scientists at any career stage are welcome to apply</li>
    </ul> 
      
      <p><h2>Cost and fees</h2></p>
      <p>The registration fee is $ 65 USD, payments should be made using this AGS-University of Notre Dame link. The registration fee includes learning materials and some meals (see agenda for details). You are expected to provide your own transportation, lodging, and additional meals. At your request, we can email you a registration acceptance letter to facilitate your application for external travel funds or scholarships.</p>

      <p><h2>Lodging</h2></p>
      <p>We have rooms reserved at the University of Notre dame dorms and hotel, please see the <a href="https://cvent.me/qvLA93">AGS website</a> for more information.</p>

      <p><h2>Weather</h2></p>
      <p>Buildings in the summer might be very cold, you might want to plan to bring a sweater. The month with the most rain in Indiana is June, it is a good idea to have an umbrella.</p>

      <p><h2>Questions</h2></p>
      <p>Please direct inquiries or questions to <a href="https://vectorbase.org/vectorbase/app/contact-us">help@vectorbase.org</a></p>

      <h2>Instructors</h2>

  
      <p>Gloria I. Giraldo-Calderón, PhD
      <br>Samuel Rund, PhD
      <br>University of Notre Dame
      <br>Notre Dame, IN, USA</p>
      
      <p>Omar S. Harb, PhD
      <br>University of Pennsylvania
      <br>Philadelphia, PA, USA</p>
      
      <p>Members of the VEuPathDB outreach team</p>
      <br>
      <br>
      
      
      <p>David S. Roos, PhD
 <br>Joint-PI, VEuPathDB NIAID-NIH Contract; WT Co-PI
 <br>University of Pennsylvania
 <br>Philadelphia, PA, USA</p>

      <p>Mary Ann McDowell, PhD
      <br>Joint-PI, VectorBase NIAID-NIH Contract
      <br>University of Notre Dame
      <br>Notre Dame, IN, USA </p>
      <br>
      <br>
      
      
      <p>Robert Waterhouse, PhD 
      <br>University of Lausanne
      <br>The Swiss Institute of Bioinformatics
      <br>Switzerland</p>
      <br>
      <br>
      

    <details open>
      <summary><b>Schedule</b></summary>
      
      <table class="hor-minimalist-a">
        <thead>
          <tr>
            <th class="break"><b>Sunday, 5th June 2022</b></th>
          </tr>
          <tr>
            <th class="time">Time</th>
            <th class="event">Topic (and handout)</th>
            <th class="author">Instructor</th>
          </tr>
        </thead>
        <tbody>
         
          <tr>
            <td>6:00 - 9:00 pm</td>
            <td><br>Registration
                <br>Welcome dinner (provided)
                <br>General introduction to omics 
                <br>Attendees introductions
                <br>Group photo
                </td>
            <td>GIGC, OH, MAM, DSR, SR</td>
          </tr>
        </tbody>
      </table>
      <br>
      <br>
      
       
      <table class="hor-minimalist-a">
        <thead>
          <tr>
            <th class="break"><b>Monday, 6th June 2022</b></th>
          </tr>
          <tr>
            <th class="time">Time</th>
            <th class="event">Topic (and handout)</th>
            <th class="author">Instructor</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>8:00 am</td>
            <td>Breakfast (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>9:00 am</td>
            <td>VectorBase general introduction: remit, data tools and resources</td>
            <td>GIGC</td>
          </tr>
          <tr>
            <td>9:30 am</td>
            <td>Search: Finding genes (with Site Search and BLAST), building search strategies and visiting gene pages</td>
            <td>GIGC<br></td>
          </tr>
          <tr>
            <td>10:00 am</td>
            <td>Coffee/Tea Break (provided)</td>
            <td></td>
          </tr>
          <tr>
            <td>10:20 am</td>
            <td>Search -- Continued</td>
            <td>GIG<br></td>
          </tr>
          <tr>
            <td>11:00 am</td>
            <td>Genome Browser I: Alignments and Comparative Genomics</td>
            <td>DSR<br></td>
          </tr>
          <tr>
            <td>12:00 m </td>
            <td>Lunch (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>1:00 pm</td>
            <td>RNA sequence data analysis via Galaxy, Part I Uploading data (Group Exercise)</td>
            <td>OSH<br></td>
          </tr>
         <tr>
            <td>1:45 pm</td>
            <td>Genome Browser II: Interpreting RNAseq data (Group Exercise)</td>
            <td>DSR<br></td>
          </tr>
          <tr>
            <td>2:30 pm </td>
            <td>Coffee/Tea Break (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>2:50 pm</td>
            <td>Homology gene relationships via OrthoMCL DB</td>
            <td>GIGC<br></td>
          </tr>
          <tr>
            <td>4:00 pm - 5:00 pm</td>
            <td>RNA sequence data analysis via Galaxy, Part II starting the workflow (Group Exercise)</td>
            <td>OSH<br></td>
          </tr>
        </tbody>
      </table>
      <br>
      <br>
      
     
      <table class="hor-minimalist-a">
        <thead>
          <tr>
            <th class="break"><b>Tuesday, 7th June 2022</b></th>
          </tr>
          <tr>
            <th class="time">Time</th>
            <th class="event">Topic (and handout)</th>
            <th class="author">Instructor</th>    
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>8:00 am</td>
            <td>Breakfast (provided)</td>
            <td> <br></td>
          </tr>
          <tr>
            <td>9:00 am</td>
            <td>Sequence Exercises: Motifs, domains and colocation</td>
            <td>GIGC</td>
          </tr>
          <tr>
            <td>9:30 am</td>
            <td>Data retrieval and download</td>
            <td>GICG<br></td>
          </tr>
          <tr>
            <td>10:00 am</td>
            <td>Coffee/Tea Break (provided)</td>
            <td></td>
          </tr>
          <tr>
            <td>10:50 am</td>
            <td>RNA sequence data analysis via Galaxy, Part III: DESeq2 and Exporting your data to VectorBase
            <br>FASTQ Groomer 
                <br>Trimmomatic
                <br>FASTQC
                <br>HISAT2 
                <br>bamCoverage
                <br>htseq-count
                <br>HTSeqCountToTPM
            </td>
            <td>OSH<br></td>
          </tr>
          <tr>
            <td>12:00 pm</td>
            <td>Lunch (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>1:00 pm</td>
            <td>RNA sequence data analysis via Galaxy, Part IV: viewing and analyzing your results, Bam to BigWig (Group Exercise)</td>
            <td>OSH<br></td>
          </tr>
          <tr>
            <td>2:30 pm</td>
            <td>Coffee/Tea Break (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>2:50 pm</td>
            <td>Functional Genomics 1: Transcriptomics, Proteomics, GO Enrichment, Metabolic Pathways</td>
            <td>GIGC<br></td>
          </tr>
          <tr>
            <td>4:00 - 5:00 pm</td>
            <td>Workshop group project, Part 1: Define your own questions and develop a method to answer them with VectorBase (Group exercise)</td>
            <td><br></td>
          </tr>
        </tbody>
      </table>
     <br>
     <br>
      
      <table class="hor-minimalist-a">
        <thead>
          <tr>
            <th class="break"><b>Wednesday, 8th June 2022</b></th>
          </tr>
          <tr>
            <th class="time">Time</th>
            <th class="event">Topic (and handout)</th>
            <th class="author">Instructor</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>8:00 am</td>
            <td>Breakfast (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>9:00 am</td>
            <td>Population data with MapVEu: population abundance, pathogen infection status, insecticide resistance genotypes and phenotypes, blood meal host, microsatellites and SNP data … etc.   </td>
            <td>SR</td>
          </tr>
          <tr>
            <td>10:00 am</td>
            <td>Coffee/Tea Break (provided)</td>
            <td></td>
          </tr>
          <tr>
            <td>10:20 am</td>
            <td>Population data with the genome browser: SNPs</td>
            <td>SR<br></td>
          </tr>
          <tr>
            <td>11:00 am</td>
            <td>RNA sequence data analysis Part V: finish data analyses </td>
            <td>OSH<br></td>
          </tr>
          <tr>
            <td>12:00 m</td>
            <td>Lunch (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>1:00 pm</td>
            <td>Functional Genomics 2: Host response data</td>
            <td>GIGC</td>
          </tr>
          <tr>
            <td>2:00 pm</td>
            <td>Workshop group project, Part 2: Data analyses (Group exercise)
              <br>Start outline of presentation, you can do a live demo and/or a few slides 
            </td>
            <td><br></td>
          </tr>
          <tr>
            <td>3:00 pm</td>
            <td>Coffee/Tea Break (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>3:20 - 5:00 pm</td>
            <td>Workshop group project, Part 3: Finalize project (Group exercise)</td>
            <td><br></td>
          </tr>
        </tbody>
      </table>
      <br>
      <br>
      
      
      <table class="hor-minimalist-a">
        <thead>
          <tr>
            <th class="break"><b>Thursday, 9th June 2022</b></th>
          </tr>
          <tr>
            <th class="time">Time</th>
            <th class="event">Topic (and handout)</th>
            <th class="author">Instructor</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>8:00 am</td>
            <td>Breakfast (provided)</td>
            <td> <br></td>
          </tr>
          <tr>
            <td>9:00 am</td>
            <td>Manual gene annotation with Apollo </td>
            <td>GIGC, RW</td>
          </tr>
          <tr>
            <td>10:30 am</td>
            <td>Coffee/Tea Break (provided)</td>
            <td><br></td>
          </tr>
          <tr>
            <td>10:50 am</td>
            <td>Workshop group project, Part 4: Present your findings</td>
            <td><br></td>
          </tr>
          <tr>
            <td>12:00 - 12:15 pm</td>
            <td>Workshop Feedback and wrap-up. Digital certificates will be provided.  </td>
            <td><br></td>
          </tr>
        </tbody>
      </table>
      <br>
      <br>

    </details>        

    <p><h2>Sponsored and organized by</h2></p>
    <img style="width: 80em; margin-top: .5em; margin-left: 4em;" src="{{ "/assets/images/workshops/wks_ags2022.png" | absolute_url }}" alt="Logos"/>

  </div>  <!-- class workshop -->

</div>  <!-- class contents -->
</div>  <!-- class static-content -->


