---
title: Cryptosporidium gp60 Resource Page
permalink: /gp60
tags: [general]
category: [CryptoDB]
---
<style>

div.method-details {
  margin: 2em;
}

</style>

<h1>Cryptosporidium <i>gp60</i> Resource Page</h1>

<div class="static-content"> 

<p><b>Importance of the <i>gp60</i> Gene in Cryptosporidium:</b> <br>
Species of the protozoan genus <i>Cryptosporidium</i> are globally important One Health pathogens, causing the gastrointestinal illness cryptosporidiosis in both humans and animals. Investigations into the molecular epidemiology of Cryptosporidium have relied heavily over the past 25 years on within-species subtyping by sequencing part of a gene encoding a 60 kDa glycoprotein (gp60). The <i>gp60</i> gene codes for an immunodominant surface glycoprotein that mediates host cell invasion and thus plays a role in host infectivity and pathogenicity, consistent with its highly polymorphic nature. <br>
gp60 subtyping is commonly used to study the diversity, transmission, and zoonotic potential of Cryptosporidium species, especially <i>C. hominis</i> (mainly infecting humans) and <i>C. parvum</i> (infecting humans and other mammals).  Since the Cryptosporidium <i>gp60</i> gene was first described, its use as a target for subtyping has been widely accepted by the scientific community and globally used in population studies as well as investigations into human and animal cryptosporidiosis, especially outbreaks.</p>

<p><b>Purpose of this Page:</b> <br>
While the <i>gp60</i> typing scheme started with a simple nomenclature, it has evolved as new features have been discovered by different investigators and has become increasingly complicated. The evolution of these new features and contradictions in their use has led to the nomenclature becoming confusing. <b>The goal of this page is to collate resources with an aim to promote standardization of this subtyping tool to serve the Cryptosporidium community.</b></p>

<p><b>Guide to gp60 Subtyping</b> <br>
The paper <i>Deciphering a cryptic minefield: A guide to Cryptosporidium gp60 subtyping</i> by Robinson et. al. outlines the details and rules for gp60 subtyping into a single publication and provides useful resources for the community. The paper is currently in press in the journal <i>Current Research in Parasitology & Vector-Borne Diseases</i> and will be linked here when available.</p>

<p><b>Summary of Current gp60 Subtyping Nomenclature:</b> <br>
The DNA, and corresponding amino acid, sequence variation seen at the <i>gp60</i> locus both within and between <i>Cryptosporidium</i> species has provided multiple features for discrimination that have been included in the nomenclature of the <i>gp60</i> subtyping scheme. These features include DNA sequence polymorphisms and mini- and microsatellite repeats. The components of the gp60 gene sequence used in the nomenclature include “R” repeats, “r” repeats, alphabetical suffixes, “variant” designations, and the use of the Greek alphabet as a family designation. The basic nomenclature is listed below. Details and references are available in Robinson et. al., 2025.
       <ul>
        <li><b>Genotype families and allelic groupings</b>: The initial <i>gp60</i> subtyping nomenclature refers to genotype I (<i>C. hominis</i>) and genotype II (<i>C.parvum</i>). Distinct allelic groupings were identified by sequence polymorphisms extending the species designations I and II to define “gp60 families” named Ia, Ib, Id, etc. for C. hominis and IIa, IId, etc. for C. parvum.</li>
        <li><b>Serine microsatellite repeats/ polyserine repeats</b>: The second main element of the gp60 subtype nomenclature describes the composition of the serine repeat located upstream of the family-defining region. The serine microsatellite is composed of TCA, TCG or TCT codons and the number of each is counted and recorded, prefixed by A, G or T, following the subtype family name. For example, IeA11G3T3 is a subtype of C. hominis belonging to the Ie allelic family with 11 TCA, 3 TCG and 3 TCT codons in the serine repeat, and IIdA23G1 is a C. parvum subtype belonging to the IId allelic family with 23 TCA and 1 TCG codons in the serine repeat.</li>
        <li><b>R repeat designation</b>: These R repeats, which follow different nomenclature rules depending on the species and gp60 family, have been the source of some confusion leading to errors and misnaming of isolates. The R repeats can be different sequences between species and subtypes and in alternative locations within the gene. Two of the R repeats that cause the most difficulty are in C. hominis subtype families Ia and If (see section below).</li>
        <li><b>The r repeat designation in IIa</b>: In C. parvum family IIa, the very rare occurrence of an ACA threonine codon within the polyserine repeat region has been designated as an “r” repeat with a digit to represent the number of repeats and an alphabetical suffix to signify its different positions within the serine tract. However, the position of this ACA r repeat is variable and it was proposed to differentiate between them using an alphabetical suffix.</li>
        <li><b>Alphabetical suffix designation</b>: The use of alphabetical suffixes in gp60 nomenclature is not restricted to variations in r motifs. Traditionally, alphabetical suffixes in gp60 subtyping are used to differentiate variants of the same subtype. The most common use of these suffixes is within the IIc C. parvum gp60 subtype family. It is not only IIc isolates that have alphabetical suffixes assigned to further differentiate subtypes, but also C. parvum IId due to individual SNPs in the post-repeat region.</li>
        <li><b>Variant designation</b>: While some publications have designated alphabetic suffixes to sequences containing variations in the position of a TCG repeat within the polyserine microsatellite region (Alsmark et al., 2018), Lebbad et al. (2021) instead reserved the alphabetic suffixes for the 3’ SNPs (e.g. IIaA16G1R1b) and added the word “variant” where this TCG variation was present (e.g. IIaA15G1R1_variant and IIaA16G1R1b_variant). So, although we do not recommend the use of this terminology as part of the standard nomenclature, it is useful that these variations are detected and compared, and we propose v1, v2, etc. for such purposes.</li>
        <li><b>Use of Greek alphabet</b>: With the recognition of the large number of <i>C. parvum gp60</i> families, the allocation of Latin alphabet letters reached its end with the submission of IIy and IIz to GenBank in 2021. To overcome this problem, we recommend that the Greek alphabet letter name is written in full (alpha, beta, gamma, etc.) rather than using the symbol. </li>
        </ul>
</p>

<p><b>Examples of alignment of complicated R repeats:</b> <br>
Another area of confusion with R repeats is their DNA sequence variability making it particularly difficult for users to identify them as repeats and hence the correct number. This is due to how the repeats have been described, based on a repeat identified from the nucleotide sequence, often using Tandem Repeat Finder, whereas investigating the resulting amino acid sequence can simplify the identification. Two of the R repeats that cause the most difficulty are in C. hominis subtype families Ia and If. Examples of alignment of these repeats are provided here: <a target="_blank" href="{{'/documents/gp60_Ia_R_repeat_alignment.fas' | absolute_url}}">Ia fasta file</a> and <a target="_blank" href="{{'/documents/gp60_If_R_repeat_alignment.fas' | absolute_url}}">If fasta file</a></p>. 

<p><b> Software for gp60 analysis: CryptoGenotyper</b> <br>
An automated tool called CryptoGenotyper was developed for analysing both gp60 and SSU rRNA (single or mixed small subunit ribosomal RNA- useful for distinguishing different Cryptosporidum species) Sanger sequences to increase the accuracy and speed of genotyping. The gp60 part of the tool accepts “.ab1” input files and determines the sequence based on fluorescent intensities. The software includes a reference database against which the sequence is BLASTed and the number of repeat regions, species and subtypes are reported. The updated CryptoGenotyper v1.5.0 now has a new added feature for direct analysis of nucleotide sequences stored in text-based FASTA files in addition to Sanger sequencing chromatogram data files in “.ab1” format. The manually curated reference database for classifying the sequences has also been updated and now includes: the multitude of newly described species and families with distinct nomenclature; updated algorithms for repeat region detection; and the standardized nomenclature described in this paper. This tool is especially useful when processing large number of sequences. <br>
The CryptoGenotyper tool is available on several platforms including the web-based platform Galaxy installable from the <a href="https://toolshed.g2.bx.psu.edu/" target="_blank">public ToolShed repository</a> (tool id: cryptogenotyper, owner: nml), from the <a href="https://usegalaxy.eu/root?tool_id=CryptoGenotyper" target="_blank">public Galaxy EU server</a>, or as a <a href="https://anaconda.org/bioconda/cryptogenotyper" target="_blank">standalone Bioconda package</a>. The source code is available <a href="https://github.com/phac-nml/CryptoGenotyper" target="_blank">here in Github</a>.

<p><b>Current gp60 Subtyping Table</b>: A curated live version of the master-list of contemporary gp60 nomenclature is accessible <a href="https://docs.google.com/spreadsheets/d/1jGudi-s34ShrlJxciSpdPd9MgyAx1aFf/edit?usp=sharing&ouid=110551133009250701521&rtpof=true&sd=true" target="_blank">on this spreadsheet.</a></p>

<p>b>A contact Form can be accessed</b> <a href="https://docs.google.com/forms/d/e/1FAIpQLSeUAcedO-I2b5IXXOW4p_iZGsoNkbROXA11_LHtVAJSZ1PTSQ/viewform?usp=header" target="_blank">at this link</a>. It can be used in different ways: 
<ul>
<li>Refer to the gp60 Master-list on the gp60 Resource Page to identify the subtype family, or to determine new subtype families or species designations, and use the contact link to notify curators of any novel designations. This will ensure curation and avoid duplication thus preserving the nomenclature. </li>
<li>Submit updates of any additional features discovered, for example, the discovery of new R repeat variation, to the gp60 Master-list by using the contact link on the gp60 Resource page. </li></p>

