---
layout: post
title: MicrosporidiaDB 60 Released
category: [MicrosporidiaDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 60.</h3>

<section>
  <h3>Release 60 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday November 17th, 2022 at 10am EST for our Release 60 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 60.  
    <a href="https://attendee.gotowebinar.com/register/8994775200159032332" target="_blank">Register here</a>  
  </div>
  <br>
</section>

<section>
  <h3>Pseudogene Classification</h3>
  <div style="margin-left: 3em;">
    In release 51 (16 March 2021) pseudogenes with an internal stop codon automatically included predicted 3'UTRs. The UTR prediction was due to an incorrect pipeline which has now been rectified. UTRs have now been removed and all pseudogenes are correctly displayed without CDS and UTR. The affected pseudogenes were shown in VEuPathDB with the Biotype classification pseudogene_with_CDS. This has now been changed to the correct biotype pseudogene. <a href="https://veupathdb.org/veupathdb/app/workspace/strategies/import/f9db1d88ac085282" target="_blank">List of affected genes</a>. </div>
</section>

<section>
  <h3>New data in {{project}} 60</h3>
  <div style="margin-left: 3em;">
    <h4>Genome Sequence</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_e5f334a759"><i>Enterocytozoon hepatopenaei</i> EHP-ID16</a> &ndash; isolated from <i>Penaeus vannamei</i> cultured at Indonesia. submitted by Ji Hyung Kim <a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_003709115.1" target="_blank">GCA_003709115.1</a></li>
     </ul>
     <br>
    <h4>Genome Sequence and Annotation</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_fba7650b2d"><i>Hamiltosporidium magnivora</i> BE-OM-2</a> &ndash; isolated from cultured <i>Daphnia magna</i> in Belgium.   <a href="http://www.ncbi.nlm.nih.gov/pubmed/31825473" target="_blank">Haag et al. 2020</a></li>

      <li><a href="/a/app/record/dataset/DS_cd8677dfb9"><i>Hamiltosporidium magnivora</i> IL-BN-2</a> &ndash; isolated from cultured <i>Daphnia magna</i> in Israel.   <a href="http://www.ncbi.nlm.nih.gov/pubmed/31825473" target="_blank">Haag et al. 2020</a></li>

      <li><a href="/a/app/record/dataset/DS_2316bd6cc9"><i>Hamiltosporidium tvaerminnensis</i> IL-G-3</a> &ndash; isolated from cultured <i>Daphnia magna</i> from Israel: Gaash Pond.   <a href="http://www.ncbi.nlm.nih.gov/pubmed/31825473" target="_blank">Haag et al. 2020</a></li>

      <li><a href="/a/app/record/dataset/DS_784ac2ef64"><i>Hamiltosporidium tvaerminnensis</i> FI-OER-3-3</a> &ndash; isolated from cultured <i>Daphnia magna</i> from Finland: Oeren Tvarminne Achipelago.   <a href="http://www.ncbi.nlm.nih.gov/pubmed/31825473" target="_blank">Haag et al. 2020</a></li>

      <li><a href="/a/app/record/dataset/DS_02354d7d63"><i>Mitosporidium daphniae</i> UGP3</a> &ndash; Updated to the new repository version to incorporate Feb 2020 changes including the supression of two contigs (contaminants: JMKJ01000282 and JMKJ01000400), and a new Assembly Name was (changed from UGP1.0 to UGP1.1).  <a href="http://www.ncbi.nlm.nih.gov/pubmed/31825473" target="_blank">Haag et al. 2020</a></li>

      <li><a href="/a/app/record/dataset/DS_535abb859b"><i>Microsporidium sp.</i> FI-F-10</a> &ndash; isolated from the infected the gut epithelium cells of its crustacean host <i>Daphnia longispina</i> sampled in Finland: skerry island on the Baltic Sea near Hanko-Tvarminne. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31825473" target="_blank">Haag et al. 2020</a></li>

      <li><a href="/a/app/record/dataset/DS_7ef7bae229"><i>Nematocida ironsii</i> ERTm5</a> &ndash; an intestinal-specific strain isolated from a wild-caught <i>C. briggsae</i> in Hawaii. This organism name was changed from <i>Nematocida sp.</i> ERTm5 to <i>Nematocida ironsii</i> ERTm5 as documented in Reinke et al. 2017. The NCBI record (PRJNA312551) reflects the original name. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31013330" target="_blank">Balla et al. 2019</a></li>
      
      <li><a href="/a/app/record/dataset/DS_2a4166bf3c"><i>Nosema apis</i> BRL 01</a> &ndash; a microsporidia infecting the honey bee.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23829473" target="_blank">Chen et al. 2013</a></li>
    </ul>
    <br>
    
  <h4>Structure Predictions</h4> 
    <ul>
      <li><a href="/a/app/record/dataset/DS_dc1c8cb24c">AlphaFold Predictions</a> &ndash; are updated to AlphaFold Release 4 (July 2022) which offers predictions for significantly more proteins than Release 3.  Use {{project}}'s <a href="/a/app/search/transcript/GenesWithAlphafold">Genes with AlphaFold Predictions</a> search to explore genes with AlphaFold predictions or visit the Structure Analysis section of your favorite gene page to view data relevant to you.</li> 
    </ul>
    <br>
  </div>
</section>

<section>
  <h3>New features in {{project}} 60</h3>
  <div style="margin-left: 3em;">
    <h4>Associated Data Sets</h4>
      <ul>
        <li>Data sets that come from the same publication are now linked.  The new 'Associated Data Sets' section on any data set record page will link to other data from the same publicaion within {{project}} as well as data from organisms supported in different VEuPathDB sites.  <a target="_blank" href="{{'/assets/images/news/AssociatedDataSets.gif' | absolute_url}}">View tutorial GIF.</a></li>
      </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
