---
layout: post
title: VectorBase 66 Released
category: [VectorBase,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 66.</h3>

<section>
  <h3>Release 66 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday Dec 14th, 2023 at 10 am Eastern for our Release 66 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 66. 
    <a href="https://attendee.gotowebinar.com/register/6066397511661949782" target="_blank">Register here</a>
  </div>
  <br>
</section>

<section>
  <h3>New features in {{project}} 66</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li>The {{project}} <a href="/a/app/static-content/webinars.html" target="_blank">webinars page</a> has a new look. 'Upcoming' webinars announce the next few webinars. 'Releases' contains the recordings for our previous release webinars.  'On-Demand Training' provides embedded videos for special topic webinars. All webinar recordings are also available from the <a href="https://www.youtube.com/@EuPathDB/playlists" target="_blank">VEuPathDB YouTube Channel</a>.  </li>
      <li><a href="https://orthomcl.org/orthomcl/app" target="_blank">OrthoMCL DB</a> offers a new preconfigured download table called Phyletic Distribution Clade Abundance that offers the OMCL group distribution of sequences in Alveolata, Archaea, Amoeba, Bacteria, Fungi, Euglenozoa, Metazoa, Viridiplantae, and other Eukaryotes. <a href="{{'/assets/images/news/Download-OMCL-table.png' | absolute_url}}"> OMCL Download Tutorial</a> </li>
      <li><a href="/a/app/workspace/maps/DS_480c976ef9/new">MapVEu &ndash; Field-based vector population studies BETA</a> &ndash; An updated version of the MapVEu is now available in Beta. This version offers all the familiar MapVEu features with signicantly improved features for filtering data, displaying variables on the map, and analyzing variables with Supporting Plots. Please try the new MapVEu and <a href="/a/app/contact-us">Contact Us</a> with questions or suggestions.  <a href="{{'/documents/MapVEu_EDA_Introduction_20230913.pdf' | absolute_url}}"> MapVEu-Beta Tutorial</a>   </li>
   </ul>
  </div>
</section>

</section>
  <h3>Contribute your expert knowledge!</h3>
  <div style="margin-left: 3em;">
    We invite all users to engage with Apollo, where you can actively edit and enhance your selected genes. Whether you're a seasoned user or a newcomer, our updated <a href="/a/app/static-content/apollo_help.html">Apollo landing page</a> provides a wealth of resources, including links to on-demand webinars, tutorials, and annotation guidelines. <br><br>
    Every edit you make contributes significantly! This open invitation is extended to all users, with a special emphasis on scientists studying groups of organisms that have recently been updated to integrate new reference genomes, and for the species that are now accessible on {{project}} for the first time. If you're unfamiliar or have any questions, don't hesitate to <a href="/a/app/contact-us">Contact Us</a>. We're here to help!
  </div>
<section>

  <h3>Reference Genome Transition</h3>
  <div style="margin-left: 3em;">
    Recent updates to <a href="https://docs.google.com/spreadsheets/d/1lqzR7jJTi_NbjTp4y4DpxAilir-ndlWnH-v9NyodGm8/edit?usp=sharing" target="_blank">mosquito and tick species</a> include new assemblies that are much less fragmented, some organized to the chromosome level. All functional data alignments (e.g., RNAseq) will be mapped to both the current and new genomes.  After approximately six months, functional data alignments to the old assembly will be removed; old assemblies will remain available on VectorBase indefinitely but with limited functionality. The new assembly will have all the omics data sets aligned and complete site functionality. A banner on the genome page of the species involved will indicate such changes (e.g., <a href="https://vectorbase.org/vectorbase/app/record/dataset/TMPTX_iscaPalLabHiFi">Ixodes scapularis PalLabHiFi</a>). If you have any feedback or questions, need help, or are not able to find your genes of interest in the new genomes, please do not hesitate to contact us <a href="/a/app/contact-us">https://vectorbase.org/vectorbase/app/contact-us</a>. 
  </div>
</section>

<section>
  <h3>New data in {{project}} 66</h3>
  <div style="margin-left: 3em;">

  <h4>Genome Sequence and Annotation</h4>
  <div style="margin-left: 3em;">
    The following organisms are represented in VectorBase for the first time. 
    <ul>
      <li><a href="/a/app/record/dataset/DS_c099af5258"><i>Lutzomyia longipalpis</i> M1 </a>&ndash; A vector of <i>Leishmania</i> parasites that cause visceral leishmaniasis. A single male (M1) specimen of <i>Lutzomyia longipalpis</i>, derived from the Jacobina strain was sequenced using PacBio HiFi and Illumina HiC technologies. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_024334085.1" target="_blank">GCF_024334085.1</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/37043542" target="_blank">Labb√© et al. 2023</a> </li>

      <li><a href="/a/app/record/dataset/DS_4582dc809c"><i>Phlebotomus papatasi</i> M1 </a>&ndash; A  vector of <i>Leishmania</i> parasites that cause cutaneous leishmaniasis. The genome of individual one male (M1) from the PPIS colony was sequenced by PacBio HiFi using the PacBio Ultralow input library prep, and additional material from the PPIS colony was used to generate HiC data for scaffolding the reads. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_024763615.1" target="_blank">GCF_024763615.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_2f761c02f0"><i>Anopheles marshallii</i> AmarGA1 </a>&ndash; This assembly (idAnoMarsDA_429_01) is based on 51x PacBio HiFi data, 10X Genomics Chromium data, and Arima Hi-C data generated by the Anopheles Reference Genomes project. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_943734725.1" target="_blank">GCF_943734725.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_2652a3167a"><i>Sabethes cyaneus</i> ScyaPA1 </a>&ndash; This assembly, idSabCyanKW18_F2, is based on 37x PacBio HiFi data and Arima Hi-C data generated by the Anopheles Reference Genomes project. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_943734655.1" target="_blank">GCF_943734655.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_89dcd52262"><i>Anopheles nili</i> AnilCM1 </a>&ndash; This assembly, idAnoNiliSN_F5_01, is based on 48x PacBio HiFi data, 10X Genomics Chromium data, and Arima Hi-C data generated by the Anopheles Reference Genomes project. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_943737925.1" target="_blank">GCF_943737925.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_b56fe6b141"><i>Biomphalaria glabrata</i> XG47 </a>&ndash; This assembly, xgBioGlab47.1, is based on 29x PacBio data and Arima2 Hi-C data generated by the <a href="https://www.sanger.ac.uk/science/programmes/tree-of-life">Wellcome Sanger Institute Tree of Life Programme</a>. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_947242115.1" target="_blank">GCF_947242115.1</a>. </li>
    </ul>
  </div>
    <br>
  
  <h4>Genome Updates</h4>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_c3e414cbf3"><i>Glossina fuscipes</i> IAEA 2018 </a>&ndash; This data set was updated with the 2022 annotation version from the INSDC repository. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCA_014805625.1" target="_blank">GCA_014805625.1</a>;  <a href="https://www.researchsquare.com/article/rs-1685795/v1" target="_blank">Saarman et al. 2022 (preprint)</a></li>
      <li></li>
    </ul>
    <br>

  <h4>Genome Reference and Strain Name Changes</h4>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_c099af5258"><i>Lutzomyia longipalpis</i> M1 </a> now serves as the VectorBase reference for <i>Lutzomyia</i> species.  All functional genomics data will be aligned to this genome.</li>

      <li><a href="/a/app/record/dataset/DS_13ae5590c9"><i>Anopheles darlingi</i> AdarGF1 </a> &ndash; The strain name was changed from idAnoDarlMG-H_01 to AdarGF1 beginning with VectorBase 66.  This organism now serves as the VectorBase reference for <i>Lutzomyia</i> species.  All functional genomics data will be aligned to this genome.</li>

      <li><a href="/a/app/record/dataset/DS_4582dc809c"><i>Phlebotomus papatasi</i> M1 </a> now serves as the VectorBase reference for <i>Phlebotomus</i> species.  All functional genomics data will be aligned to this genome. </li>
    </ul>
    <br>
  
  <h4>Annotation Updates</h4>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_d4bf7041bd"><i>Haemaphysalis longicornis</i> HaeL-2018</a> &ndash; Pfam domain names were substituted for product descriptions for genes with assigned Pfam domains and uninformative product descriptions such as 'unspecified product'.  <a href="/a/app/workspace/strategies/import/50fc3bb4bef08b53">>12,000 genes</a> were given new product descriptions.  </li>
    </ul>
  </div>
  <br>

  <h4>RNA Sequence</h4>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_a91f666e84"><i>Aedes aegypti</i> female midgut 72-h bloodmeal digestion cycle for 11-time points </a>&ndash; Female midgut 72-h bloodmeal digestion cycle for 11-time points, pre- and post-bloodmeal, focusing on the first 24 h. Four (4) independent replicates. <a href="https://doi.org/10.1101/2023.03.03.531062" target="_blank">Tsujimoto and Adelman 2023</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_b0507f5adb"><i>Haemaphysalis longicornis</i> Engorged female from a stray domestic dog </a>&ndash; Transcriptome of an engorged female from a stray domestic dog without owner, found in Pingliang, Gansu, China. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31418165" target="_blank">Niu et al. 2019</a>.  </li>
    </ul>
  </div>
    <br>
  </div>  
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (√ò) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>

