---
layout: post
title: VectorBase 66 Released
category: [VectorBase,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 66.</h3>

<section>
  <h3>Release 66 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday Dec 14th, 2023 at 10 am Eastern for our Release 66 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 66. 
    <a href="https://attendee.gotowebinar.com/register/6066397511661949782" target="_blank">Register here</a>
  </div>
  <br>
</section>

<section>
  <h3>Contribute your expert knowledge!</h3>
  <div style="margin-left: 3em;">
    We invite all users to engage with Apollo, where you can actively edit and enhance your selected genes. Whether you're a seasoned user or a newcomer, our updated <a href="/a/app/static-content/apollo_help.html">Apollo landing page</a> provides a wealth of resources, including links to on-demand webinars, tutorials, and annotation guidelines. <br><br>
    Every edit you make contributes significantly! This open invitation is extended to all users, with a special emphasis on scientists studying groups of organisms that have recently been updated to integrate new reference genomes, and for the species that are now accessible on {{project}} for the first time. If you're unfamiliar or have any questions, don't hesitate to <a href="/a/app/contact-us">Contact Us</a>. We're here to help!
  </div>
</section>

<section>
  <h3>New features in {{project}} 66</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li>New options for downloading gene features are now available from any gene record page.  Click the Download Gene link at the top of the record page to use the new BED and FASTA download options. </li>
   </ul>
  </div>
</section>

<section>
  <h3>New data in {{project}} 66</h3>
  <div style="margin-left: 3em;">

  <h4>Genome Sequence and Annotation</h4>
  <div style="margin-left: 3em;">
    The following organisms are represented in VectorBase for the first time. 
    <ul>
      <li><a href="/a/app/record/dataset/DS_c099af5258"><i>Lutzomyia longipalpis</i> M1 </a>&ndash; A vector of <i>Leishmania</i> parasites that cause visceral leishmaniasis. A single male (M1) specimen of <i>Lutzomyia longipalpis</i>, derived from the Jacobina strain was sequenced using PacBio HiFi and Illumina HiC technologies. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_024334085.1" target="_blank">GCF_024334085.1</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/37043542" target="_blank">Labbé et al. 2023</a> </li>

      <li><a href="/a/app/record/dataset/DS_4582dc809c"><i>Phlebotomus papatasi</i> M1 </a>&ndash; A  vector of <i>Leishmania</i> parasites that cause cutaneous leishmaniasis. The genome of individual one male (M1) from the PPIS colony was sequenced by PacBio HiFi using the PacBio Ultralow input library prep, and additional material from the PPIS colony was used to generate HiC data for scaffolding the reads. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_024763615.1" target="_blank">GCF_024763615.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_2f761c02f0"><i>Anopheles marshallii</i> AmarGA1 </a>&ndash; This assembly is based on 51x PacBio HiFi data, 10X Genomics Chromium data, and Arima Hi-C data generated by the Anopheles Reference Genomes project. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_943734725.1" target="_blank">GCF_943734725.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_2652a3167a"><i>Sabethes cyaneus</i> ScyaPA1 </a>&ndash; This assembly is based on 37x PacBio HiFi data and Arima Hi-C data generated by the Anopheles Reference Genomes project. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_943734655.1" target="_blank">GCF_943734655.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_89dcd52262"><i>Anopheles nili</i> AnilCM1 </a>&ndash; This assembly is based on 48x PacBio HiFi data, 10X Genomics Chromium data, and Arima Hi-C data generated by the Anopheles Reference Genomes project. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_943737925.1" target="_blank">GCF_943737925.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_b56fe6b141"><i>Biomphalaria glabrata</i> XG47 </a>&ndash; This assembly is based on 29x PacBio data and Arima2 Hi-C data generated by the <a href="https://www.sanger.ac.uk/science/programmes/tree-of-life">Wellcome Sanger Institute Tree of Life Programme</a>. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_947242115.1" target="_blank">GCF_947242115.1</a>. </li>
    </ul>
  </div>
    <br>
  
  <h4>Genome Updates</h4>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_c3e414cbf3"><i>Glossina fuscipes</i> IAEA 2018 </a>&ndash; This data set was updated with the 2022 annotation version from the INSDC repository. <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCA_014805625.1" target="_blank">GCA_014805625.1</a>;  <a href="https://www.researchsquare.com/article/rs-1685795/v1" target="_blank">Saarman et al. 2022 (preprint)</a></li>
    </ul>
  </div>
    <br>

  <h4>Genome Reference and Strain Name Changes</h4>
  <div style="margin-left: 3em;">
    In response to community input we maintain the ongoing transition to <a href="https://docs.google.com/spreadsheets/d/1lqzR7jJTi_NbjTp4y4DpxAilir-ndlWnH-v9NyodGm8/edit?usp=sharing" target="_blank">VectorBase new reference genomes</a>.  VectorBase reference genomes are chosen based on diverse metrics such as assembly level (chromosomes or scaffolds and their quantities), BUSCO, and N50, and in consideration of the designation of genomes as reference at NCBI. All functional data, including RNAseq, will be aligned to the updated reference genomes. While the previous assemblies will persist on VectorBase indefinitely, their functionality will be limited. The new assemblies will encompass aligned omics datasets and comprehensive site functionality. 
    <br><br>
    A notification banner on the genome page of relevant species, will indicate these changes. If you have feedback or inquiries, require assistance, or encounter difficulty locating your genes of interest in the new genomes, please feel free to <a href="/a/app/contact-us">Contact Us</a>.
    <br>
    <ul>
      <li><a href="/a/app/record/dataset/DS_c099af5258"><i>Lutzomyia longipalpis</i> M1 </a> now serves as the VectorBase reference for <i>Lutzomyia longipalpis</i> strains.  All functional genomics data will be aligned to this genome.</li>

      <li><a href="/a/app/record/dataset/DS_13ae5590c9"><i>Anopheles darlingi</i> AdarGF1 </a> &ndash; The strain name was changed from idAnoDarlMG-H_01 to AdarGF1 beginning with VectorBase 66.  This organism now serves as the VectorBase reference for <i>Anopheles darlingi</i> strains.  All functional genomics data will be aligned to this genome.</li>

      <li><a href="/a/app/record/dataset/DS_4582dc809c"><i>Phlebotomus papatasi</i> M1 </a> now serves as the VectorBase reference for <i>Phlebotomus</i> species.  All functional genomics data will be aligned to this genome. </li>
    </ul>
  </div>
    <br>
  
  <h4>Annotation Updates</h4>
  <div style="margin-left: 3em;">
    <a href="/a/app/record/dataset/TMPTX_cquiJHB"><i>Culex quinquefasciatus</i> JHB 2020</a> &ndash; The genome annotation for <i>Culex quinquefasciatus</i> JHB 2020 has undergone significant revisions. VectorBase users contributed changes through Apollo, which were subsequently reviewed by the VEuPathDB annotation staff and integrated into a new gene set, with the structural annotation version transitioning from Dec 04, 2020, to Jul 17, 2023.  <br><br>

    A total of 425 transcripts corresponding to <a href="/a/app/workspace/strategies/327380103/432110503">391 genes</a> were updated, including both structural (intron/exon boundaries) and functional (name/symbol and product description/putative function) annotations. The alterations encompass the creation of new genes, modifications to existing ones, deletions, merges, splits, and rectifications to ensure accurate gene annotation.<br><br>

    Attributions for these modifications go to Theresa Nicole Menna (163 Ionotropic and 53 gustatory receptors), Rong Guo (28 odorant binding proteins) from the Megan Fritz lab at the University of Maryland, and Vitor L. Dos Anjos (181 Odorant receptors) from the Carolyn (Lindy) McBride lab at Princeton University. The distribution of these changes is shown in per event type. <br><br>

    <img style="width:50em" src="{{'/assets/images/news/Culex_2023.png' | absolute_url}}" />
  </div>
  <br>

  <h4>RNA Sequence</h4>
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_0cdc83e0d4"><i>Haemaphysalis longicornis</i> treated with tetracycline and control </a>&ndash; Two days after tetracycline treatment, nymphs were collected and divided into three biological replicates with 13 ticks in each group. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34525331" target="_blank">Zhong et al. 2021</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_dd1df35b51"><i>Ixodes persulcatus</i> transcriptome </a>&ndash; RNA-Seq of Ixodes persulcatus naturally fed on patients. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35250897" target="_blank">Li et al. 2021</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_6225722d7e"><i>Lutzomyia longipalpis</i> Adult females (sugar, blood, and blood with Le. infantum) sampled at different time intervals (6, 24, and 144 hours), plus larvae samples </a>&ndash; Adult females feed with sugar. Adult females sampled at 6h, 24h, and 144h after a meal with blood or blood with Leishmania infantum. L4 larvae. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37043542" target="_blank">Labbé et al. 2023</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_1febdaa7f4"><i>Phlebotomus papatasi</i> larvae+pupae, males and females (sugar), and females (blood, and blood with Le. major sampled at 6, 24, and 96 hours) </a>&ndash; Transcriptome of Larvae and Pupae. Adult males and females feed sugar. Adult females feed blood and blood with Le. major sampled at 6, 24, and 96 hours. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37043542" target="_blank">Labbé et al. 2023</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_46552c5299"><i>Rhipicephalus microplus</i> Salivary glands of adult females with and without <i>Theileria equi</i> infection </a>&ndash; Female salivary glands, with and without <i>Theileria equi</i> infection. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33557100" target="_blank">Paulino et al. 2021</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_6597d7c577"><i>Rhipicephalus sanguineus</i> sensu lato larvae (Putignano strain) </a>&ndash; <i>Rhipicephalus sanguineus</i> larvae transcriptome. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26921517" target="_blank">De Marco et al. 2017</a>.  </li>
    </ul>
  </div>

  <h4>Population Biology</h4>
  <div style="margin-left: 3em;">
    All the datasets listed below are available in the context of 700+ other studies in the <a href="/a/app/workspace/maps/DS_480c976ef9/new">Interactive MapVEu Tool (beta)</a>.
    <ul>
      <li><a href="/a/app/record/dataset/DS_393f456fe0">Distribution of ticks in the Western Palearctic: an updated systematic review (2015-2021)</a> &ndash; The PubMed and Web of Science databases were searched for peer-reviewed papers documenting the distribution of ticks that were published between 2015 and 2021, using the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37095583" target="_blank">Noll et al. 2023</a>, <a href="https://vectorbase.org/vectorbase/app/workspace/maps/BM5aKJJ/import">View on MapVEu</a>. </li>

      <li><a href="/a/app/record/dataset/DS_44d533319c">Microsatellite-based analysis of Aedes aegypti (Diptera':' Culicidae) populations from the Kingdom of Saudi Arabia reveal forest ancestry outside Africa</a> &ndash; <i>Aedes aegypti</i> were collected from the southwestern highlands of Saudi Arabia on the border of Yemen to the northwestern in Madinah between 2019 and 2022. Samples from other countries, including Thailand (Southeast Asia) and Uganda (East Africa), were also included. Using 17 microsatellite markers. <a href="https://vectorbase.org/vectorbase/app/workspace/maps/eyQehbT/import">View on MapVEu</a>.</li>

      <li><a href="/a/app/record/dataset/DS_116198b591">Mosquito surveillance in Salt Lake County, Utah, USA. 2019</a> &ndash; Mosquito surveillance from the Salt Lake City Mosquito Abatement District program to survey mosquito populations. <a href="https://vectorbase.org/vectorbase/app/workspace/maps/75ML4HR/import">View Salt Lake Co. 2019-2022 on MapVEu</a>.</li>

      <li><a href="/a/app/record/dataset/DS_2c77428cb1">Mosquito surveillance in Salt Lake County, Utah, USA. 2020</a> &ndash; Mosquito surveillance from the Salt Lake City Mosquito Abatement District program to survey mosquito populations. <a href="https://vectorbase.org/vectorbase/app/workspace/maps/75ML4HR/import">View Salt Lake Co. 2019-2022 on MapVEu</a>. </li>

      <li><a href="/a/app/record/dataset/DS_73062eff95">Mosquito surveillance in Salt Lake County, Utah, USA. 2021</a> &ndash; Mosquito surveillance from the Salt Lake City Mosquito Abatement District program to survey mosquito populations. <a href="https://vectorbase.org/vectorbase/app/workspace/maps/75ML4HR/import">View Salt Lake Co. 2019-2022 on MapVEu</a>.</li>

      <li><a href="/a/app/record/dataset/DS_1e81493073">Mosquito surveillance in Salt Lake County, Utah, USA. 2022</a> &ndash; Mosquito surveillance from the Salt Lake City Mosquito Abatement District program to survey mosquito populations. <a href="https://vectorbase.org/vectorbase/app/workspace/maps/75ML4HR/import">View Salt Lake Co. 2019-2022 on MapVEu</a>.</li>
    </ul>
  </div>
    <br>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>

