---
layout: post
title: VectorBase 68 Released
category: [VectorBase,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 68.</h3>

<section>
  <h3>Workshops</h3>
  <div style="margin-left: 3em;">
    We have four upcoming in-person workshops scheduled for May, June, and July 2024; two still have applications open. <a href="/a/app/static-content/workshops.html">Details</a></div>
</section>

<section>
  <h3>Release 68 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday May 23rd, 2024 at 10 am Eastern for our Release 68 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 68. 
    <a href="https://attendee.gotowebinar.com/register/362861265556682325" target="_blank">Register here</a>
  </div>
  <br>
</section>

<section>
  <h3>New features in {{project}} 68</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li>Long gene product descriptions are now trucated in relation to the width of your browser window.  The full product description is available on mouseover and individual transcript product descriptions are available in the All Product Descriptions table on gene pages. <a target="_blank" href="{{'/assets/images/news/Truncated_descriptions.gif' | absolute_url}}">Truncated descriptions GIF</a>. </li>

      <li>The Orthologs and Paralogs table on {{project}} gene pages also serves as a Clustal Omega tool for retrieving alignments between orthologs. This table now includes 'Selection Count' and 'Selection Clear' options for improved tool function. <a target="_blank" href="{{'/assets/images/news/Clustal_Omega_Counts.gif' | absolute_url}}">Clustal Omega Counts GIF</a>.   </li>

      <li>The Send To tool is now offered in VEuPathDB.org Strategies. Results created in VEuPathDB.org Strategies can now be ported to {{project}} My Strategies for further investigation. Only genes present in {{project}} species will be included in the ported strategy results. <a target="_blank" href="{{'/assets/images/news/VEuPath_Port_to_Component.gif' | absolute_url}}">VEuPathDB.org Send To GIF</a>.</li>

      <li>Newly sorted strategy result tables will now default to page 1 &ndash; Search result tables can be sorted based on any column of data. When a table is sorted while viewing page 3, users are returned to page 1 to view the sorted data table. </li>
   </ul>
  </div>
</section>

<section>
  <h3>New data in {{project}} 68</h3>
  <div style="margin-left: 3em;">

  <h4>Species Complex</h4>
  <a href="https://vectorbase.org/vectorbase/app/record/dataset/TMPTX_acouAcouGA1"><i>Anopheles coustani</i> AcouGA1</a> and <a href="/a/app/record/dataset/TMPTX_azieAzieGA1"><i>Anopheles ziemanni</i> AzieGA1</a> genomes are part of a <b>species complex</b>. Based on the current evidence, <i>An. ziemanni</i> AzieGA1 may actually be the species <i>An. coustani</i> or even another species from this complex. We have banners on each gene page for both of these genomes. <a href="/a/app/static-content/aziemani.html">Additional details</a>
  <br><br>

  <h4>New Genome Sequence and Annotation</h4>
  The following organisms are represented in VectorBase for the first time.
  <div style="margin-left: 3em;"> 
    <ul>
      <li><a href="/a/app/record/dataset/DS_c8bd8b61dc"><i>Anopheles gambiae</i> Ifakara</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/pubmed/37424773" target="_blank">Habtewold et al. 2023</a>, <a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_943734735.2/" target="_blank">GCF_943734735.2</a>. 
        <ul>
          <li>This <i>Anopheles</i> genome has two public annotations available &ndash; NCBI RefSeq (<a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_943734735.2" target="_blank">GCF_943734735.2</a>) and one from Ensembl genebuild.  VectorBase integrated the RefSeq genome sequence and annotation while the Ensemble genebuild annotation is available as a JBrowse track of <a target="_blank" href="{{'/assets/images/news/Ensembl-gene-build.gif' | absolute_url}}">Ensembl annotation GIF</a>.  </li>
        </ul>
      </li>

      <li><a href="/a/app/record/dataset/DS_ab65495970"><i>Musca domestica</i> aabys 2023</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/pubmed/37608635" target="_blank">Meisel et al. 2023</a>, <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_030504385.1" target="_blank">GCF_030504385.1</a>. </li>

      <li><a href="/a/app/record/dataset/DS_b1db660eb0"><i>Phlebotomus argentipes</i> India</a> &ndash;  <a href="https://www.ncbi.nlm.nih.gov/data-hub/genome/GCF_947086385.1" target="_blank">GCF_947086385.1</a> </li>
    </ul>
    <br>
  </div>

  <h4>Genome Reference Changes</h4>
  When genomes from multiple strains of an organism are integrated into {{project}}, one strain is chosen to serve as the {{project}} reference to which all functional genomics data are aligned. The following reference changes were made in Release 68.
  <br><br>
  <div style="margin-left: 3em;">
    <style>
      #GenRefChng_Bld68 table, #GenRefChng_Bld68 td, #GenRefChng_Bld68 th, #GenRefChng_Bld68 tr {
      text-align : center;
      padding-left: 5px;
      padding-right: 5px;
      padding-top: 5px;
      padding-bottom: 5px;
      border: 1px solid black;
      }
      #GenRefChng_Bld68 {
      margin-left : 5 em;
      }
    </style>
    <table id="GenRefChng_Bld68">
      <tr>
        <th>New Reference</th>
        <th>Previous Reference</th>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on VectorBase."><a href="/a/app/record/dataset/DS_b56fe6b141"><i>Biomphalaria glabrata </i>XG47</a></td>
          <td><i>Biomphalaria glabrata </i>BB02</td>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on VectorBase."><a href="/a/app/record/dataset/DS_c3e414cbf3"><i>Glossina fuscipes </i>IAEA 2018</a></td>
        <td><i>Glossina fuscipes </i>IAEA</td>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on VectorBase."><a href="/a/app/record/dataset/DS_e7fe24aea7"><i>Anopheles sinensis </i>China</a></td>
        <td><i>Anopheles sinensis </i>SINENSIS</td>
      </tr>
    </table>
  </div>
  <br>

  To aid in the transition to a new reference genomes, we have integrated "LiftOff" tracks into the genome browser (JBrowse) and the gene manual annotation tool <a href="/a/app/static-content/apollo_help.html">Apollo</a>. The workflow to create these tracks and an explanation of their interpretation are described in the <a href="/a/app/static-content/methods.html">VectorBase Methods</a>.The existing tracks were released previously, but new funtionality now displays different colors for each transcript with a color key explanation. Additional tracks are anticipated in the future.   <a target="_blank" href="{{'/assets/images/news/Liftoff.gif' | absolute_url}}">Liftoff GIF</a>
  <br><br>

  <div style="margin-left: 3em;">
    <style>
      #Liftoff_Bld68 table, #Liftoff_Bld68 td, #Liftoff_Bld68 th, #Liftoff_Bld68 tr {
      text-align : center;
      padding-left: 5px;
      padding-right: 5px;
      padding-top: 5px;
      padding-bottom: 5px;
      border: 1px solid black;
      }
      #Liftoff_Bld68 {
      margin-left : 5 em;
      }
    </style>
    <table id="Liftoff_Bld68">
      <tr>
        <th>Genome</th>
        <th>Track Name</th>
        <th>Genome Browser Link</th>
      </tr>
      <tr>
        <td><i>Anopheles funestus</i> AfunGA1</td>
        <td><a href="/a/app/record/dataset/DS_cb12e6264c">Liftoff gene models of FUMOZ to AfunGA1</a></td>
        <td><a href="/a/app/jbrowse?data=%2Fa%2Fservice%2Fjbrowse%2Ftracks%2FafunAfunGA1&loc=AfunGA1_X%3A10796001..11454500&tracks=gene%2CafunidAnoFuneDA416_04_FUMOZ_liftoff_phase_corrected_GFF_RSRC&highlight=" target="_blank">JBrowse</a></td>
      </tr>
      <tr>
        <td><i>Lutzomyia longipalpis</i> M1</td>
        <td><a href="/a/app/record/dataset/DS_82ca3afe7e">Liftoff gene models of LlonJ1.6 to llonM1
        </a></td>
        <td><a href="/a/app/jbrowse?data=%2Fa%2Fservice%2Fjbrowse%2Ftracks%2FllonM1&loc=LlonM1_1%3A5301..137000&tracks=gene%2CllonM1_LlonJ16_liftoff_phase_corrected_GFF_RSRC&highlight=" target="_blank">JBrowse</a></td>
      </tr>
      <tr>
        <td><i>Phlebotomus papatasi </i> M1</td>
        <td><a href="/a/app/record/dataset/DS_11fc348fc4">Liftoff Gene models of PpapI1.6 to ppapM1</a>  </td>
        <td> <a href="/a/app/jbrowse?data=%2Fa%2Fservice%2Fjbrowse%2Ftracks%2FppapM1&loc=PpapM1_2%3A11856101..11987800&tracks=gene%2CppapM1_PpapI16_liftoff_phase_corrected_GFF_RSRC&highlight=" target="_blank">JBrowse</a></td>
      </tr>
    </table>
  </div>
  <br>
  
  <h4>Gene ID Updates</h4>
  For the following genomes, the gene ID formats have changed as indicated by the examples in the table below.
  <br><br>
  <div style="margin-left: 3em;">
    <style>
      #GenIdChange_Bld68 table, #GenIdChange_Bld68 td, #GenIdChange_Bld68 th, #GenIdChange_Bld68 tr {
      text-align : center;
      padding-left: 5px;
      padding-right: 5px;
      padding-top: 5px;
      padding-bottom: 5px;
      border: 1px solid black;
      }
      #GenIdChange_Bld68 {
      margin-left : 5 em;
      }
    </style>
    <table id="GenIdChange_Bld68">
      <tr>
        <th>Organism</th>
        <th>Old IDs</th>
        <th>New IDs</th>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on VectorBase."><a href="/a/app/record/dataset/DS_c15b4dcf4a"><i>Anopheles cruzii</i> AcruBR1 </a></td>
        <td>AcruBR1_000000001</td>
        <td>ACRUBR1_000001</td>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on VectorBase."><a href="/a/app/record/dataset/DS_2f761c02f0"><i>Anopheles marshallii</i> AmarGA1 </a></td>
        <td>AmarGA1_000001</td>
        <td>AMARGA1_000001</td>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on VectorBase."><a href="/a/app/record/dataset/DS_4a4d01e08e"><i>Anopheles moucheti</i> AmouCM1 </a></td>
        <td>AmouM1_0C00001</td>
        <td>AMOUCM1_000001</td>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on VectorBase."><a href="/a/app/record/dataset/TMPTX_anilAnilCM1"><i>Anopheles nili</i> AnilCM1 </a></td>
        <td>AnilCM1_037063</td>
        <td>ANILCM1_037063</td>
      </tr>
    </table>
  </div>
  <br>

  <h4>RNA Sequence</h4>
  <div style="margin-left: 3em;">
    <i>Anopheles sinensis </i>China
    <ul>
      <li><a href="/a/app/record/dataset/DS_e7d067e894">Sialotranscriptome: Female adult fed 10% glucose for salivary gland dissection</a> &ndash; Female adult mosquitoes were fed with 10% glucose solution and collected at 3-5 days post-emergence for salivary gland dissection. A total of 180 salivary glands were collected (one biological sample). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27996203" target="_blank">Fang et al. 2018</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_dae97d02f6">Male and female adults from a resistant strain</a> &ndash; Male and female adults from a resistant strain (one biological replicate). <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA339810" target="_blank">PRJNA339810</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_cd46db2ec9">Pooled male and female 30 days old</a> &ndash; Pooled male and female 30 days old, resistant vs susceptible (one biological replicate). <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA293400" target="_blank">PRJNA293400</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_b142834045"> Whole body from a 2 - 3 day old female mosquito</a> &ndash; Mosquitoes were reared under insectary in screened cages at 28 ± 2 °C, 70-75% humidity. Adult mosquitoes were fed with glucose solution and water-soaked sponges. After eclosion, adult female samples were pooled (one biological sample). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29530087" target="_blank">Feng et al. 2018</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_afe17e71d7">Pooled eggs, larvae, pupae, male and female adults</a> &ndash; Ten percent, 30%, 20%, 20%, and 20% of total RNA quantity separately from eggs, larvae, pupae, male adults, and female adults RNA samples were pooled together for cDNA preparation (one biological sample). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25000941" target="_blank">Chen et al. 2014</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_83c561e015">Antenna transcriptome pre- and post- blood meal</a> &ndash; Transcriptome analysis of the antennae of Anopheles sinensis which were kept either on sugar diet or five hours after blood feeding (three biological replicates each treatment). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28715466" target="_blank">Chen et al. 2017</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_588c80c46a">Pooled male and female larvae, pupae, and adults</a> &ndash; Produced from pooled male and female larvae, pupae, and adults (three An. sinensis samples). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25554792" target="_blank">Neafsey et al. 2015</a>.  </li>
    </ul>
    <br>

    <i>Biomphalaria glabrata </i>XG47
    <ul>
      <li><a href="/a/app/record/dataset/DS_cf5f2a3968">Long-term Schistosoma mansoni infection</a> &ndash; Snails developing schistosome parasite during long term (3,5,8 week) infection (one biological sample). <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA285250" target="_blank">PRJNA285250</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_a79c78b5c0">Embryonic and larval stages mixed</a> &ndash; Early cleavage stage embryos up through to advanced veliger larval stages until the point before hatching (one biological sample). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27793108" target="_blank">Kenny et al. 2016</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_94a1ad8c0f">Adult tentacles</a> &ndash; Adult tentacles (one biological sample). <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA282347" target="_blank">PRJNA282347</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_667b2d2a79">Inbred lines homozygous for three PTC2 alleles (R, S1, and S2) impacting susceptibility (S) and resistance (R) to S. mansoni PR-1</a> &ndash; When exposed to Schistosoma mansoni PR-1, parasite resistance in Biomphalaria glabrata 13–16-R1 depends on genotype at the PTC2 genomic region, as determined by a resistance allele (R) and two susceptibility alleles (S1 and S2). RNA-Seq data were generated from three inbred lines homozygous for these alleles: i68 (R), i172 (S1), and i4 (S2) (one biological sample for each treatment). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32845238" target="_blank">Tennessen et al 2020</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_5cd2da696f">Twelve (12) different tissues/organs from adults</a> &ndash; Twelve different tissues/organs were dissected from several individual adult BB02 B. glabrata snails (one biological replicate for each). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28508897" target="_blank">Adema et al. 2017</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_3316717fc5">Treatments (stranded RNASeq): Response to S. mansoni infection in resistance-selected snails</a> &ndash; <i>B. glabrata</i> and <i>S. mansoni</i> were collected on the island of Guadeloupe, West Indies. We selected for resistance by challenging snails with S. mansoni miracidia. Fine-mapping and RNA-Seq characterization reveal a region, the Guadeloupe Resistance Complex (GRC). We perform RNAseq on challenged and unchallenged reesistance-seleceted snails of various GRC genotypes. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25775214" target="_blank">Tennessen et al. 2015</a>.  </li>

      <li><a href="/a/app/record/dataset/DS_1d610fb8d6">Controls (unstranded RNASeq): Response to S. mansoni infection in resistance-selected snails</a> &ndash; <i>B. glabrata</i> and <i>S. mansoni</i> were collected on the island of Guadeloupe, West Indies. We selected for resistance by challenging snails with S. mansoni miracidia. Fine-mapping and RNA-Seq characterization reveal a region, the Guadeloupe Resistance Complex (GRC). We perform RNAseq on challenged and unchallenged reesistance-seleceted snails of various GRC genotypes. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25775214" target="_blank">Tennessen et al. 2015</a>.  </li>
    </ul>    
    <br>

    <i>Culicoides sonorensis</i> PIR-s-3
    <ul>
      <li><a href="/a/app/record/dataset/DS_3a8f35b31c">KC cell line under 11 different conditions</a> &ndash; <i>C. sonorensis</i> KC cell line under 11 different conditions: Prostagalndin E1, Prostaglandin A1, 20 Hydroxyecdysome, <i>Staphylococcus aureus</i> Lipoteichoic acid, <i>E. coli</i> Lipopolysaccharide, Polyinosinic:polycytidylic acid, Insulin, Transcription growth factor β1, fMLP, heat-shock at 36 C and a control (one biological replicate). <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJEB7006" target="_blank">PRJEB7006</a>.</li>
    </ul>
    <br>

    <i>Glossina fuscipes</i> IAEA 2018
    <ul>
      <li><a href="/a/app/record/dataset/DS_86bd0837c0">Reproductive tissues infection with endosymbiotic Spiroplasma</a> &ndash; Field captured (Uganda) Spiroplasma infected and uninfected male (testes, accessory glands, and ejaculatory duct) and female (ovaries, spermathecae, uterus, and oocysts) reproductive tissues. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34529715" target="_blank">Son et al. 2021</a>.</li>

      <li><a href="/a/app/record/dataset/DS_56d05f13e1">Generic RNA-Seq for gene prediction</a> &ndash;  <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA176015" target="_blank">PRJNA176015</a>.</li>
    </ul>
    <br>

    <i>Pediculus humanus</i> USDA
    <ul>
      <li><a href="/a/app/record/dataset/DS_c5e6a00365">Female accessory gland and ovary on day 0 and day 5</a> &ndash; Tissue-specific gene expression in the reproductive system of female head louse, spatial expression (accessory gland and ovary) temporal expression (day 0 and day 5) (one biological replicate). <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA694820" target="_blank">PRJNA694820</a>.</li>
    </ul>
    <br>

    <i>Rhipicephalus sanguineus</i> Rsan-2018
    <ul>
      <li><a href="/a/app/record/dataset/DS_6086c77397">Tick Cell Line RSE8 RNA-Seq</a> &ndash; Unpublished transcriptome sequencing of <i>Rhipicephalus sanguineus</i> RSE8 cell line. <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA238793" target="_blank">PRJNA238793</a>, <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA238794" target="_blank">PRJNA238794</a>, <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA238795" target="_blank">PRJNA238795</a>, <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA238796" target="_blank">PRJNA238796</a>.

      </li>
    </ul>
    <br>
  </div>

  <h4>Population Biology</h4>
  All the datasets listed below are available in the context of 700+ other studies in the <a href="/a/app/workspace/maps/DS_480c976ef9/new">Interactive MapVEu Tool (beta)</a>.
  <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_668368fe1c">Biting behavior of malaria vectors in remote communities of the Loreto region</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/pubmed/33511394" target="_blank">Morales Viteri et al. 2021</a>, <a href="/a/app/workspace/maps/kvcsgBW/import">VBP0000852 on MapVEu</a>.</li>

      <li><a href="/a/app/record/dataset/DS_f0845f0177">Dispersion of malaria vectors in a community near the city of Iquitos 2021</a> &ndash; <a href="/a/app/workspace/maps/64EHzo4/import">VBP0000856 on MapVEu</a>.</li>

      <li><a href="/a/app/record/dataset/DS_c30ff1c7b2">Fine-scale spatial distribution of deltamethrin resistance and population structure of <i>Anopheles funestus</i> and <i>Anopheles arabiensis</i> populations in Southern Mozambique</a> &ndash; Mosquitoes were collected from 11 towns in southern Mozambique, as well as from different neighbourhoods within the town of Palmeira, during the peak malaria transmission season in 2016. CDC bottle bioassay data is available in this study. <a href="http://www.ncbi.nlm.nih.gov/pubmed/36915131" target="_blank">Das et al. 2023</a>, <a href="/a/app/workspace/maps/WCNlBZZ/import">VBP0000850 on MapVEu</a>.</li> 


      <li><a href="/a/app/record/dataset/DS_c80afaadb4">Malaria vectors and their behavior during the rising and falling seasons in the community of Santa Emilia</a> &ndash; Collections were carried out during the rising season in April and the falling season in October. Three nights of collection were conducted using the HLC method to observe intra-peri bite patterns during a 12-hour collection period. <a href="/a/app/workspace/maps/R6Ia5QW/import">VBP0000854 on MapVEu</a>.</li>

      <li><a href="/a/app/record/dataset/DS_0ae0be44a6">Molecular and entomological surveillance of malaria vectors in urban and rural communities of Benguela Province, Angola</a> &ndash; Adult mosquitoes were collected indoors using CDC light traps from nine randomly selected households at two sentinel sites with distinct ecological characteristics. The samples were collected from 1 February to 30 June 2022. Pathogen data, looking at the presence of P. falciparum and blood meal source data is available for this study in VectorBase <a href="http://www.ncbi.nlm.nih.gov/pubmed/38448968" target="_blank">Alves et al. 2024</a>, <a href="/a/app/workspace/maps/aKk8taM/import">VBP0000851 on MapVEu</a>.</li>

      <li><a href="/a/app/record/dataset/DS_4d62da798f">Mosquito Surveillance For St. Joseph County, IN 2023</a> &ndash; <a href="/a/app/workspace/maps/B0qR4f3/import">VBP0000858 on MapVEu</a>.</li>
    </ul>
  </div>
    <br>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>