---
layout: plain
title: Data submission information
permalink: /dataSubmission
tags: [dataSubmission]
---
<h1>How to submit data to VEuPathDB</h1>

<div class="static-content">
    
  <p>The Eukaryotic Pathogen, Vector & Host Informatics Resources (https://VEuPathDB.org)
    is a Bioinformatics Resource Center (BRC) operated under contract from the US National Institute of Allergy and 
    Infectious Diseases (NIAID) and the Wellcome Trust. VEuPathDB is charged with ensuring that genomic (and other 
    large-scale) datasets pertaining to supported pathogens are conveniently accessible to the worldwide community of 
    biomedical researchers. This document summarizes policies associated with releasing datasets on VEuPathDB. 
    Our goal is to help the communities that we serve ensure that their data are FAIR, Findable, Accessible, 
    Interoperable and Reusable.
  </p>
  
  <p>VEuPathDB welcomes submissions of genomic-scale data concerning eukaryotic microbes, fungi, vectors of human disease, 
    and host-pathogen interactions. The VEuPathDB contract from NIAID provides support for biosecurity pathogens, including 
    <i>Babesia</i>, <i>Cryptosporidium</i>, <i>Entamoeba</i>, <i>Giardia</i>, <i>Microsporidi</i>a (various genera), 
    <i>Toxoplasma</i>, <i>Plasmodium</i>, and related taxa (<i>Acanthamoeba</i>, <i>Gregarina</i>, <i>Neospora</i>, 
    <i>Theileria</i>) and also arthropod vectors (ticks, mites, mosquitoes, kissing bugs, tsetse flies, sand flies, 
    lice, etc.) of human disease, as well as a snail that serves as an intermediate host, and comparator species. 
    Support for kinetoplastid parasites (<i>Crithidia</i>, <i>Endotrypanum</i>, <i>Leishmania</i>, <i>Trypanosoma</i>) 
    is provided by The Wellcome Trust. The FungiDB project encompasses a large (and growing) number of species supported 
    by both NIAID and the Wellcome Trust. Please <a href="/a/app/contact-us">contact us</a> if you have data from other species that should be 
    incorporated into VEuPathDB! <b>Please review our
    <a href="{{ "/documents/VEuPathDB_Data_Sub_Release_policy_rev_02April2020.pdf" | absolute_url }}">Data Submission Policy</a></b>.
  </p>
  
  <p>Our most common data types include transcriptomics, proteomics, metabolomics, epigenomics, population-level and isolate 
     information. In one form or another, VEuPathDB currently represents datasets in the following categories:
  </p>
  <ul>
    <li>Sequence (genomic [nuclear and organellar])</li>
    <li>ESTs and RNA-seq, generated on various platforms)</li>
    <li>Host-response data</li>
    <li>Comparative genomic information</li>
    <li>DNA polymorphism and population genetics data</li> 
    <li>Sequences and metadata pertaining to field and clinical isolates and collections (with geo-spatiotemporal and other metadata)</li> 
    <li>Chromatin modification data (ChIP-chip and ChIP-seq)</li> 
    <li>Manually curated and automatically generated gene models and other annotation (GO terms, InterPro domains, etc.)</li>
    <li>Transcript and proteomic profiling</li>
    <li>Host response data sets (multiple platforms)</li>
    <li>Interactome data</li>
    <li>Protein structural information</li>
    <li>Metabolic pathways and metabolomics data</li>
    <li>Phenotype information, reagents (clones, antibodies, etc.)</li>
    <li>Publication references</li>
    <li>Image data, etc.</li> 
  </ul>

  <p>We also accept other genomic-scale data and are open to suggestions. Use the <a href="/a/app/contact-us">Contact Us</a> link to make suggestions. We look 
   forward to working with you!
  </p>
  
  <h3>How to submit data for integration in VEuPathDB</h3>

  <p>To submit your data for integration, fill out the appropriate VEuPathDB Dataset nomination form listed below. If your data 
    cannot be submitted via our forms, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
    link to send a brief description (two or three sentences) of your data.
  </p>

  <p>Genomes & high throughput sequencing data (e.g. RNA-Seq, ChiP-Seq, isolates typed by Whole Genome Sequencing or by 
    sequencing limited genetic loci) must be available in The 
    International Nucleotide Sequence Database Collaboration (INSDC) such as NCBI GenBank, EMBL-EBI ENA or DDBJ.</p>
    Once the dataset is prioritised for loading, we will export the data directly from INSDC. Note: while genome sequences 
    must be available in INSDC, functional annotation (e.g. gene names, GO terms, etc.) can be submitted directly to VEuPathDB.
  </p>

  <p>Tell us about your data as early as possible, to allow ample time for scheduling into VEuPathDB release cycles. 
    Depending on the dataset type, we can provide instructions on how to transfer your data to us (e.g. formats of proteomics 
    datasets may differ depending on the nature and scale of the data to be transferred), or we may be able to facilitate data 
    submission to a repository (e.g. GenBank, GEO/ArrayExpress, etc.).
  </p>

  <p>For the gene manual annotations, the change of exon-intron boundaries and creation of new genes can be submitted via Apollo for all VectorBase genomes and some of the other VEuPathDB organisms too. The gene metadata (gene names/symbols and functional description) can be submitted by sending a spreadsheet file, follow this link (FAQ page coming soon!) for information about the 12 columns heading that are required. Before you submit a manuscript for publication, please inquire if we can generate new GeneIDs that can be linked out to within the publication; this process can take a few months, we strongly encourage you to please contact us as early as possible to allow ample time for us to process your data. Gene deletions are not handled via Apollo, please Contact Us with supporting evidence.
  Community annotations made via Apollo are now represented on gene pages in the Gene Models graphic. In addition, links from the gene page to Apollo are offered to facilitate community annotations. <a href="{{'/assets/images/51-apollo-news.png' | absolute_url}}">Example image.</a>  Please see our <a href="https://youtu.be/RMXZDBMh8Fo" target="_blank">Apollo webinar</a> for instruction. 
  </p>    

<!-- =========== Genome Sequence and Annotation =========== -->

<details>
  <summary><b>Genome Sequence and/or Annotation</b></summary>
  <p>Genomes must be available in The International Nucleotide Sequence Database Collaboration (INSDC) such as NCBI GenBank, 
        EMBL-EBI ENA or DDBJ.
  </p>
    <ul>
      <li>If your genome <b>IS</b> uploaded to a repository, complete the <a href="https://docs.google.com/forms/d/e/1FAIpQLSchN8HUyZHP6v8B505y42YZV4Elb_vly1NUgToKhyz-kUjf9g/viewform">
          Genome Sequence and/or Annotation Description Form</a> making sure to include the accession numbers of your data 
          when prompted. We will download your data from the repository.
      </li>
      <li>If the annotation file <b>is not</b> uploaded to a repository, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
          form to send us the genome annotation file only (e.g. gff file format).
      </li>
  </ul>
   <p></p>
</details>

<!-- =========== High Throughput or Next Generation Sequencing =========== -->
<details>
  <summary><b>High Throughput or Next Generation Sequencing</b> &#8211; RNA, DNA or ChIP Sequencing</summary>
  <p>We prefer to download the raw read data in FASTQ format from a sequence read archive. 
     We integrate the data into the database using the raw reads and use the raw reads during future 
     database releases to remap or update our analyses when necessary.
  </p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Upload your data to a sequence read archive such as the European Nucleotide Archive or 
          NCBI's Sequence Read Archive, and provide us the accession numbers. We will export the data directly from INSDC</li>
      <li>Complete the appropriate data description form making sure to enter your data archive accession numbers when
          prompted</li>
    </ul>
  <p>
    <a href="https://docs.google.com/a/apidb.org/forms/d/1f4UCA6tCaAuItKkh97Xg-K_mXz8Pic5SejejOlo09QY/edit?usp=sharing">RNA-Seq 
       Data Description Form</a>
  </p>
  <p><a href="https://docs.google.com/forms/d/1z4rrMp2kjJSkqz6EbwglF6oK9xp_0_VUz9AjfPg4FWc/viewform">DNA-Seq
        Data Description Form</a>
  </p>
  <p><a href="https://docs.google.com/forms/d/11Qt40E13XTmIUbdftNXVQ6lEr2VIvZPFc0__jrZILJc/viewform">ChIP-Sequencing
         Data Description Form</a>
  </p>
  <p></p>
</details>

<!-- =========== Microarray =========== -->
<details>
  <summary><b>Microarray</b></summary>
  <p>Transfer a copy of your data to VEuPathDB using one of these options:</p>
    <ul>
      <li>Upload your data to a repository such as GeneExpression Omnibus.
          Complete the data description form linked below making sure to enter your data archive accession numbers when prompted.
          We will export the data directly from a repository.
      </li>
      <li>Send your data as an attachment to an email. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a>
          form to send us an email. Files (e.g. CEL, CSV) must include expression levels and probe set information.Pay special attention to clearly 
          indicate the identity of columns in the data files you transferred to VEuPathDB.
      </li>
   </ul>
  <p><a href="https://docs.google.com/a/apidb.org/forms/d/1pzD53hCY0rH7JYomUH8cjwJXJlOSrtZRc_mFowfFfU4/edit?usp=sharing">
        Microarray Data Description Form</a>
  </p>
  <p></p>
</details>

<!-- =========== Proteomics =========== -->
<details>
  <summary><b>Proteomics</b></summary>
  <p>Excel or tab delimited text files are preferred. We can accommodate xml file format. Required columns include gene IDs, 
     peptide sequences, peptide counts and scores.</p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Send your data as an attachment to an email. Use the 
        <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
           form to send us an email.
      </li>
      <li>Complete the 
        <a href="https://docs.google.com/a/apidb.org/forms/d/1Yx9qGKDyCf2Wm5lnf1-dFHArdQAQ9tVbR8gvxdY7n5A/edit?usp=sharing">
           Proteomics Data Description Form</a> making sure to clearly indicate the content of each column in your file.
      </li>
  </ul>
  <p></p>
</details>

<!-- =========== Quantitative Proteomics =========== -->

<details>
  <summary><b>Quantitative Proteomics</b></summary>
  <p>Excel or tab delimited files are preferred. We can accommodate xml file format. Required columns include gene IDs and 
     scores.</p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Send your data as an attachment to an email. Use the 
        <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
           form to send us an email.
      </li>
      <li>Complete the 
        <a href="https://docs.google.com/a/apidb.org/forms/d/1jC1w4wWb_tNWPoL55F0zLq8o8rqOXQjtw27dt9lKups/viewform">Quantitative 
           Proteomics Data Description Form</a> making sure to include a description of data columns, for example, time course
           units and arrangement if not apparent from column headers.>
      </li>
    </ul>
  <p></p>
</details>

<!-- =========== ChIP-chip =========== -->

<details>
  <summary><b>ChIP-chip</b></summary>
  <p>Your data files should include expression levels and probe set information.</p>
  <p>Transfer a copy of your data to VEuPathDB using one of these options:</p>
    <ul>
      <li>Upload your data to a repository such as Gene Expression Omnibus and complete 
          the <a href="https://docs.google.com/forms/d/1E1QN9dKGc9rK_Bd3t-4ugIeVAyNXZoX1Z3bcSKEIMJc/viewform">ChIP-chip
          Data Description Form</a> making sure to enter the archive accession numbers (if any) for your data when prompted.
      </li>
      <li>Send your data as an attachment to an email. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact
          Us</span></a> form to send us an email.
      </li>
    </ul>
  <p></p>
</details>


<!-- =========== Isolates typed by =========== -->

<details>
  <summary><b>Isolates typed by sequencing limited genetic loci</b></summary>
  <p>If your data <b>ARE already</b> uploaded to the GenBank, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
     form to tell us about your data. Note: GeneBank Isolate records and the associated metadata are automatically
     updated with each VEuPathDB release. There is no need to complete our Isolate Submission Form.</p>
   
   <p>If your data <b>ARE NOT</b> alredady uploaded to the GenBank, we can facilitate this upload. Complete the Isolate 
      Submission Form linked below and we will use the information to generate a GenBank submission for your isolates. 
      The new isolate records will be downloaded to VEuPathDB when the records are released by the GenBank. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
      form to send us instructions for retrieving your data.</p>

   <p><a href="https://veupathdb.org/tutorials/IsolateSubmissionEuPathDBv8.xls">Isolate Submission Form</a></p>
   <p><a href="https://veupathdb.org/tutorials/EuPathDB_Isolatesubmission_genericHelp.pdf">Help for submitting Isolate Data</a></p>

</details>

<!-- =========== Isolates or Strains =========== -->

<details>
  <summary><b>Isolates or Strains typed by High-Throughput Sequencing</b></summary>

  <p>We prefer to receive the raw read data in FASTQ or FASTA file format. We integrate your data into the database using 
     the raw reads. We also use the raw reads during future database releases to remap your data when the reference genome is 
     reloaded and to update our analyses when needed.</p>

  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Upload your data to a sequence read archive such as DNA Data Bank of Japan, the European Nucleotide Archive or 
          NCBI's Sequence Read Archive. We will retrieve your data using the read archive's accession numbers for your data 
          set.
      </li>
      <li>Complete our <a href="https://docs.google.com/a/apidb.org/forms/d/1z4rrMp2kjJSkqz6EbwglF6oK9xp_0_VUz9AjfPg4FWc/viewform">  
          DNA Seq Data Description Form</a> making sure to enter the read archive accession numbers for your data when prompted.
          We also ask that you complete an abbreviated <a href="https://veupathdb.org/tutorials/IsolateSubmissionEuPathDBv8.xls">Abbreviated 
          Isolate Submission Form</a> to describe meta data associated with your isolates.
      </li>
    </ul>
  <p></p>
</details>

<!-- =========== Bulk user comments form =========== -->
<details>
  <summary><b>Bulk user comments form</b></summary>
  <p>Instructions: download the form, fill out required fields and provide any additional information, if available, and send it back to the email listed in the form</p>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/17Uu_KvfpU6FYvL0ftA7zSt1fK7OinDRZp6Dng_RPH7Y/edit?usp=sharing">
           Bulk user comments form</a>
      </li>
  <p></p>
</details>

<!-- =========== Phenotype collection form =========== -->
<details>
  <summary><b>Phenotype collection form</b></summary>
  <p>Instructions: download the form, fill out required fields and provide any additional information, if available, and send it back to the email listed in the form</p>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/1uhkr7q8QWL7ZpXLpFO3Wb5Vj9GHK47ylrzUXGbOYL9w/edit?usp=sharing">
           FungiDB phenotype collection form</a>
      </li>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/1mcoPimtWkL2nqH_5qdMerVuinIUPG0t19s_uEF8BHOQ/edit#gid=783150795">
           Parasite phenotype collection form</a>
      </li>
  <p></p>
</details>


<!-- =========== General Data Submission =========== -->
<div>
  <p><b>General Data Submission</b>
   &#8211; Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> form to tell us
   about data that does not fit any of the above categories
  </p>
</div>


</div>
