---
layout: plain
title: Data Submission and Release on VEuPathDB Databases
permalink: /dataSubmissionReleasePolicy
tags: [dataSubmissionReleasePolicy]
---
<h1>Data Submission and Release on VEuPathDB Databases</h1>
issued February 2010, most recent revision 20 April 2023

<div class="static-content">
    
  <p>The Eukaryotic Pathogen, Vector & Host Informatics Resources (<a href="https://VEuPathDB.org" target="_blank">https://VEuPathDB.org</a>)
    is a Bioinformatics Resource Center (BRC) operated under contract from the US National Institute of Allergy and 
    Infectious Diseases (NIAID) and the Wellcome Trust. VEuPathDB is charged with ensuring that genomic (and other 
    large-scale) datasets pertaining to supported pathogens are conveniently accessible to the worldwide community of 
    biomedical researchers. This document summarizes policies associated with releasing datasets on VEuPathDB. 
    Our goal is to help the communities that we serve ensure that their data are FAIR, Findable, Accessible, 
    Interoperable and Reusable.
  </p>
  
  <p><b>General principles:</b></p>
  <ul>
    <li><b>Data providers define the schedule for data release (in consultation with funders, publishers, etc).</b> While there is no point in providing VEuPathDB with data that will never become public, deposition does not in itself authorize immediate release. Data become accessible to the public only when the data providers and VEuPathDB staff agree that it is accurately represented and ready to go live. Note that knowledgebase staff are not active research scientists; they are distinct from researchers in the groups responsible for VEuPathDB, who see new data only when it becomes accessible to the general public.</li>
    <li><b>Data providers know their data best.</b> We expect to work with those who generated the underlying data to determine how best to analyze and represent new data types. This typically means taking in relatively raw data – often earlier, and in a more unprocessed form than the published dataset – and building an in-house analysis pipeline to ensure that all comparable datasets are handled similarly.</li>
    <li><b>The earlier we learn about new datasets, the easier it is to schedule timely release.</b> The nature of our knowledgebase production, and competing demands from the many communities we support, means that several months' notice are often required to prepare for release. Note that it is often possible to use a preliminary dataset for planning, which can be swapped for the final version before public release.</li>
    <li><b>Experience has shown that data not deposited prior to publication often fails to emerge at all!</b> After publication, it may be difficult to focus on tracking down the raw data, associated metadata, analysis methods, etc. It is never too early to discuss planned datasets with the VEuPathDB team!</li>
    <li><b>While not required, pre-publication data release often results in favorable attention from scientific colleagues (including journal editors and grant reviewers).</b> Note that all major scientific journals now agree that early release of genomic-scale datasets does not compromise publication.</li> 
    </ul>

 
  <h3>Why submit my data to VEuPathDB?</h3>
  <ul>
    <li>Inclusion in VEuPathDB facilitates your own analysis of the data, in the context of other genomic-scale experiments already available from researchers around the world.</li>
    <li>Electronic access permits others to analyze your data in greater depth than possible in print (even in advance of publication, if you wish to allow this).</li>
    <li>Availability within VEuPathDB keeps your data alive on a highly visible genomics knowledgebase resource: VEuPathDB is accessed by ~13,000 unique users each month.</li>
  </ul>
  
  <h3>How do I submit data to VEuPathDB?</h3>
  <ul>
    <li>Fill out the appropriate form to indicate the data availability</li>
    <li>Contact the VEuPathDB by clicking the ‘Contact Us’ link on any VEuPathDB page, or
      emailing us at <a href="https://veupathdb.org/veupathdb/app/contact-us" target="_blank"> help@VEuPathDB.org</a>.</li>
    <li>Tell us about your data as early as possible, to allow ample time for scheduling into
      VEuPathDB release cycles.</li>
    <li>Once you tell us about your data, we will provide instructions on how to transfer your
      data to us (formats may differ depending on the nature and scale of the data to be
      transferred).</li>
    <li>In order to avoid any confusion and ensure accuracy, we adhere to strict Standard
      Operating Procedures (SOPs), as outlined below.
      </li>
  </ul>  

  <h3>What data types are supported by VEuPathDB?</h3>
<p>In one form or another, VEuPathDB currently represents sequence (genomic [nuclear and organellar], ESTs and RNA-seq, generated on various platforms), host-response data, comparative genomic information, DNA polymorphism and population genomics data, sequences and metadata pertaining to field and clinical isolates and collections (with geo- spatiotemporal and other metadata), chromatin modification data (ChIP-chip and ChIP-seq), manually curated and automatically generated gene models and other annotation (GO terms, InterPro domains, etc.), transcript and proteomic profiling, host response data sets (multiple platforms), interactome data, protein structural information, metabolic pathways and metabolomics data, phenotype information, reagents (clones, antibodies, etc.), publication references, image data, etc. We can support additional data types as needed. <b>Please let us know if you have data to provide that is not currently supported!</b></p>

<h3>What species are supported by VEuPathDB?</h3>
<p>The VEuPathDB contract from NIAID provides support for biosecurity pathogens, including <i>Babesia</i>, <i>Cryptosporidium</i>, <i>Entamoeba</i>, <i>Giardia</i>, <i>Microsporidia</i> (various genera), <i>Toxoplasma</i>, <i>Plasmodium</i>, and related taxa (<i>Acanthamoeba</i>, <i>Gregarina</i>, <i>Neospora</i>, <i>Theileria</i>) and also arthropod vectors (ticks, mites, mosquitoes, kissing bugs, tsetse flies, sand flies, lice, etc.) of human disease, as well as a snail that serves as an intermediate host, and comparator species. Support for kinetoplastid parasites (<i>Crithidia</i>, <i>Endotrypanum</i>, <i>Leishmania</i>, <i>Trypanosoma</i>) is provided by The Wellcome Trust. The FungiDB project encompasses a large (and growing) number of species supported by both NIAID and the Wellcome Trust. <b>Please contact us if you have data from other species that should be incorporated into VEuPathDB!</b></p>

<h3>Data Management SOPs (Standard Operating Procedures) for VEuPathDB databases</h3>
<p>VEuPathDB routinely handles datasets provided prior to publication,in addition to those already in the public domain. In order to ensure timely and accurate data integration we strictly adhere to the following Standard Operating Procedures (SOPs):
</p>
<ul>
      <ol>
           <li><b>Datasets come to our attention in several ways, including:</b>
            <ul>
            <li>Direct contact from researchers generating the data (during the earliest stages of project design, as
              <ul>               
              <li>data is being produced, in the course of data analysis, or in the context of manuscript preparation).</li>
              <li>Information provided by members of the VEuPathDB or larger pathogen community.</li>
              <li>Information obtained by VEuPathDB staff at meetings and conferences.</li>
              <li>Publicly available information from the scientific literature, genomic dataset repositories, etc.</li>
              </ul>
            </li>
          </ul>
        </li>
      <br>
    <b>Note that VEuPathDB can often facilitate data deposition in the appropriate archival repositories (GenBank, SRA, GEO/ArrayExpress, etc.)</b> 
<br><br>
        <li><b>Decisions to include a dataset in VEuPathDB are based on value to the research community</b>. When prioritizing data for integration, we rely heavily on discussions with active researchers and the objectives and priorities of our funders. <b>Please contact us if you are interested in participating in these discussions.</b>
          <br><br>
        <li><b>Regardless of how we first learn about a given dataset, communication is established with the original data producer</b> through email, teleconference, and/or face-to-face meetings to discuss the desirability and feasibility of integration into VEuPathDB. In the course of these discussions, we consider what data are likely to be available, data formats and transfer protocols, questions the community may wish to ask of this data, and ways to represent or display such information. We also collect appropriate metadata (regarding samples, experimental protocols, etc.), and information on data sources, data providers, appropriate citation, etc. 
          <br><br>
        <li><b>Data provided to VEuPathDB is housed on secure servers and never shared outside of VEuPathDB staff without prior consent of the data provider.</b> Note that database staff are not active researchers; they are distinct from students and postdocs in the groups responsible for VEuPathDB, who see new datasets only when they become accessible to the general public. 
          <br><br>
        <li>Datasets are assigned a provisional release date, in consultation with the data provider. <b>Scheduling a dataset does not mean that it will be released without final examination and approval by the data provider!</b> We operate on the assumption that those who generate the data are best placed to evaluate its proper integration and representation in the knowledgebase. Note that this ‘golden rule’ applies to both published and unpublished data.
          <br><br>
        <li>Three to four months before the scheduled release date, the data provider is contacted by the <b>Outreach</b> team, to ensure that we have the most up-to-date version of the data, along with appropriate metadata and information on data sources and citations. The <b>Data Loading</b> team then processes and integrates this data into our internal knowledgebases. 
          <br><br>
       <li>After data loading is complete, the <b>Data Development</b> team begins to analyze and develop searches against the data. At this point we will likely communicate with the data provider, if questions arise. 
        <br><br>
       <li><b>Once data development is underway, the data provider is given access to a password-protected version of the VEuPathDB website containing their data.</b> This development site is similar to the current production knowledgebase, except that it also includes new data from the provider. We also provide instruction on how to search and view these new data, including sample searches integrating new data with relevant information already available in the knowldegebase. Important questions to consider include: 
        <ul>               
          <li>Does the knowledgebase accurately represent your data?</li>
          <li>Are the values and/or graphical displays provided appropriate?</li>
          <li>Are the questions that one can ask of your data appropriate?</li>
          <li>Are there additional questions that you would like to see implemented?</li>
          <li>Are the data appropriately described, including relevant metadata and
            reference / citation details? </li>
          </ul>
          <br>
      <li><b>A series of exchanges typically ensues,</b> in which we work iteratively with data
        providers to address any concerns, with changes reviewed on the password
        protected site so that providers can view and interrogate their data in the context of
        the rest of the database. 
        <br><br>

       <li><b>Public release is only considered after everyone is satisfied with how the data
          is represented.<b></b> If the provider is not yet ready to authorize data public release, data
          is rescheduled for a future release, and removed from the development site before it
          goes live: 
          <br><br>

       <li></b> Once data are approved for public release, a description is included in the ‘News’
            accompanying the next release, <b>highlighting new datasets and functionality, and
            acknowledging all data providers. </b> 
            <br><br>

      <li><b>Post-release quality assurance</b> provides the opportunity to modify displays and
              develop new queries if/as appropriate.  

      </li>
    </li>
  </li>
</li>
  
  </li>
</ul>
      


</div>
