---
layout: plain
title: Data Submission and Release on VEuPathDB Databases
permalink: /dataSubmissionReleasePolicy
tags: [dataSubmissionReleasePolicy]
---
<h1>Data Submission and Release on VEuPathDB Databases</h1>
issued February 2010, most recent revision 02 April 2020

<div class="static-content">
    
  <p>The Eukaryotic Pathogen, Vector & Host Informatics Resources (https://VEuPathDB.org)
    is a Bioinformatics Resource Center (BRC) operated under contract from the US National Institute of Allergy and 
    Infectious Diseases (NIAID) and the Wellcome Trust. VEuPathDB is charged with ensuring that genomic (and other 
    large-scale) datasets pertaining to supported pathogens are conveniently accessible to the worldwide community of 
    biomedical researchers. This document summarizes policies associated with releasing datasets on VEuPathDB. 
    Our goal is to help the communities that we serve ensure that their data are FAIR, Findable, Accessible, 
    Interoperable and Reusable.
  </p>
  
  <p>General principles:</p>
  <ul>
    <li>Data providers define the schedule for data release (in consultation with funders, publishers, etc). While there is no point in providing VEuPathDB with data that will never become public, deposition does not in itself authorize immediate release. Data become accessible to the public only when the data providers and VEuPathDB staff agree that it is accurately represented and ready to go live. Note that knowledgebase staff are not active research scientists; they are distinct from researchers in the groups responsible for VEuPathDB, who see new data only when it becomes accessible to the general public.</li>
    <li>Data providers know their data best. We expect to work with those who generated the underlying data to determine how best to analyze and represent new data types. This typically means taking in relatively raw data – often earlier, and in a more unprocessed form than the published dataset – and building an in-house analysis pipeline to ensure that all comparable datasets are handled similarly.</li>
    <li>The earlier we learn about new datasets, the easier it is to schedule timely release. The nature of our knowledgebase production, and competing demands from the many communities we support, means that several months' notice are often required to prepare for release. Note that it is often possible to use a preliminary dataset for planning, which can be swapped for the final version before public release.</li>
    <li>Experience has shown that data not deposited prior to publication often fails to emerge at all! After publication, it may be difficult to focus on tracking down the raw data, associated metadata, analysis methods, etc. It is never too early to discuss planned datasets with the VEuPathDB team!</li>
    <li>While not required, pre-publication data release often results in favorable attention from scientific colleagues (including journal editors and grant reviewers). Note that all major scientific journals now agree that early release of genomic-scale datasets does not compromise publication.</li> 
    </ul>

 
  <h3>Why submit my data to VEuPathDB?</h3>
  <ul>
    <li>Inclusion in VEuPathDB facilitates your own analysis of the data, in the context of other genomic-scale experiments already available from researchers around the world.</li>
    <li>Electronic access permits others to analyze your data in greater depth than possible in print (even in advance of publication, if you wish to allow this).</li>
    <li>Availability within VEuPathDB keeps your data alive on a highly visible genomics knowledgebase resource: VEuPathDB is accessed by ~13,000 unique users each month.</li>
  </ul>
  
  <h3>How do I submit data to VEuPathDB?</h3>
  <ul>
    <li>Fill out the appropriate form to indicate the data availability</li>
    <li>Contact the VEuPathDB by clicking the ‘Contact Us’ link on any VEuPathDB page, or
      emailing us at help@VEuPathDB.org.</li>
    <li>Tell us about your data as early as possible, to allow ample time for scheduling into
      VEuPathDB release cycles.</li>
    <li>Once you tell us about your data, we will provide instructions on how to transfer your
      data to us (formats may differ depending on the nature and scale of the data to be
      transferred).</li>
    <li>In order to avoid any confusion and ensure accuracy, we adhere to strict Standard
      Operating Procedures (SOPs), as outlined below.
      </li>
  </ul>  

  <h3>What data types are supported by VEuPathDB?</h3>
<p>In one form or another, VEuPathDB currently represents sequence (genomic [nuclear and organellar], ESTs and RNA-seq, generated on various platforms), host-response data, comparative genomic information, DNA polymorphism and population genomics data, sequences and metadata pertaining to field and clinical isolates and collections (with geo- spatiotemporal and other metadata), chromatin modification data (ChIP-chip and ChIP-seq), manually curated and automatically generated gene models and other annotation (GO terms, InterPro domains, etc.), transcript and proteomic profiling, host response data sets (multiple platforms), interactome data, protein structural information, metabolic pathways and metabolomics data, phenotype information, reagents (clones, antibodies, etc.), publication references, image data, etc. We can support additional data types as needed. <b>Please let us know if you have data to provide that is not currently supported!</b></p>

<h3>What species are supported by VEuPathDB?</h3>
<p>The VEuPathDB contract from NIAID provides support for biosecurity pathogens, including <i>Babesia</i>, <i>Cryptosporidium</i>, <i>Entamoeba</i>, <i>Giardia</i>, <i>Microsporidia</i> (various genera), <i>Toxoplasma</i>, <i>Plasmodium</i>, and related taxa (<i>Acanthamoeba</i>, <i>Gregarina</i>, <i>Neospora</i>, <i>Theileria</i>) and also arthropod vectors (ticks, mites, mosquitoes, kissing bugs, tsetse flies, sand flies, lice, etc.) of human disease, as well as a sail that serves as an intermediate host, and comparator species. Support for kinetoplastid parasites (<i>Crithidia</i>, <i>Endotrypanum</i>, <i>Leishmania</i>, <i>Trypanosoma</i>) is provided by The Wellcome Trust. The FungiDB project encompasses a large (and growing) number of species supported by both NIAID and the Wellcome Trust. <b>Please contact us if you have data from other species that should be incorporated into VEuPathDB!</b></p>

<h3>Data Management SOPs (Standard Operating Procedures) for VEuPathDB databases</h3>
<p>VEuPathDB routinely handles datasets provided prior to publication,in addition to those already in the public domain. In order to ensure timely and accurate data integration we strictly adhere to the following Standard Operating Procedures (SOPs):
</p>
  <ol>
    <li>Datasets come to our attention in several ways, including:</li>
      <ul>Direct contact from researchers generating the data (during the earliest stages of project design, as
        <li>a. data is being produced, in the course of data analysis, or in the context of manuscript preparation).</li>
        <li>b. Information provided by members of the VEuPathDB or larger pathogen community.
          <li>c.Information obtained by VEuPathDB staff at meetings and conferences.</li>
          <li> d. Publicly available information from the scientific literature, genomic dataset repositories, etc.</li>
      </ul>
    </ol>
      
  <b>Note that VEuPathDB can often facilitate data deposition in the appropriate archival repositories (GenBank, SRA, GEO/ArrayExpress, etc.)</b>

  <h3>Data Management SOPs (Standard Operating Procedures) for VEuPathDB databases</h3>
<p>VEuPathDB routinely handles datasets provided prior to publication,in addition to those already in the public domain. In order to ensure timely and accurate data integration we strictly adhere to the following Standard Operating Procedures (SOPs):
</p>
  <ul>Datasets come to our attention in several ways, including:
    <li>Direct contact from researchers generating the data (during the earliest stages of project design, as</li>
    <li>data is being produced, in the course of data analysis, or in the context of manuscript preparation).
      b. Information provided by members of the VEuPathDB or larger pathogen community.
      c. Information obtained by VEuPathDB staff at meetings and conferences.
      d. Publicly available information from the scientific literature, genomic dataset repositories, etc.
      </li>
  </ul>  
  <b>Note that VEuPathDB can often facilitate data deposition in the appropriate archival repositories (GenBank, SRA, GEO/ArrayExpress, etc.)</b>


<!-- =========== Genome Sequence and Annotation =========== -->

<details>
  <summary><b>Genome Sequence and/or Annotation</b></summary>
  <p>Genomes must be available in The International Nucleotide Sequence Database Collaboration (INSDC) such as NCBI GenBank, 
        EMBL-EBI ENA or DDBJ.
  </p>
    <ul>
      <li>If your genome <b>IS</b> uploaded to a repository, complete the <a href="https://docs.google.com/forms/d/e/1FAIpQLSchN8HUyZHP6v8B505y42YZV4Elb_vly1NUgToKhyz-kUjf9g/viewform">
          Genome Sequence and/or Annotation Description Form</a> making sure to include the accession numbers of your data 
          when prompted. We will download your data from the repository.
      </li>
      <li>If the annotation file <b>is not</b> uploaded to a repository, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
          form to send us the genome annotation file only (e.g. gff file format).
      </li>
  </ul>
   <p></p>
</details>

<!-- =========== High Throughput or Next Generation Sequencing =========== -->
<details>
  <summary><b>High Throughput or Next Generation Sequencing</b> &#8211; RNA, DNA or ChIP Sequencing</summary>
  <p>We prefer to download the raw read data in FASTQ format from a sequence read archive. 
     We integrate the data into the database using the raw reads and use the raw reads during future 
     database releases to remap or update our analyses when necessary.
  </p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Upload your data to a sequence read archive such as the European Nucleotide Archive or 
          NCBI's Sequence Read Archive, and provide us the accession numbers. We will export the data directly from INSDC</li>
      <li>Complete the appropriate data description form making sure to enter your data archive accession numbers when
          prompted</li>
    </ul>
  <p>
    <a href="https://docs.google.com/a/apidb.org/forms/d/1f4UCA6tCaAuItKkh97Xg-K_mXz8Pic5SejejOlo09QY/edit?usp=sharing">RNA-Seq 
       Data Description Form</a>
  </p>
  <p><a href="https://docs.google.com/forms/d/1z4rrMp2kjJSkqz6EbwglF6oK9xp_0_VUz9AjfPg4FWc/viewform">DNA-Seq
        Data Description Form</a>
  </p>
  <p><a href="https://docs.google.com/forms/d/11Qt40E13XTmIUbdftNXVQ6lEr2VIvZPFc0__jrZILJc/viewform">ChIP-Sequencing
         Data Description Form</a>
  </p>
  <p></p>
</details>

<!-- =========== Microarray =========== -->
<details>
  <summary><b>Microarray</b></summary>
  <p>Transfer a copy of your data to VEuPathDB using one of these options:</p>
    <ul>
      <li>Upload your data to a repository such as GeneExpression Omnibus.
          Complete the data description form linked below making sure to enter your data archive accession numbers when prompted.
          We will export the data directly from a repository.
      </li>
      <li>Send your data as an attachment to an email. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a>
          form to send us an email. Files (e.g. CEL, CSV) must include expression levels and probe set information.Pay special attention to clearly 
          indicate the identity of columns in the data files you transferred to VEuPathDB.
      </li>
   </ul>
  <p><a href="https://docs.google.com/a/apidb.org/forms/d/1pzD53hCY0rH7JYomUH8cjwJXJlOSrtZRc_mFowfFfU4/edit?usp=sharing">
        Microarray Data Description Form</a>
  </p>
  <p></p>
</details>

<!-- =========== Proteomics =========== -->
<details>
  <summary><b>Proteomics</b></summary>
  <p>Excel or tab delimited text files are preferred. We can accommodate xml file format. Required columns include gene IDs, 
     peptide sequences, peptide counts and scores.</p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Send your data as an attachment to an email. Use the 
        <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
           form to send us an email.
      </li>
      <li>Complete the 
        <a href="https://docs.google.com/a/apidb.org/forms/d/1Yx9qGKDyCf2Wm5lnf1-dFHArdQAQ9tVbR8gvxdY7n5A/edit?usp=sharing">
           Proteomics Data Description Form</a> making sure to clearly indicate the content of each column in your file.
      </li>
  </ul>
  <p></p>
</details>

<!-- =========== Quantitative Proteomics =========== -->

<details>
  <summary><b>Quantitative Proteomics</b></summary>
  <p>Excel or tab delimited files are preferred. We can accommodate xml file format. Required columns include gene IDs and 
     scores.</p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Send your data as an attachment to an email. Use the 
        <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
           form to send us an email.
      </li>
      <li>Complete the 
        <a href="https://docs.google.com/a/apidb.org/forms/d/1jC1w4wWb_tNWPoL55F0zLq8o8rqOXQjtw27dt9lKups/viewform">Quantitative 
           Proteomics Data Description Form</a> making sure to include a description of data columns, for example, time course
           units and arrangement if not apparent from column headers.>
      </li>
    </ul>
  <p></p>
</details>

<!-- =========== ChIP-chip =========== -->

<details>
  <summary><b>ChIP-chip</b></summary>
  <p>Your data files should include expression levels and probe set information.</p>
  <p>Transfer a copy of your data to VEuPathDB using one of these options:</p>
    <ul>
      <li>Upload your data to a repository such as Gene Expression Omnibus and complete 
          the <a href="https://docs.google.com/forms/d/1E1QN9dKGc9rK_Bd3t-4ugIeVAyNXZoX1Z3bcSKEIMJc/viewform">ChIP-chip
          Data Description Form</a> making sure to enter the archive accession numbers (if any) for your data when prompted.
      </li>
      <li>Send your data as an attachment to an email. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact
          Us</span></a> form to send us an email.
      </li>
    </ul>
  <p></p>
</details>


<!-- =========== Isolates typed by =========== -->

<details>
  <summary><b>Isolates typed by sequencing limited genetic loci</b></summary>
  <p>If your data <b>ARE already</b> uploaded to the GenBank, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
     form to tell us about your data. Note: GeneBank Isolate records and the associated metadata are automatically
     updated with each VEuPathDB release. There is no need to complete our Isolate Submission Form.</p>
   
   <p>If your data <b>ARE NOT</b> alredady uploaded to the GenBank, we can facilitate this upload. Complete the Isolate 
      Submission Form linked below and we will use the information to generate a GenBank submission for your isolates. 
      The new isolate records will be downloaded to VEuPathDB when the records are released by the GenBank. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
      form to send us instructions for retrieving your data.</p>

   <p><a href="https://veupathdb.org/tutorials/IsolateSubmissionEuPathDBv8.xls">Isolate Submission Form</a></p>
   <p><a href="https://veupathdb.org/tutorials/EuPathDB_Isolatesubmission_genericHelp.pdf">Help for submitting Isolate Data</a></p>

</details>

<!-- =========== Isolates or Strains =========== -->

<details>
  <summary><b>Isolates or Strains typed by High-Throughput Sequencing</b></summary>

  <p>We prefer to receive the raw read data in FASTQ or FASTA file format. We integrate your data into the database using 
     the raw reads. We also use the raw reads during future database releases to remap your data when the reference genome is 
     reloaded and to update our analyses when needed.</p>

  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Upload your data to a sequence read archive such as DNA Data Bank of Japan, the European Nucleotide Archive or 
          NCBI's Sequence Read Archive. We will retrieve your data using the read archive's accession numbers for your data 
          set.
      </li>
      <li>Complete our <a href="https://docs.google.com/a/apidb.org/forms/d/1z4rrMp2kjJSkqz6EbwglF6oK9xp_0_VUz9AjfPg4FWc/viewform">  
          DNA Seq Data Description Form</a> making sure to enter the read archive accession numbers for your data when prompted.
          We also ask that you complete an abbreviated <a href="https://veupathdb.org/tutorials/IsolateSubmissionEuPathDBv8.xls">Abbreviated 
          Isolate Submission Form</a> to describe meta data associated with your isolates.
      </li>
    </ul>
  <p></p>
</details>

<!-- =========== Bulk user comments form =========== -->
<details>
  <summary><b>Bulk user comments form</b></summary>
  <p>Instructions: download the form, fill out required fields and provide any additional information, if available, and send it back to the email listed in the form</p>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/17Uu_KvfpU6FYvL0ftA7zSt1fK7OinDRZp6Dng_RPH7Y/edit?usp=sharing">
           Bulk user comments form</a>
      </li>
  <p></p>
</details>

<!-- =========== Phenotype collection form =========== -->
<details>
  <summary><b>Phenotype collection form</b></summary>
  <p>Instructions: download the form, fill out required fields and provide any additional information, if available, and send it back to the email listed in the form</p>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/1uhkr7q8QWL7ZpXLpFO3Wb5Vj9GHK47ylrzUXGbOYL9w/edit?usp=sharing">
           FungiDB phenotype collection form</a>
      </li>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/1mcoPimtWkL2nqH_5qdMerVuinIUPG0t19s_uEF8BHOQ/edit#gid=783150795">
           Parasite phenotype collection form</a>
      </li>
  <p></p>
</details>


<!-- =========== General Data Submission =========== -->
<div>
  <p><b>General Data Submission</b>
   &#8211; Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> form to tell us
   about data that does not fit any of the above categories
  </p>
</div>


</div>
