<h1>About VEuPathDB</h1>

<div class="static-content">

<a name="about_general"></a>

<h3>General Information</h3>
The Eukaryotic Pathogen, Vector and Host Informatics Resource (VEuPathDB) is one of two <a target="_blank" href="https://www.niaid.nih.gov/research/bioinformatics-resource-centers">Bioinformatics Resource Centers (BRCs)</a> funded by the US National Institute of Allergy and Infectious Diseases (NIAID), with additional support from the Wellcome Trust (UK).<br><br>
These resources stem from support initially provided for the Plasmodium Genome Database by the Burroughs Wellcome Fund (2000-2) and a research grant from NIAID (2002-6). The BRC program was initiated in 2004 to provide public access to computational platforms and analysis tools enabling collection, management, integration and mining of genomic information and other large-scale datasets relevant to infectious disease pathogens including their interaction with mammalian hosts and invertebrate vectors of disease. Two BRCs are currently funded:<br>
<ul>
<li>VEuPathDB focuses on eukaryotic pathogens and invertebrate vectors of infectious diseases, , encompassing data from prior BRCs devoted to parasitic species (EuPathDB), fungi (FungiDB) and vector species (VectorBase).</li>
<li><a target="_blank" href="https://patricbrc.org">PATRIC</a> & <a target="_blank" href="https://www.viprbrc.org/">VIPR</a> focus on bacterial and viral pathogens.</li>
</ul><br>


VEuPathDB has also received funding from the Wellcome Trust (UK), Bill & Melinda Gates Foundation, and US Department of Agriculture to support informatics efforts focusing on kinetoplastida and fungal organisms with special emphasis on improving functional annotation for select genome sequences and families of genes.  
<br><br>

VEuPathDB provides access to diverse genomic and other large scale datasets related to eukaryotic pathogens and invertebrate vectors of disease (see <a href="/a/app/search/organism/GenomeDataTypes?autoRun">Data Summary</a>).
Organisms supported by this resource include (but are not limited to) the NIAID list of <a target="_blank" href="https://www.niaid.nih.gov/research/emerging-infectious-diseases-pathogens">emerging and re-emerging infectious diseases</a>.
<br><br>
Component web sites are constructed using a common infrastructure and standard data analysis and loading procedures, allowing the use of <a href="http://VEuPathDB.org">VEuPathDB</a> as a single point of entry for each (or all) of these community resources, and the opportunity to leverage orthology for searches across taxa.
<br><br><ul>
<li><a href = "/a/app/search/organism/GenomeDataTypes/result">Genome Info & Stats</a> provides a list of all organisms available in this website.
</li>
<li><a href="/a/app/search/dataset/AllDatasets/result">Data Sets</a> provides a list of all information in this website integrated into VEuPathDB, with relevant references.</li>

</ul>

<a name="about_funding"/></a>
<h3>Current Funding</h3>
The Eukaryotic Pathogen, Vector and Host Informatics Resources (VEuPathDB) is funded by the National Institute of Allergy and Infectious Diseases (NIH/DHHS) under Contract No. NIH HHS 75N93019C00077. 
<br><br>
VEuPathDB also receives funding from the Wellcome Trust (UK) to support informatics efforts focusing on kinetoplastida and fungal organisms with special emphasis on improving functional annotation of genomes. Grant numbers: 212929/Z/18/Z and 218288/Z/19/Z. 

<a name="about_use"/></a>
<h3>Data Access Policy</h3>
All data on these websites are provided freely for public use, through the contributions of many researchers involved in generating genome sequences, functional genomics datasets, and additional information. These data often derive from ongoing research, and are not guaranteed to be accurate. When using data obtained from VEuPathDB, it is important to cite the original publications and contributors. Please see <a href=#about_citing>Citing VEuPathDB</a>.


<a name="about_citing"/></a>
<h3>Citing VEuPathDB in Publications and Presentations</h3>
If you use a VEuPathDB resource, we invite you to please cite the most relevant publication. This <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/?term=%28%22eupathdb%22%5BTitle%5D+OR+%22plasmodb%22%5BTitle%5D+OR+%22toxodb%22%5BTitle%5D+OR+%22giardiadb%22%5BTitle%5D+OR+%22amoebadb%22%5BTitle%5D+OR+%22microsporidiadb%22%5BTitle%5D+OR+%22tritrypdb%22%5BTitle%5D+OR+%22vectorbase%22%5BTitle%5D+OR+%22fungidb%22%5BTitle%5D+OR+%22apidb%22%5BTitle%5D+OR+%22cryptodb%22%5BTitle%5D+OR+%22cryptodb.org%22%5Btitle%5D+OR+%22piroplasmadb%22%5BTitle%5D+OR+%22trichdb%22%5BTitle%5D+OR+%22orthomcl%22%5BTitle%5D%29+AND+%28Roos+%5Bauthor%5D+OR+kissinger+%5Bauthor%5D+OR+collins+%5Bauthor%5D%29&sort=date">PubMed filter</a> provides a list of the most recent VEuPathDB publications.<br><br>
Please note that much of the data in VEuPathDB is provided by independent researchers, please cite them if you use their data. See <a href="/a/app/search/dataset/AllDatasets/result">Data Sets</a> for a list of all information integrated into VEuPathDB, and related publications.<br><br>
For acknowledgements in presentations, you may wish to use one or more of the following logos (right/control click to copy):


<!--<br><br>
When citing EuPathDB, please use:
<br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/27903906">EuPathDB: the eukaryotic pathogen genomics database resource.</a>
Cristina Aurrecoechea; Ana Barreto; Evelina Y. Basenko; John Brestelli; Brian P. Brunk; Shon Cade; Kathryn Crouch; Ryan Doherty; Dave Falke; Steve Fischer; Bindu Gajria; Omar S. Harb; Mark Heiges; Christiane Hertz-Fowler; Sufen Hu; John Iodice; Jessica C. Kissinger; Cris Lawrence; Wei Li; Deborah F. Pinney; Jane A. Pulman; David S. Roos; Achchuthan Shanmugasundram; Fatima Silva-Franco; Sascha Steinbiss; Christian J. Stoeckert Jr; Drew Spruill; Haiming Wang; Susanne Warrenfeltz; Jie Zheng. 
Nucleic Acids Research 2017 doi: 10.1093/nar/gkw1105
<br>-->


<br><br>
<table style="width: 80%" border="0">
  <tbody>
    <tr>
      <td style="text-align: left; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/VEuPathDB.png"
          alt=""
          style="width: 334px; height: 75px;"></td>
      
      <td style="text-align: left; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/amoebaDB.png"
          alt=""
          style="width: 265px; height: 70px;"></td>
          <td style="text-align: left; vertical-align: middle; background-color: white;"><img
            src="/a/images/VEuPathDB/Logos-color-text-web/cryptoDB.png"
            alt=""
            style="width: 292px; height: 70px;"></td>
    </tr>
    <tr>
      
      <td style="text-align: left; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/fungiDB.png"
          alt=""
          style="width: 275px; height: 70px;"></td>
          <td style="text-align: left; vertical-align: middle; background-color: white;"><img
            src="/a/images/VEuPathDB/Logos-color-text-web/giardiaDB.png"
            alt=""
            style="width: 263px; height: 70px;"></td>
        <td style="text-align: left; vertical-align: middle; background-color: white;"><img
            src="/a/images/VEuPathDB/Logos-color-text-web/hostDB.png"
            alt=""
            style="width: 283px; height: 70px;"></td>
    </tr>
    <tr>
      <td style="text-align: left; vertical-align: middle; background-color: white;"><img
        src="/a/images/VEuPathDB/Logos-color-text-web/microsporidiaDB.png"
        alt=""
        style="width: 283px; height: 70px;"></td>
    <td style="text-align: left; vertical-align: middle; background-color: white;"><img
        src="/a/images/VEuPathDB/Logos-color-text-web/orthoMCL.png"
        alt=""
        style="width: 286px; "></td>
        <td style="text-align: left; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/piroplasmaDB.png"
          alt=""
          style="width: 294px; height: 70px;"></td>
    </tr>
    <tr>
      <td style="text-align: left; vertical-align: middle; background-color: white;"><img
        src="/a/images/VEuPathDB/Logos-color-text-web/plasmoDB.png"
        alt=""
        style="width: 288px; height: 70px;"></td>
        <td style="text-align: left; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/toxoDB.png"
          alt=""
          style="width: 272px; height: 70px;"></td>
      <td style="text-align: left; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/trichDB.png"
          alt=""
          style="width: 278px; height: 70px;"></td>
    </tr>
    <tr>
      <td style="text-align: left; vertical-align: middle; background-color: white;"><img
        src="/a/images/VEuPathDB/Logos-color-text-web/tritrypDB.png"
        alt=""
        style="width: 271px; height: 70px;"></td>
    <td style="text-align: left; vertical-align: middle; background-color: white;"><img
        src="/a/images/VEuPathDB/Logos-color-text-web/vectorbase.png"
        alt=""
        style="width: 270px; height: 70px;"></td>
      
    </tr>
  </tbody>
</table>
<br><br>
Additional resources leveraging the same infrastructure: <a target="_blank" href="https://microbiomedb.org">MicrobiomeDB</a> and <a target="_blank" href="https://clinepidb.org">ClinEpiDB</a>.
 
<a name="about_privacy"/></a>
<h3>Website Privacy Policy</h3>
We do not use or share any of your personal information for any purpose unrelated to the functionality of the websites; however, we do collect some information to help us understand how our sites are being used and to improve community support. See  
<a href="/a/app/static-content/privacyPolicy.html">Website Privacy Policy</a>.

</div>
<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>About OrthoMCL</h1>

<div class="static-content">
  <div class="about-body">

        <div id="release" class="section-title">
          <h2>Current Release</h2>
        </div>
        <div class="section-content">
          <p>
	    See <a href="/a/app/static-content/OrthoMCL/news.html">news about the Current Release</a>.
	  </p>
	  <p>
	    To see the current set of organisms as well as their proteome sources and orthology statistics, go to 
            <a href="/a/app/release-summary">Proteome Sources and Statistics</a>.
          </p>
          <p>
            <strong>Downloads:</strong>
            Go to the <a href="/a/app/downloads">download site</a> to obtain the protein sequences
            and ortholog groups used in this release.
          </p>
        </div>

        <div id="forming_groups" class="section-title">
          <h2>Method for Forming and Expanding Ortholog Groups</h2>
        </div>
        <div class="section-content">
          <p>
	    Proteins are placed into Ortholog Groups by the following steps:
	  <ol>
	    <li>The OrthoMCL algorithm (see below) is employed on proteins from a set of 150 <b>Core</b> species to form <b>Core</b> ortholog groups. These species were carefully chosen based on proteome quality and widespread placement across the tree of life. Each Core protein is placed by the algorithm into a <b>Core</b> ortholog group consisting of one or more proteins. Core group names have the format OG6_xxxxxx (e.g., OG6_101327). OG6 refers to OrthoMCL release 6; for each sub-release (e.g., 6.1, 6.2, etc), the Core species and the Core ortholog group names will remain constant.</li>
	    <li>The proteins from hundreds of additional organisms, termed <b>Peripheral</b> organisms, are mapped into the Core groups. To do this, NCBI BLASTP is used to compare each Peripheral protein to each Core protein in the Core groups. (Note that Peripheral proteins that were previously added to the Core group are NOT used in the BLASTP.) Then, each <b>Peripheral</b> protein is assigned to the Core group containing the Core protein with the best BLAST score, but only if the E-Value is &lt;1e-5 and the percent match length is &gt;=50%.</li>
	    <li>All Peripheral proteins that fail to map to a Core group are collected and subjected to independent OrthoMCL analysis, forming <b>Residual</b> groups consisting of one or more proteins. Residual group names have the format OG6r1_xxxxxx (e.g., OG6r1_101327), where OG6 refers to release 6 and r1 refers to sub-release 1.</li>
	    <li>For each subsequent sub-release (which will occur every ~3 months along with other VEuPathDB sites), proteomes from additional <b>Peripheral</b> organisms will be processed as in steps 2 and 3 above. However, step 3 will differ slightly because the previous set of Residual groups will be disassembled, leaving the previous unmapped Peripheral proteins to be combined with the new unmapped Peripheral proteins. All of these proteins will be used to form new Residual groups (e.g., OG6r2_xxxxxx).</li>
	    <li>During a sub-release, the proteomes of some species will be updated to the latest version. This can be easily done for a Peripheral species: the old set of proteins are removed from ortholog groups and then the new set is mapped into groups as above. However, this is not possible for Core species because these proteins are used to define Core groups. Thus, the Core species with the older proteome remains on the site but is superficially retired by appending its abbreviation with -old (e.g., aaeg becomes aaeg-old). Then, the latest version of the proteome is mapped in as a peripheral species and obtains the original species abbreviation (e.g., aaeg is a peripheral with a more recent proteome than aaeg-old). These retired species will be eliminated fully when a new set of Core species is defined, as described in the next point.</li>
	    <li>On occasion, the set of Core species will be re-defined, as more appropriate proteomes become available and/or when a large number of Core species are retired. In this case, new Core groups (e.g., OG7_xxxxxx) and Residual groups (e.g., OG7r1_xxxxxx) will be formed from the latest version of proteomes from a carefully-chosen set of core species.</li>
	  </ol>
	  </p>
	  <p>
	    This design allows for the addition of proteomes at every sub-release (e.g., 6.1, 6.2, etc). Note that <b>Core</b> groups (e.g., OG6_101327) will remain between sub-releases, though these groups will expand as Peripheral proteins are mapped in. In contrast, <b>Residual</b> groups will exist only for that sub-release; thus, Residual groups are useful in allowing the user to find proteins related to their protein(s) of interest, but are not stable groups.
	  </p>
        </div>

        <div id="orthomcl_algorithm" class="section-title">
          <h2>The OrthoMCL Algorithm</h2>
        </div>
        <div class="section-content">
          <p>
            See the <b><a href="https://docs.google.com/document/d/1RB-SqCjBmcpNq-YbOYdFxotHGuU7RK_wqxqDAMjyP_w/pub" target="_blank">OrthoMCL Algorithm Document</a></b> for a detailed description of the OrthoMCL algorithm.
          </p>
           In overview:
           <ul class="cirbulletlist">
           <li>All-v-all BLASTP of the proteins.</li>
           <li>Compute <i>percent match length</i>
             <ul class="cirbulletlist">
             <li>Select whichever is shorter, the query or subject sequence.  Call that sequence S.</li>
             <li>Count all amino acids in S that participate in any HSP.</li>
             <li>Divide that count by the length of S and multiply by 100</li>
             </ul>
           </li>
           <li>Apply thresholds to blast result.  Keep matches with E-Value &lt; 1e-5 percent match length &gt;= 50%</li>
           <li>Find potential inparalog, ortholog and co-ortholog <i>pairs</i> using the Orthomcl Pairs program.  (These are the pairs that are counted to form the <i>Average % Connectivity</i> statistic per group.)</li>
           <li>Use the <a href="http://micans.org/mcl/" target="_blank">MCL</a> program to cluster the pairs into groups</li>
           </ul>
        </div>

        <div id="background" class="section-title">
          <h2>Background on Orthology and Prediction</h2>
        </div>
        <div class="section-content">
          <p>
            Orthologs are homologs separated by speciation events.  Paralogs are homologs separated
            by duplication events. Detection of orthologs is becoming much more important with the
            rapid progress in genome sequencing (<a href="https://academic.oup.com/mbe/article/36/10/2157/5523206" target="_blank">Glover et al. 2019</a>).
          </p>
          <p>
            OrthoMCL is a genome-scale algorithm for grouping orthologous protein sequences. It
            provides not only groups shared by two or more species/genomes, but also groups
            representing species-specific gene expansion families. Thus, it serves as an important
            utility for automated eukaryotic genome annotation. OrthoMCL starts with reciprocal best
            hits within each genome as potential in-paralog/recent paralog pairs and reciprocal best
            hits across any two genomes as potential ortholog pairs.  Related proteins are interlinked
            in a similarity graph. Then, MCL (Markov Clustering algorithm; <a href="https://dspace.library.uu.nl/handle/1874/848" target="_blank">Dongen 2000</a>;
            <a href="http://micans.org/mcl/" target="_blank">www.micans.org/mcl</a>) is invoked to split mega-clusters.
            This process is analogous to the manual review in COG construction.  MCL clustering is
            based on weights between each pair of proteins, so to correct for differences in
            evolutionary distance the weights are normalized before running MCL.
          </p>
          <p>
            OrthoMCL is similar to the INPARANOID algorithm (<a href="https://www.sciencedirect.com/science/article/abs/pii/S0022283600951970?via%3Dihub" target="_blank">Remm et al. 2001</a>) but is extended
            to cluster orthologs from multiple species. OrthoMCL clusters are coherent with groups
            identified by EGO (<a href="https://genome.cshlp.org/content/12/3/493.long" target="_blank">Lee et al. 2002</a>), and an analysis using EC number suggests a
            high degree of reliability (<a href="http://www.genome.org/cgi/content/abstract/13/9/2178" target="_blank">Li et al. 2003</a>).
          </p>
          <p>
            We evaluated the performance of seven widely-used orthology detection algorithms that use three general
	    prediction strategies: phylogeny-based, evolutionary distance-based and BLAST-based (<a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0000383" target="_blank">Chen, et al. 2007</a>).
	    Specifically, we used Latent Class Analysis (LCA), a statistical technique appropriate for testing large data
	    sets when no gold standard is available. Our results show an overall trade-off between sensitivity and
	    specificity among these algorithms, with INPARANOID and OrthoMCL performing best with False Positive
            (FP) and False Negative (FN) error rates lower than 20%.
          </p>
        </div>


        <div id="software" class="section-title">
          <h2>Software</h2>
        </div>
        <div class="section-content">
          <p>
            OrthoMCL was originally implemented by Li Li.  The software was not available for download.
          </p>
          <p>
            <a href="/a/app/downloads/software" target="_blank">Version 1.4</a> was developed as publicly available software by Feng Chen (This version is now not supported).
          </p>
          <p>
            <a href="/a/app/downloads/software" target="_blank">Version 2.0</a> was re-engineered to handle large-scale datasets (hundreds of genomes) by Steve Fischer, Mark Heiges, John Iodice, and Ryan Thibodeau.
          </p>
        </div>


        <div id="pubs" class="section-title">
          <h2>Publications</h2>
        </div>
        <div class="section-content">
          <ol>
            <li>
              Li Li, Christian J. Stoeckert, Jr., and David S. Roos<br />
              OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes<br />
              Genome Res. 2003 13: 2178-2189.
                <a href="http://www.genome.org/cgi/content/abstract/13/9/2178" target="_blank">[Abstract]</a>
                <a href="http://www.genome.org/cgi/content/full/13/9/2178" target="_blank">[Full Text]</a>
            </li>
            <li>
              Feng Chen, Aaron J. Mackey, Christian J. Stoeckert, Jr., and David S. Roos<br />
              OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups <br />
              Nucleic Acids Res. 2006 34: D363-8.
                <a href="http://nar.oxfordjournals.org/cgi/content/full/34/suppl_1/D363" target="_blank">[Full Text]</a><br />
                * Please cite this paper if you publish research results benefited from OrthoMCL-DB.
            </li>
            <li>
              Feng Chen, Aaron J. Mackey, Jeroen K. Vermunt, and David S. Roos <br />
              Assessing Performance of Orthology Detection Strategies Applied to Eukaryotic Genomes<br />
              PLoS ONE 2007 2(4): e383.
                <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0000383" target="_blank">[Full Text]</a><br />
                * Recommended in <a href="http://www.f1000biology.com/article/id/1092076" target="_blank">Faculty1000</a>
            </li>
            <li>
            Fischer, S., Brunk, B. P., Chen, F., Gao, X., Harb, O. S., Iodice, J. B., Shanmugam, D., Roos, D. S. and Stoeckert, C. J.<br />
            Using OrthoMCL to Assign Proteins to OrthoMCL-DB Groups or to Cluster Proteomes Into New Ortholog Groups<br />
            Current Protocols in Bioinformatics. 2011 35:6.12.1–6.12.19.
              <a href="http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi0612s35/full" target="_blank">[Full Text]</a>
            </li>
          </ol>
        </div>


        <div id="acknowledge" class="section-title">
          <h2>Acknowledgements</h2>
        </div>
        <div class="section-content">
          <p>
            This project has been funded in whole or in part with Federal funds from the National
            Institute of Allergy and Infectious Diseases, National Institutes of Health, Department
            of Health and Human Services, under Contract No. HHSN266200400037C. The major PIs are
            Drs. David Roos and Chris Stoeckert.
          </p>
          <p>
            The OrthoMCL-DB project was initiated by Feng Chen in April 2005, and people from
            VEuPathDB, PCBI, and the Penn Center for Bioinformatics who have contributed to the
	    project include: Mark Hickman, Steve Fischer, Brian Brunk, Omar Harb, Ryan Doherty,
	    Aaron Mackey, Praveen Chakravarthula, Jerric Gao, and Charles Treatman. We'd also
	    like to thank students and postdocs from the Roos lab for valuable suggestions,
	    specifically Lucia Peixoto, and Dhanasekaran Shanmugam.
          </p>
        </div>

  </div>
</div>


<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>How to submit data to VEuPathDB</h1>

<div class="static-content">
    
  <p>The Eukaryotic Pathogen, Vector & Host Informatics Resources (https://VEuPathDB.org)
    is a Bioinformatics Resource Center (BRC) operated under contract from the US National Institute of Allergy and 
    Infectious Diseases (NIAID) and the Wellcome Trust. VEuPathDB is charged with ensuring that genomic (and other 
    large-scale) datasets pertaining to supported pathogens are conveniently accessible to the worldwide community of 
    biomedical researchers. This document summarizes policies associated with releasing datasets on VEuPathDB. 
    Our goal is to help the communities that we serve ensure that their data are FAIR, Findable, Accessible, 
    Interoperable and Reusable.
  </p>
  
  <p>VEuPathDB welcomes submissions of genomic-scale data concerning eukaryotic microbes, fungi, vectors of human disease, 
    and host-pathogen interactions. The VEuPathDB contract from NIAID provides support for biosecurity pathogens, including 
    <i>Babesia</i>, <i>Cryptosporidium</i>, <i>Entamoeba</i>, <i>Giardia</i>, <i>Microsporidi</i>a (various genera), 
    <i>Toxoplasma</i>, <i>Plasmodium</i>, and related taxa (<i>Acanthamoeba</i>, <i>Gregarina</i>, <i>Neospora</i>, 
    <i>Theileria</i>) and also arthropod vectors (ticks, mites, mosquitoes, kissing bugs, tsetse flies, sand flies, 
    lice, etc.) of human disease, as well as a snail that serves as an intermediate host, and comparator species. 
    Support for kinetoplastid parasites (<i>Crithidia</i>, <i>Endotrypanum</i>, <i>Leishmania</i>, <i>Trypanosoma</i>) 
    is provided by The Wellcome Trust. The FungiDB project encompasses a large (and growing) number of species supported 
    by both NIAID and the Wellcome Trust. Please <a href="/a/app/contact-us">contact us</a> if you have data from other species that should be 
    incorporated into VEuPathDB! <b>Please review our
    <a href="https://qa.community.eupathdb.org/documents/VEuPathDB_Data_Sub_Release_policy_rev_02April2020.pdf">Data Submission Policy</a></b>.
  </p>
  
  <p>Our most common data types include transcriptomics, proteomics, metabolomics, epigenomics, population-level and isolate 
     information. In one form or another, VEuPathDB currently represents datasets in the following categories:
  </p>
  <ul>
    <li>Sequence (genomic [nuclear and organellar])</li>
    <li>ESTs and RNA-seq, generated on various platforms)</li>
    <li>Host-response data</li>
    <li>Comparative genomic information</li>
    <li>DNA polymorphism and population genetics data</li> 
    <li>Sequences and metadata pertaining to field and clinical isolates and collections (with geo-spatiotemporal and other metadata)</li> 
    <li>Chromatin modification data (ChIP-chip and ChIP-seq)</li> 
    <li>Manually curated and automatically generated gene models and other annotation (GO terms, InterPro domains, etc.)</li>
    <li>Transcript and proteomic profiling</li>
    <li>Host response data sets (multiple platforms)</li>
    <li>Interactome data</li>
    <li>Protein structural information</li>
    <li>Metabolic pathways and metabolomics data</li>
    <li>Phenotype information, reagents (clones, antibodies, etc.)</li>
    <li>Publication references</li>
    <li>Image data, etc.</li> 
  </ul>

  <p>We also accept other genomic-scale data and are open to suggestions. Use the <a href="/a/app/contact-us">Contact Us</a> link to make suggestions. We look 
   forward to working with you!
  </p>
  
  <h3>How to submit data for integration in VEuPathDB</h3>

  <p>To submit your data for integration, fill out the appropriate VEuPathDB Dataset nomination form listed below. If your data 
    cannot be submitted via our forms, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
    link to send a brief description (two or three sentences) of your data.
  </p>

  <p>Genomes & high throughput sequencing data (e.g. RNA-Seq, ChiP-Seq, isolates typed by Whole Genome Sequencing or by 
    sequencing limited genetic loci) must be available in The 
    International Nucleotide Sequence Database Collaboration (INSDC) such as NCBI GenBank, EMBL-EBI ENA or DDBJ.</p>
    Once the dataset is prioritised for loading, we will export the data directly from INSDC. Note: while genome sequences 
    must be available in INSDC, functional annotation (e.g. gene names, GO terms, etc.) can be submitted directly to VEuPathDB.
  </p>

  <p>Tell us about your data as early as possible, to allow ample time for scheduling into VEuPathDB release cycles. 
    Depending on the dataset type, we can provide instructions on how to transfer your data to us (e.g. formats of proteomics 
    datasets may differ depending on the nature and scale of the data to be transferred), or we may be able to facilitate data 
    submission to a repository (e.g. GenBank, GEO/ArrayExpress, etc.).
  </p>

  <p>For the gene manual annotations, the change of exon-intron boundaries and creation of new genes can be submitted via Apollo for all VectorBase genomes and some of the other VEuPathDB organisms too. The gene metadata (gene names/symbols and functional description) can be submitted by sending a spreadsheet file, follow this link (FAQ page coming soon!) for information about the 12 columns heading that are required. Before you submit a manuscript for publication, please inquire if we can generate new GeneIDs that can be linked out to within the publication; this process can take a few months, we strongly encourage you to please contact us as early as possible to allow ample time for us to process your data. Gene deletions are not handled via Apollo, please Contact Us with supporting evidence.
  Community annotations made via Apollo are now represented on gene pages in the Gene Models graphic. In addition, links from the gene page to Apollo are offered to facilitate community annotations. <a href="https://qa.community.eupathdb.org/assets/images/51-apollo-news.png">Example image.</a>  Please see our <a href="https://youtu.be/RMXZDBMh8Fo" target="_blank">Apollo webinar</a> for instruction. 
  </p>    

<!-- =========== Genome Sequence and Annotation =========== -->

<details>
  <summary><b>Genome Sequence and/or Annotation</b></summary>
  <p>Genomes must be available in The International Nucleotide Sequence Database Collaboration (INSDC) such as NCBI GenBank, 
        EMBL-EBI ENA or DDBJ.
  </p>
    <ul>
      <li>If your genome <b>IS</b> uploaded to a repository, complete the <a href="https://docs.google.com/forms/d/e/1FAIpQLSchN8HUyZHP6v8B505y42YZV4Elb_vly1NUgToKhyz-kUjf9g/viewform">
          Genome Sequence and/or Annotation Description Form</a> making sure to include the accession numbers of your data 
          when prompted. We will download your data from the repository.
      </li>
      <li>If the annotation file <b>is not</b> uploaded to a repository, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
          form to send us the genome annotation file only (e.g. gff file format).
      </li>
  </ul>
   <p></p>
</details>

<!-- =========== High Throughput or Next Generation Sequencing =========== -->
<details>
  <summary><b>High Throughput or Next Generation Sequencing</b> &#8211; RNA, DNA or ChIP Sequencing</summary>
  <p>We prefer to download the raw read data in FASTQ format from a sequence read archive. 
     We integrate the data into the database using the raw reads and use the raw reads during future 
     database releases to remap or update our analyses when necessary.
  </p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Upload your data to a sequence read archive such as the European Nucleotide Archive or 
          NCBI's Sequence Read Archive, and provide us the accession numbers. We will export the data directly from INSDC</li>
      <li>Complete the appropriate data description form making sure to enter your data archive accession numbers when
          prompted</li>
    </ul>
  <p>
    <a href="https://docs.google.com/a/apidb.org/forms/d/1f4UCA6tCaAuItKkh97Xg-K_mXz8Pic5SejejOlo09QY/edit?usp=sharing">RNA-Seq 
       Data Description Form</a>
  </p>
  <p><a href="https://docs.google.com/forms/d/1z4rrMp2kjJSkqz6EbwglF6oK9xp_0_VUz9AjfPg4FWc/viewform">DNA-Seq
        Data Description Form</a>
  </p>
  <p><a href="https://docs.google.com/forms/d/11Qt40E13XTmIUbdftNXVQ6lEr2VIvZPFc0__jrZILJc/viewform">ChIP-Sequencing
         Data Description Form</a>
  </p>
  <p></p>
</details>

<!-- =========== Microarray =========== -->
<details>
  <summary><b>Microarray</b></summary>
  <p>Transfer a copy of your data to VEuPathDB using one of these options:</p>
    <ul>
      <li>Upload your data to a repository such as GeneExpression Omnibus.
          Complete the data description form linked below making sure to enter your data archive accession numbers when prompted.
          We will export the data directly from a repository.
      </li>
      <li>Send your data as an attachment to an email. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a>
          form to send us an email. Files (e.g. CEL, CSV) must include expression levels and probe set information.Pay special attention to clearly 
          indicate the identity of columns in the data files you transferred to VEuPathDB.
      </li>
   </ul>
  <p><a href="https://docs.google.com/a/apidb.org/forms/d/1pzD53hCY0rH7JYomUH8cjwJXJlOSrtZRc_mFowfFfU4/edit?usp=sharing">
        Microarray Data Description Form</a>
  </p>
  <p></p>
</details>

<!-- =========== Proteomics =========== -->
<details>
  <summary><b>Proteomics</b></summary>
  <p>Excel or tab delimited text files are preferred. We can accommodate xml file format. Required columns include gene IDs, 
     peptide sequences, peptide counts and scores.</p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Send your data as an attachment to an email. Use the 
        <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
           form to send us an email.
      </li>
      <li>Complete the 
        <a href="https://docs.google.com/a/apidb.org/forms/d/1Yx9qGKDyCf2Wm5lnf1-dFHArdQAQ9tVbR8gvxdY7n5A/edit?usp=sharing">
           Proteomics Data Description Form</a> making sure to clearly indicate the content of each column in your file.
      </li>
  </ul>
  <p></p>
</details>

<!-- =========== Quantitative Proteomics =========== -->

<details>
  <summary><b>Quantitative Proteomics</b></summary>
  <p>Excel or tab delimited files are preferred. We can accommodate xml file format. Required columns include gene IDs and 
     scores.</p>
  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Send your data as an attachment to an email. Use the 
        <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
           form to send us an email.
      </li>
      <li>Complete the 
        <a href="https://docs.google.com/a/apidb.org/forms/d/1jC1w4wWb_tNWPoL55F0zLq8o8rqOXQjtw27dt9lKups/viewform">Quantitative 
           Proteomics Data Description Form</a> making sure to include a description of data columns, for example, time course
           units and arrangement if not apparent from column headers.>
      </li>
    </ul>
  <p></p>
</details>

<!-- =========== ChIP-chip =========== -->

<details>
  <summary><b>ChIP-chip</b></summary>
  <p>Your data files should include expression levels and probe set information.</p>
  <p>Transfer a copy of your data to VEuPathDB using one of these options:</p>
    <ul>
      <li>Upload your data to a repository such as Gene Expression Omnibus and complete 
          the <a href="https://docs.google.com/forms/d/1E1QN9dKGc9rK_Bd3t-4ugIeVAyNXZoX1Z3bcSKEIMJc/viewform">ChIP-chip
          Data Description Form</a> making sure to enter the archive accession numbers (if any) for your data when prompted.
      </li>
      <li>Send your data as an attachment to an email. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact
          Us</span></a> form to send us an email.
      </li>
    </ul>
  <p></p>
</details>


<!-- =========== Isolates typed by =========== -->

<details>
  <summary><b>Isolates typed by sequencing limited genetic loci</b></summary>
  <p>If your data <b>ARE already</b> uploaded to the GenBank, use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
     form to tell us about your data. Note: GeneBank Isolate records and the associated metadata are automatically
     updated with each VEuPathDB release. There is no need to complete our Isolate Submission Form.</p>
   
   <p>If your data <b>ARE NOT</b> alredady uploaded to the GenBank, we can facilitate this upload. Complete the Isolate 
      Submission Form linked below and we will use the information to generate a GenBank submission for your isolates. 
      The new isolate records will be downloaded to VEuPathDB when the records are released by the GenBank. Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> 
      form to send us instructions for retrieving your data.</p>

   <p><a href="https://veupathdb.org/tutorials/IsolateSubmissionEuPathDBv8.xls">Isolate Submission Form</a></p>
   <p><a href="https://veupathdb.org/tutorials/EuPathDB_Isolatesubmission_genericHelp.pdf">Help for submitting Isolate Data</a></p>

</details>

<!-- =========== Isolates or Strains =========== -->

<details>
  <summary><b>Isolates or Strains typed by High-Throughput Sequencing</b></summary>

  <p>We prefer to receive the raw read data in FASTQ or FASTA file format. We integrate your data into the database using 
     the raw reads. We also use the raw reads during future database releases to remap your data when the reference genome is 
     reloaded and to update our analyses when needed.</p>

  <p>How to transfer a copy of your data to VEuPathDB:</p>
    <ul>
      <li>Upload your data to a sequence read archive such as DNA Data Bank of Japan, the European Nucleotide Archive or 
          NCBI's Sequence Read Archive. We will retrieve your data using the read archive's accession numbers for your data 
          set.
      </li>
      <li>Complete our <a href="https://docs.google.com/a/apidb.org/forms/d/1z4rrMp2kjJSkqz6EbwglF6oK9xp_0_VUz9AjfPg4FWc/viewform">  
          DNA Seq Data Description Form</a> making sure to enter the read archive accession numbers for your data when prompted.
          We also ask that you complete an abbreviated <a href="https://veupathdb.org/tutorials/IsolateSubmissionEuPathDBv8.xls">Abbreviated 
          Isolate Submission Form</a> to describe meta data associated with your isolates.
      </li>
    </ul>
  <p></p>
</details>

<!-- =========== Bulk user comments form =========== -->
<details>
  <summary><b>Bulk user comments form</b></summary>
  <p>Instructions: download the form, fill out required fields and provide any additional information, if available, and send it back to the email listed in the form</p>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/17Uu_KvfpU6FYvL0ftA7zSt1fK7OinDRZp6Dng_RPH7Y/edit?usp=sharing">
           Bulk user comments form</a>
      </li>
  <p></p>
</details>

<!-- =========== Phenotype collection form =========== -->
<details>
  <summary><b>Phenotype collection form</b></summary>
  <p>Instructions: download the form, fill out required fields and provide any additional information, if available, and send it back to the email listed in the form</p>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/1uhkr7q8QWL7ZpXLpFO3Wb5Vj9GHK47ylrzUXGbOYL9w/edit?usp=sharing">
           FungiDB phenotype collection form</a>
      </li>
      <li>Download the 
        <a href="https://docs.google.com/spreadsheets/d/1mcoPimtWkL2nqH_5qdMerVuinIUPG0t19s_uEF8BHOQ/edit#gid=783150795">
           Parasite phenotype collection form</a>
      </li>
  <p></p>
</details>


<!-- =========== General Data Submission =========== -->
<div>
  <p><b>General Data Submission</b>
   &#8211; Use the <a href="/a/app/contact-us" class="new-window" data-name="contact_us">Contact Us</a> form to tell us
   about data that does not fit any of the above categories
  </p>
</div>


</div>

<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>VEuPathDB tutorials</h1>

<div class="static-content"> 

 
<a name="Apollo:Manualgeneannotation"></a>
<h3>Apollo: Manual gene annotation</h3> 
<i style="font-size: 90%">(updated: 03 Jun 2021)</i>
<br /><br />
Structural and functional community curation with Apollo, a real time collaborative genome annotation and curation platform
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Apollo_functional_annotation_03_06_2021.pdf">Apollo: Functional annotation</a></li>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Apollo_structural_annotation_14_06_2021.pdf">Apollo: Structural annotation</a></li>
  
</ul>
<hr />
 
<a name="GenePages"></a>
<h3>Gene Pages</h3> 
<i style="font-size: 90%">(updated: 27 Oct 2020)</i>
<br /><br />
Explore the data and images on gene pages
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Exploring-the-Gene-Page.pdf">Gene Pages</a></li>
  
</ul>
<hr />
 
<a name="GeneticVariation"></a>
<h3>Genetic Variation</h3> 
<i style="font-size: 90%">(updated: 27 Oct 2020)</i>
<br /><br />
SNPs and CNV based on whole genome sequencing
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Genetic-Variation.pdf">Genetic Variation</a></li>
  
</ul>
<hr />
 
<a name="GenomeAnnotation"></a>
<h3>Genome Annotation</h3> 
<i style="font-size: 90%">(updated: 27 Oct 2020)</i>
<br /><br />
Annotate your genome against a reference using Companion, an easy and reliable parasite genome annotation resource
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Companion-1.pdf">Companion (Part 1)</a></li>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Companion-2.pdf">Companion (Part 2)</a></li>
  
</ul>
<hr />
 
<a name="GenomeBrowser"></a>
<h3>Genome Browser</h3> 
<i style="font-size: 90%">(updated: 15 Apr 2020)</i>
<br /><br />
Basic tutorial on how to effectively explore data in the genome browser including track selection and configuration
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/JBrowseBasics.pdf">Genome browser basics</a></li>
  
</ul>
<hr />
 
<a name="GenomeBrowserAdvanced"></a>
<h3>Genome Browser Advanced</h3> 
<i style="font-size: 90%">(updated: 27 Oct 2020)</i>
<br /><br />
Self-led exercise on investigating genes models and structural annotations with RNA-seq expression data
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/JBrowse-RNAseq.pdf">RNA-Seq data &amp; gene models</a></li>
  
</ul>
<hr />
 
<a name="GOEnrichment"></a>
<h3>GO Enrichment</h3> 
<i style="font-size: 90%">(updated: 27 Oct 2020)</i>
<br /><br />
Run a GO enrichment on a set of genes from a fold change RNA-Seq search
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/GO-Enrichment.pdf">GO Enrichment</a></li>
  
</ul>
<hr />
 
<a name="HostResponse"></a>
<h3>Host Response</h3> 
<i style="font-size: 90%">(updated: 27 Oct 2020)</i>
<br /><br />
Explore host responses by building a search strategy in HostDB
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Host-response.pdf">Host Response (HostDB)</a></li>
  
</ul>
<hr />
 
<a name="MapVEu"></a>
<h3>MapVEu</h3> 
<i style="font-size: 90%">(updated: 05 May 2020)</i>
<br /><br />
use the map to visualize, search, and analyze field-collected data including insecticide resistance (phenotypes and genotypes), population abundance, pathogen infection status, and blood meal source among others
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/MapVEu_part1_05_05_2020.pdf">Introduction to MapVEu</a></li>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/MapVEu_part2_05_05_2020.pdf">Additional features of MapVEu</a></li>
  
</ul>
<hr />
 
<a name="MetabolicPathways"></a>
<h3>Metabolic Pathways</h3> 
<i style="font-size: 90%">(updated: 28 Oct 2020)</i>
<br /><br />
Learn to use metabolic pathway searches and visualization tools
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Metabolic-Pathways.pdf">Metabolic Pathways</a></li>
  
</ul>
<hr />
 
<a name="OrthologyPhyleticPatterns"></a>
<h3>Orthology Phyletic Patterns</h3> 
<i style="font-size: 90%">(updated: 28 Oct 2020)</i>
<br /><br />
Leverage the transform by orthology tool, orthology tables on gene pages, OrthoMCL, and the phyletic pattern search
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Orthology-Phyletic-Patterns.pdf">Orthology and Phyletic Patterns</a></li>
  
</ul>
<hr />
 
<a name="OrthoMCLtutorial"></a>
<h3>OrthoMCL tutorial</h3> 
<i style="font-size: 90%">(updated: 17 Dec 2020)</i>
<br /><br />
Learn standard searches and ortholog group analysis within the OrthoMCL website
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/OrthoMCL_tutorial1.pdf">OrthoMCL tutorial</a></li>
  
</ul>
<hr />
 
<a name="Regularexpressions&amp;colocation"></a>
<h3>Regular expressions &amp; colocation</h3> 
<i style="font-size: 90%">(updated: 15 Apr 2020)</i>
<br /><br />
Use regular expressions to identify protein or DNA patters. Use the colocation tool to identify DNA segments based on their relative location on the genome
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/RegEXcolocate.pdf">Regular expressions &amp; genomic colocation</a></li>
  
</ul>
<hr />
 
<a name="SearchStrategies(Basics)"></a>
<h3>Search Strategies (Basics)</h3> 
<i style="font-size: 90%">(updated: 28 Oct 2020)</i>
<br /><br />
Run basic searches, extend them into strategies, and explore search results
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/SearchStrategies.pdf">Search Strategies (Basics)</a></li>
  
</ul>
<hr />
 
<a name="SearchStrategies(Advanced)"></a>
<h3>Search Strategies (Advanced)</h3> 
<i style="font-size: 90%">(updated: 28 Oct 2020)</i>
<br /><br />
Integrate diverse data types and leverage orthology or phylogenetic profiling
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Advanced-Search-Strategies.pdf">Search Strategies (Advanced)</a></li>
  
</ul>
<hr />
 
<a name="SiteSearch"></a>
<h3>Site Search</h3> 
<i style="font-size: 90%">(updated: 27 Oct 2020)</i>
<br /><br />
Explore site search and learn how to filter your results or export them to a search strategy
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/SiteSearch.pdf">Site Search</a></li>
  
</ul>
<hr />
 
<a name="TranscriptomicsandProteomics"></a>
<h3>Transcriptomics and Proteomics</h3> 
<i style="font-size: 90%">(updated: 28 Oct 2020)</i>
<br /><br />
Explore searches that return genes based on expression evidence from RNA-Seq, microaray and proteomics searches
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/Transcriptomic-proteomics.pdf">Expression searches</a></li>
  
</ul>
<hr />
 
<a name="VariantCalling"></a>
<h3>Variant Calling</h3> 
<i style="font-size: 90%">(updated: 15 May 2020)</i>
<br /><br />
Analyze DNA sequence for variants using a workflow in VEuPathDB Galaxy
<ul>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/VariantAnalysisVEuPathDBGalaxyPart1.pdf">Variant calling in VEuPathDB galaxy (Part 1)</a></li>
  
  <li>
    
    <a href="https://qa.community.eupathdb.org/documents/VariantAnalysisVEuPathDBGalaxyPart2.pdf">Variant calling in VEuPathDB galaxy (Part 2)</a></li>
  
</ul>
<hr />


</div>

<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>Related sites</h1>

<div class="static-content">

<table border="0" cellpadding="2" cellspacing="0" width="100%">

    <tr><td><ul>
    
    
      
      
      <li><a href="http://workshop.eupathdb.org/">Previous EuPathDB Workshops</a></li>
    
      
      
      <li><a href="https://companion.sanger.ac.uk">Companion</a></li>
    
      
      
      <li><a href="http://orthomcl.org/">OrthoMCL</a></li>
    
      
      
      <li><a href="http://www.genedb.org/">GeneDB</a></li>
    
      
      
      <li><a href="http://modbase.compbio.ucsf.edu/modbase-cgi/search_form.cgi">ModBase at UCSF</a></li>
    
      
      
      <li><a href="http://www.ciliate.org/">Tetrahymena Genome</a></li>
    
      
      
      <li><a href="http://www.arabidopsis.org/">The Arabidopsis Information Resource</a></li>
    
      
      
      <li><a href="https://www.oxfordjournals.org/nar/database/c/">NAR Database Summary Paper Categories</a></li>
    
    </ul></td></tr>

  


</table>

</div>
<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>VEuPathDB Publications and Citations</h1>

<div class="static-content">

<table width="100%">
   <tr> <td>
      View <b><a target="_blank" href="https://scholar.google.com/scholar?hl=en&as_sdt=0,39&q=OrthoMCL+OR+PlasmoDB+OR+ToxoDB+OR+CryptoDB+OR+TrichDB+OR+GiardiaDB+OR+TriTrypDB+OR+AmoebaDB+OR+MicrosporidiaDB+OR+%22FungiDB%22+OR+PiroplasmaDB+OR+%22vectorbase%22+OR+veupathdb+OR+ApiDB+OR+EuPathDB+-encrypt+-cryptography+-hymenoptera">
         publications that cite us</a></b> in Google Scholar  </td>
</tr>
<tr> <td>
  <b>This <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%28%22eupathdb%22%5BTitle%5D+OR+%22plasmodb%22%5BTitle%5D+OR+%22toxodb%22%5BTitle%5D+OR+%22giardiadb%22%5BTitle%5D+OR+%22amoebadb%22%5BTitle%5D+OR+%22microsporidiadb%22%5BTitle%5D+OR+%22tritrypdb%22%5BTitle%5D+OR+%22vectorbase%22%5BTitle%5D+OR+%22fungidb%22%5BTitle%5D+OR+%22apidb%22%5BTitle%5D+OR+%22cryptodb%22%5BTitle%5D+OR+%22cryptodb.org%22%5Btitle%5D+OR+%22piroplasmadb%22%5BTitle%5D+OR+%22trichdb%22%5BTitle%5D+OR+%22orthomcl%22%5BTitle%5D%29+AND+%28Roos+%5Bauthor%5D+OR+kissinger+%5Bauthor%5D+OR+collins+%5Bauthor%5D%29&sort=date">
     PubMed filter</a> provides a current list of the most recent publications about VEuPathDB resources</b></td>
</tr>
</table>

<table border=0 width=100% cellpadding=3 cellspacing=0 bgcolor=white class=thinTopBottomBorders> 
<tr><td bgcolor=white valign=top>
 
  <table border="0" cellpadding="2" cellspacing="0" width="100%"><tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      (Book Chapter)
      <br>
      Omar Harb, Jessica C. Kissinger and David S. Roos on behalf of the EuPathDB group 
      <br>
      <b>ToxoDB: the functional genomic resource for <i>Toxoplasma</i></b>
      <br>
      <i>Toxoplasma gondii</i> 3rd edition
      <br>
      Edited by Louis Weiss and Kami Kim (2020)<br>
       <a href="https://doi.org/10.1016/B978-0-12-815041-2.00023-2">https://doi.org/10.1016/B978-0-12-815041-2.00023-2</a>
      </td>
    </tr></table>

    <table border="0" cellpadding="2" cellspacing="0" width="100%"><tr><td colspan="2"><hr></td></tr>
      <tr class="rowLight"><td>
        (Book Chapter)
        <br>
        Susanne Warrenfeltz and Jessica Kissinger on behalf of the EuPathDB Consortium
        <br>
          <b><a target="_blank" href="https://link.springer.com/protocol/10.1007%2F978-1-4939-9748-0_10">Accessing <i>Cryptosporidium</i> omic & isolate data via CryptoDB.org</a></b>
        <br>
        Methods in Molecular Biology 
        <br>
        Editor, Jan Mead (2020)<br>
         <a href="https://doi.org/10.1007/978-1-4939-9748-0_22">https://doi.org/10.1007/978-1-4939-9748-0_22</a>
        
        </td>
      </tr></table>

      <table border="0" cellpadding="2" cellspacing="0" width="100%"><tr><td colspan="2"><hr></td></tr>
        <tr class="rowLight"><td>
          (Book Chapter)
          <br>
          Omar S. Harb and David S. Roos on behalf of the EuPathDB Consortium 
          <br>
            <b>ToxoDB: Functional Genomics Resource for <i>Toxoplasma</i> and Related Organisms</b>
          <br>
          Methods in Molecular Biology 
          <br>
          Editor, Christopher J. Tonkin (2020)<br>
           <a href="https://doi.org/10.1007/978-1-4939-9857-9_2">https://doi.org/10.1007/978-1-4939-9857-9_2</a>
          </td>
        </tr></table>


   <table border="0" cellpadding="2" cellspacing="0" width="100%"><tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    (Book Chapter)
    <br>
    Susanne Warrenfeltz, Evelina Y Basenko, Kathryn Crouch, Omar S. Harb, Jessica C. Kissinger, Achchuthan Shanmugasundram and Fatima Silva-Franco 
    <br>
    <b>EuPathDB: the eukaryotic pathogen genomics database resource</b>
    <br>
    Methods in Molecular Biology   
    <br>
    Editor, Martin Kollmar (2018)<br>
     <a href="https://doi.org/10.1007/978-1-4939-7737-6_5">https://doi.org/10.1007/978-1-4939-7737-6_5</a>
    </td>
  </tr></table>
    
  <table border="0" cellpadding="2" cellspacing="0" width="100%"><tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Evelina Y. Basenko, Jane A. Pulman, Achchuthan Shanmugasundram, Omar S. Harb, Kathryn Crouch, David Starns, Susanne Warrenfeltz, Cristina Aurrecoechea, Christian J. Stoeckert, Jr., Jessica C. Kissinger, David S. Roos and Christiane Hertz-Fowler
    <br>
    <b>FungiDB: An Integrated Bioinformatic Resource for Fungi and Oomycetes. (2018)</b>
    <br>
    J. Fungi 2018, 4(1), 39<br>
     <a href=" https://doi.org/10.3390/jof4010039"> https://doi.org/10.3390/jof4010039</a>
    </td>
  </tr></table>
  
  
  
  <table border="0" cellpadding="2" cellspacing="0" width="100%"><tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Francislon S Oliveira, John Brestelli, Shon Cade, Jie Zheng, John Iodice, Steve Fischer, Cristina Aurrecoechea, Jessica C Kissinger, Brian P Brunk, Christian J Stoeckert, Jr, Gabriel R Fernandes, David S Roos, Daniel P Beiting
     <br>
    <b>MicrobiomeDB: a systems biology platform for integrating, mining and analyzing microbiome experiments</b>
    <br>
    <a target="_blank" href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkx1027/4584629">Nucleic Acids Research. 2017 doi.org/10.1093/nar/gkx1027</a>

    </td>
  </tr></table>
  
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Cristina Aurrecoechea; Ana Barreto; Evelina Y. Basenko; John Brestelli; Brian P. Brunk; Shon Cade; Kathryn Crouch; Ryan Doherty; Dave Falke; Steve Fischer; Bindu Gajria; Omar S. Harb; Mark Heiges; Christiane Hertz-Fowler; Sufen Hu; John Iodice; Jessica C. Kissinger; Cris Lawrence; Wei Li; Deborah F. Pinney; Jane A. Pulman; David S. Roos; Achchuthan Shanmugasundram; Fatima Silva-Franco; Sascha Steinbiss; Christian J. Stoeckert Jr; Drew Spruill; Haiming Wang; Susanne Warrenfeltz; Jie Zheng<br>
    <b>EuPathDB: the eukaryotic pathogen genomics database resource</b><br>
    <a target="_blank" href="http://nar.oxfordjournals.org/content/early/2016/11/29/nar.gkw1105.abstract?keytype=ref&ijkey=R3xQYyrwxYzNSRA">Nucleic Acids Research 2017 doi: 10.1093/nar/gkw1105</a>
    </td>
  </tr></table>
   
  
<table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Mitraka, E., Topalis, P., Dritsou, V., Dialynas, E., & Louis, C.<br> 
   <b>Describing the breakbone fever: IDODEN, an ontology for dengue fever</b><br> 
   PLoS neglected tropical diseases, 3 February 2015; 9(2), e0003479<br> 
   <a href="https://doi.org/10.1371/journal.pntd.0003479">https://doi.org/10.1371/journal.pntd.0003479</a>
   </td>
</tr></table>
  
   
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Warren, A. S., Aurrecoechea, C., Brunk, B., Desai, P., Emrich, S., Giraldo-Calderón, G. I., Harb, O., Hix, D., Lawson, D., Machi, D., Mao, C., McClelland, M., Nordberg, E., Shukla, M., Vosshall, L. B., Wattam, A. R., Will, R., Yoo, H. S., & Sobral, B.<br> 
   <b>RNA-Rocket: an RNA-Seq analysis resource for infectious disease research</b><br> 
     Bioinformatics, 1 May 2015; 31(9), 1496–1498<br> 
     <a href="https://doi.org/10.1093/bioinformatics/btv002">https://doi.org/10.1093/bioinformatics/btv002</a>
</td>
</tr></table>

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      Giraldo-Calder&oacute;n GI, Emrich SJ, MacCallum RM, Maslen G, Dialynas E, Topalis P, Ho N, Gesing S; VectorBase Consortium, Madey G, Collins FH, Lawson D.<br>
      <b>VectorBase: an updated bioinformatics resource for invertebrate vectors and other organisms related with human diseases</b><br>
       Nucleic Acids Res. 2015 Jan;43(Database issue):D707-13<br> 
      <a target="_blank" href="https://doi.org/10.1093/nar/gku1117">https://doi.org/10.1093/nar/gku1117</a>
      </td>
    </tr></table>

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Harb OS, Roos DS.<br>
    <b>The Eukaryotic Pathogen Databases: a functional genomic resource integrating data from human and veterinary parasites</b><br>
    Methods Mol Biol. 2015;1201:1-18<br>
    [ <a href="http://link.springer.com/protocol/10.1007%2F978-1-4939-1438-8_1">Full text</a>]
    </td></tr></table>
  
   
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Dugan VG, Emrich SJ, Giraldo-Calderón GI, Harb OS, Newman RM,
Pickett BE, Schriml LM, Stockwell TB, Stoeckert CJ Jr, Sullivan
DE, Singh I, Ward DV, Yao A, Zheng J, Barrett T, Birren B,
Brinkac L, Bruno VM, Caler E, Chapman S, Collins FH, Cuomo CA, 
Di Francesco V, Durkin S, Eppinger M, Feldgarden M, Fraser C,
Fricke WF, Giovanni M, Henn MR, Hine E, Hotopp JD, Karsch-Mizrachi
I, Kissinger JC, Lee EM, Mathur P, Mongodin EF, Murphy CI,
Myers G, Neafsey DE, Nelson KE, Nierman WC, Puzak J, Rasko D, 
Roos DS, Sadzewicz L, Silva JC, Sobral B, Squires RB, Stevens
RL, Tallon L, Tettelin H, Wentworth D, White O, Will R,
Wortman J, Zhang Y, Scheuermann RH.<br>
    <b>Standardized metadata for human pathogen/vector genomic sequences</b><br>
    PLoS One. 2014 Jun 17;9(6):e99979<br>
    [ <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0099979">Full text</a> | <a href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0099979&representation=PDF">PDF</a> ]
    </td></tr></table>
 
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Topalis, P., Mitraka, E., Dritsou, V., Dialynas, E., & Louis, C.<br> 
<b>IDOMAL: the malaria ontology revisited</b><br> 
   Journal of biomedical semantics, 13 September 2013; 4(1), 16<br> 
   <a href="https://doi.org/10.1186/2041-1480-4-16">https://doi.org/10.1186/2041-1480-4-16</a>
     </td></tr></table>

   
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Aurrecoechea C, Barreto A, Brestelli J, Brunk BP, Cade S, Doherty R, Fischer
S, Gajria B, Gao X, Gingle A, Grant G, Harb OS, Heiges M, Hu S, Iodice J,
Kissinger JC, Kraemer ET, Li W, Pinney DF, Pitts B, Roos DS, Srinivasamoorthy G, 
Stoeckert CJ Jr, Wang H, Warrenfeltz S.<br>
    <b>EuPathDB: the eukaryotic pathogen database</b><br>
    Nucleic Acids Res. 2013 Jan;41(Database issue):D684-91<br>
    [ <a href="http://nar.oxfordjournals.org/content/41/D1/D684.long">Full text</a> | <a href="http://nar.oxfordjournals.org/content/41/D1/D684.full.pdf+html">PDF</a> ]
    </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Zerlotini A, Aguiar ER, Yu F, Xu H, Li Y, Young ND, Gasser RB, Protasio AV,
Berriman M, Roos DS, Kissinger JC, Oliveira G.<br>
    <b>SchistoDB: an updated genome resource for the three key schistosomes of humans</b><br>
    Nucleic Acids Res. 2013 Jan;41(Database issue):D728-31<br>
    [ <a href="http://nar.oxfordjournals.org/content/41/D1/D728.long">Full text</a> | <a href="http://nar.oxfordjournals.org/content/41/D1/D728.full.pdf+html">PDF</a> ]
    </td></tr></table>
  
<table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td> 
Megy, K., Emrich, S. J., Lawson, D., Campbell, D., Dialynas, E., Hughes, D. S., Koscielny, G., Louis, C., Maccallum, R. M., Redmond, S. N., Sheehan, A., Topalis, P., Wilson, D., & VectorBase Consortium<br> 
<b>VectorBase: improvements to a bioinformatics resource for invertebrate vector genomics</b><br> 
Nucleic acids research, 1 January 2012; 40(Database issue), D729–D734<br> 
<a href="https://academic.oup.com/nar/article/40/D1/D729/2903049">https://doi.org/10.1093/nar/gkr1089</a>  
    </td></tr></table>


  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Stajich JE, Harris T, Brunk BP, Brestelli J, Fischer S, Harb OS, Kissinger JC, Li W, Nayak V, Pinney DF, Stoeckert CJ Jr, Roos DS.<br>
    <b>FungiDB: an integrated functional genomics database for fungi</b><br>
    Nucleic Acids Res. 2012 Jan;40(Database issue):D675-81<br>
    [ <a href="http://nar.oxfordjournals.org/content/40/D1/D675.abstract">Abstract</a> | <a href="http://nar.oxfordjournals.org/content/40/D1/D675.full">Full text</a> | <a href="http://nar.oxfordjournals.org/content/40/D1/D675.full.pdf+html">PDF</a> ]
    </td></tr></table>
  
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Topalis, P., Dialynas, E., Mitraka, E., Deligianni, E., Siden-Kiamos, I., & Louis, C.<br> 
   <b>A set of ontologies to drive tools for the control of vector-borne diseases</b><br> 
   Journal of biomedical informatics, February 2011; 44(1), 42–47<br> 
     <a href="https://doi.org/10.1016/j.jbi.2010.03.012">https://doi.org/10.1016/j.jbi.2010.03.012</a>
     </td></tr></table>

      
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Maccallum, R. M., Redmond, S. N., & Christophides, G. K.<br> 
<b>An expression map for <i>Anopheles gambiae</i></b><br> 
   BMC genomics, 20 December 2011; 12, 620<br> 
   <a href="https://doi.org/10.1186/1471-2164-12-620">https://doi.org/10.1186/1471-2164-12-620</a>
 </td></tr></table>
   
  
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Fischer S, Brunk BP, Chen F, Gao X, Harb OS, Iodice JB, Shanmugam D, Roos DS, Stoeckert CJ Jr.<br>
    <b>Using OrthoMCL to assign proteins to OrthoMCL-DB groups or to cluster proteomes into new ortholog groups</b><br>
    Curr. Protoc. Bioinform. 35:6.12.1-6.12.19. © 2011 by John Wiley & Sons, Inc. PMID: 21901743<br>
    [ <a href="http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi0612s35/abstract">Abstract</a> | <a href="http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi0612s35/full">Full text</a> | <a href="http://onlinelibrary.wiley.com/doi/10.1002/0471250953.bi0612s35/pdf">PDF</a> ]
    </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Aurrecoechea C., Barreto A, Brestelli J., Brunk BP., Caler EV., Fischer S., Gajria B., Gao X., Gingle A, Grant G., Harb OH., Heiges M., Iodice J., Kissinger JC., Kraemer ET., Li W., Nayak V., Pennington C., Pinney DF., Pitts B., Roos DS., Srinivasamoorthy G., Stoeckert CJ Jr., Treatman C., and Wang H.<br>
    <b>AmoebaDB and MicrosporidiaDB: functional genomic resources for Amoebozoa and Microsporidia species</b><br>
    Nucleic Acids Research. 39(Database issue):D612-9, 2011 Jan<br>
    <a href="https://doi.org/10.1093/nar/gkq1006">https://doi.org/10.1093/nar/gkq1006</a>
     </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Fischer S, Aurrecoechea C, Brunk BP, Gao X, Harb OS, Kraemer ET, Pennington C, Treatman C, Kissinger JC, Roos DS, Stoeckert CJ.<br>
    <b>The Strategies WDK: a graphical search interface and web development kit for functional genomics databases</b><br>
    Database (Oxford). 2011 Jun 23;2011:bar027. Print 2011.  PMID: 21705364 PMCID: PMC3122067<br>
    [ <a href="http://database.oxfordjournals.org/content/2011/bar027.abstract">Abstract</a> | <a href="http://database.oxfordjournals.org/content/2011/bar027.full">Full text</a> | <a href="http://database.oxfordjournals.org/content/2011/bar027.full.pdf+html">PDF</a> ]
    </td></tr></table>
   
   

   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Topalis, P., Mitraka, E., Bujila, I., Deligianni, E., Dialynas, E., Siden-Kiamos, I., Troye-Blomberg, M., & Louis, C.<br> 
     <b>IDOMAL: an ontology for malaria</b><br> 
     Malaria journal, 10 August 2010; 9, 230<br> 
     <a href="https://doi.org/10.1186/1475-2875-9-230">https://doi.org/10.1186/1475-2875-9-230</a>
 </td></tr></table>
   
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    H. Wang, J. DeBarry, J. C. Kissinger<br>
    <b>SynView - Synteny Browser with GBrowse Integration</b><br>
    Editor, Gabriel P.C. Fung (2010)<br>
    <a href="https://www.amazon.com/Practical-Guide-Bioinformatics-Analysis/dp/1453608621">A Practical Guide to Bioinformatics Analysis- 2010 iConcept Press Ltd</a><br>
    </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Aurrecoechea C. Brestelli J. Brunk BP. Fischer S. Gajria B. Gao X. Gingle A. Grant G. Harb OS. Heiges M. Innamorato F. Iodice J. Kissinger JC. Kraemer ET. Li W. Miller JA. Nayak V. Pennington C. Pinney DF. Roos DS. Ross C. Srinivasamoorthy G. Stoeckert CJ Jr. Thibodeau R. Treatman C. Wang H.<br>
    <b>EuPathDB: a portal to eukaryotic pathogen databases</b><br>
    Nucleic Acids Research 2010 38(Database issue):D415-9<br>
    [ <a href="http://nar.oxfordjournals.org/content/38/suppl_1/D415.abstract">Abstract</a> | <a href="http://nar.oxfordjournals.org/content/38/suppl_1/D415.full">Full text</a> | <a href="http://nar.oxfordjournals.org/content/38/suppl_1/D415.full.pdf+html">PDF</a> ]
    </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Aslett M, Aurrecoechea C, Berriman M, Brestelli J, Brunk BP, Carrington M, 
        Depledge DP, Fischer S, Gajria B, Gao X, Gardner MJ, Gingle A, Grant G, 
        Harb OS, Heiges M, Hertz-Fowler C, Houston R, Innamorato F, Iodice J, 
        Kissinger JC, Kraemer E, Li W, Logan FJ, Miller JA, Mitra S, Myler PJ, 
        Nayak V, Pennington C, Phan I, Pinney DF, Ramasamy G, Rogers MB, Roos DS, 
        Ross C, Sivam D, Smith DF, Srinivasamoorthy G, Stoeckert CJ Jr., 
        Subramanian S, Thibodeau R, Tivey A, Treatman C, Velarde G, Wang H.<br>
    <b>TriTrypDB: a functional genomic resource for the Trypanosomatidae</b><br>
    Nucleic Acids Research 2010 38(Database issue):D457-D462; doi:10.1093/nar/gkp851<br>
    [ <a href="http://nar.oxfordjournals.org/cgi/content/abstract/gkp851">Abstract</a> | <a href="http://nar.oxfordjournals.org/cgi/content/full/38/suppl_1/D457">Full text</a> | <a href="http://nar.oxfordjournals.org/cgi/content/full/38/suppl_1/D457">PDF</a> ]
    </td></tr></table>
  
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Dialynas, E., Topalis, P., Vontas, J., & Louis, C.<br> 
   <b>MIRO and IRbase: IT tools for the epidemiological monitoring of insecticide resistance in mosquito disease vectors</b><br> 
   PLoS neglected tropical diseases, 23 June 2009; 3(6), e465<br> 
   <a href="https://doi.org/10.1371/journal.pntd.0000465">https://doi.org/10.1371/journal.pntd.0000465</a>
   </td></tr></table>
   
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Megy, K., Hammond, M., Lawson, D., Bruggner, R. V., Birney, E., & Collins, F. H.<br> 
   <b>Genomic resources for invertebrate vectors of human pathogens, and the role of VectorBase</b><br> 
   Infection, genetics and evolution, May 2009; 9(3), 308–313<br> 
   <a href="https://doi.org/10.1016/j.meegid.2007.12.007">https://doi.org/10.1016/j.meegid.2007.12.007</a>
      </td></tr></table>

   
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Lawson, D., Arensburger, P., Atkinson, P., Besansky, N. J., Bruggner, R. V., Butler, R., Campbell, K. S., Christophides, G. K., Christley, S., Dialynas, E., Hammond, M., Hill, C. A., Konopinski, N., Lobo, N. F., MacCallum, R. M., Madey, G., Megy, K., Meyer, J., Redmond, S., Severson, D. W., … Collins, F. H.<br> 
   <b>VectorBase: a data resource for invertebrate vector genomics</b> <br> 
   Nucleic acids research, 1 January 2009; 37(Database issue), D583–D587<br> 
   <a href="https://doi.org/10.1093/nar/gkn857">https://doi.org/10.1093/nar/gkn857</a><br>
   </td></tr></table>
  
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Aurrecoechea, C., J. Brestelli, B. P. Brunk, J. Dommer, S. Fischer, 
         B. Gajria, X. Gao, A. Gingle, G. Grant, O. S. Harb, M. Heiges, F. Innamorato, 
         J. Iodice, J. C. Kissinger, E. Kraemer, W. Li, J. A. Miller, V. Nayak, 
         C. Pennington, D. F. Pinney, D. S. Roos, C. Ross, C. J. Stoeckert, Jr., 
         C. Treatman, and H. Wang<br>
    <b>PlasmoDB: a functional genomic database for malaria parasites</b><br>
    Nucleic Acids Res. 2009. 37:D539-D543<br>
    <a href="https://doi.org/10.1093/nar/gkn814">https://doi.org/10.1093/nar/gkn814</a><br>
     </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Aurrecoechea, C., J. Brestelli, B. P. Brunk, J. M. Carlton, J. Dommer, 
         S. Fischer, B. Gajria, X. Gao, A. Gingle, G. Grant, O. S. Harb, M. Heiges, 
         F. Innamorato, J. Iodice, J. C. Kissinger, E. Kraemer, W. Li, J. A. Miller, 
         H. G. Morrison, V. Nayak, C. Pennington, D. F. Pinney, D. S. Roos, 
         C. Ross, C. J. Stoeckert, Jr., S. Sullivan, C. Treatman, and H. Wang<br>
    <b>GiardiaDB and TrichDB: integrated genomic resources for the eukaryotic protist pathogens <i>Giardia lamblia</i> and <i>Trichomonas vaginalis</i></b><br>
    Nucleic Acids Res. 2009. 37:D526-D530<br>
    <a href="https://doi.org/10.1093/nar/gkn631">https://doi.org/10.1093/nar/gkn631</a>
     </td></tr></table>
  
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Topalis, P., Tzavlaki, C., Vestaki, K., Dialynas, E., Sonenshine, D.E., Butler, R., ... & Louis, C.<br> 
   <b>Anatomical ontologies of mosquitoes and ticks, and their web browsers in VectorBase</b><br>
   Insect molecular biology, 30 January 2008; 17(1), 87-9<br> 
   <a href="https://doi.org/10.1111/j.1365-2583.2008.00781.x">https://doi.org/10.1111/j.1365-2583.2008.00781.x</a> 
         </td></tr></table>
   
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Topalis, P., Lawson, D., Collins, F.H., & Louis, C.<br> 
   <b>How can ontologies help vector biology</b><br> 
   Trends in parasitology, 1 June 2008; 24(6), 249-52<br> 
   <a href="https://doi.org/10.1016/j.pt.2008.03.002">https://doi.org/10.1016/j.pt.2008.03.002</a> 
   </td></tr></table>
   
   

   
      <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
      Lawson, D., Arensburger, P., Atkinson, P., Besansky, N. J., Bruggner, R. V., Butler, R., Campbell, K. S., Christophides, G. K., Christley, S., Dialynas, E., Emmert, D., Hammond, M., Hill, C. A., Kennedy, R. C., Lobo, N. F., MacCallum, M. R., Madey, G., Megy, K., Redmond, S., Russo, S., … Collins, F. H.<br>
      <b>VectorBase: a home for invertebrate vectors of human pathogens</b><br>
      Nucleic acids research, 1 January 2007; 35(Database issue), D503–D505<br> 
      <a href="https://doi.org/10.1093/nar/gkl960">https://doi.org/10.1093/nar/gkl960</a><br>
          </td></tr></table>
   
   
  
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    B. Gajria, A. Bahl, J. Brestelli, J. Dommer, S. Fischer, 
        X. Gao, M. Heiges, J. Iodice, J. C. Kissinger, A. J. Mackey, et al.<br>
    <b>ToxoDB: an integrated <i>Toxoplasma gondii</i> database resource</b><br>
    Nucleic Acids Res. 2007. 36:D553-6<br>
    <a href="https://doi.org/10.1093/nar/gkm981">https://doi.org/10.1093/nar/gkm981</a>
     </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Cristina Aurrecoechea, Mark Heiges, Haiming Wang, Zhiming Wang, 
        Steve Fischer, Philippa Rhodes, John Miller, Eileen Kraemer, 
        Christian J. Stoeckert, Jr., David S. Roos and Jessica C. Kissinger<br>
    <b>ApiDB: integrated resources for the apicomplexan bioinformatics resource center</b><br>
    Nucleic Acids Research. 2007. 35:D427-30<br>
    <a href="https://doi.org/10.1093/nar/gkl880">https://doi.org/10.1093/nar/gkl880</a>
     </td></tr></table>
  
   
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Christian J. Stoeckert Jr., Steve Fisher, Jessica Kissinger, 
        Mark Heiges, Cristina Aurrecoechea, Bindu Gajria, David S. Roos<br>
    <b>PlasmoDB v5: New Looks, New Genomes</b><br>
    Trends in Parasitology. 2006. 22(12): 543-546<br>
    [ <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17029963&query_hl=1&itool=pubmed_docsum">Abstract</a> ]
    </td></tr></table>
  
   


  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Patricia L. Whetzel, Shailesh V. Date, Kobby Essien, Martin J. Fraunholz, 
        Bindu Gajria, Gregory R. Grant, John Iodice, Jessica C. Kissinger, 
        Philip T. Labo, Arthur J. Milgram, David S. Roos, and Christian J. Stoeckert Jr.<br>
    <b>PlasmoDB: The Plasmodium Genomics and Functional Genomics Resource</b><br>
    <i>In silico</i> Genomics and Proteomics: Functional Annotation of Genomes and Proteins. Nicola Mulder and Rolf Apweiler (eds.). Nova Science Publishers. 2006<br>
    
    </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Haiming Wang, Yanqi Su, Aaron J. Mackey, Eileen T. Kraemer and Jessica C. Kissinger<br>
    <b>SynView: A GBrowse-compatible Approach to Visualizing Comparative Genome Data</b><br>
    Bioinformatics. 2006. 22(18):2308-2309<br>
    [ <a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/22/18/2308">Abstract</a> | <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/22/18/2308">Full text</a> |<a href="http://bioinformatics.oxfordjournals.org/cgi/reprint/22/18/2308">PDF</a> | <a href="http://eupathdb.org/apps/SynView/">Supplement</a> ]
    </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Heiges M, Wang H, Robinson E, Aurrecoechea C, Gao X, Kaluskar N, 
        Rhodes P, Wang S, He CZ, Su Y, Miller J, Kraemer E, Kissinger JC.<br>
    <b>CryptoDB: a <i>Cryptosporidium</i> bioinformatics resource update</b><br>
    Nucleic Acids Res. 2006 Jan 1;34:D419-22<br>
    [ <a href="http://nar.oxfordjournals.org/cgi/content/abstract/34/suppl_1/D419">Abstract</a> | <a href="http://nar.oxfordjournals.org/cgi/content/full/34/suppl_1/D419">Full text</a> | <a href="http://nar.oxfordjournals.org/cgi/reprint/34/suppl_1/D419">PDF</a> ]
    </td></tr></table>
  

  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Feng Chen, Aaron J. Mackey, Christian J. Stoeckert Jr. and David S. Roos<br>
    <b>OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups</b><br>
    Nucleic Acids Res. 2006 Jan 1;34:D363-8<br>
    [ <a href="http://nar.oxfordjournals.org/cgi/content/abstract/34/suppl_1/D363">Abstract</a> | <a href="http://nar.oxfordjournals.org/cgi/content/full/34/suppl_1/D363">Full text</a> | <a href="http://nar.oxfordjournals.org/cgi/reprint/34/suppl_1/D363">PDF</a> ]
    </td></tr></table>
  
  
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Kriventseva, E. V., Koutsos, A. C., Blass, C., Kafatos, F. C., Christophides, G. K., & Zdobnov, E. M.<br> 
   <b>AnoEST: toward <i>A. gambiae</i> functional genomics</b><br> 
   Genome research, 17 May 2005; 15(6), 893–899<br>
   <a href="https://doi.org/10.1101/gr.3756405">https://doi.org/10.1101/gr.3756405</a>
       </td></tr></table>

      


 
  <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    Stoeckert CJ Jr.<br>
    <b>Functional genomics databases on the web</b><br>
    Cell Microbiol. 2005 7(8), 1053-9<br>
    <a href="https://doi.org/10.1111/j.1462-5822.2005.00553.x">https://doi.org/10.1111/j.1462-5822.2005.00553.x</a>
     </td></tr></table>
   
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
    J.C. Kissinger and D.S. Roos<br>
    <b>Getting the most out of bioinformatics resources</b><br>
    <i>In</i>: Malaria Parasites, A.P. Waters & C.J. Janse, editors.  Horizon, Norfolk UK. 2004<br>
     <a href="https://www.caister.com/hsp/abstracts/mal/03.html">https://www.caister.com/hsp/abstracts/mal/03.html</a>
    </td></tr></table>
   
   <table border="0" cellpadding="2" cellspacing="0" width="100%">
  <tr><td colspan="2"><hr></td></tr>
  <tr class="rowLight"><td>
   Ribeiro, J.M., Topalis, P., & Louis, C.<br> 
   <b>AnoXcel: an Anopheles gambiae protein database</b><br>
   Insect molecular biology, 16 September 2004; 13(5), 449-57<br>
   <a href="https://doi.org/10.1111/j.0962-1075.2004.00503.x">https://doi.org/10.1111/j.0962-1075.2004.00503.x</a>  
      </td></tr></table>

 

</div> 
<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<style>

div.method-details {
  margin: 2em;
}

</style>

<h1>VEuPathDB Data Analysis Methods </h1>

<div class="static-content"> 

<p>VEuPathDB draws data from many sources.  To facilitate comparisons across data sets, we analyze all data with standardized, data type-specific analyses.  All data of one type are analyzed with the same workflow.  Although our results may show some differences from an author’s publication, our re-analysis of the data makes it feasible to compare data sets from very different sources and to update the data analysis with contemporary methods.  For transparency, the methods we use to analyze data are presented here.  </p>

<h4>Contents</h4>
<ul>
   <li><a href="#GenomeAnalyses">Genome Analyses</a></li>
      <ul>
      <li><a href="#EBIpipeline">EBI Pipeline</a></li>
      <li><a href="#Supplements-to-EBI">Supplements to the EBI Pipelines</a></li>
      <li><a href="#In-House-genome-analyses">In-house genome analyses in Lieu of the EBI Pipeline</a></li>
      </ul>
   <li><a href="#Orthology">Orthology</a></li>
      <ul>
      <li><a href="#OrthoMCL">OrthoMCL</a></li>
      <li><a href="#Orthology-on-the-gene-page">Orthology on the gene page</a></li>
      <li><a href="#Function-prediction-on-the-gene-page">Function prediction on the gene page</a></li>
      <li><a href="#Searches-for-genes-based-on-orthology">Searches for genes based on orthology</a></li>
      </ul>
   <li><a href="#Proteomics">Proteomics</a></li>
   <li><a href="#RNA-Sequence">RNA-Sequence</a></li>
   <li><a href="#ChIP-Sequence">ChIP-Sequence</a></li>
   <li><a href="#Copy-Number-Variation">Copy Number Variation</a></li>
      <ul>
      <li><a href="#Searches-Copy-Number-Variation">Searches for genes based on Copy Number Variation</a></li>
      </ul>
   <li><a href="#Genetic-Variation-and-SNP-calling">Genetic Variation and SNP calling</a></li>
   <li><a href="#Microarray">Microarray Data</a></li>
   <li><a href="#Protein-Array">Protein Array Data</a></li>
   <li><a href="#Metabolic-Pathways">Metabolic Pathways</a></li>
</ul>


<hr />
<div class="anchor"><a name="GenomeAnalyses"></a></div>
<h2>Genome analyses</h2>

<p>Genome sequence and annotation are analyzed by the <a href="#EBIpipeline">EBI Pipeline</a> supplemented with <a href="#Supplements-to-EBI">three in-house analyses</a>.  In the rare case that the EBI pipeline cannot be applied to a genome, we use a <a href="#In-House-genome-analyses">series of in-house analyses</a> in lieu of the EBI Pipelines. </p>

  <div class="anchor"><a name="EBIpipeline"></a></div>
  <h4>EBI Pipeline</h4>
  <div class="method-details">
   <p>VEuPathDB employs the <a href="http://uswest.ensembl.org/info/genome/genebuild/index.html" target="_blank">Ensembl genome analysis</a>  for analyzing genomic sequence to enhance annotations. While most of the genomic sequence (FASTA) are integrated into VEuPathDB from an INSDC repository, genome annotation (GFF3) may come from either the INSDC repository or a community submission. </p>

   <p><u>Core database pipelines</u> (figure 1)- Primary genomic sequence and structural annotation data are loaded into a <a href="http://uswest.ensembl.org/info/docs/api/core/index.html" target="_blank">core database</a> and run through 6 pipelines: core statistics, DNA feature annotation, <a href="http://uswest.ensembl.org/info/genome/genebuild/xrefs.html" target="_blank">external cross reference</a> annotation, <a href="http://uswest.ensembl.org/info/genome/genebuild/ncrna.html" target="_blank">RNA gene</a> annotation, <a href="http://uswest.ensembl.org/info/genome/genebuild/assembly_repeats.html" target="_blank">repeat feature</a> annotation, and <!-- <a href="http://ensemblgenomes.org/info/data/protein_features">protein feature</a>--> protein feature annotation.  The main pipelines applied to the core database and their components are listed in table 1. </p>

   <p>Configuration details for each pipeline are determined in Ensembl hive pipeline config files for each pipeline. Since the Ensemble pipelines may change to accommodate bioinformatic advances, pipeline component programs (e.g. Interpro for protein features), versions, and parameters are recorded in the core db analysis table. Final data check results are saved to disk and manually reviewed to determine if the final core db is suitable for release to be loaded into the GUS system.</p>

   <img style="width: 40em; margin-top: .5em; margin-left: 4em;" src="https://qa.community.eupathdb.org/assets/images/Core-analyses-image.png" alt="Searches" />

   <p> <a href="https://qa.community.eupathdb.org/documents/Methods-Core-Table.pdf">Core database analysis pipelines and hive components </a> </p>

   <p><a href="https://qa.community.eupathdb.org/documents/Methods-Ex-Pipelines.pdf">Example ehive pipelines, modules, programs and parameter data from coredb analysis table</a></p>
  </div>


  <div class="anchor"><a name="Supplements-to-EBI"></a></div>
  <h4>Supplements to the EBI Pipelines</h4>
  <div class="method-details">
   <p>VEuPath DB supplements the EBI pipeline with workflows that produce data for EST alignments, Open reading frames, and synteny (Table).</p>
   
   <p><u>EST alignments</u>: BLAT is applied to EST sequences that have been blocked using RepeatMasker. </p>
   
   <p><u>Open reading frame generation</u>: Open reading frames are generated from genomic DNA or EST sequences.  The analysis produces a gff file containing the ORFs (50 or more amino acid translations of the input nucleic acid) for the 6 reading frames. The translations in all 6 reading frames do not necessarily begin with MET but always end at a stop codon. ORF names are in the form template-frame-start-end, e.g. AAEL01000396-5-5847-4366.</p>
   
   <p><u>Synteny</u>: VEuPathDB uses an in-house script called runMercator to run pair-wise alignments that employs Mercator and MAVID for comparative genome analysis. Mercator generates orthology maps using genomes and exon coordinates to create exon translations for protein BLAT alignments. The orthology maps are used as a guide for MAVID which also uses a phylogenetic newick tree to generate gene alignments.</p>
   
   <p><a href="https://qa.community.eupathdb.org/documents/Methods-Supplements-Genomes.pdf">Details for the supplements to the EBI pipelines</a></p>
  </div>

  
  <div class="anchor"><a name="In-House-genome-analyses"></a></div>
  <h4>In-house genome analyses in Lieu of the EBI Pipeline</h4>
  <div class="method-details">
   <p>On rare occasions the EBI pipeline cannot be applied to a genome.  For example, genomes that are not housed at an INSDC repository cannot be analyzed by the EBI pipeline.  VEuPathDB uses the following in-house analyses in lieu of the EBI pipeline. </p>
   
   <p><u>BLAT against NRDB</u>: For every genome, VEuPathDB runs BLAT alignements of the annotated proteins against the GenBank Non-Redundant Protein Sequence Database (NRDB) to identify possible relationships and alignments outside the scope of VEuPathDB-supported organisms. </p>
   
   <p><u>Compute open reading frames</u>: Open reading frames are generated from genomic DNA or EST sequences.  The analysis produces a gff file containing the ORFs (50 or more amino acid translations of the input nucleic acid) for the 6 reading frames. The translations in all 6 reading frames do not necessarily begin with MET but always end at a stop codon. ORF names are in the form template-frame-start-end, e.g. AAEL01000396-5-5847-4366. </p>
   
   <p><u>DNA repeats</u>: The Tandem Repeats Finder program locates and displays tandem repeats in genomic sequences.</p>
   
   <p><u>EST alignments</u>: BLAT is applied to EST sequences that have been blocked using RepeatMasker. </p>
   
   <p><u>Protein domain annotations</u>: InterProScan scans protein sequences against the protein signatures of the InterPro member databases and generates a file containing the domain matched, description of the InterPro entry, GO descriptions and E-values.</p>
   
   <p><u>Signal peptide prediction</u>: Signal P is used to identify signal peptides and their likely cleavage sites.  A signal peptide is a short peptide present at the N-terminus of most newly synthesized proteins that are destined towards the secretory pathway.</p>
   
   <p><u>Syntenic sequences</u>: VEuPathDB uses an in-house script called runMercator to run pair-wise alignments that employs Mercator and MAVID for comparative genome analysis. Mercator generates orthology maps using genomes and exon coordinates to create exon translations for protein BLAT alignments. The orthology maps are used as a guide for MAVID which also uses a phylogenetic newick tree to generate gene alignments.</p>
   
   <p><u>Transmembrane domain prediction</u>: TMHMM is used to predict transmembrane domain presence and topology from protein sequences. </p>
   
   <p><u>tRNA gene prediction</u>: tRNAScan identifies transfer RNA genes in transcript or genome sequences.  </p>
   
   <p><a href="https://qa.community.eupathdb.org/documents/Methods-Details-In-Lieu-genomes.pdf">Details for the VEuPathDB in-house pipelines</a></p>
  </div>

<hr />
<div class="anchor"><a name="Orthology"></a></div>
<h2>Orthology</h2>

<p>The identification of orthologs is an important concept in gene evolution, commonly used to infer likely gene function(s) in newly-sequenced genomes.  Orthologs are proteins in different species that have diverged solely through speciation, and are therefore relatively likely to retain similar function (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5877793/" target="_blank">Gabaldón &amp; Koonin 2013</a>).  Paralogs are proteins that arose through gene duplication; while initially identical, duplication provides the opportunity for functional divergence and evolutionary selection.  “In-paralogs” refer to recent duplications that are likely to retain similar function; “out-paralogs” derive from more ancient duplications predating divergence of the species of interest, and are more likely to have diverged in function.  See <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1849888/" target="_blank">Chen 2007</a> and the <a href="https://orthology.benchmarkservice.org/proxy/" target="_blank">Orthology Benchmarking Service (Altenhoff 2020)</a> for descriptions and assessments of widely-used ortholog detection algorithms</p>

  <div class="anchor"><a name="OrthoMCL"></a></div>
  <h4><a href="https://orthomcl.org/" target="_blank"><b>OrthoMCL</b></a></h4>
  <div class="method-details">
   <p>VEuPathDB websites use the OrthoMCL algorithm (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC403725/" target="_blank">Li 2003</a>) to infer orthology and paralogy, employing protein sequence similarity (BLASTP) to relate proteins across the tree of life, several normalization steps to improve performance on complex eukaryotic genomes, and Markov clustering to group these proteins into ortholog groups.  Each protein in every species is assigned to precisely one OrthoMCL ortholog group (e.g. OG6_135465).  The <a href="https://orthomcl.org/" target="_blank">OrthoMCL website</a> allows users to explore ortholog groups and the proteins they contain, including sequences, similarity metrics, alignments, domain architecture, and phyletic pattern profiles.  Users can also assign putative ortholog groups to proteins from additional genomes of interest (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3196566/" target="_blank">Fisher 2011</a>).  Because of the large and rapidly growing number of available genomes, the most recent implementation of OrthoMCL has been modified to improve scalability and usability, as described <a href="https://orthomcl.org/orthomcl/app/static-content/OrthoMCL/about.html" target="_blank">here</a>. </p>
  </div>

  <div class="anchor"><a name="Orthology-on-the-gene-page"></a></div>
  <h4>Orthology on the Gene Page</h4>
  <div class="method-details">
   <p>All Gene Pages in VEuPathDB databases include an 'Orthology and Synteny' section providing: a link to the ortholog group on the OrthoMCL database, a list of orthologs within the same VEuPathDB database component (e.g. PlasmoDB; see the OrthoMCL database for orthologs in other species), and a JBrowse genome browser display of syntenic chromosomes in other strains and species (shaded to indicate orthology).</p>
   </div>

  <div class="anchor"><a name="Function-prediction-on-the-gene-page"></a></div>
  <h4>Orthology-based functional prediction</h4>
  <div class="method-details">
   <p>Many genes have yet to be functionally characterized, but function can often be inferred using orthology and protein feature information.  VEuPathDB assigns putative Enzyme Commission (EC) numbers (e.g. 2.7.4.3) based on a heuristic scoring algorithm that considers the length, sequence similarity, and domain architecture of other proteins in the same ortholog group that have previously been annotated with this EC number.  Scoring details and other statistics are provided in a table in the 'Functional Prediction' section of the Gene Page; further details on EC prediction are available <a href="https://qa.community.eupathdb.org/documents/Inferred_EC_numbers_algorithm_document.pdf" target="_blank">here</a>.  The inferred EC number is also represented in metabolic pathways graphs and tables accessible via the 'Pathways and Interactions’ section of the Gene Page. </p>
   </div>

  <div class="anchor"><a name="Searches-for-genes-based-on-orthology"></a></div>
  <h4>Searches for genes based on orthology</h4>
  <div class="method-details">
   <p>Several searches employ orthology to identify genes of possible interest.  The <a href="/a/app/search/transcript/GenesByOrthologPattern" target="_blank">Orthology Phylogenetic Profile Search</a>, identifies genes based on the taxonomic groups in which orthologs are or are <b>not</b> found (i.e. genes displaying a specific pattern of conservation across the tree of life).  For example, users seeking a pathogen-specific therapeutic target may wish to search for genes present in the pathogen(s) of interest, but not in non-pathogenic species or host species (e.g. mammals).  The <a href="/a/app/search/transcript/GenesByParalogCount" target="_blank">Paralog Count Search</a> allows users to identify genes based on the number of paralogs identified, e.g. genes that have undergone a recent expansion through one or more duplications.  Finally, any list of genes identified through previous searches or Search Strategies (<a href="https://static-content.veupathdb.org/documents/Orthology-Phyletic-Patterns.pdf" target="_blank">tutorial</a>) may be converted into a list of orthologs in other species.  This is particularly useful when seeking to identify genes in an understudied species based on functional information in more intensively characterized species (e.g. essentiality data derived from mutational screens conducted in model organisms).</p>
   </div>

<hr />
<div class="anchor"><a name="Proteomics"></a></div>
<h2>Proteomics</h2>

  <div class="method-details">
   <p>VEuPathDB integrates the results of proteomics experiments as peptides aligned to a reference genome or as abundance data assigned to a gene.   We do not reanalyze the raw mass spec data but instead use an in-house plugin that loads found peptides or abundance data from tab delimited input files of a specific format.</p>
   
   <p><a href="https://qa.community.eupathdb.org/documents/Methods-Proteomics.pdf">Details for the VEuPathDB in-house proteomics pipeline</a></p>
  </div>

<hr />
<div class="anchor"><a name="RNA-Sequence"></a></div>
<h2>RNA-Sequence</h2>
<div class="method-details">
   <p>VEuPathDB integrates RNA-Seq data from many different experiments and analyzes all data with the same EBI RNA-Seq analysis pipeline. The RNA sequence data that we integrate is processed at EBI. </p>

   
   <p>The following is a general outline of the analysis process.  </p>
    <ul>
     <li>Trim poor quality data (Trimmomatic)</li>
     <li>HiSAT2 alignment to a reference genome</li>
     <li>HT-Seq-count to tally aligned reads per gene</li>
     <li>Convert to transcripts per kilobase million (TPM)</li>
     <li>DESeq2 to determine differential expression</li>
    </ul>
    
   <p><a href="https://docs.google.com/document/d/1igsvk73s3pNjeaJvwLtitfFLexhNfr6-Vk3OIKqq3bw/edit?usp=sharing">EBI RNA-Seq pipeline details</a></p>
 </div>

<hr />
<div class="anchor"><a name="ChIP-Sequence"></a></div>
<h2>ChIP-Sequence</h2>
<div class="method-details">
   <p>VEuPathDB integrates ChIP-Seq data from many different experiments and sources. DNA seq data are aligned to the reference genome using Bowtie2.  Alignment results are converted to bigwig and displayed in JBrowse. </p>
</div>   

<hr />
<div class="anchor"><a name="Copy-Number-Variation"></a></div>
<h2>Copy Number Variation </h2>
<div class="method-details">
   <p>VEuPathDB uses coverage from whole genome sequencing data to estimate gene and chromosome copy numbers in sequenced strains. The bowtie2 alignments generated during SNP analysis are used as a starting point. HTseq-count is used to count the number of reads that align to each gene and the values are converted to transcripts per million (TPM). Assuming that the median TPM value represents a single copy gene on a chromosome of constitutive ploidy, we can infer gene or chromosome duplications by comparing the TPM values for individual genes or the median TPM for individual chromosomes to the whole genome median using custom scripts based on the method described in <a href="https://pubmed.ncbi.nlm.nih.gov/22038252/" target="_blank">PMID: 22038252</a>. Additionally, coverage is calculated in 1 kb bins across the genome, normalised to the constitutive ploidy and converted to bigwig format for visualisation in JBrowse.   </p>
   
   
   <p>Haploid number and gene dose are metrics used to define copy number in VEuPathDB. Haploid number is the number of genes on an individual chromosome. Gene dose is the total number of genes in an organism, accounting for copy number of the chromosome. For example, a single-copy gene in a diploid organism has a haploid number of 1 and a gene dose of 2. </p>
</div> 

   <div class="anchor"><a name="Searches-Copy-Number-Variation"></a></div>
  <h4>Searches for genes based on Copy Number Variation</h4>
   <div class="method-details">
    <p>There are two searches that query copy number data in VEuPathDB.  The first, <a href="/a/app/search/transcript/GenesByCopyNumber">Identify Genes based on Copy Number (CNV)</a> returns genes that are present at copy numbers within a range that you specify. The search can be configured to return genes based on the haploid number or gene dose. The second search, <a href="/a/app/search/transcript/GenesByCopyNumberComparison">Identify Genes based on Copy Number Comparison (CNV)</a>, returns genes for which the copy number varies between the reference and your chosen isolates. This search compares the estimated copy number of a gene in the resequenced strain(s) with the copy number in the reference annotation. The copy number in the reference annotation is calculated as the number of genes that are both on the same chromosome and in the same ortholog group as the gene of interest. We infer that these genes have arisen as a result of tandem duplication of a common ancestor. In this search, the metric for copy number is the haploid number, which is the number of copies of a gene on a single chromosome.</p>
   </div>
 

<hr />
<div class="anchor"><a name="Genetic-Variation-and-SNP-calling"></a></div>
<h2>Genetic Variation and SNP calling</h2>
<div class="method-details">
   <p>VEuPathDB analyzes whole genome resequencing data to call single nucleotide polymorphisms of isolates. The method employed by VEuPathDB to call SNPs from short read sequencing like Illumina reads, follows these steps:<br />
   <ul>
   <li>Reads are aligned to the reference genome using bowtie2</li>
<li>The resulting BAM file from bowtie2 is sorted and a pileup file using samtools is generated</li>
<li>Reads around indels are realigned using GATK</li>
<li>SNPs and indels are called consensus sequence using VarScan is generated:
<ul>
<li>P value &lt;= 0.01</li>
<li>minimum aligned reads &gt;= 5</li>
<li>minimum read frequency &gt;= 0.8</li>
</ul>
</li>
<li>SNP calls where coverage is &gt;2.5x the median coverage are removed to limit erroneous calls in repeat regions</li>
<li>At each SNP position “like reference” calls are generated for each strain that is identical to the reference to give the full picture of each SNP</li>
   </ul> </p>
</div> 


<hr />
<div class="anchor"><a name="Microarray"></a>
<h2>Microarray data</h2>
<div class="method-details">
   <p>VEuPathDB integrates microarray data from high density oligonucleotide as well as spotted arrays.  In general, the data comes to us as intensities associated with probes.  VEuPathDB does not reanalyze the original fluorescence data. We process the data we receive according to the following outline:</p>

   <ul>
      <li>Map the array probes to the reference genome's transcriptome</li>
      <li>Filter the data to remove outliers.</li>
      <li>Normalize</li>
         <ul>
         <li>For one channel data we perform a robust multi-array average (RMA) normalizations.</li>
         <li>For two channel data we perform a Loess normalization</li>
         </ul>
      <li>Compute the average probe intensity per gene.</li>
      <li>Compute the expression average per gene.</li>
         <ul>
         <li>First, average the technical replicates.</li>
         <li>Second, average the biological replicates (if any).</li>
         </ul>
      <li>Optional: perform differential expression analysis if there is a sufficient number of biological replicates.</li>
   </ul>
</div> 


<hr />
<div class="anchor"><a name="Protein-Array"></a></div>
<h2>Protein Array data</h2>
<div class="method-details">
   <p>VEuPathDB integrates protein array data from serm antibody microarray experiments.  In general, the data comes to us as intensities associated with probes.  VEuPathDB does not reanalyze the original fluorescence data. Although each experiment and data set can have special considerations, we process the data according to the following outline:</p>

   <ul>
      <li>Map the array probes to the reference genome's transcriptome</li>
      <li>Filter the data to remove outliers.</li>
      <li>Normalize</li>
         <ul>
         <li>For one channel data we perform a robust multi-array average (RMA) normalizations.</li>
         <li>For two channel data we perform a Loess normalization</li>
         </ul>
      <li>Compute the average probe intensity per gene.</li>
      <li>Compute the expression average per gene.</li>
         <ul>
         <li>First, average the technical replicates.</li>
         <li>Second, average the biological replicates (if any).</li>
         </ul>
      <li>Optional: perform differential expression analysis if there is a sufficient number of biological replicates.</li>
   </ul>
</div> 

<hr />
<div class="anchor"><a name="Metabolic-Pathways"></a></div>
<h2>Metabolic Pathways</h2>
<div class="method-details">
   <p>VEuPathDB integrates metabolic pathways from <a href="https://www.genome.jp/kegg/" target="_blank">KEGG</a> and <a href="https://metacyc.org/" target="_blank">MetaCyc</a>. For TriTrypDB, pathways are also integrated from <a href="http://vm-trypanocyc.toulouse.inra.fr/LEISH/organism-summary" target="_blank">LeishCyc</a> and <a href="http://vm-trypanocyc.toulouse.inra.fr/" target="_blank">TrypanoCyc</a>. Metabolic pathways are associated with genes via Enzyme Commission annotations. </p>
</div> 

</div></div>
<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>VEuPathDB Infrastructure Document</h1>

<div class="static-content">

<p>This document provides a high-level overview of the software infrastructure utilized by the VEuPathDB BRC to load, 
  integrate and provide data to users. Please check <a target="_blank" href="https://veupathdb.org/veupathdb/app/search/dataset/AllDatasets/result">
  a list of all the data sets</a> loaded in our VEuPathDB sites utilizing this infrastructure.</p>

<a name="browser"></a>
<h3>Browser Compatibility Statement</h3>
<p>We recognize that our users access VEuPathDB using various Internet Browsers and Operating Systems. Our goal is to ensure that you have the best possible experience on VEuPathDB, but it is impossible to develop applications that work identically, efficiently and effectively on all web browsers.</p>
<p>Based on our site usage statistics we support the following browsers used by greater than 95% of our visitors:</p>
<ul>
  <li>Firefox</li>
  <li>Safari</li>
  <li>Chrome</li>
</ul>
<p>Feel free to <a href="/a/app/contact-us">contact us</a> about any browsing issues you might come across.</p>


<a name="data"></a>
<h3>Data Loading and Database Schema</h3>
<p>We use the <a target="_blank" href="https://github.com/EuPathDB/GusSchema"><b>G</b>enomics <b>U</b>nified <b>S</b>chema (GUS) database schema</a> and 
  data loading infrastructure and its framework available at <a target="_blank" href="https://github.com/EuPathDB/GusAppFramework">GusAppFramework</a>.  
  This includes not only a comprehensive database schema for integrating and representing genomic and functional (or post) 
  genomic data but also tools for loading said data into that system.  We have made some extensions to the schema and tools for 
  VEuPathDB specific purposes primarily to generate de-normalized views of the data for query optimization purposes.</p>
<p>Most of the heavy data processing and compute is done by our data processing team at the European Bioinformatics Institute (EBI) where we utilize the large compute facility available there.</p>
<p>Our data are all stored in Oracle12c databases.  Our software infrastructure also supports PostgreSQL but we have some Oracle 
  specific SQL constructs in our model that would need to be changed in order to run successfully in PostgreSQL.  </p>
<p>We load all data using an in house engineered workflow system called <a target="_blank" href="https://github.com/EuPathDB/ReFlow">
  ReFlow</a>.  Briefly, ReFlow is engineered to be an efficient graph-based workflow system. In it each step (node in the graph) has 
  the ability to be undone and subsequently rerun with updated data.  This was a significant requirement as it enables us to undo entire
  genomes when the annotation or underlying sequence changes.  This results in automated removal of all data dependent on that genome. 
  When the step is re-run with the new annotation, all dependent data are recomputed and reloaded automatically, thus greatly improving
  our ability to keep these complex databases up-to-date.</p>
<p>The ReFlow workflow system utilizes another piece of software developed at the University of Pennsylvania to schedule, manage and
  monitor running tasks called 
  <a target="_blank" href="http://www.google.com/url?q=https%3A%2F%2Fdocs.google.com%2Fdocument%2Fpub%3Fid%3D1BixZ5t2c0hnOZES-Rk2wG2loAQqclcmRj7AeKQjZMHA">DistribJob</a>.  
  DistribJob distributes tasks generated from a large input dataset such as a set of sequences to compute nodes in a cluster for analysis
  and retrieves and collates the results in an efficient manner. We automate the running of large compute tasks on compute clusters 
  located at the University of Pennsylvania and the University of Georgia.</p>

<details>
  <summary style="font-size:110%;font-weight:bold;margin-bottom:5px;cursor:pointer;color:#069;"> >>> Github Repositories</summary>
  <div style="margin-left:1em;">
  These are the repositories needed to generate data and load the databases:

  <ul>
    <li><a target="_blank" href="https://github.com/Ensembl/ensembl-production-imported">EBI main processing pipeline</a>: Pipelines and scripts from the EBI Ensembl repository utilized by VEuPathDB data processing.</li>
    <li><a target="_blank" href="https://github.com/Ensembl/ensembl-production-imported/blob/trunk/lib/perl/Bio/EnsEMBL/EGPipeline/PrivateConfDetails/Impl.pm.example">Example configurations</a>: Example configuration for the EBI data processing pipelines.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ApiCommonData">ApiCommonData</a>: Defines the workflow graph for the Genomic sites and contains SQL to generate denormalized tables.
        (Also: <a target="_blank" href="https://github.com/VEuPathDB/OrthoMCLData">OrthoMCLData</a>,
               <a target="_blank" href="https://github.com/VEuPathDB/ClinEpiData">ClinEpiData</a>.)</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/ApiCommonWorkflow">ApiCommonWorkflow</a>: Supports the workflow graph for the Genomic sites.
        (Also: <a target="_blank" href="https://github.com/VEuPathDB/OrthoMCLWorkflow">OrthoMCLWorkflow</a>,
               <a target="_blank" href="https://github.com/VEuPathDB/ClinEpiWorkflow">ClinEpiWorkflow</a>,
               <a target="_blank" href="https://github.com/VEuPathDB/MicrobiomeWorkflow">MicrobiomeWorkflow</a>.)</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/ReFlow">ReFlow</a>: ReFlow is a reversible workflow system for building and maintaining large functional genomics databases. It is based on a dependency graph written in XML.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/ExampleDatasets">ExampleDatasets</a>: Example of the Data Sets to be loaded in the database by the Workflows. Also needed at site build time for model injection.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/ExamplePresenters">ExamplePresenters</a>: Example of the Data Set metadata to be loaded in denormalized tables. Also needed at site build time for model injection.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/TuningManager">TuningManager</a>: Engine to generate denormalized tables in a database.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/CBIL">CBIL</a>: scripts to generate GUS perl objects .</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/DJob">DJob</a>: DistribJob (see explanation above).</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/GusSchema">GusSchema</a>: Relational Database schema definition for Genomics data.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/GusAppFramework">GusAppFramework</a>: GUS application framework.</li>
  </ul>
  </div>
</details>
     
<a name="wdk"></a>
<h3>Web Presentation System and User Interfaces</h3>
<p>Our websites are based on code that we developed and have released to the community called the Strategies-WDK (Strategies Web 
  Development Kit) which enables the graphical strategies search system.  You can download the software and see documentation for this
  toolkit at <a target="_blank" href="https://github.com/EuPathDB/WDK">Strategies-WDK</a>.  This toolkit enables us to represent our 
  data as an XML model which is then turned into the web interfaces that are presented to users using these tools. As we develop new
  tools and services, we are transitioning towards a cloud-ready Restful architecture with user workspaces accessing tools 
  in containerized services.</p>

<details>
  <summary style="font-size:110%;font-weight:bold;margin-bottom:5px;cursor:pointer;color:#069;"> >>> Github Repositories</summary>
  <div style="margin-left:1em;">
  These are the repositories needed to build a website:

  <p><b>Infrastructure</b></p>
  <ul>
    <li><a target="_blank" href="https://github.com/EuPathDB/install">install</a>: starter.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/FgpUtil">FgpUtil</a>: Basic utilities used by the WDK such as access to database, and SQL runs.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/OAuth2Server">OAuth2Server</a>: authentication to one site used by all sites within VEuPathDB including the galaxy site.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/WDK">WDK</a>: The Web Development Kit. It reads the model (xml files) and loads into java structures. Engine for search runs, step and strategy creation, baskets and favorites and user workspace.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/WDKClient">WDKClient</a>: a React-based frontend for WDK (js, jsx, css).</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/WSF">WSF</a>: The WDK Services Framework: web service architecture to support searches that instead of accessing the database run plugins/processes.</li>
  </ul>

  <p><b>Search strategy system</b></p>
  <ul>
    <li><a target="_blank" href="https://github.com/EuPathDB/EbrcModelCommon">EbrcModelCommon</a>: xml files with basic data types definitions, and their structured searches, common to all sites (genomic, epidemiology, microbiome).
        Specialization for specific websites at: 
               <a target="_blank" href="https://github.com/VEuPathDB/ApiCommonModel">ApiCommonModel</a>,
               <a target="_blank" href="https://github.com/VEuPathDB/OrthoMCLModel">OrthoMCLModel</a>,
    <li><a target="_blank" href="https://github.com/VEuPathDB/EbrcWebsiteCommon">EbrcWebsiteCommon</a>: frontend pages (html, jsp, js, css) that add or modify WDKClient, common to all sites.
        Specialization for specific websites at:
               <a target="_blank" href="https://github.com/VEuPathDB/ApiCommonWebsite">ApiCommonWebsite</a>, 
               <a target="_blank" href="https://github.com/VEuPathDB/OrthoMCLWebsite">OrthoMCLWebsite</a>,
    <li><a target="_blank" href="https://github.com/VEuPathDB/OrthoMCLClient">OrthoMCLClient</a>: Website client code, extending WDKClient.. 
    <li><a target="_blank" href="https://github.com/VEuPathDB/OrthoMCLService">OrthoMCLService</a>: Website REST services, extending WDK.
    <li><a target="_blank" href="https://github.com/VEuPathDB/EbrcWebSvcCommon">EbrcWebSvcCommon</a>: plugins common to all sites (eg: Blast, Text search).
        Specialization for specific websites at: 
               <a target="_blank" href="https://github.com/VEuPathDB/ApiCommonWebService">ApiCommonWebService</a>,
               <a target="_blank" href="https://github.com/VEuPathDB/OrthoMCLWebService">OrthoMCLWebService</a></li>
  </ul>
   
  <p><b>Other tools</b></p>
  <ul>
    <li><a target="_blank" href="https://github.com/VEuPathDB/StaticContent">Static Content</a>: static content in jekyll (help, tutorials, etc.)</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/Apollo">Apollo</a>: annotation tool.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/JBrowse">JBrowse</a>: modern genome browser, replaces GBrowse.</li>
    <li><a target="_blank" href="https://github.com/VEuPathDB/popbio-map">PopBio Map</a>: map visualization tool for genomic, phenotypic and population data (only in Vectorbase in this release.</li>
    <li>Site Search: comprehensive site search, integrated with the strategies workspace.
      <ul>
      <li><a target="_blank" href="https://github.com/VEuPathDB/SiteSearchService">SiteSearchService</a></li>
      <li><a target="_blank" href="https://github.com/VEuPathDB/SiteSearchData">SiteSearchData</a></li>
      <li><a target="_blank" href="https://github.com/VEuPathDB/jekyll-site-search">Static Content Indexing</a></li> 
      </ul>
    </li>
    <li>User Workspace: enables users to upload their own user datasets and explore their data integrated in our websites.
      <ul>
      <li><a target="_blank" href="https://github.com/VEuPathDB/EuPathGalaxy">EuPathGalaxy</a>: code that runs in the VEuPathDB Galaxy, which includes the VEuPathDB written tools and workflows.</li>
      <li><a target="_blank" href="https://github.com/VEuPathDB/EuPathDBIrods">EuPathDBIrods</a>: Irods storage for the user workspace, which allows users to export data sets from the VEuPathDB Galaxy site and run analyses on them.</li>
      </ul>
    </li>
  </ul>


  </div>
</details>    
  
<a name="hardware"></a>
<h3>System Hardware and Third Party Software</h3>
<p>VEuPathDB maintains redundant database and content web servers at the University of Pennsylvania and the University of Georgia to 
  minimize interruptions for our users during maintenance periods. Additionally VEuPathDB compute and data loading servers are located at
  the University of Pennsylvania.</p> 
<p>Server configurations are coordinated and deployed through Puppet automation software (<a href="http://puppetlabs.com/" target="_blank">http://puppetlabs.com/</a>). 
  Custom infrastructure software is versioned and deployed through standard RPM/YUM mechanisms. When appropriate, software builds are 
  automated using Jenkins Continuous Integration Server (<a href="http://jenkins-ci.org/" target="_blank">http://jenkins-ci.org/</a>)</p>
<p>System infrastructure statistics (CPU load, I/O, etc) are gathered with collectd (<a href="http://collectd.org/" target="_blank">http://collectd.org/</a>) 
  and in-house applications and feed to Graphite (<a href="http://graphite.wikidot.com/" target="_blank">http://graphite.wikidot.com/</a>) 
  for human review. Nagios (<a href="http://www.nagios.org/" target="_blank">http://www.nagios.org/</a>) provides notifications of system
  degradations.</p>
<p>Both Universities also maintain large compute clusters that are heavily utilized by VEuPathDB in order to analyze and load incoming
  data in a timely fashion. The linked document below describes our actual hardware and includes a list of third-party software required
  in order to analyze, load and present data via our websites.</p>
<!--
<div id="cirbulletlist">
<ul><li><a href="/documents/EuPathDB_Hardware_3P_Software.pdf">Hardware and Third Party Software</a></ul>
</div>
-->
<a name="community"></a>
<h3>Community Interactions and Data Submission Policies</h3>
<p>VEuPathDB serves a global scientific community that demands direct active support and community involvement.   VEuPathDB outreach 
  activities include:</p>
<div id="cirbulletlist">
<ul>
<li>Organizing and running hands on training workshops and webinars (<a href="https://drive.google.com/open?id=1o063oJ6arjelbDBb_TnTo3MsBN0&usp=sharing">Google Map</a>).
<li>Developing educational material in the form of <a href="/a/app/static-content/landing.html">exercises and online tutorials</a>.
<li>Responding to support emails for users who contact us directly by clicking the "Contact Us" links in the header or footer of any VEuPathDB webpage (average response time is 48 hours).
<li>Holding open community meetings/forums with our diverse user base.  These meetings are held in person at scientific conferences or using an online conferencing platform.
<li>Attending national and international meetings with active participation in the form of posters, presentations or help desks.
<li>Authoring <a target="_blank" href="/a/app/static-content/veupathPubs.html">peer reviewed manuscripts</a>.
<li>Maintaining active social media presence in the form of a <a target="_blank" href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook page</a> and <a href="https://twitter.com/veupathdb">Twitter feed</a>.
<li>Providing a clear <a href="https://qa.community.eupathdb.org/documents/VEuPathDB_Data_Sub_Release_policy_rev_02April2020.pdf">data handling and release policy</a> to investigators to encourage submission of data prepublication.
</ul>
</div>

</div>
<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1 id="eupathdb-website-privacy-policy">VEuPathDB Website Privacy Policy</h1>

<div class="static-content">

<p>UPDATED: April 12, 2020</p>
<h2 id="introduction">Introduction</h2>
<p>VEuPathDB (also referred to as ‘we’ throughout this document) is committed to protecting its users’ data and privacy. The purpose of this page is to provide you with information about how the data we collect from users of VEuPathDB websites is used or shared. We may update this Privacy Notice from time to time. We encourage you to visit this page frequently and take note of the date updated field above.</p>
<p>We do not use or share any of your personal information for any purpose unrelated to the functionality of the websites; however, we do collect some information to help us understand how our sites are being used in order to improve community support and to enhance the VEuPathDB community’s experience when visiting our sites.</p>
<h2 id="information-automatically-collected">Information Automatically Collected</h2>
<p>When you browse VEuPathDB sites, certain information about your visit will be collected. We automatically collect and store the following type of information about your visit:</p>
<ul>
<li><p>The IP address of the client making the request. Often the IP address is that of your personal computer or smart phone; however, it might be that of a firewall or proxy your internet provider manages.</p></li>
<li><p>The operating system and information about the browser used when visiting the site.</p></li>
<li><p>The date and time of each visit.</p></li>
<li><p>Pages visited.</p></li>
<li><p>The address of a referring page. If you click a link on a website that directs you to a VEuPathDB page, the address of that originating web page will be collected. This “referrer” information is transmitted as part of the browser and server communications; it is not based on any marketing or partnering agreements with the referring site.</p></li>
</ul>
<p>This automatically collected information does not identify you personally unless you include personally identifying information in a support form request; see the “Contact Us” policy below for details. We use this information to measure the number of visitors to our site. The aggregate data may be included in prospectuses and reports to funding agencies.</p>
<h2 id="information-you-directly-provide">Information You Directly Provide</h2>
<p>The Basket, Favorites, Public Strategies, Gene/Sequence Comment and GBrowse Track features of the VEuPathDB websites require that you register for an account. A valid email address is required so we can send you your temporary account password. An anonymous email service can be used if you do not want to provide personally identifying information.</p>
<p>Your email address will be used to send you infrequent alerts if you subscribe to receive them. We do not sell or distribute email addresses to third parties.</p>
<p>We also ask for your name and institution during account registration. If you add a comment to a Gene or a Sequence, your name and institution will be displayed with the comment. If you make one of your strategies public, your name and institution will be displayed with it. We do not routinely verify the validity of names and institutions associated with comments or public strategies; however, we will delete accounts or comments if we believe them to be fraudulent based on inappropriate activity or posted content. We will not sell or distribute your name or institution to third parties.</p>
<p>When you log in, the client IP address is recorded. This IP address can be correlated with the address automatically collected as noted above. If your user profile personally identifies you, then it may be possible to associate you with your detailed activity on VEuPathDB web sites.</p>
<h3 id="contact-us-form">“Contact Us” Form</h3>
<p>The header on each web page includes a “Contact Us” link to a form where users can submit questions, error reports, feature requests, and dataset proposals. Submissions through this form are emailed to VEuPathDB staff and recorded in a project management application accessible only by VEuPathDB staff.</p>
<p>The form includes a field for an email address. If the email address identifies you personally, say if you use your institutional email, then your correspondence with us will likewise be linked to you. A valid email is not strictly required, although we cannot reply to you without one.</p>
<p>When you submit the form, your IP address and browser version will be recorded for internal use. In the case of reported bugs or other site errors, this information may be used by technical staff to help locate your session in the server logs to aid in troubleshooting the issue. This does have the side effect of making it possible to associate an IP address with an email address which may, in turn, personally identify you. However, VEuPathDB does not publicly release this information.</p>
<h2 id="how-eupathdb-uses-cookies">How VEuPathDB Uses Cookies</h2>
<p>VEuPathDB uses cookies to associate multiple requests by your web browser into a stateful session. Cookies are essential to track the state of query strategies, gene baskets and authentication.</p>
<p>Some cookies persist only for a single session. The information is recorded temporarily and is erased when the user quits the session or closes the browser. Others may be persistently stored on the hard drive of your computer until you manually delete them from a browser folder or until they expire, which can be months after they were last used.</p>
<p>Cookies can be disabled in your browser (refer to your browser’s documentation for instructions); however, the majority of the website functionality will be unavailable if cookies are disabled.</p>
<h3 id="google-analytics">Google Analytics</h3>
<p>Google Analytics provides aggregate measurements of website traffic including counts of page hits and unique users along with statistics on countries of origin.</p>
<p>The raw measurements and statistics are only available to approved VEuPathDB staff. Aggregated data may be included in prospectuses and reports to funding agencies.</p>
<h2 id="third-party-websites-and-applications">Third-Party Websites and Applications</h2>
<p>Third-party websites and applications are not exclusively operated or controlled by VEuPathDB. By using these third-party websites, individuals may be providing nongovernmental third-parties with access to personally identifying information.</p>
<h3 id="twitter">Twitter</h3>
<p>VEuPathDB maintains a presence on Twitter in the form of a <a target="_blank" href="https://twitter.com/VEuPathDB">VEuPathDB branded page</a>. This page allows for a direct connection with end users to promote information related VEuPathDB services and to disseminate educational information on research publications, news and events related to the biology of eukaryotic pathogens. Postings may also include information about planned service maintenance and outages.</p>
<p>Twitter collects profile information such as name and email address about users who register to use this third-party website. Depending on the user’s privacy settings, this information may be displayed on the user’s profile page or in the user’s tweets which may be retweeted on VEuPathDB’s page. The VEuPathDB Twitter account may post the authors and institutions of publicly published scientific papers and news articles. VEuPathDB does not actively collect or maintain personally identifying information through its use of Twitter. VEuPathDB will redact or refrain from retweeting a posting that contains obviously identifiable personal information. A Twitter account is not required to read VEuPathDB postings on Twitter. VEuPathDB does not collect or use personal information outside of Twitter’s site.</p>
<p>Twitter is hosted and maintained by a third party which may use browser tracking and related technologies to collect information about visitors to twitter.com and its affiliates. Refer to Twitter’s privacy statement, <a target="_blank" href="https://twitter.com/en/privacy" class="uri">https://twitter.com/en/privacy</a>, for more information.</p>
<h3 id="facebook">Facebook</h3>
<p>VEuPathDB maintains a presence on Facebook in the form of a <a target="_blank" href="https://facebook.com/EuPathDB">VEuPathDB branded page</a>. This page allows for a direct connection with end users to promote information related VEuPathDB services and to disseminate educational information on research publications, news and events related to the biology of eukaryotic pathogens. Postings may also include information about planned service maintenance and outages.</p>
<p>Like Twitter, Facebook collects profile information, including name and email address, from its users. Depending on the user’s privacy settings, this information may be displayed on the user’s profile page along with any activity such as comments or “likes” on the VEuPathDB Facebook page or in posts that VEuPathDB shares on Facebook. VEuPathDB does not collect or use any personally identifying information outside of our Facebook page. To understand how Facebook collects and uses personal information, refer to their data policy page, https://www.facebook.com/policy.php.</p>
<h3 id="youtube">YouTube</h3>
<p>VEuPathDB maintains a presence on YouTube in the form of a <a target="_blank" href="https://www.youtube.com/user/EuPathDB">VEuPathDB branded page</a>. This page provides tutorials on the use of our websites.</p>
<p>YouTube also requires some information when users create an account, including an email address, and users may choose to provide a name and other identifying information in their public profile. Depending on their individual privacy settings, some personally identifiable information may be available to other users, including VEuPathDB. However, VEuPathDB does not collect or use any of that information outside of its YouTube interactions. You can view videos without signing in to an account, but you must be a registered user in order to comment. As YouTube is a Google service, the VEuPathDB youtube channel is subject to <a target="_blank" href="https://www.google.co.uk/intl/en-GB/policies/privacy/">Google’s privacy policy</a>.</p>
<h3 id="globus-genomics">Globus Genomics</h3>
<p>The VEuPathDB Galaxy Data Analysis Service is a workspace for large-scale data analyses. Developed in partnership with <a target="_blank" href="http://globusgenomics.org/genomics/">Globus Genomics</a>, workspaces offer a private analysis platform with published workflows and pre-loaded annotated genomes. The workspace is accessed through the “Analyze My Experiment” tab on the home page of any VEuPathDB resource and can be used to upload your own data, compose and run custom workflows, retrieve results and share workflows and data analyses with colleagues.</p>
<p>The VEuPathDB Galaxy Data Analysis Service is hosted by Globus Genomics, an affiliate of Globus. The first time you visit VEuPathDB Galaxy you will be asked to sign up with Globus in order to set up your private Galaxy workspace. Linking your Globus account with your VEuPathDB account is necessary so that input data and analysis results can be transferred between the two systems. We encrypt data transfers and storage, but ultimately, we cannot guarantee the security of data transmissions among VEuPathDB, Globus and affiliates, Amazon Cloud Services, and the user. It is your responsibility to back up your data and obtain any required permissions from your study and/or institution prior to uploading data for analyses on the VEuPathDB Galaxy platform. Do not use, transmit, upload or share any human identifiable information in the files you analyze. VEuPathDB, Globus and affiliates, the University of Georgia, the University of Pennsylvania, the University of Liverpool, and Amazon Cloud Services do not take any responsibility and are not liable for the loss and/or release of any data you analyze via the VEuPathDB Galaxy platform. We encourage you to review the <a target="_blank" href="https://www.globus.org/legal/privacy">Globus’ privacy policy</a>.</p>
<h2 id="your-rights-based-on-the-general-data-protection-regulation-gdpr">Your Rights based on the General Data Protection Regulation (GDPR)</h2>
<p>To read more about GDPR please check the <a target="_blank" href="https://gdpr-info.eu">GDPR website</a>.</p>
<ol type="1">
<li>The right of transparency and modalities. The privacy policy should be clear and easy to follow in explaining what data we collect and how we use it.</li>
<li>The right to be informed about when data is gathered. This is described in the privacy policy, during the registration process (if you choose to register), site banner and an email sent out to all registered users on May 25, 2018.</li>
<li>The right of access. You can ask for what sepecific data we have about you and how we use it.</li>
<li>The right to rectification. We will correct any errors in your personal data that you point out to us.</li>
<li>The right to be forgotten. We are happy to delete your account and info when you make such a request.</li>
<li>The right to restrict processing. You have the right to request that we restrict the use of your data.</li>
<li>The right for notification obligation regarding rectification/erasure/restriction.</li>
<li>The right to data portability.</li>
<li>The right to object to the processing of your personal data at any time.</li>
<li>The right in relation to automated decision making and profiling. Basically, you have the right not to be subject to decisions based solely on automated processing which significantly affect you.</li>
</ol>
<p>To make any of the above stated requests or if you have any questions please email us at help@VEuPathDB.org</p>
</div>

<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>VEuPathDB personnel</h1>

<div class="static-content">

<h3>VEuPathDB Management</h3><br />

<div>
Beatrice Amos, Annotation Manager<br />
Cristina Aurrecoechea, User Interface and Portal Manager<br />
Bob Belnap, Systems and Databases Manager<br />
John Brestelli, Data Development Manager<br />
Brian Brunk, VEuPathDB Senior Manager<br />
George Christophides, Co-I, NIAID BRC Contract<br />
Kathryn Crouch, Co-I, Wellcome Trust<br />
Jeremy DeBarry, VEuPathDB Project Manager<br />
Steve Fischer, Software and Infrastructure Manager<br />
Paul Flicek, Co-I, NIAID BRC Contract<br />
Omar Harb, Director of Scientific Outreach &amp; Education<br />
Andy Jones, Wellcome Trust PI; Co-I NIAID BRC Contract <br />
Jessica C Kissinger, Joint-PI, NIAID BRC Contract; WT Co-PI<br />
Wei Li, Data Loading Manager<br />
Gareth Maslen, EBI Data Processing Manager<br />
Mary Ann McDowell, Joint-PI, NIAID BRC Contract<br />
David S Roos, Joint-PI, NIAID BRC Contract; WT Co-PI<br />
Chris J Stoeckert, Co-I, NIAID BRC Contract<br /><br />

To contact any one of us please use the <a href="/a/app/contact-us">contact us form</a>.<br /><br />

<h3>Current VEuPathDB Team members</h3>
    <br />
    Beatrice Amos<sup>4</sup>, Rachel Ankirskiy<sup>1</sup>, Cristina Aurrecoechea<sup>1</sup>, Matthieu Barba<sup>9</sup>, Ana Barreto<sup>3</sup>, Evelina Basenko<sup>4</sup>, Wojtek Bazant<sup>2</sup>, Dan Beiting<sup>2</sup>, Bob Belnap<sup>1</sup>, Ann Blevins<sup>2</sup>, Ulrike B&#246;hme<sup>4</sup>, John Brestelli<sup>3</sup>, Brian Brunk<sup>2</sup>, Danielle Callan<sup>2</sup>, Mikkel Christensen<sup>9</sup>, George Christophides<sup>8</sup>, Kathryn Crouch<sup>6</sup>, Katie Cybulski<sup>7</sup>, Jeremy DeBarry<sup>1</sup>, Ryan Doherty<sup>3</sup>, Yikun Duan<sup>2</sup>, Dave Falke<sup>1</sup>, Steve Fischer<sup>3</sup>, Paul Flicek<sup>9</sup>, Bindu Gajria<sup>2</sup>, Gloria I. Giraldo-Calderón<sup>7</sup>, Omar S. Harb<sup>2</sup>, Elizabeth Harper<sup>2</sup>, Danica Helb<sup>2</sup>, Mark Hickman<sup>2</sup>, Connor Howington<sup>7</sup>, Sufen Hu<sup>2</sup>, Jay Humphrey<sup>1</sup>, John Iodice<sup>3</sup>, Andy Jones<sup>4</sup>, John Judkins<sup>2</sup>, Sarah Kelly<sup>8</sup>, Jessica C. Kissinger<sup>1</sup>, Dae Kun Kwon<sup>7</sup>, Kris Lamoureux<sup>1</sup>, Wei Li<sup>2</sup>, Disha Lodha<sup>9</sup>, Jamie Long<sup>2</sup>, Bob MacCallum<sup>8</sup>, Gareth Maslen<sup>9</sup>, Mary Ann McDowell<sup>7</sup>, Greg Milewski<sup>2</sup>,  Jarek Nabrzyski<sup>7</sup>, David S. Roos<sup>2</sup>, Samuel Rund<sup>7</sup>, Steph Wever Schulman<sup>2</sup>, Achchuthan Shanmugasundram<sup>4</sup>, Vasili Sitnik<sup>9</sup>, Drew Spruill<sup>1</sup>, David Starns<sup>4</sup>, Christian J. Stoeckert Jr.<sup>3</sup>, Sheena Shah Tomko<sup>2</sup>, Haiming Wang<sup>1</sup>, Susanne Warrenfeltz<sup>1</sup>, Robert Wieck<sup>7</sup>, Paul Wilkinson<sup>4</sup>, Mariann Winkelman<sup>2</sup>, Lin Xu<sup>2</sup>, Jie Zheng<sup>3</sup>.
    <br /><br />
<h3>Previous VEuPathDB Team members, 2004-2020</h3>
<br />
    Antelmo Aguilar<sup>7</sup>, James Allen<sup>9</sup>, Alexis Allot<sup>9</sup>, Nora Besansky<sup>7</sup>, Austin Billings<sup>2</sup>, Sanjay Boddu<sup>9</sup>, Steve Bogol<sup>7</sup>, Ewan Birney<sup>9</sup>, Andrew Brockman<sup>8</sup>, Robert Bruggner<sup>7</sup>, Ja'Shon Cade<sup>3</sup>, Mark Caddick (Principal Investigator, WT 2019-2020)<sup>4</sup>, David Campbell<sup>7</sup>, Cristian Cocos<sup>2</sup>, Frank Collins (VectorBase Principal Investigator, 2004-2018)<sup>7</sup>, Kathy Couch<sup>1</sup>, Greg Davies<sup>7</sup>, Elaine Daugan<sup>4</sup>, Ale Diaz Miranda<sup>2</sup>, Emmanuel Dialynas, Jennifer Dommer<sup>3</sup>, Vicky Dritsou, Scott Emrich<sup>10</sup>, Xin Gao<sup>2</sup>, William Gelbart<sup>12</sup>, Sandra Gesing<sup>7</sup>, Alan Gingle<sup>1</sup>, Greg Grant<sup>3</sup>, Matt Guidry<sup>1</sup>, Martin Hammond<sup>9</sup>, Mark Heiges<sup>1</sup>, Christiane Hertz-Fowler (Principal Investigator, WT 2008-2019)<sup>4</sup>, Nicholas Ho<sup>8</sup>, Daniel Hughes<sup>9</sup>, Frank Innamorato<sup>3</sup>, San James<sup>14</sup>, Amie Jaye<sup>8</sup>, Fotis Kafatos<sup>8</sup>, Paul Kersey<sup>9</sup>, Ioannis Kimitzoglou<sup>8</sup>, Nathan Konopinski<sup>7</sup>, Carolyn Knoll<sup>2</sup>, Eileen T. Kraemer<sup>1</sup>, Nick Langridge<sup>9</sup>, Cris Lawrence<sup>2</sup>, Daniel Lawson<sup>8</sup>, Brianna Lindsay<sup>2</sup>, Neil Lobo<sup>7</sup>, Christos (Kitsos) Louis<sup>11</sup>, Ross Madden<sup>6</sup>, Greg Madey<sup>7</sup>, Elisabetta Manduchi<sup>3</sup>, Karine Megy<sup>9</sup>, John A. Miller<sup>6</sup>, Elvira Mitraka<sup>11</sup>, Vishal Nayak<sup>3</sup>, Cary Pennington<sup>1</sup>, Deborah F. Pinney<sup>3</sup>, Brian Pitts<sup>1</sup>, Jane A.Pulman<sup>4</sup>, Caleb Reinking<sup>7</sup>, Seth Redmon, Chris Ross<sup>1</sup>, Andrew Sheehan<sup>7</sup>, Fatima Silva<sup>4</sup>, Ganesh Srinivasamoorthy<sup>1</sup>, Scott Szakonyi<sup>7</sup>, Pantelis Toplais<sup>11</sup>, Ryan Thibodeau<sup>1</sup>, Charles Treatman<sup>2</sup>, Betsy Wenthe<sup>1</sup>, Matt Vander Werf<sup>7</sup>, Maggie Werner-Washburne<sup>13</sup>, Patricia L. Whetzel<sup>3</sup>, Derek Wilson<sup>9</sup>, Andrew Yates<sup>9</sup>
    <br /><br />
    <sup>1</sup>University of Georgia, Athens, GA 30602, USA<br />
<sup>2</sup>University of Pennsylvania, Philadelphia, PA 19104, USA<br />
<sup>3</sup>University of Pennsylvania School of Medicine, Philadelphia, PA 19104, USA<br />
<sup>4</sup>University of Liverpool, L69 3BX, UK<br />
<sup>5</sup>Wellcome Sanger Institute, Hinxton, CB10 1RQ, UK<br />
<sup>6</sup>Wellcome Centre for Integrative Parasitology, University of Glasgow, UK<br />
<sup>7</sup>University of Notre Dame, Notre Dame, IN  46556, USA<br />
<sup>8</sup>Imperial College London, South Kensington, London SW7 2BU, UK<br />
<sup>9</sup>European Bioinformatics Institute, Hinxton, CB10 1SD, UK<br />
<sup>10</sup>University of Tennessee, Knoxville, TN 37996, USA<br />
<sup>11</sup>Institute of Molecular Biology and Biotechnology-FORTH, Heraklion, Crete, Greece<br />
<sup>12</sup>Harvard University, Cambridge, MA 02138, USA<br />
<sup>13</sup>University of New Mexico, Albuquerque, NM 87131, USA<br />
<sup>14</sup>Makerere University and Infectious Diseases Research Collaboration (IDRC), Kampala, Uganda<br />
</div>
</div>
<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<h1>VEuPathDB wishes to acknowledge</h1>

<div class="static-content">

  <ul class="cirbulletlist">
  <li>All the community members who have contributed data (often pre-publication), entered user comments or sent us their suggestions.</li>
  <br />
  <li>Scientists who provided images to be used as a template for the logos in our websites and for images used in the header section of our sites:
  </li>
</ul>

  <table>
    <tbody>
      <tr>
        <td>   </td>
        <td>   <b>AmoebaDB:</b>   </td>
        <td>   <b>MicrosporidiaDB:</b>   </td>
        <td>   <b>ToxoDB:</b>   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   William Petri   </td>
        <td>   Gira Bhabha   </td>
        <td>   David Roos   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Serge Ankri   </td>
        <td>   Pattana Jaroenlak (Michael)   </td>
        <td>   <b>TrichDB:</b>   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Craig Roberts   </td>
        <td>   Damian Ekiert   </td>
        <td>   Antonio Pereira-Neves   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Fiona Henriquez   </td>
        <td>   Michael Cammer   </td>
        <td>   Marlene Benchimol   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Hugo Aguilar-Díaz   </td>
        <td>   <b>PiroplasmaDB:</b>   </td>
        <td>   <b>TriTrypDB:</b>   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Julio C. Carrero   </td>
        <td>   Ellen Yeh   </td>
        <td>   Rick Tarleton   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   <b>CryptoDB:</b>   </td>
        <td>   Lowell Kappemeyer   </td>
        <td>   Richard Wheeler   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Boris Striepen   </td>
        <td>   Audrey Lau   </td>
        <td>   Leandro Lemgruber Soares   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   <b>FungiDB:</b>   </td>
        <td>   Dirk Dobbelaere   </td>
        <td>   Margaret Mullin   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   <a href="http://zygolife.org">ZyGoLife</a>   </td>
        <td>   Manoj Duraisingh   </td>
        <td>   Camila Silva Gonçalves   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Jason Stajich   </td>
        <td>   Brendan Elsworth   </td>
        <td>   Wanderley de Souza   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Zachary Lewis   </td>
        <td>   Caroline Keroack   </td>
        <td>   Maria Cristina Machado Motta   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   <b>GiardiaDB:</b>   </td>
        <td>   Isabelle Coppens   </td>
        <td>   </td>
        <td> </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Fran Gillin   </td>
        <td>   <b>PlasmoDB:</b>   </td>
        <td>   </td>
        <td> </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Tineke Lauwaet   </td>
        <td>   Lawrence Bannister   </td>
        <td>   </td>
        <td> </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Barbara Davids   </td>
        <td>   Lewis Tilney   </td>
        <td>   </td>
        <td> </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Scott Dawson   </td>
        <td>   Pedro Moura   </td>
        <td>   </td>
        <td> </td>
      </tr>
    </tbody>
  </table>

  <h3>VEuPathDB Community Representatives</h3>

  <p>VEuPathDB encourages community members to provide feedback about our resources.  We get feedback from many community members including those listed below who have been active in our open community meetings. We encourage you to get involved. Feel free to <a href="/a/app/contact-us">contact us</a> any time. <br /><br />
<b>Amoeba</b>: Open community call and Carol Gilchrist, Upi Singh<br />
<b>Cryptosporidium</b>: Gregory Buck, Guy Robinson, Karin Troell, Sumiti Vinayak, Jonathon Wastling, Giovanni Widmer, Lihua Xiao, Guan Zhu<br />
<b>Fungi</b>: Bridget Barker, Elaine Bignell, Katherine Borkovich, Michael Bromley, Christina Cuomo, Tamara Doering, Jay Dunlap, Michael Freitag, Louise Glass, Kim Hammond-Kosack, Guilhem Janbon, Seogchan Kang, Theo Kirkland, Corby Kistler, Jennifer Lodge, Robin May, Jessie Uehling, Sinem Beyhan, Douglas Lake, Natalie Mitchell, Maureen Donlin, Vera Meyer, Marc Orbach, Nadia Ponts, Antonis Rokas, Jason Stajich, Matt Sachs, George R. Thompson, Martin Urban, Nathan P. Wiederhold<br />
<b>Giardia</b>: Scott Dawson, Fran Gillin, Adrian Hehl, Aaron Jex, Hilary Morrison, John Samuelson, Cornelia Spycher, Staffan Svard <br />
<b>Microsporidia</b>: James Becnel, Nicolas Corradi, Elizabeth Didier,  Patrick Keeling, Emily Troemel, Louis Weiss<br />
<b>Piroplasma</b>: Open community call and Choukri Mamoun<br />
<b>Plasmodium</b>: John Adams, Chris Janse, Rays H.Y. Jiang, Shahid Khan, Stuart Ralph, Akhil Vaidya, Andy Waters<br />
<b>Toxoplasma</b>: John Boothroyd, Jon P. Boyle, Vern B. Carruthers, Marc-Jan Gubbels, Kami Kim, 
Markus Meissner, Jeroen Saeij, Lilach Sheiner, Ross Waller, Michael White<br />
<b>Trichomonas</b>: Jane Carlton, Patricia Johnson, Steven Sullivan, Jan Tachazy<br />
<b>Trypanosoma/Kinetoplastids</b>: Fernan Aguero, Vivian Bellofatto, Richard Burchmore, George Cross, Angela Cruz, Antonio Estevez, Mark Field, Catarina Gadelha, Eva Gluenz,Keith Gull, John Kelly, Annette MacLeod, Jeremy Mottram, Torsten Ochsenreiter, Marc Ouellette, Barbara Papadopoulou, Laurie Read, Sergio Schenkman, Rick Tarleton, Brent Weatherly, Bill Wickstead, Michael (Mick) Urbaniak <br />
<b>Vectors</b>: Gregory Dasch, Jeff Grabowski, María de Lourdes Muñoz, Monika Gulia-Nuss, Sukanya Narasimhan, Kristin Michel, Michael Povelones, Igor Sharakhov, Ronald van Rij, Rob Waterhouse</p>
  <h3>Previous Scientific Working Group</h3>

  <p>VEuPathDB wishes to acknowledge previous scientific working group members. They provided regular feedback oversight and guidance.</p>

  <table>
    <tbody>
      <tr>
        <td> </td>
        <td> </td>
        <td> </td>
        <td> </td>
        <td> </td>
        <td> </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Lyric Bartholomay   </td>
        <td>   Michael Gottlieb   </td>
        <td>   Malcolm McConville   </td>
        <td>   John Taylor   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Matt Berriman   </td>
        <td>   Keith Gull   </td>
        <td>   Nicola Mulder   </td>
        <td>   Jake Tu   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Bill Black   </td>
        <td>   Matthew Hahn   </td>
        <td>   Ull Munderloh   </td>
        <td>   Brett Tyler   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   John Boothroyd   </td>
        <td>   Adrian Hehl   </td>
        <td>   Daniel Neafsey   </td>
        <td>   Kenneth Vernick   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Greg Buck   </td>
        <td>   Steve Higgs   </td>
        <td>   Kenneth Olson   </td>
        <td>   Sarah Volkman   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Geraldine Butler   </td>
        <td>   Catherine Hill   </td>
        <td>   Bill Petri   </td>
        <td>   Jonathan Wastling   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Angela Cruz   </td>
        <td>   Marcelo Jacobs-Lorena   </td>
        <td>   Barry Pittendrigh   </td>
        <td>   Scott Weaver   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   George Dimopoulos   </td>
        <td>   Anthony (Tony) James   </td>
        <td>   Jeffrey Powell   </td>
        <td>   Louis Weiss   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Martin Donnelly   </td>
        <td>   Pedro Lagerblad de Oliveira   </td>
        <td>   Hillary Ranson   </td>
        <td>   Dyann Wirth   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Patrick Duffy   </td>
        <td>   Greg Lanzaro   </td>
        <td>   Alexander Raikhel   </td>
        <td>   Jennifer Wortman   </td>
        <td>   </td>
      </tr>
      <tr>
        <td>   </td>
        <td>   Pascale Gaudet   </td>
        <td>   Daniel Masiga   </td>
        <td>   Lincoln Stein   </td>
        <td>   Guiyun Yan   </td>
        <td>   </td>
      </tr>
    </tbody>
  </table>

</div>

<div style="break-before: always;"></div>
<div style="break-before: always;"></div>
<style>

ul.glossary {
   list-style-type: none;
}
ul.glossary li {
  padding: 0.5em;
}
ul.glossary li div.item-term {
  margin: 1em 0;
  font-weight: bold;
}

</style>

<h1>VEuPathDB Glossary</h1>

<div class="static-content">

<p>Please check the <a href="https://www.genome.gov/genetics-glossary" target="_blank">NCBI glossary</a></p>

<ul class="glossary">
  
    <a name="3-Frame translation (forward)"></a>
      <li>
        <div class="item-term">
          3-Frame translation (forward)
        </div>
        <div class="item-descr">
          <a href="#Translation">Translation</a> of a nucleotide sequence in all three possible reading frames in one direction, usually "on the top <a href="#Strand">strand</a>" of DNA.
        </div>
      </li>
  
    <a name="3-Frame translation (reverse)"></a>
      <li>
        <div class="item-term">
          3-Frame translation (reverse)
        </div>
        <div class="item-descr">
          <a href="#Translation">Translation</a> of a nucleotide sequence in all three possible reading frames in the reverse direction, usually "on the bottom <a href="#Strand">strand</a>" of DNA.
        </div>
      </li>
  
    <a name="AA sequence"></a>
      <li>
        <div class="item-term">
          AA sequence
        </div>
        <div class="item-descr">
          Amino acid sequence.
        </div>
      </li>
  
    <a name="Affymetrix genotyped SNP probes"></a>
      <li>
        <div class="item-term">
          Affymetrix genotyped SNP probes
        </div>
        <div class="item-descr">
          Probes on Affymetrix <a href="#SNP">SNP</a> (single nucleotide polymorphism) arrays, which are used for <a href="#SNP genotyping">SNP genotyping</a>. See <a href="#Affymetrix microarray technology">Affymetrix microarray technology</a> and <a href="http://www.affymetrix.com">www.affymetrix.com</a>
        </div>
      </li>
  
    <a name="Affymetrix microarray technology"></a>
      <li>
        <div class="item-term">
          Affymetrix microarray technology
        </div>
        <div class="item-descr">
          <a href="#Microarray">Microarray</a> manufacturing technology developed by Affymetrix. Combines semiconductor fabrication techniques, solid phase chemistry, combinatorial chemistry, molecular biology, and robotics to generate a photolithographic manufacturing process in which oligonucleotides are synthesized directly on a chip. See <a href="http://www.affymetrix.com">www.affymetrix.com</a>
        </div>
      </li>
  
    <a name="Affymetrix probes"></a>
      <li>
        <div class="item-term">
          Affymetrix probes
        </div>
        <div class="item-descr">
          Probe on an Affymetrix <a href="#Microarray">microarray</a> designed to determine whether or not the complementary sequence of RNA or DNA is present in a sample. Generally 25 nucleotides in length (25-mers), their short length provides higher specificity than longer probes. See <a href="#Affymetrix microarray technology">Affymetrix microarray technology</a> and <a href="http://www.affymetrix.com">www.affymetrix.com</a>
        </div>
      </li>
  
    <a name="Amitochondriate"></a>
      <li>
        <div class="item-term">
          Amitochondriate
        </div>
        <div class="item-descr">
          Eukaryotic organism that lacks a <a href="#Mitochondrion">mitochondrion</a>. Examples include Giardia and other parasites such as Trachipleistophora and Entamoeba. However, most of these organisms contain what appear to be mitochondrial remnants as well as mitochondrial <a href="#Gene">genes</a> in their nuclear genomes.
        </div>
      </li>
  
    <a name="Annotation"></a>
      <li>
        <div class="item-term">
          Annotation
        </div>
        <div class="item-descr">
          Identified feature within a sequence, such as a known or predicted <a href="#Gene">gene</a>, domain, motif, post-translational modification, etc.
        </div>
      </li>
  
    <a name="Annotation density"></a>
      <li>
        <div class="item-term">
          Annotation density
        </div>
        <div class="item-descr">
          Level to which a nucleotide or protein sequence has been annotated. See <a href="#Annotation">Annotation</a>.
        </div>
      </li>
  
    <a name="ApiCyc"></a>
      <li>
        <div class="item-term">
          ApiCyc
        </div>
        <div class="item-descr">
          Database/utility on EuPathDB used for searching and visualizing metabolic pathway information for organisms in EuPathDB; derivative database generated by analyzing various genomes (for example from Plasmodium, Cryptosporidium, and Toxoplasma) with SRI International's pathway tools.
        </div>
      </li>
  
    <a name="ApiDots alignments"></a>
      <li>
        <div class="item-term">
          ApiDots alignments
        </div>
        <div class="item-descr">
          Consensus sequences found in the ApiDots database and generated by clustering and assembling Apicomplexan mRNA and <a href="#EST">EST</a> sequences. These consensus sequences were subjected to database searches against protein and protein domain sequences.
        </div>
      </li>
  
    <a name="Apicoplast"></a>
      <li>
        <div class="item-term">
          Apicoplast
        </div>
        <div class="item-descr">
          Nonphotosynthetic plastid found in almost all protozoan parasites belonging to the phylum Apicomplexa that have been examined. The apicoplast is surrounded by four membranes, giving rise to the theory that its presence in the Apicomplexa is the result of a secondary endosymbiosis (acquired by the engulfment of an ancestral alga and retention of the algal plastid). Similar to other endosymbiotic organelles (mitochondria, chloroplasts), the apicoplast contains its own genome as well as proteins that are encoded in the nucleus and post-translationally imported. The apicoplast is a vital organelle to the parasite's long-term survival.
        </div>
      </li>
  
    <a name="Attribute"></a>
      <li>
        <div class="item-term">
          Attribute
        </div>
        <div class="item-descr">
          Inherent characteristic or feature; in a database, a data item related to a database object. For example, attributes of <a href="#Gene">genes</a> can include features such as introns and untranslated regions (UTRs).
        </div>
      </li>
  
    <a name="BLAST"></a>
      <li>
        <div class="item-term">
          BLAST
        </div>
        <div class="item-descr">
          Basic local alignment search tool, a <a href="#Sequence similarity">sequence similarity</a> search tool used to quickly find local alignments between a <a href="#Query">query</a> sequence and sequences in nucleotide or protein databases. Different versions of this search tool are available to match the types of query sequence and database used. See <a href="#blastn">blastn</a>, <a href="#blastp">blastp</a>, <a href="#blastx">blastx</a>, <a href="#tblastn">tblastn</a>, and <a href="#tblastx">tblastx</a>.
        </div>
      </li>
  
    <a name="Boolean"></a>
      <li>
        <div class="item-term">
          Boolean
        </div>
        <div class="item-descr">
          System of logical thought developed by George Boole (1815-1864). In Boolean searching, an "and" operator between two words or values (for example, "apple AND orange") generates a search for items in a database containing both of the words or values. Similarly, an "or" operator between two words or values (for example, "apple OR orange") generates a search for items containing either word.
        </div>
      </li>
  
    <a name="CDS"></a>
      <li>
        <div class="item-term">
          CDS
        </div>
        <div class="item-descr">
          Coding sequence. Region of nucleotides that corresponds to the sequence of amino acids in a predicted protein and that includes start and stop codons. Unexpressed sequences (for example, the 5'-UTR, the 3'-UTR, and introns) are not included within a CDS. The CDS usually does not correspond to the actual mRNA sequence.
        </div>
      </li>
  
    <a name="Centromere"></a>
      <li>
        <div class="item-term">
          Centromere
        </div>
        <div class="item-descr">
          Region of the <a href="#Chromosome">chromosome</a> or chromosomal structure essential for division and retention of the chromosome within the cell; point of a chromosome where the spindle fibers attach to pull the chromosome apart during cell division.
        </div>
      </li>
  
    <a name="Chromosome"></a>
      <li>
        <div class="item-term">
          Chromosome
        </div>
        <div class="item-descr">
          Macromolecule of DNA constituting the physical organization of DNA in a cell.
        </div>
      </li>
  
    <a name="Coil"></a>
      <li>
        <div class="item-term">
          Coil
        </div>
        <div class="item-descr">
          Three-dimensional spiral structure in protein macromolecules.
        </div>
      </li>
  
    <a name="Contig"></a>
      <li>
        <div class="item-term">
          Contig
        </div>
        <div class="item-descr">
          Contiguous genomic sequence assembled from overlapping primary sequences representing overlapping regions of a particular <a href="#Chromosome">chromosome</a>.
        </div>
      </li>
  
    <a name="CryptoCyc"></a>
      <li>
        <div class="item-term">
          CryptoCyc
        </div>
        <div class="item-descr">
          Database/utility built by analyzing the Cryptosporidium genome with SRI International's pathway tools; used for searching and visualizing Cryptosporidium metabolic pathway information.
        </div>
      </li>
  
    <a name="Curated annotation"></a>
      <li>
        <div class="item-term">
          Curated annotation
        </div>
        <div class="item-descr">
          <a href="#Annotation">Annotation</a> made under the supervision of a curator as opposed to a purely computational prediction. Curated predictions often contain combinations of different types of evidence to support the annotation.
        </div>
      </li>
  
    <a name="DNA/GC content"></a>
      <li>
        <div class="item-term">
          DNA/GC content
        </div>
        <div class="item-descr">
          Content of guanine (G) and cytosine (C) in a fragment of DNA or a genome. Because GC pairs are more thermostable compared to the AT pairs, it was commonly believed that GC content played a vital part in adaptation to high temperatures, a hypothesis that has been refuted. In the genome browser, the DNA/GC content track displays a GC content graph of the reference sequence at low magnifications and the DNA sequence itself at higher magnifications.
        </div>
      </li>
  
    <a name="Dalton"></a>
      <li>
        <div class="item-term">
          Dalton
        </div>
        <div class="item-descr">
          Unit of mass abbreviated Da and used to express atomic and molecular masses.
        </div>
      </li>
  
    <a name="Deprecated gene"></a>
      <li>
        <div class="item-term">
          Deprecated gene
        </div>
        <div class="item-descr">
          <a href="#Gene">Genes</a> with little or no evidence (similarities / expression) that overlapped (or were subsumed by) larger genes for which there was evidence such as protein similarities, expression evidence from <a href="#EST alignments">EST alignments</a>, SAGE or <a href="#Proteomics">proteomics</a> data were marked as deprecated.  These genes will likely be removed in a subsequent release (and in GenBank) unless additional evidence is provided indicating they should be moved into the real gene category.
        </div>
      </li>
  
    <a name="EC numbers"></a>
      <li>
        <div class="item-term">
          EC numbers
        </div>
        <div class="item-descr">
          Enzyme Commission numbers. EC numbers constitute the numerical classification scheme for enzymes based on the chemical reactions they catalyze. EC numbers do not refer to the enzymes, but to the reactions they catalyze.
        </div>
      </li>
  
    <a name="EST"></a>
      <li>
        <div class="item-term">
          EST
        </div>
        <div class="item-descr">
          Expressed sequence tag. Short (typically 100-500 base pairs) partial <a href="#cDNA">cDNA</a> produced by single-shot sequencing of a cloned mRNA (cDNA) and often used to identify <a href="#Gene">gene</a> transcripts.
        </div>
      </li>
  
    <a name="EST alignments"></a>
      <li>
        <div class="item-term">
          EST alignments
        </div>
        <div class="item-descr">
          Alignments of expressed sequence tags (ESTs) with a corresponding genomic region. For example, in ToxoDB you can visualize <a href="#EST">EST</a> alignments by clicking on the "View this sequence in the genome browser" link and turning on the EST Alignments track. Useful for identifying intron boundaries.
        </div>
      </li>
  
    <a name="EST clusters"></a>
      <li>
        <div class="item-term">
          EST clusters
        </div>
        <div class="item-descr">
          Groups of homologous, overlapping <a href="#EST">EST</a> sequences created to reduce redundancy of the EST database.
        </div>
      </li>
  
    <a name="Expression level"></a>
      <li>
        <div class="item-term">
          Expression level
        </div>
        <div class="item-descr">
          Level at which an mRNA or protein is present in a sample. Value can be absolute or relative to other mRNA or protein species in the sample.
        </div>
      </li>
  
    <a name="Expression profile"></a>
      <li>
        <div class="item-term">
          Expression profile
        </div>
        <div class="item-descr">
          Pattern of expression of one or more <a href="#Gene">genes</a> or proteins over time or over a set of experimental conditions (for example, during development or treatment, or as a result of a genetic mutation such as a knockout).
        </div>
      </li>
  
    <a name="Expression profile correlation"></a>
      <li>
        <div class="item-term">
          Expression profile correlation
        </div>
        <div class="item-descr">
          Method for correlation of <a href="#Gene">gene</a> <a href="#Expression profile">expression profiles</a> with gene ontology (GO) <a href="#Annotation">annotations</a> developed for the purpose of identifying groups of genes, pathways, and processes reacting in concert to experimental perturbations.
        </div>
      </li>
  
    <a name="Expression timing"></a>
      <li>
        <div class="item-term">
          Expression timing
        </div>
        <div class="item-descr">
          Timing of <a href="#Gene">gene</a> expression during a developmental, metabolic, regulatory, or other biological process or response.
        </div>
      </li>
  
    <a name="GBrowse"></a>
      <li>
        <div class="item-term">
          GBrowse
        </div>
        <div class="item-descr">
          Interactive genome browser developed by the Generic Model Organism Database (GMOD) project (<a href="http://www.gmod.org">www.gmod.org</a>) that can be customized to show selected chromosomal features as well as display user-provided <a href="#Annotation">annotations</a>.
        </div>
      </li>
  
    <a name="GBrowse track"></a>
      <li>
        <div class="item-term">
          GBrowse track
        </div>
        <div class="item-descr">
          In the <a href="#GBrowse">GBrowse</a> viewer, a line of data that corresponds to a particular type of genomic information or feature and that is distinguished by a particular shape or color.
        </div>
      </li>
  
    <a name="Gametocyte"></a>
      <li>
        <div class="item-term">
          Gametocyte
        </div>
        <div class="item-descr">
          Eukaryotic germ cell that divides by mitosis to generate other gametocytes or by meiosis to generate gametes. Male gametocytes are called spermatocytes, and female gametocytes are called oocytes. Term often used to describe gametocytes of Plasmodium.
        </div>
      </li>
  
    <a name="GenBank protein record"></a>
      <li>
        <div class="item-term">
          GenBank protein record
        </div>
        <div class="item-descr">
          Protein sequence file in the GenBank database generally derived by <a href="#Translation">translation</a> of a related nucleotide record.
        </div>
      </li>
  
    <a name="GenPept protein"></a>
      <li>
        <div class="item-term">
          GenPept protein
        </div>
        <div class="item-descr">
          Protein record from the GenPept database at the <a href="#NCBI">NCBI</a> GenBank, which contains inferred <a href="#Translation">translations</a> of <a href="#Protein-coding">protein-coding</a> sequences.
        </div>
      </li>
  
    <a name="Gene"></a>
      <li>
        <div class="item-term">
          Gene
        </div>
        <div class="item-descr">
          Fundamental physical and functional unit of heredity. Ordered sequence of nucleotides located in a particular position on a particular <a href="#Chromosome">chromosome</a> that encodes a specific functional product, such as a protein or RNA molecule. A gene may have a number of parts, including the <a href="#Promoter">promoter</a> region, untranslated regions (5' and 3' <a href="#UTR">UTRs</a>), introns, and exons.
        </div>
      </li>
  
    <a name="GeneDB"></a>
      <li>
        <div class="item-term">
          GeneDB
        </div>
        <div class="item-descr">
          Project developed by the Sanger Institute Pathogen Sequencing Unit (PSU) and aimed at developing and maintaining curated database resources for all projects handled by the PSU. The database is accessible at <a href="http://www.genedb.org">www.genedb.org</a>
        </div>
      </li>
  
    <a name="Genetic markers"></a>
      <li>
        <div class="item-term">
          Genetic markers
        </div>
        <div class="item-descr">
          Known DNA sequences that can be identified by a simple assay. Generally genetic variations caused by mutation or alterations in loci that can be observed, examples include restriction length polymorphisms (RFLPs), short tandem repeats (STRs), variable number tandem repeats (VNTRs), short DNA sequences surrounding single base-pair changes (single nucleotide polymorphisms, or <a href="#SNP">SNPs</a>), or longer <a href="#Microsatellite">microsatellite</a> sequences.
        </div>
      </li>
  
    <a name="Genomic context"></a>
      <li>
        <div class="item-term">
          Genomic context
        </div>
        <div class="item-descr">
          Location of a <a href="#Gene">gene</a> in the genome, which can influence the expression of the gene and functional interactions of the gene expression products. In this database, genes are depicted on individual gene pages with their surrounding genomic region and <a href="#Annotation">annotations</a>.
        </div>
      </li>
  
    <a name="Genome Sequence"></a>
      <li>
        <div class="item-term">
          Genome Sequence
        </div>
        <div class="item-descr">
          The inherited nucleic acid component of the genome. In eukaryotes, organized into linear <a href="#Chromosome">chromosomes</a> that are capped with <a href="#Telomere">telomeres</a> and located in the nucleus.
        </div>
      </li>
  
    <a name="Genotyped SNPs"></a>
      <li>
        <div class="item-term">
          Genotyped SNPs
        </div>
        <div class="item-descr">
          Single nucleotide polymorphisms (SNPs) identified during <a href="#Genotyping">genotyping</a> of individual organism strains. See <a href="#SNP genotyping">SNP genotyping</a>.
        </div>
      </li>
  
    <a name="Genotyping"></a>
      <li>
        <div class="item-term">
          Genotyping
        </div>
        <div class="item-descr">
          Process of determining the genotype of an individual with a biological assay using PCR, DNA sequencing, or hybridization to DNA <a href="#Microarray">microarrays</a> or beads. Provides a measurement of the genetic variation between members of a species.
        </div>
      </li>
  
    <a name="GLEAN gene"></a>
      <li>
        <div class="item-term">
          GLEAN gene
        </div>
        <div class="item-descr">
          Predicted <a href="#Gene">gene</a> sequence generated by GLEAN, an algorithm that integrates different sources of gene structure evidence (for example, gene model predictions, <a href="#EST">EST</a> and protein sequence alignments to the genome, and SAGE or peptide tags) to produce a consensus gene prediction in the absence of known genes.
        </div>
      </li>
  
    <a name="GO"></a>
      <li>
        <div class="item-term">
          GO
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> Ontology project. Collaborative project that has developed three structured, controlled vocabularies (ontologies) to describe gene products in terms of their associated biological processes, cellular components, and molecular functions in a species-independent manner. The use of a consistent vocabulary allows genes from different species to be compared based on their GO <a href="#Annotation">annotations</a>.
        </div>
      </li>
  
    <a name="GO component"></a>
      <li>
        <div class="item-term">
          GO component
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> Ontology term used to describe a cellular component, or the location where a gene product may act, rather than physical features of proteins or RNAs. For example, membrane (GO:0016020), extrinsic to membrane (GO:0019898), and integral to membrane (GO:0016021).
        </div>
      </li>
  
    <a name="GO function"></a>
      <li>
        <div class="item-term">
          GO function
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> Ontology term used to describe the molecular function of a gene product, the jobs that it performs, or the "abilities" that it has (for example, transporting compounds, binding to things, holding things together, and changing one thing into another). This is different from the biological processes the gene product is involved in, which involve more than one activity.
        </div>
      </li>
  
    <a name="GO process"></a>
      <li>
        <div class="item-term">
          GO process
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> Ontology term used to describe a biological process, a recognized series of events, or molecular functions associated with a gene product. A biological process is not equivalent to a pathway, though some <a href="#GO term">GO terms</a> do describe pathways.
        </div>
      </li>
  
    <a name="GO term"></a>
      <li>
        <div class="item-term">
          GO term
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> Ontology term. The building blocks of the Gene Ontology, each term is assigned to one of the three ontologies: molecular function, cellular component, or biological process. Each <a href="#GO">GO</a> term consists of a unique alphanumerical identifier, a common name, synonyms (if applicable), and a definition. When a term has multiple meanings depending on species, the GO uses a "sensu" tag to differentiate among them. For example, the enzyme fumarase has the GO term GO:0004333, fumarate hydratase activity (fumarase activity), catalysis of the reaction: (S)-malate = fumarate + H2O.
        </div>
      </li>
  
    <a name="GPI anchor"></a>
      <li>
        <div class="item-term">
          GPI anchor
        </div>
        <div class="item-descr">
          C-terminal post-translational modification of many eukaryotic proteins. The two fatty acids within the glycophosphatidylinositol (GPI) moiety anchor the protein to the outer leaflet of the plasma membrane. GPI-anchored proteins are believed to be involved in signal transduction and immune responses, as well as the pathobiology of many parasites.
        </div>
      </li>
  
    <a name="HMM"></a>
      <li>
        <div class="item-term">
          HMM
        </div>
        <div class="item-descr">
          Hidden Markov model. Statistical model in which the system being modeled is assumed to be a Markov process with unknown parameters, and the challenge is to determine the hidden parameters from the observable parameters. The extracted model parameters can then be used to perform further analysis, for example for pattern recognition applications.
        </div>
      </li>
  
    <a name="Homolog"></a>
      <li>
        <div class="item-term">
          Homolog
        </div>
        <div class="item-descr">
          Related by evolutionary descent, either between species (ortholog) or within a species (paralog).
        </div>
      </li>
  
    <a name="Hydropathy"></a>
      <li>
        <div class="item-term">
          Hydropathy
        </div>
        <div class="item-descr">
          Hydropathicity. Degree to which a peptide or protein is likely to be soluble in water. Protein hydropathy plots can be useful in predicting <a href="#Transmembrane domain">transmembrane domains</a>, potential antigenic sites, and regions that are likely to be exposed on the protein's surface.
        </div>
      </li>
  
    <a name="Interactome"></a>
      <li>
        <div class="item-term">
          Interactome
        </div>
        <div class="item-descr">
          The complete set of physical interactions between proteins within a specific biological context such as an organelle, cell, or organism. Can also refer to the complete set of genetic interactions between genes. Usually determined using genome-wide techniques, although the complete set of interactions is difficult to capture due to experimental noise and limits of detection.
        </div>
      </li>
  
    <a name="Intergenic region"></a>
      <li>
        <div class="item-term">
          Intergenic region
        </div>
        <div class="item-descr">
          Stretch of DNA located between <a href="#Gene">genes</a> that may act to regulate gene expression.
        </div>
      </li>
  
    <a name="Intersect"></a>
      <li>
        <div class="item-term">
          Intersect
        </div>
        <div class="item-descr">
          Similar to the <a href="#Boolean">Boolean</a> operator "AND", an action used in the <a href="#Query">query</a> history to find items that are common to two query result sets. For example, to search for items that appear in both result set X and result Y, type, "X INTERSECT Y".
        </div>
      </li>
  
    <a name="Join"></a>
      <li>
        <div class="item-term">
          Join
        </div>
        <div class="item-descr">
          Similar to the <a href="#Boolean">Boolean</a> operator "UNION", an action used in the <a href="#Query">query</a> history to combine query sets. For example, to combine result sets X and Y, type, "X JOIN Y".
        </div>
      </li>
  
    <a name="JBrowse"></a>
      <li>
        <div class="item-term">
          JBrowse
        </div>
        <div class="item-descr">
          Interactive genome browser (<a href="https://jbrowse.org/">jbrowse.org</a>) developed by the Generic Model Organism Database (GMOD) project (<a href="http://www.gmod.org">www.gmod.org</a>) that can be customized to show selected chromosomal features as well as display user-provided <a href="#Annotation">annotations</a>.
        </div>
      </li>
  
    <a name="KEGG map"></a>
      <li>
        <div class="item-term">
          KEGG map
        </div>
        <div class="item-descr">
          Metabolic or regulatory interaction pathway generated by the Kyoto Encyclopedia of <a href="#Gene">Genes</a> and Genomes (KEGG) or by the use of their tools (<a href="http://www.genome.jp/kegg/">www.genome.jp/kegg</a>).
        </div>
      </li>
  
    <a name="Locus"></a>
      <li>
        <div class="item-term">
          Locus
        </div>
        <div class="item-descr">
          Position on a <a href="#Chromosome">chromosome</a> of a <a href="#Gene">gene</a>, feature (such as a <a href="#Telomere">telomere</a>), or other chromosomal marker; also, the DNA at that position. Use of this term is sometimes restricted to mean expressed DNA regions.
        </div>
      </li>
  
    <a name="Low complexity"></a>
      <li>
        <div class="item-term">
          Low complexity
        </div>
        <div class="item-descr">
          Pertaining to sequence regions that have an unusually repetitive nature (for example, a protein sequence of low complexity might look like PPTDPPPPKKDGGPPL, and a low-complexity nucleotide sequence might be AAATAAAAAAAAATAAAAAAATTA). Low-complexity regions can create problems in <a href="#Sequence similarity">sequence similarity</a> searching by causing artifactual hits. For this reason, filters are often used to remove low-complexity sequences. Low-complexity regions also contribute to antigenic variation in apicomplexan parasites.
        </div>
      </li>
  
    <a name="Mass spec data"></a>
      <li>
        <div class="item-term">
          Mass spec data
        </div>
        <div class="item-descr">
          Mass spectrometry data. Mass spectrometry is an analytical technique used to measure the mass-to-charge ratios of small molecules in several applications, including identification of proteins or peptides. In our databases, the "Identify <a href="#Gene">Genes</a> by Mass Spec Evidence" <a href="#Query">query</a> is used to identify genes that have evidence of protein expression based on mass spec data.
        </div>
      </li>
  
    <a name="Metabolic pathways"></a>
      <li>
        <div class="item-term">
          Metabolic pathways
        </div>
        <div class="item-descr">
          Series of chemical reactions occurring within a cell and often catalyzed by enzymes. In a pathway, a molecule is often changed or modified into another product, which can be stored by the cell, used as a metabolic product, or used to initiate another pathway.
        </div>
      </li>
  
    <a name="Microarray"></a>
      <li>
        <div class="item-term">
          Microarray
        </div>
        <div class="item-descr">
          Microscopic array of biological molecules (for example, DNA or protein) used to determine the presence and/or amount (referred to as quantitation) of other biomolecules (other proteins, transcripts, etc.) in biological samples.
        </div>
      </li>
  
    <a name="Microsatellite"></a>
      <li>
        <div class="item-term">
          Microsatellite
        </div>
        <div class="item-descr">
          Polymorphic <a href="#Locus">locus</a> in nuclear and organellar DNA that consists of repeating units of 1-4 base pairs in length. Mostly neutral and codominant, microsatellites are used as molecular markers and to study <a href="#Gene">gene</a> dosage (looking for duplications or deletions of a particular genetic region). Also known as simple sequence repeats (SSRs).
        </div>
      </li>
  
    <a name="Microsatellite map"></a>
      <li>
        <div class="item-term">
          Microsatellite map
        </div>
        <div class="item-descr">
          Map of <a href="#Microsatellite">microsatellite</a> locations and linkages on a genome.
        </div>
      </li>
  
    <a name="Mitochondrion"></a>
      <li>
        <div class="item-term">
          Mitochondrion
        </div>
        <div class="item-descr">
          Organelle responsible for respiration in a eukaryotic cell. Proteins required for mitochondrial function are encoded both in the nucleus and within the smaller mitochondrial genome.
        </div>
      </li>
  
    <a name="Motif search"></a>
      <li>
        <div class="item-term">
          Motif search
        </div>
        <div class="item-descr">
          Tool used to identify and locate sequence patterns (motifs) in protein and nucleic acid sequences. In our databases, this flexible search can be based on the general characteristics of the pattern and not solely on specific sequences (for example, Cys-[9-11 amino acids]-Cys or Leu-Leu-[basic residue]-Val). This allows the user to <a href="#Query">query</a> using previously undescribed motifs.
        </div>
      </li>
  
    <a name="NCBI"></a>
      <li>
        <div class="item-term">
          NCBI
        </div>
        <div class="item-descr">
          National Center for Biotechnology Information. Established in 1988 as a national resource for molecular biology information, NCBI creates public databases, conducts research in computational biology, develops software tools for analyzing genome data, and disseminates biomedical information - all for the better understanding of molecular processes affecting human health and disease.
        </div>
      </li>
  
    <a name="Nonredundant protein DB (NRDB)"></a>
      <li>
        <div class="item-term">
          Nonredundant protein DB (NRDB)
        </div>
        <div class="item-descr">
          Peptide sequence database containing all nonidentical protein sequences from the same species extracted from GenPept, <a href="#NCBI">NCBI</a> RefSeq, Swiss-Prot, and PRF databases. Used for <a href="#BLAST">BLAST</a> protein database searches because its smaller size results in shorter search times and more meaningful statistics.
        </div>
      </li>
  
    <a name="Nucmer"></a>
      <li>
        <div class="item-term">
          Nucmer
        </div>
        <div class="item-descr">
          NUCleotide MUMmer. Part of the MUMmer alignment package, an alignment tool used for the rapid alignment of very large DNA and amino acid sequences.
        </div>
      </li>
  
    <a name="ORF"></a>
      <li>
        <div class="item-term">
          ORF
        </div>
        <div class="item-descr">
          Amino acid sequences computed by translating the six frames of raw genomic sequence using the standard genetic code. We save the translated sequences having at least 50 amino acids. The sequences are not annotated nor human reviewed. ORF's do not necessarily begin with methionine residues, but they all terminate with a stop codons.
        </div>
      </li>
  
    <a name="Oligo"></a>
      <li>
        <div class="item-term">
          Oligo
        </div>
        <div class="item-descr">
          Oligonucleotide. Short sequence of DNA or RNA, typically of 20-70 nucleotides.
        </div>
      </li>
  
    <a name="Oligonucleotide microarray"></a>
      <li>
        <div class="item-term">
          Oligonucleotide microarray
        </div>
        <div class="item-descr">
          Collection of microscopic oligonucleotide spots arrayed on a solid surface by covalent attachment to chemically suitable matrices. Used for expression profiling, the monitoring of the <a href="#Expression level">expression levels</a> of hundreds or thousands of <a href="#Gene">genes</a> simultaneously. Probes for oligonucleotide <a href="#Microarray">microarrays</a> are designed to match parts of known or predicted mRNAs.
        </div>
      </li>
  
    <a name="Organellar genome sequence"></a>
      <li>
        <div class="item-term">
          Organellar genome sequence
        </div>
        <div class="item-descr">
          The genome sequence contained in an organelle i.e. the <a href="#Mitochondrion">mitochondria</a> and/or <a href="#Apicoplast">apicoplast</a> organelles.  These sequences may be circular or linear and exist in addition to the much larger nuclear <a href="#Genome Sequence">genome sequence</a>.
        </div>
      </li>
  
    <a name="Open Reading Frame"></a>
      <li>
        <div class="item-term">
          Open Reading Frame
        </div>
        <div class="item-descr">
          See <a href="#ORF">ORF</a>
        </div>
      </li>
  
    <a name="OrthoMCL"></a>
      <li>
        <div class="item-term">
          OrthoMCL
        </div>
        <div class="item-descr">
          Genome-scale algorithm for grouping orthologous protein sequences. It provides <a href="#Gene">genes</a> shared by two or more species/genomes and also genes representing species-specific gene expansion families. Therefore, it serves as a utility for automated eukaryotic genome <a href="#Annotation">annotation</a> and phylogenetic profiling. Available at <a href="https://orthomcl.org">orthomcl.org</a>
        </div>
      </li>
  
    <a name="Ortholog"></a>
      <li>
        <div class="item-term">
          Ortholog
        </div>
        <div class="item-descr">
          Same <a href="#Gene">gene</a> in different organisms; having evolved from the same ancestral <a href="#Locus">locus</a>.
        </div>
      </li>
  
    <a name="Ortholog group"></a>
      <li>
        <div class="item-term">
          Ortholog group
        </div>
        <div class="item-descr">
          Orthologous <a href="#Gene">genes</a> shared by group of organisms; the group of genes can also contain <a href="#Paralog">paralogs</a>.
        </div>
      </li>
  
    <a name="Orthology-based phylogenetic profile"></a>
      <li>
        <div class="item-term">
          Orthology-based phylogenetic profile
        </div>
        <div class="item-descr">
          Tool used to find <a href="#Gene">genes</a> that are present or not present in a desired group of organisms on the tree of life (currently computed for 81 complete genomes). The user has control over the profile and over whether or not genes must be found in any particular group of organisms (for example, in Apicomplexa but not in mammals). Taxa can also be marked as indifferent (for example, it does not matter if the gene is also found in plants). <a href="#Ortholog">Ortholog</a> and <a href="#Paralog">paralog</a> relationships are determined using the <a href="#OrthoMCL">OrthoMCL</a> algorithm.
        </div>
      </li>
  
    <a name="PATS"></a>
      <li>
        <div class="item-term">
          PATS
        </div>
        <div class="item-descr">
          Neural network analysis tool that identifies amino acid sequences within a <a href="#Query">query</a> sequence that are potentially targeted to the <a href="#Apicoplast">apicoplast</a> matrix of Plasmodium falciparum.
        </div>
      </li>
  
    <a name="PDB 3D structure"></a>
      <li>
        <div class="item-term">
          PDB 3D structure
        </div>
        <div class="item-descr">
          Three-dimensional macromolecular structure in the Protein Data Bank (PDB) (<a href="http://www.pdb.org">www.pdb.org</a>) obtained by one of three methods: X-ray crystallography (over 80%), solution nuclear magnetic resonance (NMR) (about 16%), or theoretical modeling (2%). A few structures were determined by other methods.
        </div>
      </li>
  
    <a name="PROSITE motif"></a>
      <li>
        <div class="item-term">
          PROSITE motif
        </div>
        <div class="item-descr">
          Protein sequence pattern or profile derived from multiple alignments of homologous sequences and stored in the PROSITE database (<a href="https://prosite.expasy.org/">prosite.expasy.org</a>), an annotated collection of motif descriptors dedicated to the identification of protein families and domains.
        </div>
      </li>
  
    <a name="Paralog"></a>
      <li>
        <div class="item-term">
          Paralog
        </div>
        <div class="item-descr">
          Related by <a href="#Gene">gene</a> duplication within a genome; originated by duplication and then diverged from the parent copy by mutation and selection or drift.
        </div>
      </li>
  
    <a name="Pearson correlation"></a>
      <li>
        <div class="item-term">
          Pearson correlation
        </div>
        <div class="item-descr">
          Pearson Product Moment Correlation, the most common measure of the correlation between two variables. Reflects the degree of linear relationship between two variables and ranges from +1 to -1, with a correlation of +1 indicating a perfect positive linear relationship between variables.
        </div>
      </li>
  
    <a name="Peptide mass fingerprinting"></a>
      <li>
        <div class="item-term">
          Peptide mass fingerprinting
        </div>
        <div class="item-descr">
          Analytical technique for protein identification wherein an unknown protein of interest is cleaved into peptides by a protease such as trypsin, and the peptides resulting from this cleavage are analyzed using a mass spectrometric method such as MALDI-TOF or ESI-TOF. The masses derived for the peptides are then compared to a database containing known protein sequences or even to the genome. Computer programs theoretically cut the protein sequences in the database into peptides with the same protease (for example trypsin), and calculate the absolute masses of the peptides from each protein. They then compare the masses of the peptides of the unknown protein to the theoretical peptide masses of each protein encoded in the genome. The results are statistically analyzed to find the best match.
        </div>
      </li>
  
    <a name="Pfam domain"></a>
      <li>
        <div class="item-term">
          Pfam domain
        </div>
        <div class="item-descr">
          Conserved protein region in the Pfam database (<a href="http://pfam.xfam.org">Pfam.org</a>), a collection of multiple sequence alignments and hidden Markov models covering many common protein families. The alignments may represent evolutionarily conserved structures that may shed light on protein function. Profile hidden Markov models (profile <a href="#HMM">HMMs</a>) built from the Pfam alignments can be useful for associating a new protein to a known protein family, even if the homology is weak. Unlike standard pairwise alignment methods (for example, <a href="#BLAST">BLAST</a> and FASTA), Pfam HMMs deal sensibly with multidomain proteins.
        </div>
      </li>
  
    <a name="Phylogeny"></a>
      <li>
        <div class="item-term">
          Phylogeny
        </div>
        <div class="item-descr">
          Historical relationships among lineages of organisms or their parts, including their <a href="#Gene">genes</a>.
        </div>
      </li>
  
    <a name="PlasmoAP"></a>
      <li>
        <div class="item-term">
          PlasmoAP
        </div>
        <div class="item-descr">
          Algorithm/tool that predicts the likelihood that a protein sequence is targeted to the <a href="#Apicoplast">apicoplast</a>. It provides the position of <a href="#Signal peptide">signal peptide</a> cleavage sites in amino acid sequences if targeting is predicted.
        </div>
      </li>
  
    <a name="PlasmoCyc"></a>
      <li>
        <div class="item-term">
          PlasmoCyc
        </div>
        <div class="item-descr">
          Database/utility built by analyzing the genomes of the Plasmodium species in EuPathDB with SRI International's pathway tools; used for searching and visualizing Plasmodium metabolic pathway information.
        </div>
      </li>
  
    <a name="ProDom"></a>
      <li>
        <div class="item-term">
          ProDom
        </div>
        <div class="item-descr">
          Database of protein domain families generated from the global comparison of all available protein sequences (<a href="http://prodom.prabi.fr">prodom.prabi.fr</a>).
        </div>
      </li>
  
    <a name="Promoter"></a>
      <li>
        <div class="item-term">
          Promoter
        </div>
        <div class="item-descr">
          Regulatory region of DNA located upstream (towards the 5' region) of a <a href="#Gene">gene</a> and providing a control point for regulated gene transcription.
        </div>
      </li>
  
    <a name="Protein-coding"></a>
      <li>
        <div class="item-term">
          Protein-coding
        </div>
        <div class="item-descr">
          Capable of encoding a protein sequence; generally refers to a sequence of DNA.
        </div>
      </li>
  
    <a name="Proteomics"></a>
      <li>
        <div class="item-term">
          Proteomics
        </div>
        <div class="item-descr">
          The large-scale study of proteins, particularly of the full set of proteins encoded by a genome.
        </div>
      </li>
  
    <a name="Pseudogene"></a>
      <li>
        <div class="item-term">
          Pseudogene
        </div>
        <div class="item-descr">
          Defunct relatives of known <a href="#Gene">genes</a> that have lost their <a href="#Protein-coding">protein-coding</a> ability or are otherwise no longer expressed in the cell. Although they may have some gene-like features (such as <a href="#Promoter">promoters</a>, CpG islands, and splice sites), they are nonetheless considered nonfunctional due to their lack of protein-coding ability resulting from various genetic disablements (stop codons, frameshifts, or a lack of transcription) or their inability to function as an RNA (such as with <a href="#rRNA">rRNA</a> pseudogenes).
        </div>
      </li>
  
    <a name="PubCrawler"></a>
      <li>
        <div class="item-term">
          PubCrawler
        </div>
        <div class="item-descr">
          Free service that scans daily updates to the <a href="#NCBI">NCBI</a> Medline (PubMed) and GenBank databases and alerts users to any relevant updates. Available at <a href="http://pubcrawler.gen.tcd.ie">pubcrawler.gen.tcd.ie</a>
        </div>
      </li>
  
    <a name="Query"></a>
      <li>
        <div class="item-term">
          Query
        </div>
        <div class="item-descr">
          Sequence or term used in a database search. For example, the sequence submitted for a <a href="#BLAST">BLAST</a> search is the query sequence.
        </div>
      </li>
  
    <a name="RNA predictions"></a>
      <li>
        <div class="item-term">
          RNA predictions
        </div>
        <div class="item-descr">
          Predictions of <a href="#Gene">genes</a> that encode nonprotein-encoding RNA's such as <a href="#tRNA">tRNA</a>, snoRNA, <a href="#rRNA">rRNA</a>, etc.
        </div>
      </li>
  
    <a name="RefSeq mRNA"></a>
      <li>
        <div class="item-term">
          RefSeq mRNA
        </div>
        <div class="item-descr">
          Nonredundant mRNA sequence in the RefSeq database. RefSeq mRNA sequences with an NM_XXXXXX accession are curated sequences and are, therefore, considered more reliable than those with XM_XXXXXX accessions (predicted mRNA sequences).
        </div>
      </li>
  
    <a name="Reference genome sequence"></a>
      <li>
        <div class="item-term">
          Reference genome sequence
        </div>
        <div class="item-descr">
          The community gold standard genome sequence. Usually the most complete assembly including an annotation. Reference genome sequences do change as superior assemblies and annotation become available.
        </div>
      </li>
  
    <a name="RefSeq noncoding RNA"></a>
      <li>
        <div class="item-term">
          RefSeq noncoding RNA
        </div>
        <div class="item-descr">
          Nonredundant noncoding RNA (ncRNA) sequence in the RefSeq database. RefSeq <a href="#ncRNA">ncRNA</a> sequences with an NR_XXXXXX accession are curated sequences and are, therefore, considered more reliable than those with XR_XXXXXX accessions (predicted ncRNA sequences).
        </div>
      </li>
  
    <a name="RefSeq protein"></a>
      <li>
        <div class="item-term">
          RefSeq protein
        </div>
        <div class="item-descr">
          Nonredundant protein sequence in the RefSeq database. RefSeq protein sequences with NP_XXXXXX accessions are curated sequences and are, therefore, considered more reliable than those with XP_XXXXXX accessions (predicted protein sequences).
        </div>
      </li>
  
    <a name="RefSeq"></a>
      <li>
        <div class="item-term">
          RefSeq
        </div>
        <div class="item-descr">
          <a href="#NCBI">NCBI</a> reference sequences. A curated nonredundant collection of sequences representing genomes, transcripts, and proteins as annotated by NCBI (available at <a href="https://www.ncbi.nlm.nih.gov/refseq/">www.ncbi.nlm.nih.gov/refseq</a>). The <a href="#Annotation">annotation</a> in these records is often different from the original GenBank submission, which may not be updated every time new information is obtained.
        </div>
      </li>
  
    <a name="Repeat regions"></a>
      <li>
        <div class="item-term">
          Repeat regions
        </div>
        <div class="item-descr">
          Sequences present in many identical or highly similar copies in the genome.
        </div>
      </li>
  
    <a name="SAGE tags"></a>
      <li>
        <div class="item-term">
          SAGE tags
        </div>
        <div class="item-descr">
          Serial analysis of <a href="#Gene">gene</a> expression (SAGE) tags. Short (14-nucleotide) sequences found within mRNA, the relative abundance of which indicates the level of expression of the mRNA containing that tag.
        </div>
      </li>
  
    <a name="SNP"></a>
      <li>
        <div class="item-term">
          SNP
        </div>
        <div class="item-descr">
          Single nucleotide polymorphism. Small genetic changes or variations that can occur within a DNA sequence, for example when a single nucleotide, such as an A replacing one of the other three nucleotide letters C, G, or T. Most SNPs are found outside of coding sequences, but SNPs found within a coding sequence are more likely to alter the biological function of a protein. SNPs may be synonymous (generating a conservative change not altering the amino acid sequence) or they can be nonsynonymous and change the amino acid that is encoded.
        </div>
      </li>
  
    <a name="SNP density"></a>
      <li>
        <div class="item-term">
          SNP density
        </div>
        <div class="item-descr">
          Amount or number of single nucleotide polymorphisms (SNPs) in a region of the genome.
        </div>
      </li>
  
    <a name="SNP genotyping"></a>
      <li>
        <div class="item-term">
          SNP genotyping
        </div>
        <div class="item-descr">
          Identifying and mapping single nucleotide polymorphisms (SNPs) in an effort to determine the genotype members of a species. <a href="#SNP">SNPs</a> usually consist of two alleles (where the rare allele frequency is less than 1%), are evolutionarily conserved, and are the most common type of genetic variation. See <a href="#Genotyping">Genotyping</a>.
        </div>
      </li>
  
    <a name="Scaffolds"></a>
      <li>
        <div class="item-term">
          Scaffolds
        </div>
        <div class="item-descr">
          In genomic mapping, a series of <a href="#Contig">contigs</a> that are in the right order and orientation, but not necessarily connected in one continuous stretch of sequence.
        </div>
      </li>
  
    <a name="Sequence similarity"></a>
      <li>
        <div class="item-term">
          Sequence similarity
        </div>
        <div class="item-descr">
          Degree of similarity between two or more protein or nucleotide sequences.
        </div>
      </li>
  
    <a name="Signal peptide"></a>
      <li>
        <div class="item-term">
          Signal peptide
        </div>
        <div class="item-descr">
          Short (3-60 amino acids) peptide sequence that directs the co-translational import of a protein to certain organelles or for secretion.
        </div>
      </li>
  
    <a name="SignalP"></a>
      <li>
        <div class="item-term">
          SignalP
        </div>
        <div class="item-descr">
          Program that predicts the presence and location of <a href="#Signal peptide">signal peptide</a> cleavage sites in amino acid sequences from Gram-positive and -negative prokaryotes and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/nonsignal peptide prediction based on a combination of several artificial neural networks and hidden Markov models (HMMs).
        </div>
      </li>
  
    <a name="Strand"></a>
      <li>
        <div class="item-term">
          Strand
        </div>
        <div class="item-descr">
          One half of the DNA helix; string or stretch of covalently linked nucleotides.
        </div>
      </li>
  
    <a name="Subtract"></a>
      <li>
        <div class="item-term">
          Subtract
        </div>
        <div class="item-descr">
          Similar to the <a href="#Boolean">Boolean</a> operator "NOT", an action used in the <a href="#Query">query</a> history to remove items from one result set that occur in another result set. For example, to remove items that exist in set Y from those in set X, type, "X SUBTRACT Y".
        </div>
      </li>
  
    <a name="Syntenic"></a>
      <li>
        <div class="item-term">
          Syntenic
        </div>
        <div class="item-descr">
          Loci located on the same <a href="#Chromosome">chromosome</a> but not necessarily linked. For example, <a href="#Gene">genes</a> that are part of a syntenic group share a common chromosomal location. Also used to refer to conservation of gene order across species.
        </div>
      </li>
  
    <a name="TIGR"></a>
      <li>
        <div class="item-term">
          TIGR
        </div>
        <div class="item-descr">
          The Institute for Genomic Research, a nonprofit center dedicated to deciphering and analyzing genomes.
        </div>
      </li>
  
    <a name="TM domains"></a>
      <li>
        <div class="item-term">
          TM domains
        </div>
        <div class="item-descr">
          See <a href="#Transmembrane domain">transmembrane domain</a>.
        </div>
      </li>
  
    <a name="TWINSCAN gene models"></a>
      <li>
        <div class="item-term">
          TWINSCAN gene models
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> models generated using TWINSCAN, a tool that integrates traditional probability models (such as those underlying GENSCAN and FGENESH) with information from alignments between two genomes. TWINSCAN is based on the idea that functional sequences show different patterns of evolutionary conservation than sequences under little selective pressure, such as the central regions of introns. TWINSCAN is designed for the analysis of high-throughput genomic sequences containing an unknown number of genes.
        </div>
      </li>
  
    <a name="Telomere"></a>
      <li>
        <div class="item-term">
          Telomere
        </div>
        <div class="item-descr">
          Nucleoprotein complexes that constitute the physical ends of linear eukaryotic <a href="#Chromosome">chromosomes</a> and that have important functions, primarily in the protection, replication, and stabilization of the chromosome ends. Telomeres often contain lengthy stretches of tandemly repeated simple DNA sequences composed of a G-rich <a href="#Strand">strand</a> and a C-rich strand (called terminal repeats). These terminal repeats are highly conserved. Sequences adjacent to the telomeric repeats are often highly polymorphic and rich in repetitive elements (termed subtelomeric repeats); in some cases, <a href="#Gene">genes</a> have been found in the proterminal regions of chromosomes.
        </div>
      </li>
  
    <a name="TigrScan gene"></a>
      <li>
        <div class="item-term">
          TigrScan gene
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> model generated using TigrScan, a gene-finding tool based on the generalized hidden Markov model (HMM) framework, similar to GENSCAN and Genie. It is highly reconfigurable and includes software for retraining.
        </div>
      </li>
  
    <a name="ToxoCyc"></a>
      <li>
        <div class="item-term">
          ToxoCyc
        </div>
        <div class="item-descr">
          Database/utility built by analyzing the Toxoplasma gondii genome with SRI International's pathway tools; used for searching and visualizing Toxoplasma metabolic pathway information.
        </div>
      </li>
  
    <a name="Translation"></a>
      <li>
        <div class="item-term">
          Translation
        </div>
        <div class="item-descr">
          Synthesis of protein from an mRNA template.
        </div>
      </li>
  
    <a name="Transmembrane domain"></a>
      <li>
        <div class="item-term">
          Transmembrane domain
        </div>
        <div class="item-descr">
          Three-dimensional protein structure that is thermodynamically stable in a membrane. This may be a single alpha helix, a stable complex of several transmembrane alpha helices, a transmembrane beta barrel, a beta-helix of gramicidin A, or any other structure. Transmembrane domains average 20 amino acid residues in length, though they may be much smaller or much longer.
        </div>
      </li>
  
    <a name="Transmembrane protein"></a>
      <li>
        <div class="item-term">
          Transmembrane protein
        </div>
        <div class="item-descr">
          Protein that spans an entire biological membrane.
        </div>
      </li>
  
    <a name="UTR"></a>
      <li>
        <div class="item-term">
          UTR
        </div>
        <div class="item-descr">
          Untranslated region. Section of messenger RNA (mRNA) that either precedes (5' UTR) or follows (3' UTR) the coding region and is not itself translated. The UTR contains several regulatory regions, including the polyadenylation (polyA) site in the 3' UTR, sequences involved in the initiation of <a href="#Translation">translation</a> (in the 5' UTR), and binding regions for proteins and other regulatory molecules in both the 3' and 5' UTR.
        </div>
      </li>
  
    <a name="UniGene"></a>
      <li>
        <div class="item-term">
          UniGene
        </div>
        <div class="item-descr">
          Project and database at <a href="#NCBI">NCBI</a> aimed at defining <a href="#Gene">gene</a>-oriented clusters of expressed sequence tags (ESTs). Sets of <a href="#EST">ESTs</a> are clustered based on strong sequence homology in an attempt to define a specific, nonredundant cluster for each transcript in a tissue or genome. Each UniGene cluster contains sequences that represent a unique gene in addition to information about the tissue types in which the gene has been expressed and map location.
        </div>
      </li>
  
    <a name="Wildcard character"></a>
      <li>
        <div class="item-term">
          Wildcard character
        </div>
        <div class="item-descr">
          Character used to substitute for any other character(s) in a string.
        </div>
      </li>
  
    <a name="Xenolog"></a>
      <li>
        <div class="item-term">
          Xenolog
        </div>
        <div class="item-descr">
          <a href="#Gene">Gene</a> found in an unrelated species and that is related by gene transfer rather than common vertical descent.
        </div>
      </li>
  
    <a name="blastn"></a>
      <li>
        <div class="item-term">
          blastn
        </div>
        <div class="item-descr">
          Version of the basic local alignment search tool (BLAST) used to compare a nucleotide <a href="#Query">query</a> sequence against a nucleotide sequence database.
        </div>
      </li>
  
    <a name="blastp"></a>
      <li>
        <div class="item-term">
          blastp
        </div>
        <div class="item-descr">
          Version of the basic local alignment search tool (BLAST) used to compare a protein <a href="#Query">query</a> sequence against a protein sequence database.
        </div>
      </li>
  
    <a name="blastx"></a>
      <li>
        <div class="item-term">
          blastx
        </div>
        <div class="item-descr">
          Version of the basic local alignment search tool (BLAST) used to compare a nucleotide <a href="#Query">query</a> sequence translated in all reading frames against a protein sequence database.
        </div>
      </li>
  
    <a name="cDNA"></a>
      <li>
        <div class="item-term">
          cDNA
        </div>
        <div class="item-descr">
          Complementary DNA. DNA molecule synthesized by the enzyme reverse transcriptase using an mRNA as template.
        </div>
      </li>
  
    <a name="cDNA microarray"></a>
      <li>
        <div class="item-term">
          cDNA microarray
        </div>
        <div class="item-descr">
          Collection of microscopic <a href="#cDNA">cDNA</a> spots commonly representing single <a href="#Gene">genes</a> and arrayed on a solid surface (commonly glass slides) by covalent attachment to chemically suitable matrices. Used for expression profiling, the monitoring of the <a href="#Expression level">expression levels</a> of hundreds or thousands of genes simultaneously.
        </div>
      </li>
  
    <a name="ePCR"></a>
      <li>
        <div class="item-term">
          ePCR
        </div>
        <div class="item-descr">
          Electronic PCR (polymerase chain reaction). Computational procedure used to check for uniqueness in spacing and number of primer binding sites within DNA sequences. Searches for subsequences that closely match a set of PCR primers and that have the correct order, orientation, and spacing to make a PCR product. Used to check the expected length of a PCR product, which can provide information regarding unexpected repetitive sequences.
        </div>
      </li>
  
    <a name="ncRNA"></a>
      <li>
        <div class="item-term">
          ncRNA
        </div>
        <div class="item-descr">
          Noncoding RNA. Any RNA that is not translated into a protein. Includes transfer RNA (tRNA), ribosomal RNA (rRNA), small RNAs such as snoRNAs, microRNAs, siRNAs and piRNAs, as well as long ncRNAs.
        </div>
      </li>
  
    <a name="rRNA"></a>
      <li>
        <div class="item-term">
          rRNA
        </div>
        <div class="item-descr">
          Ribosomal RNA. Component of the ribosomes, which function in protein synthesis.
        </div>
      </li>
  
    <a name="snRNA"></a>
      <li>
        <div class="item-term">
          snRNA
        </div>
        <div class="item-descr">
          Small nuclear RNA. Class of small RNA molecules found within the nucleus, transcribed by RNA polymerase II or III, and involved in a variety of important processes such as RNA splicing (removal of introns from hnRNA), regulation of transcription factors (7SK RNA) or RNA polymerase II (B2 RNA), and maintaining <a href="#Telomere">telomeres</a>. They are always associated with specific proteins, and the complexes are referred to as small nuclear ribonucleoproteins (snRNP) or snurps. These elements are rich in uridine. A large group of snRNAs known as small nucleolar RNAs (snoRNAs) are small RNA molecules that play an essential role in RNA biogenesis and chemical modification of ribosomal RNAs (rRNAs) and other RNA <a href="#Gene">genes</a> (tRNA and snRNAs). They are located in the nucleus and the cajal bodies of eukaryotic cells (the major sites of RNA synthesis).
        </div>
      </li>
  
    <a name="tRNA"></a>
      <li>
        <div class="item-term">
          tRNA
        </div>
        <div class="item-descr">
          Transfer RNA. Small RNA chain (73-93 nucleotides) that transfers a specific amino acid to a growing polypeptide chain at the ribosomal site of protein synthesis during <a href="#Translation">translation</a>. A three-base region, the anticodon, pairs to the corresponding three-base codon region on the template mRNA. Each type of tRNA molecule can be attached to only one type of amino acid, but because the genetic code contains multiple codons that specify the same amino acid, tRNA molecules bearing different anticodons may also carry the same amino acid.
        </div>
      </li>
  
    <a name="tblastn"></a>
      <li>
        <div class="item-term">
          tblastn
        </div>
        <div class="item-descr">
          Version of the basic local alignment search tool (BLAST) used to compare a protein <a href="#Query">query</a> sequence against a translated nucleotide sequence database.
        </div>
      </li>
  
    <a name="tblastx"></a>
      <li>
        <div class="item-term">
          tblastx
        </div>
        <div class="item-descr">
          Version of the basic local alignment search tool (BLAST) used to compare the six-frame <a href="#Translation">translations</a> of a nucleotide <a href="#Query">query</a> sequence against the six-frame translations of a nucleotide sequence database.
        </div>
      </li>
  
</ul>

</div>
