---
layout: post
title: AmoebaDB 59 Released
category: [AmoebaDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 59.</h3>

<section>
  <h3>Annual VEuPathDB Workshop (Virtual)</h3>
    The 15th Annual VEuPathDB workshop will be a virtual event taking place daily on <b>September 6 - 9th, 2022</b> from <b>9am - 1pm Eastern</b>. Join us for these 4 half-day sesssions to learn how to effectively use VEuPathDB tools and features that support your hypothesis driven research. Short lectures will precede hands-on exercises via informal Zoom sessions where discussion and conversation are encouraged. <a href="/a/app/static-content/workshopSept2022.html">Details here</a> or <a href="https://ugeorgia.ca1.qualtrics.com/jfe/form/SV_elEnAmi5PvtUNwO" target="_blank">Register here</a>.
</section>

<section>
  <h3>Release 59 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday September 1st, 2022 at 10am EDT for our Release 59 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 59.  
    <a href="https://attendee.gotowebinar.com/register/3393660656180049424" target="_blank">Register here</a>  
  </div>
  <br>
</section>

<section>
  <h3>New data in {{project}} 59</h3> 
  <div style="margin-left: 3em;">
    <h4>Genome Updates:</h4>  The following genomes were updated to the INSDC versions to avoid small discrepancies between public versions.
      <ul>
        <li><a href="/a/app/record/dataset/DS_16f5925e6e"><i>Acanthamoeba astronyxis</i> Unknown</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826245.1/" target="_blank"> GCA_000826245.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_6798c8801d"><i>Acanthamoeba castellanii </i>Ma</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826485.1" target="_blank"> GCA_000826485.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_b8d35e3357"><i>Acanthamoeba culbertsoni</i> A1</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826265.1/" target="_blank"> GCA_000826265.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_f659d3f712"><i>Acanthamoeba lenticulata</i> PDS2</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826285.1/" target="_blank"> GCA_000826285.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_a9890f8831"><i>Acanthamoeba lugdunensis</i> L3a</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826425.1/" target="_blank"> GCA_000826425.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_187ad209cc"><i>Acanthamoeba mauritaniensis</i> 1652</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826465.1/" target="_blank"> GCA_000826465.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_7207e44a8f"><i>Acanthamoeba palestinensis</i> Reich</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826305.1/" target="_blank"> GCA_000826305.1</a> (In Genbank as <i>A. healyi</i>).</li>

        <li><a href="/a/app/record/dataset/DS_30083445f0"><i>Acanthamoeba quina</i> Vil3</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826445.1/" target="_blank"> GCA_000826445.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_d173b9881a"><i>Acanthamoeba rhysodes</i> Singh</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826385.1/" target="_blank"> GCA_000826385.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_7a3d45c781"><i>Acanthamoeba sp.</i> T4B-type</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826345.1/" target="_blank"> GCA_000826345.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_8ca217d969"><i>Acanthamoeba sp.</i> Incertae_sedis</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826365.1/" target="_blank"> GCA_000826365.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_93b3580697"><i>Acanthamoeba sp.</i> Galka</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826365.1/" target="_blank"> GCA_000826505.1</a>.</li>

        <li><a href="/a/app/record/dataset/DS_828b3a972f"><i>Acanthamoeba triangularis</i> SH621</a> &ndash;  <a href="http://www.ncbi.nlm.nih.gov/assembly/GCA_000826325.1/" target="_blank"> GCA_000826325.1</a> (in GenBank as <i>A. palestinensis</i>).</li>
      </ul>
      <br>
    <h4>RNA Sequence</h4> 
    <h4><i>Entamoeba dispar</i> SAW760 </h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_0e8ecebdf0">Small RNA and antisense transcription in E dispar</a> &ndash; Transcriptome analysis <i>E dispar</i> SAW760 to include small RNA sequences for determining antisense transcription and transcription start sites. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35208849" target="_blank">Mornico et al. 2022</a>.</li> 
      </ul>
    <h4><i>Entamoeba histolytica</i> HM1:IMSS</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_2c2c87bf2c">Anexic (normal, attenuated, virulent) and colon cultured E histolytica</a> &ndash; Transcriptome analysis of anexic normal, attenuated and virulent cultured <i>E histolytica</i> HM1:IMSS as well as trphozoites cultured on colon samples. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33214622" target="_blank">Mornico et al. 2020</a>.</li> 

        <li><a href="/a/app/record/dataset/DS_67f526fac0">Heat Shock effects on <i>E histolytic</i> transcriptome</a> &ndash; Transcriptome analysis E histolytica HM1:IMSS during 8 hour heat shock time course. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33214622" target="_blank">Mornico et al. 2020</a>.</li> 

        <li><a href="/a/app/record/dataset/DS_9707d17ce4">Initiation of natural antisense transcription in E histolytica</a> &ndash; Transcriptome analysis <i>E histolytica</i> HM1:IMSS to illucidate natural antisense transcription. Control, csRNA and TEX+ libraries were sequenced. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33214622" target="_blank">Mornico et al. 2020</a>.</li> 

        <li><a href="/a/app/record/dataset/DS_0e8ecebdf0">Small RNA and antisense transcription in <i>E histolytica</i></a> &ndash; Transcriptome analysis <i>E histolytica</i> HM1:IMSS to include small RNA sequences for determining antisense transcription and transcription start sites. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35208849" target="_blank">Mornico et al. 2022</a>.</li> 
      </ul>
    
    <h4><i>Entamoeba invadens</i> IP1</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_86e4ee56d2">Small RNA and antisense transcription in <i>E invadens</i></a> &ndash; Transcriptome analysis <i>E invadens</i> IP1 to include small RNA sequences for determining antisense transcription and transcription start sites. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35208849" target="_blank">Mornico et al. 2022</a>.</li> 
      </ul>
    <h4><i>Entamoeba moshkovskii</i> Laredo</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_1911713b77">Small RNA and antisense transcription in <i>E moshkovskii</i></a> &ndash; Transcriptome analysis <i>E moshkovskii</i> Laredo to include small RNA sequences for determining antisense transcription and transcription start sites. <a href="http://www.ncbi.nlm.nih.gov/pubmed/35208849" target="_blank">Mornico et al. 2022</a>.</li> 
      </ul>
      <br>

    <h4>Genome Browser Tracks</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_0d1304c893">Transcription start sites for E dispar</a> &ndash; <a href="http://www.ncbi.nlm.nih.gov/pubmed/35208849" target="_blank">Mornico et al. 2022</a>. <a href ="/a/app/jbrowse?data=%2Fa%2Fservice%2Fjbrowse%2Ftracks%2FedisSAW760&amp%3Bloc=DS548800%3A113601..225500&tracks=gene%2CedisSAW760_TSS&highlight=&loc=DS548800%3A34122..305321">View data in JBrowse</a></li>
      </ul>

  </div>
</section>

<section>
  <h3>New features in {{project}} 59</h3>
  <div style="margin-left: 3em;">
  <h4>'Send To' actions for the results of a gene search</h4>
    <ul>
      <li>Send to My Data Sets:  Save your search result as an ID list in My Data Sets. Open the 'Send To' dropdown menu and choose 'My Data Sets' to create your private data set with the IDs from the search result.  Your new data set record will appear on your My Data Sets page as a downloadable text file. You must be a registered {{project}} user and logged in to use this feature. <a target="_blank" href="{{'/assets/images/MyDataSets/SaveListMyDataSets.gif' | absolute_url}}">View tutorial GIF.</a></li><br>
      
      <li>Send to VEuPathDB: Transfer a gene list from {{project}} to VEuPathDB and analyze in context with all VEuPathDB organisms.  From the {{project}} gene result, open the 'Send To' dropdown menu and choose 'VEuPathDB.org Strategy'.  The VEuPathDB My Strategies page will open in a new window displaying a one-step strategy that returns the gene list just transferred from {{project}}. Once transferred, use the 'Add a step' tool to extend the strategy and take advantage of orthology across all VEuPathDB organisms. <a target="_blank" href="{{'/assets/images/MyDataSets/SendToVeupathdb.gif' | absolute_url}}">View tutorial GIF.</a>  </li>
    </ul>
  <h4>Upload a gene list to My Data Sets</h4>
    <ul>
      <li>In preparation to provide gene set functionality such as gene set enrichment analysis we are introducing this new feature. The 'New upload' tab of <a href="/a/app/workspace/datasets">My Data Sets</a> allows for the upload of a gene list into {{project}} directly from a file or URL. Your new list will appear in the All tab and will remain available between sessions. If the results of your last experiment or the last article you read included a gene list, you can upload it here for further analysis. <a target="_blank" href="{{'/assets/images/MyDataSets/UploadListMyDataSets.gif' | absolute_url}}">View tutorial GIF.</a></li>
    </ul>
  <h4>Sequence Retrieval Tool improvements</h4>
    <ul>
      <li>The <a href="/a/app/fasta-tool/genesequence">Sequence Retrieval Tool</a> now remembers your parameter settings during your session. You can leave and return to the tool without having to set every parameter again. </li>
    </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
