---
layout: post
title: ToxoDB 57 Released
category: [ToxoDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 57.</h3>

<section>
  <h3>Release 57 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday April 28th, 2022 at 10am EDT for our Release 57 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 57.  
    <a href="https://attendee.gotowebinar.com/register/1664706106928504331" target="_blank">Register here</a>  
  </div>
</section>

<section>
  <h3>New data in {{project}} 57</h3>
  <div style="margin-left: 3em;">
    <h4>RNA Sequence</h4>
    <ul>
      <li><i>Eimeria tenella</i> Houghton</li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_38f0241270">Transcriptome of <i>E. tenella</i> from infected chicken caecal tissues</a> &ndash; Transcriptome of <i>Eimeria tenella</i> from infected chicken caecal tissues. Chickens were infected with purified sporozoites and RNA-Seq was performed on samples collected 3, 4, and 10 days post-infection <a href="http://www.ncbi.nlm.nih.gov/pubmed/34521339" target="_blank">Sandholt et al. 2021</a>.  The host-aligned data analysis of this dual RNA seq dataset appears in HostDB: <a href="https://hostdb.org/hostdb/app/record/dataset/DS_f61d8b2d63" target="_blank">Chicken caecal tissue during <i>E tenella</i> infection</a>.</li>
        <li>
          <a href="/a/app/record/dataset/DS_16c0d0f21d"><i>Eimeria tenella</i> transcriptome during infection in chicken macrophage-like cells</a> &ndash; Transcriptome of Eimeria tenella during infection in macrophage-like cell line HD11 cultures infected in vitro. Triplicate cultures were harvested at 2, 4, 12, 24, 48 and 72 hr post-infection and mRNA was extracted and sequenced. <a href="http://www.ncbi.nlm.nih.gov/pubmed/33536090" target="_blank">Sandholt et al. 2021</a>.  The host-aligned data analysis of this dual RNA seq dataset appears in HostDB: <a href="https://hostdb.org/hostdb/app/record/dataset/DS_0ef6c17001" target="_blank">Chicken macrophage-like cells (HD11) infected in vitro with <i>Eimeria tenella</i> infection</a>.</li>          
      </ul>
      <li><i>Toxoplasma gondii</i> ME49</li>
      <ul>
        <li>
          <a href="/a/app/record/dataset/DS_79e1396920">RNA-Seq of MORC WT and MORC KD parasites</a> &ndash; Microrchidia (MORC) protein transcriptome changes at different time points and FR235222 treatment. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32094587" target="_blank">Farhat et al. 2020</a>. </li> 
        <li>
          <a href="/a/app/record/dataset/DS_4a81d92b98">RNA-Seq of CPSF4 WT and CPSF4 KD parasites</a> &ndash; RNA profiles were generated from HFF cells infected by RHDku80 CPSF4-mAID-HA T. gondii strain for untreated conditions, auxin induced KD at 7h, 24h, and 48h of CPSF4. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34263725" target="_blank">Farhat et al. 2021</a>. </li> 
        <li>
          <a href="/a/app/record/dataset/DS_386bac2860"><i>T. gondii</i> during spontaneous differentiation in primary rat brain cells.</a> &ndash; Transcriptome changes during the parasite spontaneous differentiation after infection of primary rat brain cells. Triplicate RNA samples of infected primary CNS cell culture at 1d, 2d, 4d, 7d, and 14d post-infection. <a href="http://www.ncbi.nlm.nih.gov/pubmed/34610266" target="_blank">Mouveaux et al. 2021</a>. The host-aligned data analysis of this dual RNA seq dataset appears in HostDB: <a href="https://hostdb.org/hostdb/app/record/dataset/DS_23f5fa7e60" target="_blank">Rat brain primary cells transcriptome during infection with <i>T. gondii</i></a>.</li> 
      </ul>
    </ul>

    <h4>ChIP Sequence</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_426c5d331c">ChIP-Seq in MORC WT and MORC KD parasites</a> &ndash; Genome-wide MORC and HDAC3 occupancy profiles in Pruku80_MORC_mAID_HA_KD left untreated (UT) and treated with IAA (indole-3-acetic acid) for 30 hours. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32094587" target="_blank">Farhat et al. 2020</a>.</li>
    </ul>
  </div>
</section>

<section>
  <h3>New features in {{project}} 57</h3>
  <div style="margin-left: 3em;">
  <h4>New 'Unannotated Intron Junctions' Search</h4>
    <ul>
    <li>The new <a href="/a/app/search/transcript/GenesByIntronJunctions">Unannotated Intron Junctions</a> search enables users to identify genes that contain, or are flanked by, unannotated high confidence intron junction-spanning reads from RNA-seq data. These genes may be incompletely or inaccurately annotated due to missing introns/exons and/or alternative splice variants.  The search is categorized under Search for Genes, Function Prediction.  See our <a href="https://qa.static-content.veupathdb.org/documents/Intron_junctions_search_16_04_2022.pdf" target="_blank">PDF Tutorial</a> or our <a href="https://www.youtube.com/watch?v=-kjjsB3eqxA" target="_blank">Video Tutorial</a> for more information. </li>
    </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>

