---
layout: post
title: ToxoDB 67 Released
category: [ToxoDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 67.</h3>

<section>
  <h3>Release 67 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on Thursday Feb 29th, 2024 at 10 am Eastern for our Release 67 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 67. 
    <a href="https://attendee.gotowebinar.com/register/473566692359573344" target="_blank">Register here</a>
  </div>
  <br>
</section>

<section>
  <h3>VEuPathDB named as an Elixir Core Data Resource</h3>
  <div style="margin-left: 3em;">
    <a href="https://elixiruknode.org/news/2023/4-uk-services-cdrs/" target="_blank">Inducted in December 2023</a>, VEuPathDB is honored to join the select group of <a href="https://elixir-europe.org/platforms/data/core-data-resources" target="_blank">Elixir Core Data Resources</a> (CDR). With this distinction, VEuPathDB is identified as a resource of fundamental importance to the life science community and the preservation of biological data, dedicated to the principle of open access for publicly funded research. The CDR program recognizes resources that promote excellence and provide infrastructure vital for the advancement of the life sciences.  Evaluation criteria include scientific focus, quality of science and community service, governance, impact, and ability to facilitate scientific research. 
  </div>
</section>

<section>
  <h3>Contribute your expert knowledge!</h3>
  <div style="margin-left: 3em;">
    We invite all users to engage with Apollo, where you can actively edit and enhance your selected genes. Whether you're a seasoned user or a newcomer, our updated <a href="/a/app/static-content/apollo_help.html">Apollo landing page</a> provides a wealth of resources, including links to on-demand webinars, tutorials, and annotation guidelines. <br><br>
    Every edit you make contributes significantly! This open invitation is extended to all users, with a special emphasis on scientists studying groups of organisms that have recently been updated to integrate new reference genomes, and for the species that are now accessible on {{project}} for the first time. If you're unfamiliar or have any questions, don't hesitate to <a href="/a/app/contact-us">Contact Us</a>. We're here to help!
  </div>
</section>

<section>
  <h3>New data in {{project}} 67</h3>
  <div style="margin-left: 3em;">
    <h4>Transcriptomics</h4>
    <div style="margin-left: 3em;">
    <ul>
      <li><a href="/a/app/record/dataset/DS_60c5730a33">Early infection of <i>Besnoitia besnoiti</i> tachyzoites in Bovine monocyte-derived macrophages</a> &ndash; Early interaction between <i>Besnoitia besnoiti</i> tachyzoites and primary bovine monocyte-derived macrophages <i>in vitro</i>. <i>B. besnoiti</i> portion of a dual transcriptomic profiling of <i>B. taurus</i> macrophage.  <a href="http://hostdb.org/hostdb/app/record/dataset/DS_0cadd21ec8" target="_blank"><i>B. taurus</i> data set</a> &ndash; <a href="http://www.ncbi.nlm.nih.gov/pubmed/37207972" target="_blank">Fernández-Álvarez et al. 2023</a></li>

      <li><a href="/a/app/record/dataset/DS_ef5599ef46"><i>Eimeria tenella</i> response to toltrazuril</a> &ndash; RNA-Seq transcriptomes of merozoites treated with toltrazuril, a widely used frontline anticoccidial drug that is experiencing drug resistance, for 0, 1, 2, and 4-hours. <a href="http://www.ncbi.nlm.nih.gov/pubmed/37176073" target="_blank">Zhang et al. 2023</a></li>
    </ul>
    </div>
    <br>

    <h4>Updated Functional Annotation</h4>
    Functional annotations in ToxoDB have been updated for <i>T. gondii</i> ME49. For details please see this <a href="https://docs.google.com/spreadsheets/d/1bKqwhyaa-oigTZ1myBcEQOXPb33odcOzWg39SdeQ1Mc" target="_blank">Summary of annotation updates</a> which include products, GO terms, gene names, synonyms and PMID.
    <br><br>

    <style>
      #FuncAnnot_Bld67 table, #FuncAnnot_Bld67 td, #FuncAnnot_Bld67 th, #FuncAnnot_Bld67 tr {
      text-align : center;
      padding-left: 5px;
      padding-right: 5px;
      padding-top: 5px;
      padding-bottom: 5px;
      border: 1px solid black;
      }
      #FuncAnnot_Bld67 {
      margin-left : 5 em;
      }
    </style>
    <table id="FuncAnnot_Bld67">
      <tr>
        <th>ToxoDB Genome Version</th>
        <th>Genes Updated</th>
      </tr>
      <tr>
        <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on ToxoDB."><a href="/a/app/record/dataset/NCBITAXON_508771"><i>Toxoplasma gondii </i>ME49</a></td>
        <td>495</td>
      </tr>
    </table>
    <br>
  </div>
</section>

<section>
  <h3>New features in {{project}} 67</h3>
  <div style="margin-left: 3em;">
    <ul>
      <li>The <a href="/a/app/fasta-tool/transcript">Sequence Retrieval Tool</a> offers more input options such as uploading from a text file or URL. New options for configuring or choosing the type of sequence returned are also available. </li>
      <li>The Product Descriptions table on gene pages is now called All Product Descriptions and contains product descriptions from all relevant sources. The most commonly used product description also appears at the top of the gene page. </li>
      <li>The <a href="/a/app/search/transcript/GenesByText">Text Search</a> 'Fields' parameter values have changed. Choose 'Product descriptions (all)' to search the All Product Descriptions table on gene pages.  Choose 'Product descriptions' to search only the commonly used product descriptions that appear at the top of gene pages.  </li>
      <li>The Organisms parameter in the <a href="/a/app/search/transcript/GenesByNgsSnps">Identify Genes based on SNP Characteristics</a> search has a new look. Choose your organism of interest to load the Set of Samples parameter with that organism's WGS isolates.</li>
      <li>Compressed files in the <a href="/a/app/downloads">Download Data Files</a> tool are now in .gz format.</li>
   </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>

